
/****************************************************************

   This file was created automatically by `FlexCat 2.4'
   from "VEGA.cd" using VEGA.sd 1.0 (02.01.2002)
   Template created by Alessandro Pedretti
   

   Do NOT edit by hand!

****************************************************************/


#define  MSG_TRANSLATION  0
#define  MSG_ERR_ARGPAR_MATCH  1
#define  MSG_ERR_ARGPAR_INPFILE  2
#define  MSG_ERR_ARGPAR_ATMNUM  3
#define  MSG_ERR_ARGPAR_DUPDESC  4
#define  MSG_ERR_ARGPAR_INCDESC  5
#define  MSG_ERR_ARGPAR_TOOLONG  6
#define  MSG_ERR_ARGPAR_ILLATMNUM  7
#define  MSG_ERR_ARGPAR_ILLNUMMES  8
#define  MSG_ERR_ARGPAR_UNKMES  9
#define  MSG_ERR_ARGPAR_TOOFEW  10
#define  MSG_ERR_BIOSYM_NOTSUPPORTED  11
#define  MSG_CHARGE_ASSPROG  12
#define  MSG_ERR_CHARGE_TEMDESCDUP  13
#define  MSG_CHARGE_TOTCHG  14
#define  MSG_ERR_CHARGE_TEMRESEXP  15
#define  MSG_CHARGE_PROGPMT  16
#define  MSG_ERR_CHARGE_TEMFORMAT  17
#define  MSG_ERR_CHARGE_TEMRESLONG  18
#define  MSG_ERR_CHARGE_TEMRESMIS  19
#define  MSG_CONNECT_PROGPMT  20
#define  MSG_CONNECT_NORMPROG  21
#define  MSG_ERR_CONNECT_UNKVDW  22
#define  MSG_ERR_CONNECT_UNACCBD  23
#define  MSG_DCD_STARTAREA  24
#define  MSG_DCD_STARTSURFDIA  25
#define  MSG_DCD_STARTPSA  26
#define  MSG_DCD_ASSATMTYPES  27
#define  MSG_DCD_STARTVLOGP  28
#define  MSG_DCD_STARTVOL  29
#define  MSG_DCD_STARTVOLDIA  30
#define  MSG_DCD_TRJINFO  31
#define  MSG_DCD_LOWESTENE  32
#define  MSG_ECD_SAVPLOT  33
#define  MSG_DCD_PROCFRAME  34
#define  MSG_DCD_RESULTS  35
#define  MSG_ERR_DCD_UNCHARGED  36
#define  MSG_ERR_DCD_UNSUPPOUT  37
#define  MSG_ERR_DCD_UNKTRAJ  38
#define  MSG_ENER_ASSPROG  39
#define  MSG_ENER_CALCPROG  40
#define  MSG_ENER_SORTPROG  41
#define  MSG_ENER_RESULTS  42
#define  MSG_ERR_ENER_MORERES  43
#define  MSG_ERR_ENER_RESNOTFOUND  44
#define  MSG_ERR_ENER_UNKATMTYPE  45
#define  MSG_ERR_ENER_NONBOND  46
#define  MSG_ERR_ENER_FFNOTFOUND  47
#define  MSG_ERR_FASTA_NOPROT  48
#define  MSG_FFIELD_FIXPROG  49
#define  MSG_FFIELD_RINGPROG  50
#define  MSG_ERR_FFIELD_ILLNUM  51
#define  MSG_ERR_FFIELD_UNBPAR  52
#define  MSG_ERR_FFIELD_SUBLVL  53
#define  MSG_ERR_FFIELD_ATYPTOOLONG  54
#define  MSG_ERR_FFIELD_INCATM  55
#define  MSG_ERR_FFIELD_INCSUBATM  56
#define  MSG_ERR_FFIELD_NOTFOUND  57
#define  MSG_ERR_FFIELD_CANTASSIGN  58
#define  MSG_ERR_FFIELD_FFERRLINE  59
#define  MSG_ERR_FFIELD_FFERRLINETEMP  60
#define  MSG_ERR_FFIELD_ILLAROM  61
#define  MSG_ERR_FFIELD_ILLRING  62
#define  MSG_ERR_FFIELD_ILLBONDORD  63
#define  MSG_GASTEIGER_TOTCHG  64
#define  MSG_ERR_IFF_INVFILE  65
#define  MSG_ERR_IFF_NOTIFF  66
#define  MSG_ERR_IFF_CORRUPTED  67
#define  MSG_INFO_TITLE  68
#define  MSG_INFO_PROP1  69
#define  MSG_INFO_CHARGE  70
#define  MSG_INFO_SURFAREA  71
#define  MSG_INFO_PSA  72
#define  MSG_INFO_VOLUME  73
#define  MSG_INFO_OVALITY  74
#define  MSG_INFO_OVALNOTAVAIL  75
#define  MSG_INFO_LOGPCRIPPEN  76
#define  MSG_INFO_LOGPBROTO  77
#define  MSG_INFO_VLOGP  78
#define  MSG_INFO_NOTAVAIL  79
#define  MSG_INFO_AACOMP  80
#define  MSG_INFO_TOOATM  81
#define  MSG_LOADER_MSF  82
#define  MSG_LOADER_CRD  83
#define  MSG_LOADER_SYBREG  84
#define  MSG_LOADER_IFF  85
#define  MSG_LOADER_CAR  86
#define  MSG_LOADER_PDB  87
#define  MSG_LOADER_CSSR  88
#define  MSG_LOADER_QMC  89
#define  MSG_LOADER_HIN  90
#define  MSG_LOADER_MOPARC  91
#define  MSG_LOADER_MOPINT  92
#define  MSG_LOADER_MOL2  93
#define  MSG_LOADER_GROMOS  94
#define  MSG_LOADER_XYZ  95
#define  MSG_ERR_LOADER_CRDASSFILE  96
#define  MSG_ERR_LOADER_CRDMESURE  97
#define  MSG_ERR_LOADER_UNKFORMAT  98
#define  MSG_ERR_LOADER_NOTFOUND  99
#define  MSG_ERR_LOCALE_ERROR  100
#define  MSG_ERR_LOGP_UNRECFILE  101
#define  MSG_ERR_LOGP_NOTALLPAR  102
#define  MSG_ERR_LOGP_NOTFOUND  103
#define  MSG_MAIN_DCDPROG  104
#define  MSG_MAIN_DONEPROG  105
#define  MSG_MAIN_VLOGP  106
#define  MSG_MAIN_OSCPU  107
#define  MSG_MAIN_SYNOPSIS  108
#define  MSG_ERR_MAIN_UNKOPT  109
#define  MSG_ERR_MAIN_ILLSTRRES  110
#define  MSG_ERR_MAIN_UNSUPCHGTEMP  111
#define  MSG_ERR_MAIN_CHGFILENOTFOUND  112
#define  MSG_ERR_MAIN_ILLDIEL  113
#define  MSG_ERR_MAIN_ILLRESNAME  114
#define  MSG_ERR_MAIN_UNKOUTFOR  115
#define  MSG_ERR_MAIN_ILLPROBERAD  116
#define  MSG_ERR_MAIN_ILLSOLRAD  117
#define  MSG_ERR_MAIN_UNKSHAPE  118
#define  MSG_ERR_MAIN_SHELLFILE  119
#define  MSG_ERR_MAIN_FFNOTSTD  120
#define  MSG_ERR_MAIN_FFNOTFOUND  121
#define  MSG_ERR_MAIN_ILLSASDEN  122
#define  MSG_ERR_MAIN_NOSTDOUT  123
#define  MSG_ERR_MAIN_NOFFTAG  124
#define  MSG_ERR_MAIN_NOCHARGE  125
#define  MSG_ERR_MAIN_CSSR  126
#define  MSG_ERR_MAIN_FILEMISS  127
#define  MSG_ERR_MAIN_FLOATNUM  128
#define  MSG_ERR_MAIN_INTNUM  129
#define  MSG_ERR_MAIN_SUBARG  130
#define  MSG_ERR_MAIN_RGNNOMOL  131
#define  MSG_ERR_MAIN_ABORT  132
#define  MSG_MLP_VLOGP  133
#define  MSG_ERR_MLP_NOVLOGP  134
#define  MSG_MOL2_ASSIGNPROG  135
#define  MSG_ERR_MOL2_CORRUPTED  136
#define  MSG_ERR_MOL2_ILLTEMP  137
#define  MSG_ERR_MOL2_NOTFOUND  138
#define  MSG_ERR_MOPAC_ATMFAULT  139
#define  MSG_ERR_MOPAC_BADCONV  140
#define  MSG_ERR_MSF_NOTCOMP  141
#define  MSG_ERR_MSF_NOTMSF  142
#define  MSG_PDB_FATFOUND  143
#define  MSG_SAVER_SAVEPROG  144
#define  MSG_SAVER_ATTYPROG  145
#define  MSG_SAVER_LOGPBROTO  146
#define  MSG_SAVER_WARN_NOTCHARGE  147
#define  MSG_SAVER_WARN_FFNOTASS  148
#define  MSG_ERR_SAVER_PLOT  149
#define  MSG_SOLVATE_SOLVATING  150
#define  MSG_SOLVATE_ADDED  151
#define  MSG_ERR_SOLVATE_SHELLDIM  152
#define  MSG_ERR_SOLVATE_READING  153
#define  MSG_ERR_SOLVATE_CANTOPEN  154
#define  MSG_ERR_SOLVATE_NOTHING  155
#define  MSG_SURFACE_BUILDILM  156
#define  MSG_SURFACE_BUILDMEP  157
#define  MSG_SURFACE_BUILDVLOGP  158
#define  MSG_SURFACE_BUILDPSA  159
#define  MSG_SURFACE_BUILDVDW  160
#define  MSG_SURFACE_RANGES  161
#define  MSG_ERR_SURFACE_ILM  162
#define  MSG_UTILS_LOADPROG  163
#define  MSG_UTILS_RMHPROG  164
#define  MSG_UTILS_RMWPROG  165
#define  MSG_UTILS_RENRESPROG  166
#define  MSG_ERR_UTILS_DOSERR  167
#define  MSG_ERR_UTILS_OUTOFMEM  168
#define  MSG_ERR_UTILS_VEGADIR  169
#define  MSG_WARN_UTILS_WARN  170
#define  MSG_ERR_SYBRGN_POINTS  171
#define  MSG_ERR_SYBRGN_NUMSTPS  172
#define  MSG_ERR_SYBRGN_STPSIZE  173
#define  MSG_ERR_SYBRGN_CORRUPTED  174
#define  MSG_ERR_PKPP_ERROR  175
#define  MSG_ERR_PKPP_NOPPLIB  176
#define  MSG_ERR_PKPP_TOOSHORT  177
#define  MSG_ERR_GZIP_ERROR  178
#define  MSG_ERR_BZIP2_ERROR  179
#define  MSG_ERR_Z32_ERROR  180
#define  MSG_ERR_COMFA_RGN  181
#define  MSG_ERR_COMFA_FIELD  182
#define  MSG_PK_PROG  183
#define  MSG_ERR_MAIN_PKNOSTDOUT  184
#define  MSG_ERR_MAIN_UNKPACKMET  185
#define  MSG_ERR_MAIN_CANTPACK  186
#define  MSG_ERR_PKPP_NOCRUNCH  187
#define  MSG_ERR_ENER_FFNOTASS  188
#define  MSG_MAIN_OPENGLSTART  189
#define  MSG_MAIN_ASSVDW  190
#define  MSG_GLMAIN_EXITREQ  191
#define  MSG_MAIN_MENU_FILE  192
#define  MSG_MAIN_MENU_NEW  193
#define  MSG_MAIN_MENU_OPEN  194
#define  MSG_MAIN_MENU_SAVE  195
#define  MSG_MAIN_MENU_SAVE2  196
#define  MSG_MAIN_MENU_PRINT  197
#define  MSG_MAIN_MENU_SAVEBMP  198
#define  MSG_MAIN_MENU_COMP  199
#define  MSG_MAIN_MENU_CMPNONE  200
#define  MSG_MAIN_MENU_CMPBZIP2  201
#define  MSG_MAIN_MENU_CMPGZIP  202
#define  MSG_MAIN_MENU_CMPZCOMP  203
#define  MSG_MAIN_MENU_CONN  204
#define  MSG_MAIN_MENU_DEMO  205
#define  MSG_MAIN_MENU_DMRUN  206
#define  MSG_MAIN_MENU_DMSTOP  207
#define  MSG_MAIN_MENU_DMMUSIC  208
#define  MSG_MAIN_MENU_TEXT  209
#define  MSG_MAIN_MENU_EXIT  210
#define  MSG_MAIN_MENU_EDIT  211
#define  MSG_MAIN_MENU_CUT  212
#define  MSG_MAIN_MENU_COPY  213
#define  MSG_MAIN_MENU_PASTE  214
#define  MSG_MAIN_MENU_SELECTALL  215
#define  MSG_MAIN_MENU_COPYALL  216
#define  MSG_MAIN_MENU_FIND  217
#define  MSG_MAIN_MENU_REMOVE  218
#define  MSG_MAIN_MENU_RMHYD  219
#define  MSG_MAIN_MENU_RMWAT  220
#define  MSG_MAIN_MENU_RMSURF  221
#define  MSG_MAIN_MENU_RENRES  222
#define  MSG_MAIN_MENU_NORM  223
#define  MSG_MAIN_MENU_COPYBMP  224
#define  MSG_MAIN_MENU_VIEW  225
#define  MSG_MAIN_MENU_DISPLAY  226
#define  MSG_MAIN_MENU_DSPWIRE  227
#define  MSG_MAIN_MENU_DSPVDW  228
#define  MSG_MAIN_MENU_DSPVDWS  229
#define  MSG_MAIN_MENU_DSPCPK  230
#define  MSG_MAIN_MENU_DSPCPKS  231
#define  MSG_MAIN_MENU_COLOR  232
#define  MSG_MAIN_MENU_COLBYATM  233
#define  MSG_MAIN_MENU_COLBYMOL  234
#define  MSG_MAIN_MENU_LBLATM  235
#define  MSG_MAIN_MENU_LBLATM_OFF  236
#define  MSG_MAIN_MENU_LBLATM_NAME  237
#define  MSG_MAIN_MENU_LBLATM_ELEM  238
#define  MSG_MAIN_MENU_LBLATM_NUM  239
#define  MSG_MAIN_MENU_LBLATM_TYPE  240
#define  MSG_MAIN_MENU_LBLATM_CHARGE  241
#define  MSG_MAIN_MENU_RESETVIEW  242
#define  MSG_MAIN_MENU_ANIM  243
#define  MSG_MAIN_MENU_LIGHT  244
#define  MSG_MAIN_MENU_MOLINFO  245
#define  MSG_MAIN_MENU_CALC  246
#define  MSG_MAIN_MENU_CALCCHRG  247
#define  MSG_MAIN_MENU_CALCCLUST  248
#define  MSG_MAIN_MENU_CALCMOPAC  249
#define  MSG_MAIN_MENU_ANALYSIS  250
#define  MSG_MAIN_MENU_BIODOCK  251
#define  MSG_MAIN_MENU_SURFACE  252
#define  MSG_MAIN_MENU_SRFCALC  253
#define  MSG_MAIN_MENU_SRFSAVE  254
#define  MSG_MAIN_MENU_SRFCOLOR  255
#define  MSG_MAIN_MENU_SRFREMOVE  256
#define  MSG_MAIN_MENU_TOOLS  257
#define  MSG_MAIN_MENU_TLSGRAPH  258
#define  MSG_MAIN_MENU_TLSMINIED  259
#define  MSG_MAIN_MENU_TLSWINDD  260
#define  MSG_MAIN_MENU_TLSTASK  261
#define  MSG_MAIN_MENU_TLSPRFED  262
#define  MSG_MAIN_MENU_HELP  263
#define  MSG_MAIN_MENU_CONTENTS  264
#define  MSG_MAIN_MENU_MOPACHELP  265
#define  MSG_MAIN_MENU_HELPKEYS  266
#define  MSG_MAIN_MENU_WEB  267
#define  MSG_MAIN_MENU_ABOUT  268
#define  MSG_POPUP_MENU_MOUSE  269
#define  MSG_POPUP_MENU_MOUSEROT  270
#define  MSG_POPUP_MENU_MOUSETR  271
#define  MSG_POPUP_MENU_MOUSESCL  272
#define  MSG_GLDEMO_ERR_MCI  273
#define  MSG_MENU_NEW  274
#define  MSG_MENU_CALCSTOP  275
#define  MSG_MENU_OPEN  276
#define  MSG_MENU_SAVE  277
#define  MSG_MENU_SAVEBMP  278
#define  MSG_MENU_ERR_HELP  279
#define  MSG_MENU_ERR_CANTEXE  280
#define  MSG_LOADER_QNTSRF  281
#define  MSG_ERR_LODER_OPENGLONLY  282
#define  MSG_ERR_GL_RUNNING  283
#define  MSG_ERR_GL_CONTEXT  284
#define  MSG_ERR_GL_PIXELFORM1  285
#define  MSG_ERR_GL_PIXELFORM2  286
#define  MSG_ERR_GL_CREATECONTEXT  287
#define  MSG_ERR_GL_ACTCONTEXT  288
#define  MSG_ERR_GL_FAIL  289
#define  MSG_ERR_GL_WNDCLASS  290
#define  MSG_ERR_GL_WNDCREAT  291
#define  MSG_DLGSRFCLC_TITLE  292
#define  MSG_DLGSRFCLC_LBL_TYPE  293
#define  MSG_DLGSRFCLC_LBL_PROBERAD  294
#define  MSG_DLGSRFCLC_LBL_DENSITY  295
#define  MSG_DLGSRFCLC_BT_CALC  296
#define  MSG_BT_ABORT  297
#define  MSG_BT_CANCEL  298
#define  MSG_BT_CLOSE  299
#define  MSG_BT_DONE  300
#define  MSG_BT_FIX  301
#define  MSG_BT_OK  302
#define  MSG_BT_RUN  303
#define  MSG_BT_SHOW  304
#define  MSG_BT_START  305
#define  MSG_BT_STOP  306
#define  MSG_BT_EDIT  307
#define  MSG_DLGSRFCOL_TITLE  308
#define  MSG_DLGSRFCOL_RG_TITLE  309
#define  MSG_DLGSRFCOL_RG_PROPERTY  310
#define  MSG_DLGSRFCOL_RG_ATOM  311
#define  MSG_DLGSRFCOL_RG_COLOR  312
#define  MSG_DLGSRFCOL_BT_CHANGE  313
#define  MSG_DLGMOPAC_TITLE  314
#define  MSG_DLGMOPAC_LBL_CALC  315
#define  MSG_DLGMOPAC_LBL_CHARGE  316
#define  MSG_DLGMOPAC_LBL_OTHER  317
#define  MSG_DLGMOPAC_GRP_PARAM  318
#define  MSG_DLGMOPAC_GRP_SWITCHES  319
#define  MSG_DLGMOPAC_START  320
#define  MSG_ERR_DLGMOPAC_MOPACEXE  321
#define  MSG_ERR_DLGMOPAC_NOARC  322
#define  MSG_ERR_DLGMOPAC_TOOATM  323
#define  MSG_DLGCHG_TITLE  324
#define  MSG_DLGCHG_FFIELD  325
#define  MSG_DLGCHG_CHARGES  326
#define  MSG_DLGCHG_ERR_NOTASS  327
#define  MSG_DLGABOUT_TITLE  328
#define  MSG_DLGABOUT_COPYRIGHT  329
#define  MSG_DLGABOUT_MUSIC  330
#define  MSG_DLGCLUSTER_TITLE  331
#define  MSG_DLGCLUSTER_LBL_SOLVENT  332
#define  MSG_DLGCLUSTER_RG_BOX  333
#define  MSG_DLGCLUSTER_RG_SPHERE  334
#define  MSG_DLGCLUSTER_RG_LAYER  335
#define  MSG_DLGCLUSTER_LBL_SIDE  336
#define  MSG_DLGCLUSTER_LBL_RADIUS  337
#define  MSG_DLGCLUSTER_LBL_THICKNESS  338
#define  MSG_DLGKEYS_TITLE  339
#define  MSG_DLGKEYS_TITDES  340
#define  MSG_DLGKEYS_DESC  341
#define  MSG_PRFED_TITLE  342
#define  MSG_MOLINFO_TITLE  343
#define  MSG_MOLINFO_MAXATM  344
#define  MSG_EDFFIELD_TITLE  345
#define  MSG_EDCHARGE_TITLE  346
#define  MSG_GRAPH_YAXTITLE  347
#define  MSG_GRAPH_BT_PLOT1  348
#define  MSG_GRAPH_BT_PLOT2  349
#define  MSG_GRAPH_DLG_SAVE  350
#define  MSG_GRAPH_DLG_OPEN  351
#define  MSG_GRAPH_MM_ZOOM  352
#define  MSG_GRAPH_MM_ZOOMIN  353
#define  MSG_GRAPH_MM_ZOOMOUT  354
#define  MSG_GRAPH_MM_ZOOMRESET  355
#define  MSG_GRAPH_MM_CLUSTGRAPH  356
#define  MSG_GRAPH_MM_STATVAL  357
#define  MSG_GRAPHVAL_TITLE  358
#define  MSG_GRAPHVAL_LBL_SAMPLES  359
#define  MSG_GRAPHVAL_LBL_MIN  360
#define  MSG_GRAPHVAL_LBL_MAX  361
#define  MSG_GRAPHVAL_LBL_AVERAGE  362
#define  MSG_GRAPHVAL_LBL_STDDEV  363
#define  MSG_GRAPHCLUST_TITLE  364
#define  MSG_GRAPHCLUST_MODE  365
#define  MSG_GRAPHCLUST_MAX  366
#define  MSG_GRAPHCLUST_MIN  367
#define  MSG_GRAPHCLUST_BT_ADD  368
#define  MSG_GRAPHCLUST_BT_UPDATE  369
#define  MSG_GRAPHCLUST_BT_REMOVE  370
#define  MSG_GRAPHCLUST_BT_LOAD  371
#define  MSG_GRAPHCLUST_BT_REMALL  372
#define  MSG_GRAPHCLUST_BT_SAVE  373
#define  MSG_GRAPHCLUST_NUMBER  374
#define  MSG_GRAPHCLUST_RANGES  375
#define  MSG_GRAPHFILT_TITLE  376
#define  MSG_GRAPHFILT_SKIP  377
#define  MSG_GRAPHFILT_SEPARATOR  378
#define  MSG_GRAPHFILT_COLUMNX  379
#define  MSG_GRAPHFILT_COLUMNY  380
#define  MSG_ANA_TITLE  381
#define  MSG_ANA_GRP_FILE  382
#define  MSG_ANA_GRP_INFO  383
#define  MSG_ANA_LBL_FRAMES  384
#define  MSG_ANA_LBL_FRMSTEP  385
#define  MSG_ANA_LBL_SIMTIME  386
#define  MSG_ANA_LBL_LOWENE  387
#define  MSG_ANA_LBL_RANGE  388
#define  MSG_ANA_LBL_TEMP  389
#define  MSG_ANA_LBL_STARTTIME  390
#define  MSG_ANA_TS_SELECT  391
#define  MSG_ANA_TS_CALC  392
#define  MSG_ANA_TS_MEASURE  393
#define  MSG_ANA_TS_ANIM  394
#define  MSG_ANA_LBL_FRAME  395
#define  MSG_ANA_BT_FIRST  396
#define  MSG_ANA_BT_LAST  397
#define  MSG_ANA_BT_LOWEST  398
#define  MSG_ANA_LBL_OUTPUT  399
#define  MSG_ANA_LBL_PROPERTY  400
#define  MSG_ANA_CB_IGNOREWAT  401
#define  MSG_ANA_CB_SHOWPLOT  402
#define  MSG_ANA_LBL_MEASURE  403
#define  MSG_ANA_LBL_START  404
#define  MSG_ANA_LBL_END  405
#define  MSG_ANA_LBL_SKIP  406
#define  MSG_ANA_CB_LOOP  407
#define  MSG_ANA_LBL_SPEED  408
#define  MSG_ANA_CB_DISTANCE  409
#define  MSG_ANA_CB_ANGLE  410
#define  MSG_ANA_CB_TORSION  411
#define  MSG_ANA_CB_PLANE  412
#define  MSG_ANA_CB_MULTIPLE  413
#define  MSG_ANA_CB_DIPOLE  414
#define  MSG_ANA_CB_MLP  415
#define  MSG_ANA_CB_PSA  416
#define  MSG_ANA_CB_SRF  417
#define  MSG_ANA_CB_SRFDIA  418
#define  MSG_ANA_CB_VOL  419
#define  MSG_ANA_CB_VOLDIA  420
#define  MSG_ANA_OPENTRJ  421
#define  MSG_ERR_ANA_SELECT  422
#define  MSG_PRGBAR_TITLE  423
#define  MSG_RENRES_TITLE  424
#define  MSG_RENRES_LBL_START  425
#define  MSG_RENRES_BT_REN  426
#define  MSG_SEL_TITLE  427
#define  MSG_SEL_BT_DWNATM  428
#define  MSG_SEL_BT_NEW  429
#define  MSG_SEL_BT_REMATM  430
#define  MSG_SEL_BT_REMOVE  431
#define  MSG_SEL_BT_UPATM  432
#define  MSG_SEL_LBL_NAME  433
#define  MSG_SEL_LBL_SELATM  434
#define  MSG_SEL_LBL_SELECT  435
#define  MSG_BIODOCK_TITLE  436
#define  MSG_BIODOCK_OPENREC  437
#define  MSG_BIODOCK_OPENLIG  438
#define  MSG_BIODOCK_OPENPRJ  439
#define  MSG_BIODOCK_SAVEPRJ  440
#define  MSG_BIODOCK_MM_EXIT  441
#define  MSG_BIODOCK_MM_NOW  442
#define  MSG_BIODOCK_MM_OPEN  443
#define  MSG_BIODOCK_MM_PROJ  444
#define  MSG_BIODOCK_MM_RESET  445
#define  MSG_BIODOCK_MM_SAVEAS  446
#define  MSG_BIODOCK_MM_RUN  447
#define  MSG_BIODOCK_LBL_PRJNAME  448
#define  MSG_BIODOCK_LBL_RECNAME  449
#define  MSG_BIODOCK_LBL_LIGNAME  450
#define  MSG_BIODOCK_LBL_FFIELD  451
#define  MSG_BIODOCK_LBL_DIEL  452
#define  MSG_BIODOCK_LBL_FRAMES  453
#define  MSG_BIODOCK_CB_HBOND  454
#define  MSG_BIODOCK_CB_DISTDEP  455
#define  MSG_BIODOCK_CB_AUTOPAR  456
#define  MSG_BIODOCK_GB_RANGES  457
#define  MSG_BIODOCK_LBL_TRANS  458
#define  MSG_BIODOCK_LBL_ROT  459
#define  MSG_BIODOCK_CB_CENTER  460
#define  MSG_BIODOCK_LBL_ACTIONS  461
#define  MSG_BIODOCK_CB_DOCK  462
#define  MSG_BIODOCK_CB_EXTRA  463
#define  MSG_BIODOCK_CB_OPTIM  464
#define  MSG_BIODOCK_GB_ADVANCED  465
#define  MSG_BIODOCK_LBL_OVERLAP  466
#define  MSG_BIODOCK_LBL_CUTOFF  467
#define  MSG_BIODOCK_LBL_CUT  468
#define  MSG_BIODOCK_LBL_TPRI  469
#define  MSG_BIODOCK_LBL_ACLUST  470
#define  MSG_BIODOCK_LBL_TRASH  471
#define  MSG_BIODOCK_LBL_REFRESH  472
#define  MSG_BIODOCK_RG_METHOD  473
#define  MSG_BIODOCK_LBL_STEPS  474
#define  MSG_BIODOCK_LBL_RMS  475
#define  MSG_BIODOCK_LBL_SAVEMIN  476
#define  MSG_BIODOCK_LBL_CONV  477
#define  MSG_BIODOCK_LBL_INTER  478
#define  MSG_BIODOCK_CB_EXBESTE  479
#define  MSG_BIODOCK_CB_EXBESTF  480
#define  MSG_BIODOCK_CB_EXTRASH  481
#define  MSG_BIODOCK_CB_EXSEL  482
#define  MSG_BIODOCK_CB_EXALL  483
#define  MSG_BIODOCK_LBL_MODE  484
#define  MSG_BIODOCK_LBL_ENER  485
#define  MSG_BIODOCK_LBL_FRMNUM  486
#define  MSG_BIODOCK_LBL_SEL  487
#define  MSG_BIODOCK_STRBOX_TIT  488
#define  MSG_BIODOCK_ERR_PROMPT  489
#define  MSG_BIODOCK_ERR_TOOLINES  490
#define  MSG_BIODOCK_ERR_UNKCOM  491
#define  MSG_BIODOCK_ERR_BEGMIS  492
#define  MSG_BIODOCK_ERR_DUPCOM  493
#define  MSG_BIODOCK_ERR_ARGMIS  494
#define  MSG_BIODOCK_ERR_MACROOPENED  495
#define  MSG_BIODOCK_ERR_MACROCLOSED  496
#define  MSG_BIODOCK_ERR_MACRONOTOPENED  497
#define  MSG_BIODOCK_ERR_MACRONOTCLOSED  498
#define  MSG_BIODOCK_ERR_MACROFREQ  499
#define  MSG_BIODOCK_ERR_NUMINT  500
#define  MSG_BIODOCK_ERR_NUMPOS  501
#define  MSG_BIODOCK_ERR_NUMFLOAT  502
#define  MSG_BIODOCK_ERR_NUMBOOL  503
#define  MSG_BIODOCK_ERR_NULLNUM  504
#define  MSG_BIODOCK_ERR_NULLSTR  505
#define  MSG_BIODOCK_ERR_UNTERMSTR  506
#define  MSG_BIODOCK_ERR_TRANSPOS  507
#define  MSG_BIODOCK_ERR_ROTPOS  508
#define  MSG_BIODOCK_ERR_CONVFACT  509
#define  MSG_BIODOCK_ERR_MODULE  510
#define  MSG_BIODOCK_ERR_MISCOM  511
#define  MSG_BIODOCK_ERR_NUMFRM  512
#define  MSG_BIODOCK_ERR_FRMSELMIS  513
#define  MSG_BIODOCK_ERR_FRMSELINV  514
#define  MSG_BIODOCK_ERR_UNKEXTRA  515
#define  MSG_BIODOCK_ERR_UNKOPTMET  516
#define  MSG_STRBOX_TITLE  517
#define  MSG_TASK_TITLE  518
#define  MSG_TASK_BT_KILL  519
#define  MSG_TASK_LBL_NAME  520
#define  MSG_TASK_LBL_PID  521
#define  MSG_TASK_MM_FILE  522
#define  MSG_TASK_MM_REFRESH  523
#define  MSG_TASK_MM_EXIT  524
#define  MSG_TASK_MM_OPTIONS  525
#define  MSG_TASK_MM_FORCE  526
#define  MSG_TASK_MM_FILTER  527
#define  MSG_TASK_MM_ACTIONS  528
#define  MSG_TASK_MM_KILL  529
#define  MSG_GLPRF_ERR_DEFAULT  530
#define  MSG_MAIN_MENU_DSPVDWD  531
#define  MSG_MAIN_MENU_DSPLIQS  532
#define  MSG_POPUP_MENU_WINDOW  533
#define  MSG_GLMAIN_WINTITLE  534
#define  MSG_ERR_MAIN_INVALIDPORT  535
#define  MSG_ERR_MAIN_WIN32ONLY  536
#define  MSG_DLGCLUSTER_RG_SHAPE  537
#define  MSG_ERR_DEVIL_PROMPT  538
#define  MSG_GLMAIN_CONTITLE  539
#define  MSG_MAIN_MENU_COLBYCHAIN  540
#define  MSG_MAIN_MENU_COLBYCHARGE  541
#define  MSG_MAIN_MENU_COLBYRES  542
#define  MSG_MAIN_MENU_SELECT  543
#define  MSG_MAIN_MENU_SELALL  544
#define  MSG_MAIN_MENU_SELBACK  545
#define  MSG_MAIN_MENU_SELNOHYD  546
#define  MSG_MAIN_MENU_SELINV  547
#define  MSG_MAIN_MENU_SELNOWAT  548
#define  MSG_MAIN_MENU_DSPSET  549
#define  MSG_PDB_ATDLFOUND  550
#define  MSG_PDB_ORACFOUND  551
#define  MSG_VIEWPREFS_CYLRAD  552
#define  MSG_VIEWPREFS_CYLRES  553
#define  MSG_VIEWPREFS_DEFAULT  554
#define  MSG_VIEWPREFS_DOTSIZE  555
#define  MSG_VIEWPREFS_SPHERERES  556
#define  MSG_VIEWPREFS_SPHERESCALE  557
#define  MSG_VIEWPREFS_TITLE  558
#define  MSG_VIEWPREFS_WIRESMOOTH  559
#define  MSG_VIEWPREFS_WIRETICK  560
#define  MSG_VIEWPREFS_PGCPK  561
#define  MSG_VIEWPREFS_PGLIQ  562
#define  MSG_VIEWPREFS_PGVDW  563
#define  MSG_VIEWPREFS_PGWIREFRAME  564
#define  MSG_VIEWPREFS_ZCLIPFAR  565
#define  MSG_DLGABOUT_MAIL  566
#define  MSG_DLGABOUT_MORE  567
#define  MSG_ATMSEL_GB_RANGE  568
#define  MSG_ATMSEL_LBL_ATOMCOL  569
#define  MSG_ATMSEL_LBL_CHAINCOL  570
#define  MSG_ATMSEL_LBL_RESCOL  571
#define  MSG_ATMSEL_LBL_RESNCOL  572
#define  MSG_ATMSEL_LBL_SELECT  573
#define  MSG_ATMSEL_RG_ATMNUM  574
#define  MSG_ATMSEL_RG_MODE  575
#define  MSG_ATMSEL_RG_RESNUM  576
#define  MSG_ATMSEL_TITLE  577
#define  MSG_ATMSEL_TS_ATOM  578
#define  MSG_ATMSEL_TS_RANGE  579
#define  MSG_COLTBL_TITLE  580
#define  MSG_PLUG_TITLE  581
#define  MSG_PLUG_BT_ABOUT  582
#define  MSG_PLUG_BT_CONFIG  583
#define  MSG_PLUG_BT_HELP  584
#define  MSG_GLDEMO_ERR_NOTINST  585
#define  MSG_ERR_COM_COLUNK  586
#define  MSG_ERR_COM_EXCRANGEI  587
#define  MSG_ERR_COM_EXCRANGEF  588
#define  MSG_ERR_COM_FEWARGS  589
#define  MSG_ERR_COM_TOOARGS  590
#define  MSG_ERR_COM_ILLARG  591
#define  MSG_ERR_COM_NOCOM  592
#define  MSG_ERR_COM_NOMOL  593
#define  MSG_ERR_COM_NOTIMP  594
#define  MSG_ERR_COM_NOTRJ  595
#define  MSG_ERR_COM_UNKCOM  596
#define  MSG_ERR_COM_UNKERR  597
#define  MSG_ERR_COM_UNKKEYWORD  598
#define  MSG_ERR_DCD_NOMOL  599
#define  MSG_ERR_FMOD_PROMPT  600
#define  MSG_GLMAIN_CPUTYPE  601
#define  MSG_GLMAIN_ENABLEOPT  602
#define  MSG_GLMEAS_ANGLE  603
#define  MSG_GLMEAS_DIST  604
#define  MSG_GLMEAS_PICKATM  605
#define  MSG_GLMEAS_PLANG  606
#define  MSG_GLMEAS_TORSION  607
#define  MSG_MAIN_MENU_LASTERR  608
#define  MSG_MAIN_MENU_PLUGCONF  609
#define  MSG_MAIN_MENU_PLUGMANAGE  610
#define  MSG_MAIN_MENU_COLSEL  611
#define  MSG_MAIN_MENU_SELNONE  612
#define  MSG_MAIN_MENU_SELCUSTOM  613
#define  MSG_POPUP_MENU_JOYENABLE  614
#define  MSG_POPUP_MENU_PICK  615
#define  MSG_POPUP_MENU_PICKNONE  616
#define  MSG_POPUP_MENU_PICKANG  617
#define  MSG_POPUP_MENU_PICKATOM  618
#define  MSG_POPUP_MENU_PICKDIST  619
#define  MSG_POPUP_MENU_PICKPLANG  620
#define  MSG_POPUP_MENU_PICKTOR  621
#define  MSG_LOADER_MDLMOL  622
#define  MSG_ERR_LOCALE_ERRCODE  623
#define  MSG_ERR_LOCALE_LASTERR  624
#define  MSG_ERR_LOCALE_NOERROR  625
#define  MSG_ERR_WIN_CANTLOADLIB  626
#define  MSG_ERR_WIN_CANTGETADDR  627
#define  MSG_DLGCOPY_TITLE  628
#define  MSG_DLGCOPY_BT_COPY  629
#define  MSG_DLGCOPY_RG_MODE  630
#define  MSG_DLGINTER_TITLE  631
#define  MSG_DLGINTER_BT_EVALUATE  632
#define  MSG_DLGINTER_CB_COLLIGATM  633
#define  MSG_DLGINTER_CB_COLMOL  634
#define  MSG_DLGINTER_CB_COLRES  635
#define  MSG_DLGINTER_CB_LIGAND  636
#define  MSG_DLGINTER_CB_RESIDUES  637
#define  MSG_DLGINTER_CB_WATER  638
#define  MSG_DLGINTER_GRP_COLOR  639
#define  MSG_DLGINTER_GRP_ENERGY  640
#define  MSG_DLGINTER_GRP_VISUAL  641
#define  MSG_DLGINTER_LBL_CUTOFF  642
#define  MSG_DLGINTER_LBL_DIEL  643
#define  MSG_DLGINTER_LBL_RESIDUE  644
#define  MSG_DLGINTER_LBL_TRASH  645
#define  MSG_ERR_SRFNOMOL  646
#define  MSG_LOADER_GAMESS  647
#define  MSG_LOADER_INSSRF  648
#define  MSG_MAIN_MENU_CONSOLE  649
#define  MSG_MAIN_MENU_INTERACT  650
#define  MSG_ERR_COM_EXCRANGEUI  651
#define  MSG_ERR_COM_NOSRF  652
#define  MSG_ERR_CON_SAVEBUF  653
#define  MSG_BT_NO  654
#define  MSG_BT_YES  655
#define  MSG_MAIN_MENU_EDITATM  656
#define  MSG_DLGEDITATM_TITLE  657
#define  MSG_DLGEDITATM_BT_APPLY  658
#define  MSG_DLGEDITATM_LBL_ATMTYPE  659
#define  MSG_DLGEDITATM_LBL_CHAIN  660
#define  MSG_DLGEDITATM_LBL_CHARGE  661
#define  MSG_DLGEDITATM_LBL_CURCHAIN  662
#define  MSG_DLGEDITATM_LBL_CURRES  663
#define  MSG_DLGEDITATM_LBL_ELEM  664
#define  MSG_DLGEDITATM_LBL_NUMBER  665
#define  MSG_DLGEDITATM_LBL_NAME  666
#define  MSG_DLGEDITATM_LBL_RESNAME  667
#define  MSG_DLGEDITATM_LBL_RESSEQ  668
#define  MSG_DLGEDITATM_RG_ALLATM  669
#define  MSG_DLGEDITATM_RG_APPLYTO  670
#define  MSG_DLGEDITATM_RG_VIS  671
#define  MSG_DLGEDITATM_TS_ATOM  672
#define  MSG_DLGEDITATM_TS_CHAIN  673
#define  MSG_DLGEDITATM_TS_RESIDUE  674
#define  MSG_DLGMULTISEL_TITLE  675
#define  MSG_VIEWPREFS_SMARTATOMS  676
#define  MSG_VIEWPREFS_SMARTMOVE  677
#define  MSG_ANA_BT_ENEGRAPH  678
#define  MSG_ANA_CB_ILM  679
#define  MSG_ANA_CB_VISONLY  680
#define  MSG_ANA_INTENERGY  681
#define  MSG_ANA_LBL_ENERGY  682
#define  MSG_ANA_POTENERGY  683
#define  MSG_ANA_SELCONF  684
#define  MSG_BIODOCK_END  685
#define  MSG_BIODOCK_START  686
#define  MSG_ERR_BIODOCK_BIODOCKEXE  687
#define  MSG_ERR_BIODOCK_BIOOUT  688
#define  MSG_ERR_BIODOCK_BIOPRJ  689
#define  MSG_ERR_BIODOCK_STRUCTOUT  690
#define  MSG_GLMEAS_CENTER  691
#define  MSG_MAIN_MENU_COLBYSEG  692
#define  MSG_POPUP_MENU_PICKCENTER  693
#define  MSG_GRAPH_MM_SELCONF  694
#define  MSG_ERR_GRAPH_POINTRANGE  695
#define  MSG_LOADER_ALCHEMY  696
#define  MSG_LOADER_CSVSRF  697
#define  MSG_MINIED_SAVECHG  698
#define  MSG_DLG3DCTRL_BT_ROTX  699
#define  MSG_DLG3DCTRL_BT_ROTY  700
#define  MSG_DLG3DCTRL_BT_ROTZ  701
#define  MSG_DLG3DCTRL_BT_SCALE  702
#define  MSG_DLG3DCTRL_TITLE  703
#define  MSG_DLG3DCTRL_BT_TRANSX  704
#define  MSG_DLG3DCTRL_BT_TRANSY  705
#define  MSG_DLG3DCTRL_BT_TRANSZ  706
#define  MSG_BT_ADD  707
#define  MSG_BT_ALL  708
#define  MSG_BT_APPLY  709
#define  MSG_BT_AUTO  710
#define  MSG_BT_BUILD  711
#define  MSG_BT_MERGE  712
#define  MSG_BT_MOVE  713
#define  MSG_BT_NEW  714
#define  MSG_BT_NONE  715
#define  MSG_BT_REMOVE  716
#define  MSG_BT_RESET  717
#define  MSG_BT_SELECT  718
#define  MSG_BT_SWAP  719
#define  MSG_OVERWRITE  720
#define  MSG_ANA_CB_LIPBROTO  721
#define  MSG_ANA_CB_LIPCRIPPEN  722
#define  MSG_ERR_ANA_GRAPH  723
#define  MSG_DLGATMADD_LBL_OTHER  724
#define  MSG_DLGATMADD_LBL_SELECTED  725
#define  MSG_DLGATMADD_RG_ELEM  726
#define  MSG_DLGATMADD_RG_HYBRID  727
#define  MSG_DLGATMADD_TITLE  728
#define  MSG_ATMSEL_BT_HYDROG  729
#define  MSG_ATMSEL_BT_WATER  730
#define  MSG_ATMSEL_LBL_MOLCOL  731
#define  MSG_ATMSEL_LBL_PROXRAD  732
#define  MSG_ATMSEL_RG_PROXMODE  733
#define  MSG_ATMSEL_RG_PROXMODEATM  734
#define  MSG_ATMSEL_RG_PROXMODECHAIN  735
#define  MSG_ATMSEL_RG_PROXMODEMOL  736
#define  MSG_ATMSEL_RG_PROXMODERES  737
#define  MSG_ATMSEL_RG_PROXWHAT  738
#define  MSG_ATMSEL_RG_PROXWHATATM  739
#define  MSG_ATMSEL_RG_PROXWHATRES  740
#define  MSG_ATMSEL_RG_SELMODE  741
#define  MSG_ATMSEL_RG_SELMODEADD  742
#define  MSG_ATMSEL_RG_SELMODENORM  743
#define  MSG_ATMSEL_TS_PROXIMITY  744
#define  MSG_BOND_ASSIGNPROG  745
#define  MSG_DLGBOND_LBL_CONNTOL  746
#define  MSG_DLGBOND_LBL_NONE  747
#define  MSG_DLGBOND_LBL_SELBOND  748
#define  MSG_DLGBOND_RG_ALLATM  749
#define  MSG_DLGBOND_RG_BND  750
#define  MSG_DLGBOND_RG_BNDDOUBLE  751
#define  MSG_DLGBOND_RG_BNDPARDOUBLE  752
#define  MSG_DLGBOND_RG_BNDSINGLE  753
#define  MSG_DLGBOND_RG_BNDTRIPLE  754
#define  MSG_DLGBOND_RG_CONNALLATM  755
#define  MSG_DLGBOND_RG_CONNSELATM  756
#define  MSG_DLGBOND_RG_FINDALLATM  757
#define  MSG_DLGBOND_RG_FINDSELATM  758
#define  MSG_DLGBOND_RG_ONEBOND  759
#define  MSG_DLGBOND_RG_SELATM  760
#define  MSG_DLGBOND_RG_WHAT  761
#define  MSG_DLGBOND_TITLE  762
#define  MSG_DLGBOND_TS_ADD  763
#define  MSG_DLGBOND_TS_CHANGE  764
#define  MSG_DLGBOND_TS_REMOVE  765
#define  MSG_WARN_DLGBOND_ALREADYCONN  766
#define  MSG_WARN_DLGBOND_ALREADYNOTCONN  767
#define  MSG_WARN_DLGBOND_NOTCONN  768
#define  MSG_DLGCENTR_HC_ATOMS  769
#define  MSG_DLGCENTR_LBL_NAME  770
#define  MSG_DLGCENTR_TITLE  771
#define  MSG_WARN_DLGCENTR_ALREADY  772
#define  MSG_WARN_DLGCENTR_NOCENTR  773
#define  MSG_DLGCLUSTBLD_BEGIN  774
#define  MSG_DLGCLUSTBLD_CB_ENABLE  775
#define  MSG_DLGCLUSTBLD_CB_MINIMIZE  776
#define  MSG_DLGCLUSTBLD_CB_REMOVE  777
#define  MSG_DLGCLUSTBLD_CB_SAME  778
#define  MSG_DLGCLUSTBLD_CB_VIEWCELL  779
#define  MSG_DLGCLUSTBLD_GRP_BOXSIZE  780
#define  MSG_DLGCLUSTBLD_GRP_COPIES  781
#define  MSG_DLGCLUSTBLD_GRP_EXTRA  782
#define  MSG_DLGCLUSTBLD_GRP_OPTIM  783
#define  MSG_DLGCLUSTBLD_GRP_RNDROT  784
#define  MSG_DLGCLUSTBLD_LBL_OVERLAP  785
#define  MSG_DLGCLUSTBLD_LBL_RMS  786
#define  MSG_DLGCLUSTBLD_LBL_STEPS  787
#define  MSG_DLGCLUSTBLD_PROGBAR  788
#define  MSG_DLGCLUSTBLD_RMMOL  789
#define  MSG_DLGCLUSTBLD_TITLE  790
#define  MSG_DLGCLUSTBLD_TOTMOL  791
#define  MSG_DLGCLUSTBLD_UNABLEOPTIM  792
#define  MSG_ERR_COM_EXCRANGEC  793
#define  MSG_DCD_STARTLIPOLE  794
#define  MSG_ERR_DCD_NOOUT  795
#define  MSG_ERR_ESCHER_CORRFILE  796
#define  MSG_ERR_ESCHER_NUMSOL  797
#define  MSG_ERR_ESCHER_UNKFORMAT  798
#define  MSG_DLGEDITATM_CB_COLOR  799
#define  MSG_DLGFIXATM_GRP_COLORS  800
#define  MSG_DLGFIXATM_GRP_PARAM  801
#define  MSG_DLGFIXATM_LBL_VALUE  802
#define  MSG_DLGFIXATM_RG_MODE  803
#define  MSG_DLGFIXATM_RG_MODEFIX  804
#define  MSG_DLGFIXATM_RG_MODEFREE  805
#define  MSG_DLGFIXATM_RG_MODEVALUE  806
#define  MSG_DLGFIXATM_RG_SELECTION  807
#define  MSG_DLGFIXATM_RG_SELALLATM  808
#define  MSG_DLGFIXATM_RG_SELBACKBONE  809
#define  MSG_DLGFIXATM_RG_SELVISATM  810
#define  MSG_DLGFIXATM_TITLE  811
#define  MSG_GAMESS_ENERGY  812
#define  MSG_GLMEAS_CENTERWARN  813
#define  MSG_MAIN_MENU_BIOTOOLS  814
#define  MSG_MAIN_MENU_BIORES  815
#define  MSG_MAIN_MENU_SIMIL  816
#define  MSG_MAIN_MENU_CHANGE  817
#define  MSG_MAIN_MENU_CHGTOR  818
#define  MSG_MAIN_MENU_ADD  819
#define  MSG_MAIN_MENU_ATOM  820
#define  MSG_MAIN_MENU_BONDS  821
#define  MSG_MAIN_MENU_CENTROID  822
#define  MSG_MAIN_MENU_IONS  823
#define  MSG_MAIN_MENU_BUILD  824
#define  MSG_MAIN_MENU_BLDCLUST  825
#define  MSG_MAIN_MENU_COORDS  826
#define  MSG_MAIN_MENU_APPLYTRANS  827
#define  MSG_MAIN_MENU_INVERTZ  828
#define  MSG_MAIN_MENU_RMCENTR  829
#define  MSG_MAIN_MENU_RMINVISATM  830
#define  MSG_MAIN_MENU_RMMOL  831
#define  MSG_MAIN_MENU_RMSEG  832
#define  MSG_MAIN_MENU_SEG  833
#define  MSG_MAIN_MENU_SEGFIX  834
#define  MSG_MAIN_MENU_SEGMERGE  835
#define  MSG_MAIN_MENU_FIXSAV  836
#define  MSG_MAIN_MENU_MERGE  837
#define  MSG_MAIN_MENU_3DCTRL  838
#define  MSG_MAIN_MENU_COLBYCONSTR  839
#define  MSG_MAIN_MENU_LBLATM_FIX  840
#define  MSG_MAIN_MENU_LBLATM_RESNAME  841
#define  MSG_MAIN_MENU_LBLATM_RESNAMSEQ  842
#define  MSG_MAIN_MENU_LBLATM_RESSEQ  843
#define  MSG_MAIN_MENU_SELMOL  844
#define  MSG_MAIN_MENU_SELSEG  845
#define  MSG_INFO_PREDCHARGE  846
#define  MSG_ION_EXCLUSIONMAP  847
#define  MSG_ION_GRIDNODES  848
#define  MSG_ION_GRIDSIZE  849
#define  MSG_ION_MOLDIM  850
#define  MSG_ION_PLACED  851
#define  MSG_ION_POSITION  852
#define  MSG_ION_START  853
#define  MSG_DLGEDITATM_LBL_FIX  854
#define  MSG_DLGIONS_BT_NEUTRALIZE  855
#define  MSG_DLGIONS_FIXCHARGES  856
#define  MSG_DLGIONS_LBL_BOXTHICK  857
#define  MSG_DLGIONS_LBL_CHARGE  858
#define  MSG_DLGIONS_LBL_ELEMENT  859
#define  MSG_DLGIONS_LBL_EXCLATOMRAD  860
#define  MSG_DLGIONS_LBL_EXCLIONRAD  861
#define  MSG_DLGIONS_LBL_GRIDSTEP  862
#define  MSG_DLGIONS_LBL_ION  863
#define  MSG_DLGIONS_LBL_MOLCHARGE  864
#define  MSG_DLGIONS_LBL_NIONS  865
#define  MSG_DLGIONS_TITLE  866
#define  MSG_DLGMERGE_CB_ATMNAMES  867
#define  MSG_DLGMERGE_CB_ATMTYPES  868
#define  MSG_DLGMERGE_CB_CHAINID  869
#define  MSG_DLGMERGE_CB_CHARGES  870
#define  MSG_DLGMERGE_CB_CONN  871
#define  MSG_DLGMERGE_CB_CONSTRAINTS  872
#define  MSG_DLGMERGE_CB_COORD  873
#define  MSG_DLGMERGE_CB_ELEM  874
#define  MSG_DLGMERGE_CB_MOLID  875
#define  MSG_DLGMERGE_CB_RESNAMES  876
#define  MSG_DLGMERGE_CB_RESSEQ  877
#define  MSG_DLGMERGE_CB_SEGMENTS  878
#define  MSG_DLGMERGE_LBL_SEL  879
#define  MSG_DLGMERGE_OPEN  880
#define  MSG_DLGMERGE_TITLE  881
#define  MSG_DLGMERGE_ERR_NOTMERGE  882
#define  MSG_DLGMERGE_ERR_UNALLELEM  883
#define  MSG_DLGMERGE_ERR_UNCOMPMOL  884
#define  MSG_DLGMERGE_WARN_CONT  885
#define  MSG_DLGMVMOL_CONNMOL  886
#define  MSG_DLGMVMOL_CONNWORLD  887
#define  MSG_DLGMVMOL_TITLE  888
#define  MSG_DLGOVER_BT_RETRY  889
#define  MSG_DLGOVER_BT_SUPERIMP  890
#define  MSG_DLGOVER_HC_SEL  891
#define  MSG_DLGOVER_HC_SOURCE  892
#define  MSG_DLGOVER_HC_TARGET  893
#define  MSG_DLGOVER_RMS  894
#define  MSG_DLGOVER_TITLE  895
#define  MSG_WARN_PDB_NOCONN  896
#define  MSG_DLGRMMOL_HC_ATOMS  897
#define  MSG_DLGRMMOL_HC_END  898
#define  MSG_DLGRMMOL_HC_MOLECULE  899
#define  MSG_DLGRMMOL_HC_RESIDUE  900
#define  MSG_DLGRMMOL_HC_SEGMENT  901
#define  MSG_DLGRMMOL_HC_START  902
#define  MSG_DLGRMMOL_TITLE  903
#define  MSG_DLGRMSEG_TITLE  904
#define  MSG_DLGSELMOL_TITLE  905
#define  MSG_DLGSELSEG_TITLE  906
#define  MSG_DLGTOR_BT_CHANGE  907
#define  MSG_DLGTOR_HC_TORSION  908
#define  MSG_DLGTOR_LBL_CONTROL  909
#define  MSG_DLGTOR_LBL_TORVAL  910
#define  MSG_DLGTOR_TITLE  911
#define  MSG_GASTEIGER_ERR_NOPAR  912
#define  MSG_GLMEAS_ALREADYPICKED  913
#define  MSG_GLMEAS_RESET  914
#define  MSG_GLMEAS_ERR_ATMMULTI  915
#define  MSG_GLMEAS_ERR_ATMNOTFND  916
#define  MSG_ION_PROGBAR_ENER  917
#define  MSG_ION_PROGBAR_INIT  918
#define  MSG_ION_PROGBAR_PLACE  919
#define  MSG_POPUP_MENU_MOVE  920
#define  MSG_POPUP_MENU_MOVEALL  921
#define  MSG_POPUP_MENU_MOVEMOL  922
#define  MSG_POPUP_MENU_PICKNOMON  923
#define  MSG_POPUP_MENU_PICKRESET  924
#define  MSG_REM_ATOMS  925
#define  MSG_REM_NOATOM  926
#define  MSG_SEG_FOUND  927
#define  MSG_SEG_NOTFOUND  928
#define  MSG_SEL_OPENTITLE  929
#define  MSG_SEL_SAVETITLE  930
#define  MSG_SEL_ERR_DONTMATCH  931
#define  MSG_SEL_ERR_ILLNUMARG  932
#define  MSG_SEL_ERR_UNKTAG  933
#define  MSG_VIEWPREFS_WIRESMULTIVEC  934
#define  MSG_DLG3DCTRL_LBL_WRKSPC  935
#define  MSG_DLG3DCTRL_PM_WKSNEW  936
#define  MSG_DLG3DCTRL_PM_WKSREMOVE  937
#define  MSG_DLG3DCTRL_PM_WKSREMOVEALL  938
#define  MSG_ADDH_PROG  939
#define  MSG_ADDH_WARN  940
#define  MSG_BT_CLOSEALL  941
#define  MSG_BT_GET  942
#define  MSG_BT_FINISH  943
#define  MSG_BT_NEXT  944
#define  MSG_BT_OPEN  945
#define  MSG_BT_PREVIOUS  946
#define  MSG_BT_PUT  947
#define  MSG_BT_RENAME  948
#define  MSG_BT_RESCAN  949
#define  MSG_BT_UPDATE  950
#define  MSG_DBENGINE_CREATE  951
#define  MSG_DBENGINE_GET  952
#define  MSG_DBENGINE_OPEN  953
#define  MSG_DBENGINE_OPENMOL  954
#define  MSG_DBENGINE_PUT  955
#define  MSG_DBENGINE_REMOVE  956
#define  MSG_DBENGINE_REN  957
#define  MSG_DBENGINE_UPDATE  958
#define  MSG_ERR_DBENGINE_CREATE  959
#define  MSG_ERR_DBENGINE_GET  960
#define  MSG_ERR_DBENGINE_MOLLIST  961
#define  MSG_ERR_DBENGINE_OPEN  962
#define  MSG_ERR_DBENGINE_PUT  963
#define  MSG_ERR_DBENGINE_REMOVE  964
#define  MSG_ERR_DBENGINE_REN  965
#define  MSG_ERR_DBENGINE_UNKFOR  966
#define  MSG_ERR_DBENGINE_UPDATE  967
#define  MSG_ERR_DBENGINE_ZIP32INIT  968
#define  MSG_WARN_DBENGINE_PUT  969
#define  MSG_WARN_DBENGINE_UPDATE  970
#define  MSG_DLGADDFRAG_HC_DBASE  971
#define  MSG_DLGADDFRAG_HC_FRAG  972
#define  MSG_DLGADDFRAG_LBL_FRAGMENT  973
#define  MSG_DLGADDFRAG_LBL_PICKFRAGH  974
#define  MSG_DLGADDFRAG_LBL_PICKEDFRAG  975
#define  MSG_DLGADDFRAG_LBL_PICKEDFRAGH  976
#define  MSG_DLGADDFRAG_LBL_PICKMOLH  977
#define  MSG_DLGADDFRAG_LBL_PICKEDMOLH  978
#define  MSG_DLGADDFRAG_TITLE  979
#define  MSG_DLGADDFRAG_LBL_TOR  980
#define  MSG_DLGADDFRAG_LBL_TORSION  981
#define  MSG_DLGADDHYD_CB_IUPAC  982
#define  MSG_DLGADDHYD_RG_HYDPOS  983
#define  MSG_DLGADDHYD_RG_HYDPOS_ATM  984
#define  MSG_DLGADDHYD_RG_HYDPOS_RES  985
#define  MSG_DLGADDHYD_RG_MOLTYPE  986
#define  MSG_DLGADDHYD_RG_MOLTYPE_GEN  987
#define  MSG_DLGADDHYD_RG_MOLTYPE_NA  988
#define  MSG_DLGADDHYD_RG_MOLTYPE_PROT  989
#define  MSG_DLGADDHYD_TITLE  990
#define  MSG_ANA_CB_RMSD  991
#define  MSG_DLGANA_OPENTRJFILTER  992
#define  MSG_DLGCENTR_CB_FIXED  993
#define  MSG_DLGCLUSTER_GRP_COORD  994
#define  MSG_DLGCLUSTER_RG_CLUSTPOS  995
#define  MSG_DLGCLUSTER_RG_CLUSTPOSCUST  996
#define  MSG_DLGCLUSTER_RG_CLUSTPOSGEO  997
#define  MSG_DLGDBEXP_DEFMOLNAME  998
#define  MSG_DLGDBEXP_HC_DBASE  999
#define  MSG_DLGDBEXP_HC_MOL  1000
#define  MSG_DLGDBEXP_RG_MODE  1001
#define  MSG_DLGDBEXP_RG_MODEADD  1002
#define  MSG_DLGDBEXP_RG_MODEREP  1003
#define  MSG_DLGDBEXP_RG_MODEWKS  1004
#define  MSG_DLGDBEXP_REMOVE  1005
#define  MSG_DLGDBEXP_TITLE  1006
#define  MSG_DLGDBOPEN_BT_CREATE  1007
#define  MSG_DLGDBOPEN_CB_FDIR  1008
#define  MSG_DLGDBOPEN_CB_FSDF  1009
#define  MSG_DLGDBOPEN_CB_FZIP  1010
#define  MSG_DLGDBOPEN_CK_MOLCONN  1011
#define  MSG_DLGDBOPEN_CK_MOLFIX  1012
#define  MSG_DLGDBOPEN_FILTER  1013
#define  MSG_DLGDBOPEN_LBL_FORMAT  1014
#define  MSG_DLGDBOPEN_LBL_MOLCOMPR  1015
#define  MSG_DLGDBOPEN_LBL_MOLTYPE  1016
#define  MSG_DLGDBOPEN_LBL_NEWDB  1017
#define  MSG_DLGDBOPEN_TITLE  1018
#define  MSG_GRAPH_OPENFILTER  1019
#define  MSG_GRAPH_SAVEFILTER  1020
#define  MSG_ERR_GRAPH_ILLID  1021
#define  MSG_ERR_GRAPH_NOCUR  1022
#define  MSG_DLGINTER_CB_RESWORSE  1023
#define  MSG_DLGINTER_CB_COLRESENE  1024
#define  MSG_DLGMOLPLACE_BT_APPEND  1025
#define  MSG_DLGMOLPLACE_BT_REPLACE  1026
#define  MSG_DLGMOLPLACE_BT_WORKSPACE  1027
#define  MSG_DLGMOLPLACE_CB_SAME  1028
#define  MSG_DLGMOLPLACE_TITLE  1029
#define  MSG_DLGREMRES_CB_KEEPOPEN  1030
#define  MSG_DLGREMRES_HC_CHAIN  1031
#define  MSG_DLGREMRES_HC_MOLECULE  1032
#define  MSG_DLGREMRES_HC_RESNAM  1033
#define  MSG_DLGREMRES_HC_RESSEQ  1034
#define  MSG_DLGREMRES_TITLE  1035
#define  MSG_GLDEMO_LBLATOM  1036
#define  MSG_GLDEMO_LBLCHG  1037
#define  MSG_GLDEMO_LBLELEM  1038
#define  MSG_GLDEMO_LOADING  1039
#define  MSG_GLDEMO_MOL1  1040
#define  MSG_GLDEMO_MOL2  1041
#define  MSG_GLDEMO_MOL3  1042
#define  MSG_GLDEMO_MOL3_ATOMS  1043
#define  MSG_GLDEMO_MOL3_VECTORS  1044
#define  MSG_GLDEMO_MOL3_WATERS  1045
#define  MSG_GLDEMO_SOLVAT  1046
#define  MSG_GLDEMO_SONGTITLE  1047
#define  MSG_GLDEMO_SURFMEP  1048
#define  MSG_GLDEMO_SURFVDW  1049
#define  MSG_GLDEMO_VIEWCPKSOLID  1050
#define  MSG_GLDEMO_VIEWCPKWIRE  1051
#define  MSG_GLDEMO_VIEWLIQUORICE  1052
#define  MSG_GLDEMO_VIEWVDWDOT  1053
#define  MSG_GLDEMO_VIEWVDWSOLID  1054
#define  MSG_GLDEMO_VIEWVDWWIRE  1055
#define  MSG_ERR_HYB_ATMNAMEMIS  1056
#define  MSG_ERR_HYB_ATMNAMETOOLONG  1057
#define  MSG_ERR_HYB_ATMTYPMIS  1058
#define  MSG_ERR_HYB_ATMTYPTOOLONG  1059
#define  MSG_ERR_HYB_RESNAMETOOLONG  1060
#define  MSG_ERR_HYB_UNKFORMAT  1061
#define  MSG_ERR_HYB_WARNCODEILL  1062
#define  MSG_ERR_HYB_WARNCODEMIS  1063
#define  MSG_ERR_MAIN_MOLTYPE  1064
#define  MSG_MAIN_MENU_ADDFRAG  1065
#define  MSG_MAIN_MENU_ADDHYD  1066
#define  MSG_MAIN_MENU_MOLECULES  1067
#define  MSG_MAIN_MENU_RMRES  1068
#define  MSG_MAIN_MENU_DATABASE  1069
#define  MSG_MAIN_MENU_DBEXPLORE  1070
#define  MSG_MAIN_MENU_DBOPEN  1071
#define  MSG_WKS_CLIP  1072
#define  MSG_WKS_EMPTY  1073
#define  MSG_WKS_REMCONT  1074
#define  MSG_WKS_REMALLCONT  1075
#define  MSG_LOADER_CHEM3D  1076
#define  MSG_MOL_FOUND  1077
#define  MSG_MOL_MERGE  1078
#define  MSG_MOL_NOMERGE  1079
#define  MSG_MOL_NOTFOUND  1080
#define  MSG_SEG_MERGE  1081
#define  MSG_SEG_NOMERGE  1082
#define  MSG_WARN_SOLVATE_SHELLDIM  1083
#define  MSG_BT_DEFAULT  1084
#define  MSG_CHKCHIR_HEADER  1085
#define  MSG_CHKCHIR_NODFOUND  1086
#define  MSG_CHKCHIR_WARNING  1087
#define  MSG_CHKINTER_HEADER  1088
#define  MSG_CHKINTER_NODFOUND  1089
#define  MSG_CHKRESSEQ_TOTINTER  1090
#define  MSG_CHKPEPT_HEADER  1091
#define  MSG_CHKPEPT_NOCISFOUND  1092
#define  MSG_CHKRESSEQ_HEADER  1093
#define  MSG_CHKRESSEQ_NOMIS  1094
#define  MSG_CHKRESSEQ_TOTMIS  1095
#define  MSG_ERR_COM_SRFINVNUM  1096
#define  MSG_BT_ADVANCED  1097
#define  MSG_BT_HIDEALL  1098
#define  MSG_BT_LOAD  1099
#define  MSG_BT_REMALL  1100
#define  MSG_BT_SAVE  1101
#define  MSG_BT_SHOWALL  1102
#define  MSG_LBL_NO  1103
#define  MSG_LBL_YES  1104
#define  MSG_OPEN_FILTER  1105
#define  MSG_OPEN_FILTER_TEXT  1106
#define  MSG_DLGACT_BT_COPY  1107
#define  MSG_DLGACT_BT_CONNECT  1108
#define  MSG_DLGACT_LBL_MES0  1109
#define  MSG_DLGACT_LBL_MES1  1110
#define  MSG_DLGACT_LBL_MES2  1111
#define  MSG_DLGACT_LBL_MES3  1112
#define  MSG_DLGACT_LBL_MES4  1113
#define  MSG_DLGACT_LBL_MES5  1114
#define  MSG_DLGACT_LICMES  1115
#define  MSG_DLGACT_TITLE  1116
#define  MSG_ERR_DLGACT_EXPIRED  1117
#define  MSG_ERR_DLGACT_INVCRC  1118
#define  MSG_ERR_DLGACT_INVHOST  1119
#define  MSG_ERR_DLGACT_INVSIGN  1120
#define  MSG_ANA_CB_GYRRAD  1121
#define  MSG_DLGATMADD_RG_PLACE  1122
#define  MSG_DLGATMADD_RG_PLACEATM  1123
#define  MSG_DLGATMADD_RG_PLACEMOL  1124
#define  MSG_DLGATMADD_RG_PLACERES  1125
#define  MSG_DLGADDFRAG_WKS_NAME  1126
#define  MSG_ATMSEL_CB_COLOR  1127
#define  MSG_ATMSEL_CB_DISNOCHANGE  1128
#define  MSG_ATMSEL_GRP_DISPLAY  1129
#define  MSG_DLGBOND_LBL_SELANGLE  1130
#define  MSG_DLGBOND_TS_SWAP  1131
#define  MSG_DLGCLUSTER_SRFNAME  1132
#define  MSG_DLGCLUSTER_CB_SHOWSHAPE  1133
#define  MSG_DLGCOLORSET_CB_LOGOP_BOTTOMC  1134
#define  MSG_DLGCOLORSET_CB_LOGOP_BOTTOML  1135
#define  MSG_DLGCOLORSET_CB_LOGOP_BOTTOMR  1136
#define  MSG_DLGCOLORSET_CB_LOGOP_CENTERC  1137
#define  MSG_DLGCOLORSET_CB_LOGOP_CENTERL  1138
#define  MSG_DLGCOLORSET_CB_LOGOP_CENTERR  1139
#define  MSG_DLGCOLORSET_CB_LOGOP_NONE  1140
#define  MSG_DLGCOLORSET_CB_LOGOP_TOPC  1141
#define  MSG_DLGCOLORSET_CB_LOGOP_TOPL  1142
#define  MSG_DLGCOLORSET_CB_LOGOP_TOPR  1143
#define  MSG_DLGCOLORSET_CB_SKYENABLED  1144
#define  MSG_DLGCOLORSET_GRP_ATMLABELS  1145
#define  MSG_DLGCOLORSET_GRP_SKYVISION  1146
#define  MSG_DLGCOLORSET_LBL_ATMLABELS  1147
#define  MSG_DLGCOLORSET_LBL_ATMUNKPOT  1148
#define  MSG_DLGCOLORSET_LBL_BACKCOL  1149
#define  MSG_DLGCOLORSET_LBL_COLANGLE  1150
#define  MSG_DLGCOLORSET_LBL_COLBR  1151
#define  MSG_DLGCOLORSET_LBL_COLC  1152
#define  MSG_DLGCOLORSET_LBL_COLCL  1153
#define  MSG_DLGCOLORSET_LBL_COLDIST  1154
#define  MSG_DLGCOLORSET_LBL_COLF  1155
#define  MSG_DLGCOLORSET_LBL_COLGEN  1156
#define  MSG_DLGCOLORSET_LBL_COLH  1157
#define  MSG_DLGCOLORSET_LBL_COLI  1158
#define  MSG_DLGCOLORSET_LBL_COLN  1159
#define  MSG_DLGCOLORSET_LBL_COLO  1160
#define  MSG_DLGCOLORSET_LBL_COLP  1161
#define  MSG_DLGCOLORSET_LBL_COLPLANG  1162
#define  MSG_DLGCOLORSET_LBL_COLPLANGCUBE  1163
#define  MSG_DLGCOLORSET_LBL_COLS  1164
#define  MSG_DLGCOLORSET_LBL_COLSHAPE  1165
#define  MSG_DLGCOLORSET_LBL_COLSUPLINE  1166
#define  MSG_DLGCOLORSET_LBL_COLTEXT  1167
#define  MSG_DLGCOLORSET_LBL_COLTOR  1168
#define  MSG_DLGCOLORSET_LBL_COLXC  1169
#define  MSG_DLGCOLORSET_LBL_LOGO  1170
#define  MSG_DLGCOLORSET_LBL_SKYOFFSETX  1171
#define  MSG_DLGCOLORSET_LBL_SKYOFFSETY  1172
#define  MSG_DLGCOLORSET_LBL_SKYROTX  1173
#define  MSG_DLGCOLORSET_LBL_SKYROTY  1174
#define  MSG_DLGCOLORSET_LBL_SKYROTZ  1175
#define  MSG_DLGCOLORSET_LBL_SKYTEX  1176
#define  MSG_DLGCOLORSET_TITLE  1177
#define  MSG_DLGCOLORSET_TS_ATOMS  1178
#define  MSG_DLGCOLORSET_TS_BACKGROUND  1179
#define  MSG_DLGCOLORSET_TS_MONITORS  1180
#define  MSG_DLGCOLORSET_TS_RESIDUES  1181
#define  MSG_DLGDBEXP_MOLNAME  1182
#define  MSG_DLGENETYPE_ANGLE  1183
#define  MSG_DLGENETYPE_BOND  1184
#define  MSG_DLGENETYPE_BOUNDARY  1185
#define  MSG_DLGENETYPE_DIHEDRAL  1186
#define  MSG_DLGENETYPE_ELECTROSTATIC  1187
#define  MSG_DLGENETYPE_IMPROPER  1188
#define  MSG_DLGENETYPE_KINETIC  1189
#define  MSG_DLGENETYPE_MISC  1190
#define  MSG_DLGENETYPE_POTENTIAL  1191
#define  MSG_DLGENETYPE_TEMPERATURE  1192
#define  MSG_DLGENETYPE_TITLE  1193
#define  MSG_DLGENETYPE_TOTAL  1194
#define  MSG_DLGENETYPE_VANDERWAALS  1195
#define  MSG_DLGFIXATM_RG_SELINVISATM  1196
#define  MSG_DLGIMGSAVE_CB_RES_CUSTOM  1197
#define  MSG_DLGIMGSAVE_CK_EMBOSS  1198
#define  MSG_DLGIMGSAVE_CK_GAUSSBLUR  1199
#define  MSG_DLGIMGSAVE_CK_GRAYSCALE  1200
#define  MSG_DLGIMGSAVE_GRP_EFFECTS  1201
#define  MSG_DLGIMGSAVE_GRP_RESOLUTION  1202
#define  MSG_DLGIMGSAVE_RG_RMODE  1203
#define  MSG_DLGIMGSAVE_RG_RMODE_HARD  1204
#define  MSG_DLGIMGSAVE_RG_RMODE_SNAP  1205
#define  MSG_DLGIMGSAVE_RG_RMODE_SOFT  1206
#define  MSG_LIGHT_CB_AMBIENT  1207
#define  MSG_LIGHT_CB_LIGHTEN  1208
#define  MSG_LIGHT_CB_LIGHTINGEN  1209
#define  MSG_LIGHT_CB_VECTORSPEC  1210
#define  MSG_LIGHT_GRP_AMBIENT  1211
#define  MSG_LIGHT_GRP_COLORS  1212
#define  MSG_LIGHT_GRP_MATERIAL  1213
#define  MSG_LIGHT_GRP_POSITION  1214
#define  MSG_LIGHT_LBL_AMBIENTCOL  1215
#define  MSG_LIGHT_LBL_DIFFUSECOL  1216
#define  MSG_LIGHT_LBL_MATSHINY  1217
#define  MSG_LIGHT_LBL_MATSPECULAR  1218
#define  MSG_LIGHT_LBL_SPECULARCOL  1219
#define  MSG_LIGHT_RG_LIGHTS  1220
#define  MSG_LIGHT_RG_LIGHT  1221
#define  MSG_LIGHT_TITLE  1222
#define  MSG_DLGMAIN_OD_FILTER  1223
#define  MSG_DLGMAIN_OD_FILTERMRG  1224
#define  MSG_DLGMAIN_MOVE_ALL  1225
#define  MSG_DLGMAIN_MOVE_MOL  1226
#define  MSG_DLGMAIN_MOVE_SEG  1227
#define  MSG_DLGMAIN_MOVE_RES  1228
#define  MSG_DLGMAIN_SB_ANGLE  1229
#define  MSG_DLGMAIN_SB_ATOM  1230
#define  MSG_DLGMAIN_SB_CENTER  1231
#define  MSG_DLGMAIN_SB_DISTANCE  1232
#define  MSG_DLGMAIN_SB_PLANG  1233
#define  MSG_DLGMAIN_SB_ROTATE  1234
#define  MSG_DLGMAIN_SB_SCALE  1235
#define  MSG_DLGMAIN_SB_TORSION  1236
#define  MSG_DLGMAIN_SB_TRANSLATE  1237
#define  MSG_DLGMAIN_SD_FILTERBMP  1238
#define  MSG_DLGMAIN_SD_FILTERSAVE  1239
#define  MSG_DLGMAIN_SD_FILTERSRF  1240
#define  MSG_DLGMAIN_TBT_MENU_TITLE  1241
#define  MSG_DLGMAIN_TBT_STANDARD_TITLE  1242
#define  MSG_DLGMAIN_TBT_TOOLS_TITLE  1243
#define  MSG_DLGMINIED_OD_FILTER  1244
#define  MSG_DLGMINIED_OD_TITLE  1245
#define  MSG_DLGMINIED_SD_FILTER  1246
#define  MSG_DLGMINIED_SD_TITLE  1247
#define  MSG_MOLSAVE_GRP_OPTIONS  1248
#define  MSG_DLGMVRES_CONNRES  1249
#define  MSG_DLGMVRES_TITLE  1250
#define  MSG_DLGMVSEG_CONNSEG  1251
#define  MSG_DLGMVSEG_TITLE  1252
#define  MSG_DLGPROGBAR_WAIT  1253
#define  MSG_DLGRMMOL_CB_VISSEL  1254
#define  MSG_DLGRMMOL_HC_RESEND  1255
#define  MSG_DLGRMMOL_HC_RESSTART  1256
#define  MSG_DLGRMMOL_LBL_SELECTED  1257
#define  MSG_DLGRMMOL_LBL_UNSELECTED  1258
#define  MSG_SEL_TBI_TORALL  1259
#define  MSG_SEL_TBI_TORFLEX  1260
#define  MSG_DLGSRFCLC_BT_COLGRADRANGE  1261
#define  MSG_DLGSRFCLC_CB_COLBYGRAD  1262
#define  MSG_DLGSRFCLC_CB_COLGRADRANGE  1263
#define  MSG_DLGSRFCLC_CB_DEFOGLLISTS  1264
#define  MSG_DLGSRFCLC_CB_DEFTRANSEN  1265
#define  MSG_DLGSRFCLC_CB_OGLLIST  1266
#define  MSG_DLGSRFCLC_CB_TRANSEN  1267
#define  MSG_DLGSRFCLC_HC_SRFNAME  1268
#define  MSG_DLGSRFCLC_HC_SRFNUM  1269
#define  MSG_DLGSRFCLC_HC_SRFPOINTS  1270
#define  MSG_DLGSRFCLC_HC_SRFVIS  1271
#define  MSG_DLGSRFCLC_LBL_COLCUST  1272
#define  MSG_DLGSRFCLC_LBL_COLGRADNUM  1273
#define  MSG_DLGSRFCLC_LBL_DEFCOLOR  1274
#define  MSG_DLGSRFCLC_LBL_DOTSIZE  1275
#define  MSG_DLGSRFCLC_LBL_MESHSIZE  1276
#define  MSG_DLGSRFCLC_LBL_TRANSVAL  1277
#define  MSG_DLGSRFCLC_NEWSRFNAM  1278
#define  MSG_DLGSRFCLC_PM_COLGRADINV  1279
#define  MSG_DLGSRFCLC_PM_COLGRADSHL  1280
#define  MSG_DLGSRFCLC_PM_COLGRADSHR  1281
#define  MSG_DLGSRFCLC_PM_COLGRADPRESETS  1282
#define  MSG_DLGSRFCLC_PM_COLGRADDEF  1283
#define  MSG_DLGSRFCLC_PM_SRFHIDE  1284
#define  MSG_DLGSRFCLC_PM_SRFREMOVE  1285
#define  MSG_DLGSRFCLC_PM_SRFREN  1286
#define  MSG_DLGSRFCLC_PM_SRFSHOW  1287
#define  MSG_DLGSRFCLC_RG_COLMET  1288
#define  MSG_DLGSRFCLC_RG_COLMETATM  1289
#define  MSG_DLGSRFCLC_RG_COLMETCHAIN  1290
#define  MSG_DLGSRFCLC_RG_COLMETCUST  1291
#define  MSG_DLGSRFCLC_RG_COLMETMOL  1292
#define  MSG_DLGSRFCLC_RG_COLMETRES  1293
#define  MSG_DLGSRFCLC_RG_COLMETSEG  1294
#define  MSG_DLGSRFCLC_RG_COLMETSRF  1295
#define  MSG_DLGSRFCLC_RG_VISMODDOT  1296
#define  MSG_DLGSRFCLC_RG_VISMODMESH  1297
#define  MSG_DLGSRFCLC_RG_VISMODSOLID  1298
#define  MSG_DLGSRFCLC_TS_COLOR  1299
#define  MSG_DLGSRFCLC_TS_GRADIENT  1300
#define  MSG_DLGSRFCLC_TS_NEW  1301
#define  MSG_DLGSRFCLC_TS_SETTINGS  1302
#define  MSG_DLGSRFCLC_TS_TRANS  1303
#define  MSG_DLGTOR_LBL_ANGVAL  1304
#define  MSG_DLGTRJADV_BT_SETFRMCUR  1305
#define  MSG_DLGTRJADV_GRP_FILENAME  1306
#define  MSG_DLGTRJADV_GRP_REFFRAME  1307
#define  MSG_DLGTRJADV_RG_RMSDFRAME  1308
#define  MSG_DLGTRJADV_RG_RMSDMODE  1309
#define  MSG_DLGTRJADV_RG_RMSDMOLFILE  1310
#define  MSG_DLGTRJADV_RG_RMSDPREV  1311
#define  MSG_DLGTRJADV_TITLE  1312
#define  MSG_VIEWPREFS_ANTIALIAS  1313
#define  MSG_VIEWPREFS_CB_FONTTYPEBMP  1314
#define  MSG_VIEWPREFS_CB_FONTTYPESTROKE  1315
#define  MSG_VIEWPREFS_CB_FONTTYPETT  1316
#define  MSG_VIEWPREFS_CB_STEREOACTIVEGLASSES  1317
#define  MSG_VIEWPREFS_CB_STEREOANAFBLUERED  1318
#define  MSG_VIEWPREFS_CB_STEREOANAFCYANRED  1319
#define  MSG_VIEWPREFS_CB_STEREOANAFGREENRED  1320
#define  MSG_VIEWPREFS_CB_STEREOANAFREDBLUE  1321
#define  MSG_VIEWPREFS_CB_STEREOANAFREDCYAN  1322
#define  MSG_VIEWPREFS_CB_STEREOANAFREDGREEN  1323
#define  MSG_VIEWPREFS_CB_STEREOANAGLYPH  1324
#define  MSG_VIEWPREFS_CB_STEREONONE  1325
#define  MSG_VIEWPREFS_CB_TUBEBIQUAD  1326
#define  MSG_VIEWPREFS_DEPTHCUE  1327
#define  MSG_VIEWPREFS_LBL_FONT  1328
#define  MSG_VIEWPREFS_LBL_FONTRES  1329
#define  MSG_VIEWPREFS_LBL_FONTSIZE  1330
#define  MSG_VIEWPREFS_LBL_FONTSIZETT  1331
#define  MSG_VIEWPREFS_LBL_FONTTYPE  1332
#define  MSG_VIEWPREFS_LBL_SPLINERES  1333
#define  MSG_VIEWPREFS_LBL_STEREOANAFILTER  1334
#define  MSG_VIEWPREFS_LBL_STEREOEYEADJUST  1335
#define  MSG_VIEWPREFS_LBL_STEREOEYEOFFSET  1336
#define  MSG_VIEWPREFS_LBL_STEREOMODE  1337
#define  MSG_VIEWPREFS_PGFONTS  1338
#define  MSG_VIEWPREFS_PGMAIN  1339
#define  MSG_VIEWPREFS_PGSTEREO  1340
#define  MSG_VIEWPREFS_PGTRACE  1341
#define  MSG_VIEWPREFS_PGTUBE  1342
#define  MSG_VIEWPREFS_RESTART  1343
#define  MSG_ERR_DCD_RMSDNOSRF  1344
#define  MSG_ERR_DCD_RMSDNOTRAJ  1345
#define  MSG_DCD_ERR_RMSDUNCOMPMOL  1346
#define  MSG_MENU_SAVESURF  1347
#define  MSG_MLP_DOTONLY  1348
#define  MSG_MAIN_MENU_PROCHECK  1349
#define  MSG_MAIN_MENU_CHKAACHIR  1350
#define  MSG_MAIN_MENU_PEPTBOND  1351
#define  MSG_MAIN_MENU_PROMISRES  1352
#define  MSG_MAIN_MENU_RINGINTER  1353
#define  MSG_MAIN_MENU_SWAPBONDS  1354
#define  MSG_MAIN_MENU_DSPLIQ  1355
#define  MSG_MAIN_MENU_DSPTRACES  1356
#define  MSG_MAIN_MENU_DSPTUBES  1357
#define  MSG_ERR_PBUFFER_CANTALLOCATE  1358
#define  MSG_ERR_PBUFFER_CANTINIT  1359
#define  MSG_ERR_PBUFFER_EXT  1360
#define  MSG_ERR_PBUFFER_NOOGLEXTENSIONS  1361
#define  MSG_ERR_PBUFFER_NOPIXELFORMAT  1362
#define  MSG_ERR_PBUFFER_NOTAVAIL  1363
#define  MSG_ERR_PBUFFER_UNKPROB  1364
#define  MSG_POPUP_MENU_MOVESEG  1365
#define  MSG_POPUP_MENU_MOVERES  1366
#define  MSG_GLPRINT_JOB  1367
#define  MSG_LOADER_CRT  1368
#define  MSG_ERR_MAIN_MDLMOL  1369
#define  MSG_MAIN_MENU_ADDCHAINS  1370
#define  MSG_MAIN_MENU_HLPCHKUPD  1371
#define  MSG_MAIN_MENU_TBSTANDARD  1372
#define  MSG_MAIN_MENU_TBTOOLS  1373
#define  MSG_MAIN_MENU_TOOLBARS  1374
#define  MSG_WARN_MAIN_STEREO  1375
#define  MSG_MOL2_MOREMOL  1376
#define  MSG_SURFACE_BUILDDEEP  1377
#define  MSG_TBTOOLS_ADDATM  1378
#define  MSG_TBTOOLS_ADDFRAG  1379
#define  MSG_TBTOOLS_ANALYSIS  1380
#define  MSG_TBTOOLS_CALCCHRG  1381
#define  MSG_TBTOOLS_CALCMOPAC  1382
#define  MSG_TBTOOLS_CALCSRF  1383
#define  MSG_TBTOOLS_EDITATM  1384
#define  MSG_TBTOOLS_SELCUSTOM  1385
#define  MSG_ATMSEL_LBL_SLICEPOS  1386
#define  MSG_ATMSEL_LBL_SLICETHICK  1387
#define  MSG_ATMSEL_TS_SLICE  1388
#define  MSG_ATMSEL_RG_SLICEPLANE  1389
#define  MSG_GRAPH_EXCEL_STEP  1390
#define  MSG_GRAPH_EXCEL_VALUE  1391
#define  MSG_GRAPH_MM_EXCEL  1392
#define  MSG_ERR_GRAPH_EXCEL  1393
#define  MSG_ERR_GRAPH_EXCEL_TOOMANYREC  1394
#define  MSG_ERR_GRAPH_OLE  1395
#define  MSG_MAIN_MENU_EXCEL  1396
#define  MSG_EXCEL_EXPORT  1397
#define  MSG_EXCEL_INIT  1398
#define  MSG_EXCEL_MOLLBL_ATOM  1399
#define  MSG_EXCEL_MOLLBL_CHAIN  1400
#define  MSG_EXCEL_MOLLBL_CHARGE  1401
#define  MSG_EXCEL_MOLLBL_CONSTRINT  1402
#define  MSG_EXCEL_MOLLBL_ELEMENT  1403
#define  MSG_EXCEL_MOLLBL_NUMBER  1404
#define  MSG_EXCEL_MOLLBL_POTENTIAL  1405
#define  MSG_EXCEL_MOLLBL_RESIDUE  1406
#define  MSG_EXCEL_MOLLBL_RESNUM  1407
#define  MSG_BT_APPEND  1408
#define  MSG_BT_FIND  1409
#define  MSG_BT_OVERWRITE  1410
#define  MSG_BT_SYNCHRONIZE  1411
#define  MSG_ERR_AVI_ADD  1412
#define  MSG_ERR_AVI_CREATE  1413
#define  MSG_ERR_AVI_SET  1414
#define  MSG_DCD_SAVEPROG  1415
#define  MSG_ERR_DCD_SAMEFILE  1416
#define  MSG_DLGADDFRAG_CB_INHERITH  1417
#define  MSG_ANA_CB_SHOWFRAME  1418
#define  MSG_DLGCODEC_CB_ASPRATIO  1419
#define  MSG_DLGCODEC_DEPTHS  1420
#define  MSG_DLGCODEC_DEPTHWARN  1421
#define  MSG_DLGCODEC_GRP_INFO  1422
#define  MSG_DLGCODEC_GRP_RATE  1423
#define  MSG_DLGCODEC_HC_CODEC  1424
#define  MSG_DLGCODEC_LB_CODEC_UNCOMP  1425
#define  MSG_DLGCODEC_LBL_BITRATE  1426
#define  MSG_DLGCODEC_LBL_DELTAFRAME  1427
#define  MSG_DLGCODEC_LBL_DRIVERNAME  1428
#define  MSG_DLGCODEC_LBL_FOURCC  1429
#define  MSG_DLGCODEC_LBL_QUALITY  1430
#define  MSG_DLGCODEC_LBL_RATE  1431
#define  MSG_DLGCODEC_LBL_REST  1432
#define  MSG_DLGCODEC_RESTHEIGHT  1433
#define  MSG_DLGCODEC_RESTWIDTH  1434
#define  MSG_DLGCODEC_RESTUNKHEIGHT  1435
#define  MSG_DLGCODEC_RESTUNKWIDTH  1436
#define  MSG_DLGCODEC_SIZEWARN  1437
#define  MSG_DLGCODEC_TITLE  1438
#define  MSG_ERR_DLGCODEC_FORMAT  1439
#define  MSG_ERR_DLGCODEC_MULTX  1440
#define  MSG_ERR_DLGCODEC_MULTY  1441
#define  MSG_ERR_DLGCODEC_SIZE  1442
#define  MSG_DLGCOLORSET_GRP_COLHBTYPES  1443
#define  MSG_DLGCOLORSET_LBL_COLHBACC  1444
#define  MSG_DLGCOLORSET_LBL_COLHBDON  1445
#define  MSG_DLGCOLORSET_LBL_COLHBOTHER  1446
#define  MSG_DLGCOLORSET_TS_HBONDS  1447
#define  MSG_DLGDBEXP_CB_COLBYATM  1448
#define  MSG_DLGDBEXP_FD_MOLTITLE  1449
#define  MSG_DLGDBEXP_LARGEDB  1450
#define  MSG_DLGDBEXP_NEWMOLNAME  1451
#define  MSG_DLGDBEXP_SB_DATABASES  1452
#define  MSG_DLGDBEXP_SB_MOLECULES  1453
#define  MSG_GRAPH_AMPLITUDE  1454
#define  MSG_GRAPH_ANGLE  1455
#define  MSG_GRAPH_AREA  1456
#define  MSG_GRAPH_CLUSTERS  1457
#define  MSG_GRAPH_DERIVATIVE  1458
#define  MSG_GRAPH_DISTANCE  1459
#define  MSG_GRAPH_DIAMETER  1460
#define  MSG_GRAPH_LOGP  1461
#define  MSG_GRAPH_NEWXLABEL  1462
#define  MSG_GRAPH_NEWYLABEL  1463
#define  MSG_GRAPH_NEWTITLE  1464
#define  MSG_GRAPH_POPULATION  1465
#define  MSG_GRAPH_RADIUS  1466
#define  MSG_GRAPH_RMSD  1467
#define  MSG_GRAPH_SPECTRUM  1468
#define  MSG_GRAPH_SPECTRUMAX  1469
#define  MSG_GRAPH_TIME  1470
#define  MSG_GRAPH_TORSION  1471
#define  MSG_GRAPH_VOLUME  1472
#define  MSG_GRAPH_MM_LABELS  1473
#define  MSG_GRAPH_MM_CALCDER  1474
#define  MSG_GRAPH_MM_CALCNR  1475
#define  MSG_GRAPH_MM_CALCSPEC  1476
#define  MSG_GRAPH_MM_LBLTITLE  1477
#define  MSG_GRAPH_MM_LBLXAXIS  1478
#define  MSG_GRAPH_MM_LBLYAXIS  1479
#define  MSG_ERR_GRAPH_NODATA  1480
#define  MSG_DLGHBOND_CB_LBLDIST  1481
#define  MSG_DLGHBOND_CB_LBLENE  1482
#define  MSG_DLGHBOND_CB_VISONLY  1483
#define  MSG_DLGHBOND_GRP_ACC  1484
#define  MSG_DLGHBOND_GRP_DON  1485
#define  MSG_DLGHBOND_GRP_LABELS  1486
#define  MSG_DLGHBOND_GRP_PARAM  1487
#define  MSG_DLGHBOND_LBL_PARAEXP  1488
#define  MSG_DLGHBOND_LBL_PARCOSEXPAA  1489
#define  MSG_DLGHBOND_LBL_PARCOSEXPHA  1490
#define  MSG_DLGHBOND_LBL_PARCUTOFF  1491
#define  MSG_DLGHBOND_LBL_PARREXP  1492
#define  MSG_DLGHBOND_LBL_PARTHRES  1493
#define  MSG_DLGHBOND_RESULTS  1494
#define  MSG_DLGHBOND_START  1495
#define  MSG_DLGHBOND_TITLE  1496
#define  MSG_DLGIMGSAVE_LBL_ANTIALIAS  1497
#define  MSG_DLGMAIN_FRAME  1498
#define  MSG_DLGMAIN_SD_FILTERAVI  1499
#define  MSG_DLGMAIN_SD_FILTERTRJ  1500
#define  MSG_DLGMOPAC_FILEFILTER  1501
#define  MSG_ERR_DLGMOPAC_FRACCHARGE  1502
#define  MSG_DLGMPEG_CB_AUTOMAXBR  1503
#define  MSG_DLGMPEG_CB_EXTREME  1504
#define  MSG_DLGMPEG_CB_GOPFORCE  1505
#define  MSG_DLGMPEG_CB_HQ  1506
#define  MSG_DLGMPEG_CB_INTERLACE  1507
#define  MSG_DLGMPEG_CB_SCENEDETECT  1508
#define  MSG_DLGMPEG_CB_PRODEFAULT  1509
#define  MSG_DLGMPEG_CB_PRODVDNTSC  1510
#define  MSG_DLGMPEG_CB_PRODVDPAL  1511
#define  MSG_DLGMPEG_CB_PROSVCDNTSC  1512
#define  MSG_DLGMPEG_CB_PROSVCDPAL  1513
#define  MSG_DLGMPEG_CB_PROVCDNTSC  1514
#define  MSG_DLGMPEG_CB_PROVCDPAL  1515
#define  MSG_DLGMPEG_CB_TOPFIELD  1516
#define  MSG_DLGMPEG_CB_TRELLIS  1517
#define  MSG_DLGMPEG_CB_VBR  1518
#define  MSG_DLGMPEG_GRP_ADVANCED  1519
#define  MSG_DLGMPEG_GRP_BITRATE  1520
#define  MSG_DLGMPEG_GRP_GOP  1521
#define  MSG_DLGMPEG_GRP_PROFILES  1522
#define  MSG_DLGMPEG_LBL_GOPSIZE  1523
#define  MSG_DLGMPEG_LBL_MAXBFRM  1524
#define  MSG_DLGMPEG_LBL_MAXBITRATE  1525
#define  MSG_DLGMPEG_LBL_PROFILE  1526
#define  MSG_DLGMPEG_RG_ASPECTRATIO  1527
#define  MSG_DLGMPEG_TITLE  1528
#define  MSG_TRJSAVE_CK_ACTIVEONLY  1529
#define  MSG_TRJSAVE_CK_SWAPENDIAN  1530
#define  MSG_TRJSAVE_GRP_ADVANCED  1531
#define  MSG_TRJSAVE_GRP_FRAMES  1532
#define  MSG_TRJSAVE_GRP_OPTIONS  1533
#define  MSG_TRJSAVE_GRP_PRECISION  1534
#define  MSG_TRJSAVE_LBL_END  1535
#define  MSG_TRJSAVE_LBL_SKIP  1536
#define  MSG_TRJSAVE_LBL_START  1537
#define  MSG_TRJSAVE_OVERWRITE  1538
#define  MSG_TRJSAVE_PROGRESS  1539
#define  MSG_VIEWPREFS_CB_FASTREFRESH  1540
#define  MSG_VIEWPREFS_LBL_ANIMSPEED  1541
#define  MSG_VIEWPREFS_LBL_FRAMERATE  1542
#define  MSG_VIEWPREFS_PGREFRESH  1543
#define  MSG_MENU_SAVETRJ  1544
#define  MSG_MAIN_MENU_AACONTACTS  1545
#define  MSG_MAIN_MENU_SAVETRAJ  1546
#define  MSG_MAIN_MENU_COLBYHBOND  1547
#define  MSG_MAIN_MENU_SELSUPATM  1548
#define  MSG_POPUP_MENU_HBONDS  1549
#define  MSG_WKS_MOLECULE  1550
#define  MSG_LOADER_TINKER_XYZ  1551
#define  MSG_SURFACE_BUILDHBACC  1552
#define  MSG_SURFACE_BUILDHBDON  1553
#define  MSG_ERR_DLGACT_IFFCSIZE  1554
#define  MSG_ERR_DLGACT_IFFFORMAT  1555
#define  MSG_ERR_DLGACT_IFFSIZE  1556
#define  MSG_ERR_DLGACT_INVCRYP  1557
#define  MSG_ERR_DLGACT_NOIFF  1558
#define  MSG_ERR_DLGACT_READ  1559
#define  MSG_ERR_DLGACT_UNKNOWN  1560
#define  MSG_DLGDBEXP_PM_MOLCOPYLIST  1561
#define  MSG_DLGDBEXP_PM_MOLSAVELIST  1562
#define  MSG_DLGDBEXP_SD_FILEFILTER  1563
#define  MSG_DLGDBEXP_SD_TITLE  1564
#define  MSG_ERR_ENER_PARKEYUNK  1565
#define  MSG_DLGJOY_ASS_AR  1566
#define  MSG_DLGJOY_ASS_AU  1567
#define  MSG_DLGJOY_ASS_AV  1568
#define  MSG_DLGJOY_ASS_AX  1569
#define  MSG_DLGJOY_ASS_AY  1570
#define  MSG_DLGJOY_ASS_AZ  1571
#define  MSG_DLGJOY_ASS_BUTTON  1572
#define  MSG_DLGJOY_ASS_CENTER  1573
#define  MSG_DLGJOY_ASS_NO  1574
#define  MSG_DLGJOY_ASS_NONE  1575
#define  MSG_DLGJOY_ASS_POVX  1576
#define  MSG_DLGJOY_ASS_POVY  1577
#define  MSG_DLGJOY_ASS_YES  1578
#define  MSG_DLGJOY_LBL_CAPSAXIS  1579
#define  MSG_DLGJOY_LBL_CAPSBUT  1580
#define  MSG_DLGJOY_LBL_CAPSDRIVER  1581
#define  MSG_DLGJOY_LBL_CAPSMID  1582
#define  MSG_DLGJOY_LBL_CAPSOEM  1583
#define  MSG_DLGJOY_LBL_CAPSPID  1584
#define  MSG_DLGJOY_LBL_CAPSPOV  1585
#define  MSG_DLGJOY_LBL_CAPSPOV4D  1586
#define  MSG_DLGJOY_LBL_CAPSPOVCTS  1587
#define  MSG_DLGJOY_LBL_CAPSR  1588
#define  MSG_DLGJOY_LBL_CAPSU  1589
#define  MSG_DLGJOY_LBL_CAPSV  1590
#define  MSG_DLGJOY_LBL_CAPSZ  1591
#define  MSG_DLGJOY_LBL_CFGAXL  1592
#define  MSG_DLGJOY_LBL_CFGAXR  1593
#define  MSG_DLGJOY_LBL_CFGAYD  1594
#define  MSG_DLGJOY_LBL_CFGAYU  1595
#define  MSG_DLGJOY_LBL_CFGAZL  1596
#define  MSG_DLGJOY_LBL_CFGAZR  1597
#define  MSG_DLGJOY_LBL_CFGMAN  1598
#define  MSG_DLGJOY_LBL_CFGSD  1599
#define  MSG_DLGJOY_LBL_CFGSI  1600
#define  MSG_DLGJOY_LBL_CFGTXL  1601
#define  MSG_DLGJOY_LBL_CFGTXR  1602
#define  MSG_DLGJOY_LBL_CFGTYD  1603
#define  MSG_DLGJOY_LBL_CFGTYU  1604
#define  MSG_DLGJOY_LBL_CFGTZF  1605
#define  MSG_DLGJOY_LBL_CFGTZN  1606
#define  MSG_DLGJOY_LBL_JOYDEVICE  1607
#define  MSG_DLGJOY_LBL_JOYPARSAMP  1608
#define  MSG_DLGJOY_LBL_JOYPARSENSAX  1609
#define  MSG_DLGJOY_LBL_JOYPARSENSAY  1610
#define  MSG_DLGJOY_LBL_JOYPARSENSAZ  1611
#define  MSG_DLGJOY_LBL_JOYPARSENSSC  1612
#define  MSG_DLGJOY_LBL_JOYPARSENSTX  1613
#define  MSG_DLGJOY_LBL_JOYPARSENSTY  1614
#define  MSG_DLGJOY_LBL_JOYPARSENSTZ  1615
#define  MSG_DLGJOY_LBL_JOYPARTRESH  1616
#define  MSG_DLGJOY_LBL_MOUSEPARSENSA  1617
#define  MSG_DLGJOY_LBL_MOUSEPARSENSS  1618
#define  MSG_DLGJOY_LBL_MOUSEPARSENSW  1619
#define  MSG_DLGJOY_LBL_MOUSEPOSX  1620
#define  MSG_DLGJOY_LBL_MOUSEPOSY  1621
#define  MSG_DLGJOY_LBL_TESTAR  1622
#define  MSG_DLGJOY_LBL_TESTAU  1623
#define  MSG_DLGJOY_LBL_TESTAV  1624
#define  MSG_DLGJOY_LBL_TESTAX  1625
#define  MSG_DLGJOY_LBL_TESTAY  1626
#define  MSG_DLGJOY_LBL_TESTAZ  1627
#define  MSG_DLGJOY_LBL_TESTBT  1628
#define  MSG_DLGJOY_LBL_TESTPOV  1629
#define  MSG_DLGJOY_LBL_TESTPX  1630
#define  MSG_DLGJOY_LBL_TESTPY  1631
#define  MSG_DLGJOY_TS_JOYCAPS  1632
#define  MSG_DLGJOY_TS_JOYCONFIG  1633
#define  MSG_DLGJOY_TS_JOYPARAM  1634
#define  MSG_DLGJOY_TS_JOYSTICK  1635
#define  MSG_DLGJOY_TS_JOYTEST  1636
#define  MSG_DLGJOY_TS_MOUSE  1637
#define  MSG_DLGJOY_TITLE  1638
#define  MSG_VIEWPREFS_CB_AXISSTYLESOLID  1639
#define  MSG_VIEWPREFS_CB_AXISSTYLEVECTOR  1640
#define  MSG_VIEWPREFS_LBL_AXISCOLL  1641
#define  MSG_VIEWPREFS_LBL_AXISPOS  1642
#define  MSG_VIEWPREFS_LBL_AXISRES  1643
#define  MSG_VIEWPREFS_LBL_AXISSCALE  1644
#define  MSG_VIEWPREFS_LBL_AXISSTYLE  1645
#define  MSG_VIEWPREFS_LBL_AXISWIDTH  1646
#define  MSG_VIEWPREFS_LBL_MULTISAMPLE  1647
#define  MSG_VIEWPREFS_TS_AXIS  1648
#define  MSG_MAIN_MENU_TLSHID  1649
#define  MSG_ERR_GL2VRML_PROMPT  1650
#define  MSG_CHARGE_TEMRESFIX  1651
#define  MSG_ERR_CHARGE_TEMATMLONG  1652
#define  MSG_ERR_CHARGE_TEMATMMIS  1653
#define  MSG_ERR_CHARGE_TEMBONDINV  1654
#define  MSG_ERR_CHARGE_TEMBONDMIS  1655
#define  MSG_ERR_CHARGE_TEMCALLITS  1656
#define  MSG_ERR_CHARGE_TEMDEL  1657
#define  MSG_ERR_CHARGE_TEMDESCLONG  1658
#define  MSG_ERR_CHARGE_TEMGRPINV  1659
#define  MSG_ERR_CHARGE_TEMGRPMIS  1660
#define  MSG_ERR_CHARGE_TEMIDDUP  1661
#define  MSG_ERR_CHARGE_TEMIDLONG  1662
#define  MSG_ERR_CHARGE_TEMIDMIS  1663
#define  MSG_ERR_CHARGE_TEMIDUNI  1664
#define  MSG_ERR_CHARGE_TEMIDUNK  1665
#define  MSG_ERR_DLGCLUSTBLD_TOOBIG  1666
#define  MSG_DLGADDFRAG_CB_RESSEQ  1667
#define  MSG_DLGADDHYD_CB_BNDORD  1668
#define  MSG_DLGDBOPEN_CK_MOL64BIT  1669
#define  MSG_DLGPREFS_CB_CPUS  1670
#define  MSG_DLGPREFS_CB_GLASSWIN  1671
#define  MSG_DLGPREFS_CB_SOUND  1672
#define  MSG_DLGPREFS_LBL_CPUS  1673
#define  MSG_DLGPREFS_LBL_ENECUTOFF  1674
#define  MSG_DLGPREFS_LBL_ENEDIEL  1675
#define  MSG_DLGPREFS_LBL_ENENONBOND  1676
#define  MSG_DLGPREFS_LBL_ENETRESH  1677
#define  MSG_DLGPREFS_LBL_GLASSWINMAX  1678
#define  MSG_DLGPREFS_LBL_GLASSWINMIN  1679
#define  MSG_DLGPREFS_LBL_GLASSWINOPEN  1680
#define  MSG_DLGPREFS_LBL_GLASSWINSPEED  1681
#define  MSG_DLGPREFS_LBL_GRAPHZOOM  1682
#define  MSG_DLGPREFS_LBL_LANGUAGE  1683
#define  MSG_DLGPREFS_LBL_MOPACKEY  1684
#define  MSG_DLGPREFS_LBL_SOLRAD  1685
#define  MSG_DLGPREFS_LBL_SRFDOTS  1686
#define  MSG_DLGPREFS_LBL_SRFMESH  1687
#define  MSG_DLGPREFS_LBL_SURFACE  1688
#define  MSG_DLGPREFS_LBL_VOLUME  1689
#define  MSG_DLGPREFS_LBL_SRFPROBERAD  1690
#define  MSG_DLGPREFS_TITLE  1691
#define  MSG_DLGPREFS_TS_ENERGY  1692
#define  MSG_DLGPREFS_TS_GLASSWIN  1693
#define  MSG_DLGPREFS_TS_MAIN  1694
#define  MSG_DLGPREFS_TS_MISC  1695
#define  MSG_DLGPREFS_TS_SURFACE  1696
#define  MSG_DLGPROFIT_BT_WAIT  1697
#define  MSG_DLGPROFIT_MESSAGE  1698
#define  MSG_ERR_GAUSS_CORR  1699
#define  MSG_MAIN_MENU_RMGRAPHOBJ  1700
#define  MSG_MAIN_MENU_UNDO  1701
#define  MSG_ERR_IFF_READ  1702
#define  MSG_ERR_IFF_WRITE  1703
#define  MSG_LOADER_GAUSSIN  1704
#define  MSG_LOADER_GAUSSOUT  1705
#define  MSG_ERR_VGL_BEGINGROUPEXP  1706
#define  MSG_ERR_VGL_INVGROUPID  1707
#define  MSG_ERR_VGL_MTXSTACK  1708
#define  MSG_ERR_VGL_PRIMITIVE  1709
#define  MSG_ERR_VGL_TOOMANYGROUPS  1710
#define  MSG_MAIN_MENU_CALCAMMP  1711
#define  MSG_AMMP_BACKRUN  1712
#define  MSG_ERR_AMMP_COMLOST  1713
#define  MSG_ERR_AMMP_CONNECT  1714
#define  MSG_ERR_AMMP_DATA  1715
#define  MSG_ERR_AMMP_EXE  1716
#define  MSG_ERR_AMMP_HOSTUSED  1717
#define  MSG_ERR_AMMP_NOFREEHOST  1718
#define  MSG_ERR_AMMP_MOLECULE  1719
#define  MSG_ERR_AMMP_PIPEINIT  1720
#define  MSG_ERR_AMMP_SAVE_SP4  1721
#define  MSG_WARN_AMMP_BONDORD  1722
#define  MSG_WARN_AMMP_SAVE_ATT  1723
#define  MSG_BT_SEND  1724
#define  MSG_ERR_COM_CALCRUN  1725
#define  MSG_ERR_DLGACT_LIVE  1726
#define  MSG_DLGAMMP_BT_EDITPAR  1727
#define  MSG_DLGAMMP_BT_EDITTEM  1728
#define  MSG_DLGAMMP_CB_POTHYBRID  1729
#define  MSG_DLGAMMP_CB_POTTORSION  1730
#define  MSG_DLGAMMP_FILEFILTER  1731
#define  MSG_DLGAMMP_GRP_EDIT  1732
#define  MSG_DLGAMMP_GRP_POTOTHER  1733
#define  MSG_DLGAMMP_LBL_CURHOST  1734
#define  MSG_DLGAMMP_LBL_CURHOSTDESC  1735
#define  MSG_DLGAMMP_LBL_CUTOFF  1736
#define  MSG_DLGAMMP_LBL_DTOLER  1737
#define  MSG_DLGAMMP_LBL_HOSTS  1738
#define  MSG_DLGAMMP_LBL_HOSTPOOLS  1739
#define  MSG_DLGAMMP_LBL_LAMBDA  1740
#define  MSG_DLGAMMP_LBL_MINRMS  1741
#define  MSG_DLGAMMP_LBL_MINSTEPS  1742
#define  MSG_DLGAMMP_LBL_MINUPDATE  1743
#define  MSG_DLGAMMP_LBL_MXCUT  1744
#define  MSG_DLGAMMP_LBL_MXDQ  1745
#define  MSG_DLGAMMP_LBL_NCNGDEL  1746
#define  MSG_DLGAMMP_LBL_NDEEP  1747
#define  MSG_DLGAMMP_LBL_NSTEEP  1748
#define  MSG_DLGAMMP_LBL_SEED  1749
#define  MSG_DLGAMMP_LBL_SIGMA  1750
#define  MSG_DLGAMMP_LBL_VSTART  1751
#define  MSG_DLGAMMP_RG_CONST  1752
#define  MSG_DLGAMMP_RG_CONST_ATMFIX  1753
#define  MSG_DLGAMMP_RG_CONST_NONE  1754
#define  MSG_DLGAMMP_RG_CONST_TETHER  1755
#define  MSG_DLGAMMP_RG_MINMETHOD_CONJG  1756
#define  MSG_DLGAMMP_RG_MINMETHOD_GENETIC  1757
#define  MSG_DLGAMMP_RG_MINMETHOD_NEWTO  1758
#define  MSG_DLGAMMP_RG_MINMETHOD_POLY  1759
#define  MSG_DLGAMMP_RG_MINMETHOD_RIGID  1760
#define  MSG_DLGAMMP_RG_MINMETHOD_SINPT  1761
#define  MSG_DLGAMMP_RG_MINMETHOD_STEEP  1762
#define  MSG_DLGAMMP_RG_MINMETHOD_TRUNC  1763
#define  MSG_DLGAMMP_RG_MINMETHOD_TRUST  1764
#define  MSG_DLGAMMP_RG_POTANGLE  1765
#define  MSG_DLGAMMP_RG_POTANGLE_UFFC  1766
#define  MSG_DLGAMMP_RG_POTBOND  1767
#define  MSG_DLGAMMP_RG_POTBOND_HOMO  1768
#define  MSG_DLGAMMP_RG_POTBOND_MM3  1769
#define  MSG_DLGAMMP_RG_POTBOND_NONE  1770
#define  MSG_DLGAMMP_RG_POTBOND_STD  1771
#define  MSG_DLGAMMP_RG_POTNONBOND  1772
#define  MSG_DLGAMMP_RG_POTNONBOND_DEBYE  1773
#define  MSG_DLGAMMP_RG_POTNONBOND_HSHELL  1774
#define  MSG_DLGAMMP_RG_POTNONBOND_SCREEN  1775
#define  MSG_DLGAMMP_RG_POTNONBOND_SHADOW  1776
#define  MSG_DLGAMMP_SP4PARAM  1777
#define  MSG_DLGAMMP_STOPCALC  1778
#define  MSG_DLGAMMP_TITLE  1779
#define  MSG_DLGAMMP_TS_CONSOLE  1780
#define  MSG_DLGAMMP_TS_HOSTS  1781
#define  MSG_DLGAMMP_TS_MIN  1782
#define  MSG_DLGAMMP_TS_PARAM  1783
#define  MSG_DLGAMMP_TS_POT  1784
#define  MSG_DLGHOST_BT_KEYGEN  1785
#define  MSG_DLGHOST_CB_ENCRYPTION  1786
#define  MSG_DLGHOST_CB_TYPE_UNK  1787
#define  MSG_DLGHOST_LBL_ADDRESS  1788
#define  MSG_DLGHOST_LBL_DESC  1789
#define  MSG_DLGHOST_LBL_KEY  1790
#define  MSG_DLGHOST_LBL_NAME  1791
#define  MSG_DLGHOST_LBL_PASSWD1  1792
#define  MSG_DLGHOST_LBL_PASSWD2  1793
#define  MSG_DLGHOST_LBL_PORT  1794
#define  MSG_DLGHOST_LBL_THREADS  1795
#define  MSG_DLGHOST_LBL_TYPE  1796
#define  MSG_DLGHOST_LBL_USER  1797
#define  MSG_DLGHOST_SAVE  1798
#define  MSG_DLGHOST_TITLE  1799
#define  MSG_DLGHOST_ERR_ADDRESS  1800
#define  MSG_DLGHOST_ERR_DUPNAME  1801
#define  MSG_DLGHOST_ERR_NAME  1802
#define  MSG_DLGHOST_ERR_PORT  1803
#define  MSG_DLGHOST_ERR_PWDDIFF  1804
#define  MSG_DLGHOST_ERR_PWDREQ  1805
#define  MSG_DLGHOST_ERR_REMOVE  1806
#define  MSG_DLGHOST_ERR_SAVE  1807
#define  MSG_DLGHOST_ERR_THREADS  1808
#define  MSG_DLGHOST_ERR_KEYSIZE  1809
#define  MSG_DLGHOST_WARN_SAVE  1810
#define  MSG_DLGJOY_LBL_MOUSEBT  1811
#define  MSG_DLGJOY_LBL_MOUSEBTL  1812
#define  MSG_DLGJOY_LBL_MOUSEBTM  1813
#define  MSG_DLGJOY_LBL_MOUSEBTR  1814
#define  MSG_DLGJOY_LBL_WHEEL  1815
#define  MSG_DLGJOY_LBL_WHEELB  1816
#define  MSG_DLGJOY_LBL_WHEELF  1817
#define  MSG_DLGJOY_LBL_WHEELVAL  1818
#define  MSG_ERR_UNCHARGED  1819
#define  MSG_MAIN_MENU_STOPCALC  1820
#define  MSG_MAIN_MENU_TLSHOST  1821
#define  MSG_HOSTS_POOL_ALL  1822
#define  MSG_HOSTS_POOL_LOCAL  1823
#define  MSG_HOSTS_POOL_REMOTE  1824
#define  MSG_LOADER_AMMP  1825
#define  MSG_LOADER_CPMD_XYZ  1826
#define  MSG_LOADER_STOPCALC  1827
#define  MSG_BT_CONTINUE  1828
#define  MSG_ERR_AMMP_SAVE_ASSFF  1829
#define  MSG_ERR_AMMP_SAVE_FF  1830
#define  MSG_ERR_CML_ABORT  1831
#define  MSG_ERR_CML_PARSE  1832
#define  MSG_ERR_CML_PARSER  1833
#define  MSG_ERR_CML_STACK  1834
#define  MSG_MAIN_MENU_CALCAMMPMIN  1835
#define  MSG_MAIN_MENU_CALCAMMPTSRC  1836
#define  MSG_DLGAMMP_BT_TSRCEDIT  1837
#define  MSG_DLGAMMP_CB_TSCRMETHOD_BJUMP  1838
#define  MSG_DLGAMMP_CB_TSCRMETHOD_RANDOM  1839
#define  MSG_DLGAMMP_CB_TSCRMETHOD_SYSTEM  1840
#define  MSG_DLGAMMP_CK_TSRCKEEPMIN  1841
#define  MSG_DLGAMMP_CK_TSRCKEEPPREV  1842
#define  MSG_DLGAMMP_CK_TSRCMIN  1843
#define  MSG_DLGAMMP_GRP_TSRC  1844
#define  MSG_DLGAMMP_GRP_TSRCKEEP  1845
#define  MSG_DLGAMMP_GRP_TSRCPAR  1846
#define  MSG_DLGAMMP_HINT_BT_CLONE  1847
#define  MSG_DLGAMMP_HINT_BT_OPEN  1848
#define  MSG_DLGAMMP_LBL_FFIELD  1849
#define  MSG_DLGAMMP_LBL_FILEENE  1850
#define  MSG_DLGAMMP_LBL_FILEOUT  1851
#define  MSG_DLGAMMP_LBL_FILETRAJ  1852
#define  MSG_DLGAMMP_LBL_FILEVEL  1853
#define  MSG_DLGAMMP_LBL_TSRCBASE  1854
#define  MSG_DLGAMMP_LBL_TSRCITER  1855
#define  MSG_DLGAMMP_LBL_TSRCMETHOD  1856
#define  MSG_DLGAMMP_LBL_TSRCRMSD  1857
#define  MSG_DLGAMMP_LBL_TSRCSTEPS  1858
#define  MSG_DLGAMMP_LBL_TSRCTEMP  1859
#define  MSG_DLGAMMP_LBL_TSRCWIN  1860
#define  MSG_DLGAMMP_LV_TSRCTOR_BASE  1861
#define  MSG_DLGAMMP_LV_TSRCTOR_STEPS  1862
#define  MSG_DLGAMMP_LV_TSRCTOR_TOR  1863
#define  MSG_DLGAMMP_LV_TSRCTOR_WIN  1864
#define  MSG_DLGAMMP_PM_TSRCCHKALL  1865
#define  MSG_DLGAMMP_PM_TSRCCHKSEL  1866
#define  MSG_DLGAMMP_PM_TSRCSELALL  1867
#define  MSG_DLGAMMP_PM_TSRCUNCHKALL  1868
#define  MSG_DLGAMMP_PM_TSRCUNCHKSEL  1869
#define  MSG_DLGAMMP_PM_TSRCUNSELALL  1870
#define  MSG_DLGAMMP_SD_FILTER_ENE  1871
#define  MSG_DLGAMMP_SD_FILTER_OUT  1872
#define  MSG_DLGAMMP_SD_FILTER_VEL  1873
#define  MSG_DLGAMMP_SD_TITLE_ENE  1874
#define  MSG_DLGAMMP_SD_TITLE_OUT  1875
#define  MSG_DLGAMMP_SD_TITLE_VEL  1876
#define  MSG_DLGAMMP_TITLETSRC  1877
#define  MSG_DLGAMMP_TS_TSRC  1878
#define  MSG_DLGANA_CK_CLUSTACTIVE  1879
#define  MSG_DLGANA_CK_CLUSTENE  1880
#define  MSG_DLGANA_CK_CLUSTSORTENE  1881
#define  MSG_DLGANA_CLUST_HDR  1882
#define  MSG_DLGANA_CLUST_NUM  1883
#define  MSG_DLGANA_RG_CLUSTMETCOORD  1884
#define  MSG_DLGANA_RG_CLUSTMETTOR  1885
#define  MSG_DLGANA_RG_CLUSTMETTORRMSD  1886
#define  MSG_DLGANA_TS_CLUST  1887
#define  MSG_DLGCODEC_LBL_SLOWMOTION  1888
#define  MSG_DLGCOLORSET_LBL_COLHBOND  1889
#define  MSG_DLGMOLOPEN_CB_FFIELD_NONE  1890
#define  MSG_DLGMOLOPEN_LBL_FFIELD  1891
#define  MSG_DLGMOPAC_BT_CHGCALC  1892
#define  MSG_DLGPARMIS_MM_AUTOASSIGN  1893
#define  MSG_DLGPARMIS_MM_VIEW_COLOR  1894
#define  MSG_DLGPARMIS_MM_VIEW_LABELS  1895
#define  MSG_DLGPARMIS_MM_VIEW_WIRE  1896
#define  MSG_DLGPARMIS_NBOND_HPOLAR  1897
#define  MSG_DLGPARMIS_NBOND_CAROM  1898
#define  MSG_DLGPARMIS_NBOND_NAROM  1899
#define  MSG_DLGPARMIS_NBOND_METAL  1900
#define  MSG_DLGPARMIS_PM_ANGLE_SP3  1901
#define  MSG_DLGPARMIS_PM_ANGLE_SP2  1902
#define  MSG_DLGPARMIS_PM_ANGLE_SP  1903
#define  MSG_DLGPARMIS_PM_DIHEDRAL_DOUBLE  1904
#define  MSG_DLGPARMIS_PM_DIHEDRAL_PARDOUBLE  1905
#define  MSG_DLGPARMIS_PM_DIHEDRAL_SINGLE  1906
#define  MSG_DLGPARMIS_PM_DIHEDRAL_TRIPLE  1907
#define  MSG_DLGPARMIS_PM_IMPROPER_FREE  1908
#define  MSG_DLGPARMIS_PM_IMPROPER_PLANAR  1909
#define  MSG_DLGPARMIS_ROW_ANGLE  1910
#define  MSG_DLGPARMIS_ROW_BOND  1911
#define  MSG_DLGPARMIS_ROW_DIHEDRAL  1912
#define  MSG_DLGPARMIS_ROW_IMPROPER  1913
#define  MSG_DLGPARMIS_ROW_NBOND  1914
#define  MSG_DLGPARMIS_SD_PARAM_FILTER  1915
#define  MSG_DLGPARMIS_SD_PARAM_TITLE  1916
#define  MSG_DLGPARMIS_SG_ATMTYP  1917
#define  MSG_DLGPARMIS_SG_PARAM  1918
#define  MSG_DLGPARMIS_TITLE  1919
#define  MSG_DLGSEL_MM_REMALL  1920
#define  MSG_DLGSEL_OD_FILTER  1921
#define  MSG_DLGSRFCLC_CB_DEFOGLBUFFER  1922
#define  MSG_DLGSRFCLC_CB_OGLBUFFER  1923
#define  MSG_ERR_HOST_CONN  1924
#define  MSG_ERR_HOST_CREATEPIPE  1925
#define  MSG_ERR_HOST_RECV  1926
#define  MSG_ERR_HOST_RUN  1927
#define  MSG_ERR_HOST_SEND  1928
#define  MSG_ERR_HOST_SOCKET  1929
#define  MSG_ERR_HOST_USER  1930
#define  MSG_ERR_HOST_UNKNOWN  1931
#define  MSG_LOADER_CHEMSOL  1932
#define  MSG_LOADER_CML  1933
#define  MSG_LOADER_PQRXML  1934
#define  MSG_LOADER_PSFX  1935
#define  MSG_MAIN_MENU_DELETE  1936
#define  MSG_TRJCLUST_PROGRMSD  1937
#define  MSG_TRJCLUST_PROGMEASTOR  1938
#define  MSG_TRJCLUST_PROGTOR  1939
#define  MSG_WARN_PSFX  1940
#define  MSG_BOND_FIXAROMATIC  1941
#define  MSG_ERR_BOND_FIXTYPE  1942
#define  MSG_INFOXML_AREA  1943
#define  MSG_INFOXML_AREAPRORAD  1944
#define  MSG_INFOXML_ASA  1945
#define  MSG_INFOXML_ATOMS  1946
#define  MSG_INFOXML_CENTGEO  1947
#define  MSG_INFOXML_CENTMASS  1948
#define  MSG_INFOXML_CENTROIDS  1949
#define  MSG_INFOXML_CHAINS  1950
#define  MSG_INFOXML_CHARGE  1951
#define  MSG_INFOXML_DIMENSIONS  1952
#define  MSG_INFOXML_DIPOLE  1953
#define  MSG_INFOXML_FORMULA  1954
#define  MSG_INFOXML_GYRRAD  1955
#define  MSG_INFOXML_HEAVYATOMS  1956
#define  MSG_INFOXML_LIPOLEBROTO  1957
#define  MSG_INFOXML_LOGPBROTO  1958
#define  MSG_INFOXML_LIPOLECRIPPEN  1959
#define  MSG_INFOXML_LOGPCRIPPEN  1960
#define  MSG_INFOXML_LOGPVIRTUAL  1961
#define  MSG_INFOXML_MOLECULES  1962
#define  MSG_INFOXML_OVALITY  1963
#define  MSG_INFOXML_PSA  1964
#define  MSG_INFOXML_RESIDUES  1965
#define  MSG_INFOXML_SDIAM  1966
#define  MSG_INFOXML_SEGMENTS  1967
#define  MSG_INFOXML_VDIAM  1968
#define  MSG_INFOXML_VOLUME  1969
#define  MSG_INFOXML_WATERS  1970
#define  MSG_INFOXML_WEIGHT  1971
#define  MSG_INFOXML_HYDROGENS  1972
#define  MSG_ERR_INFOXML_NOTHING  1973
#define  MSG_LOADER_CIF  1974
#define  MSG_LOADER_MDLMOLV3000  1975
#define  MSG_DLGADDFRAG_BT_FINISH  1976
#define  MSG_DLGADDFRAG_BT_NEXT  1977
#define  MSG_DLGADDFRAG_BT_PREV  1978
#define  MSG_DLGBOND_RG_FIXALLAROM  1979
#define  MSG_DLGBOND_RG_FIXSELAROM  1980
#define  MSG_MAIN_MENU_MOPAC2007HELP  1981
#define  MSG_MAIN_MENU_TLSVEGAOL  1982
#define  MSG_ADDCHAIN_ADDPROG  1983
#define  MSG_APPLYTRANS_PROG  1984
#define  MSG_CHIRAL_WARN_BONDORD  1985
#define  MSG_BT_MORE  1986
#define  MSG_BT_TERMINATE  1987
#define  MSG_ANA_CB_BOND  1988
#define  MSG_DLGAMMP_CB_FILEENE  1989
#define  MSG_DLGAMMP_CB_FILEOUT  1990
#define  MSG_DLGAMMP_CB_FILETRAY  1991
#define  MSG_DLGAMMP_CB_FILEVEL  1992
#define  MSG_DLGAMMP_GRP_OUTFILES  1993
#define  MSG_DLGEXCEPT_LBL_ERROR  1994
#define  MSG_DLGEXCEPT_MESSAGE_1  1995
#define  MSG_DLGEXCEPT_MESSAGE_2  1996
#define  MSG_DLGEXCEPT_TITLE  1997
#define  MSG_DLGMOPAC_CK_UPDATE  1998
#define  MSG_DLGMOPAC_HINT_BT_CHGCALC  1999
#define  MSG_DLGMOPAC_HINT_CK_1SCF  2000
#define  MSG_DLGMOPAC_HINT_CK_GEOK  2001
#define  MSG_DLGMOPAC_HINT_CK_MMOK  2002
#define  MSG_DLGMOPAC_HINT_CK_PRECISE  2003
#define  MSG_DLGSSTR_CB_ADDCHAINS  2004
#define  MSG_DLGSSTR_CB_ADDHYDROG  2005
#define  MSG_DLGSSTR_CB_FIXANG  2006
#define  MSG_DLGSSTR_RG_TYP  2007
#define  MSG_DLGSSTR_PEPTIDE  2008
#define  MSG_DLGSSTR_PROG  2009
#define  MSG_DLGSSTR_RG_TYP310  2010
#define  MSG_DLGSSTR_RG_TYPALPHA  2011
#define  MSG_DLGSSTR_RG_TYPBETA  2012
#define  MSG_DLGSSTR_RG_TYPBETAANTI  2013
#define  MSG_DLGSSTR_RG_TYPBETAPAR  2014
#define  MSG_DLGSSTR_RG_TYPCUSTOM  2015
#define  MSG_DLGSSTR_RG_TYPLEFT  2016
#define  MSG_DLGSSTR_RG_TYPPI  2017
#define  MSG_DLGSSTR_TITLE_BUILD  2018
#define  MSG_DLGSSTR_TITLE_CHANGE  2019
#define  MSG_MAIN_MENU_BUILDPEPT  2020
#define  MSG_MAIN_MENU_BOND_TYPE  2021
#define  MSG_MAIN_MENU_REDO  2022
#define  MSG_MAIN_MENU_CHGSECSTRUCT  2023
#define  MSG_MAIN_MENU_SELNOCONSTR  2024
#define  MSG_ERR_MAIN_NOTHINGTOSAVE  2025
#define  MSG_LOADER_FASTA  2026
#define  MSG_DLGOVER_TBI_SELACTIVEONLY  2027
#define  MSG_DLGOVER_TBI_SELALL  2028
#define  MSG_DLGOVER_TBI_SELDIRASC  2029
#define  MSG_DLGOVER_TBI_SELDIRBOTH  2030
#define  MSG_DLGOVER_TBI_SELDIRDESC  2031
#define  MSG_DLGOVER_TBI_SELSIM  2032
#define  MSG_DLGOVER_TBS_SELDIR  2033
#define  MSG_DLGOVER_TBS_SELECT  2034
#define  MSG_DLGPREFS_CB_UNDOWORLD  2035
#define  MSG_DLGPREFS_LBL_UNDOLVL  2036
#define  MSG_SEL_TBS_TORPROT  2037
#define  MSG_SSTR_PROGPMT  2038
#define  MSG_ERR_SSTR_NOPROT  2039
#define  MSG_ERR_SSTR_NOTOR  2040
#define  MSG_ERR_SSTR_TORNAME  2041
#define  MSG_DLGTOR_BT_DEFAULT  2042
#define  MSG_DLGTOR_CK_MOVEALL  2043
#define  MSG_DLGTOR_LBL_BNDLEN  2044
#define  MSG_LOADER_MOPCAR  2045
#define  MSG_UNDO_ADDATOM  2046
#define  MSG_UNDO_AMMP  2047
#define  MSG_UNDO_AMMPBOND  2048
#define  MSG_UNDO_ANIM  2049
#define  MSG_UNDO_APPLYTRANSF  2050
#define  MSG_UNDO_ATMCHANGE  2051
#define  MSG_UNDO_BONDADD  2052
#define  MSG_UNDO_BONDCHANGE  2053
#define  MSG_UNDO_BONDCHANGEALL  2054
#define  MSG_UNDO_BONDCONN  2055
#define  MSG_UNDO_BONDFIXAROM  2056
#define  MSG_UNDO_BONDFIXTYPES  2057
#define  MSG_UNDO_BONDRM  2058
#define  MSG_UNDO_BONDRMCONN  2059
#define  MSG_UNDO_BONDSWAP  2060
#define  MSG_UNDO_CENTROID  2061
#define  MSG_UNDO_CHAINCHANGE  2062
#define  MSG_UNDO_CHARGE  2063
#define  MSG_UNDO_CHARGEPOT  2064
#define  MSG_UNDO_CHGANGLE  2065
#define  MSG_UNDO_CHGBOND  2066
#define  MSG_UNDO_CHGTORSION  2067
#define  MSG_UNDO_CHGSECSTRUCT  2068
#define  MSG_UNDO_CHKAACHIR  2069
#define  MSG_UNDO_CHKPEPTBOND  2070
#define  MSG_UNDO_CLUSTBUILD  2071
#define  MSG_UNDO_CLUSTER  2072
#define  MSG_UNDO_CHKRINGS  2073
#define  MSG_UNDO_COLBYATM  2074
#define  MSG_UNDO_COLBYCHAIN  2075
#define  MSG_UNDO_COLBYCHARGE  2076
#define  MSG_UNDO_COLBYCONSTR  2077
#define  MSG_UNDO_COLBYATMHBOND  2078
#define  MSG_UNDO_COLBYMOL  2079
#define  MSG_UNDO_COLBYRES  2080
#define  MSG_UNDO_COLBYSEG  2081
#define  MSG_UNDO_COLSEL  2082
#define  MSG_UNDO_DBGET  2083
#define  MSG_UNDO_DSPCPK  2084
#define  MSG_UNDO_DSPCPKW  2085
#define  MSG_UNDO_DSPLIQ  2086
#define  MSG_UNDO_DSPLIQW  2087
#define  MSG_UNDO_DSPTRACE  2088
#define  MSG_UNDO_DSPTUBE  2089
#define  MSG_UNDO_DSPVDW  2090
#define  MSG_UNDO_DSPVDWD  2091
#define  MSG_UNDO_DSPVDWW  2092
#define  MSG_UNDO_DSPWIRE  2093
#define  MSG_UNDO_FIX  2094
#define  MSG_UNDO_FRAGMENT  2095
#define  MSG_UNDO_HBOND  2096
#define  MSG_UNDO_HYDROG  2097
#define  VG_UNDO_INTER  2098
#define  MSG_UNDO_INVZCOORD  2099
#define  MSG_UNDO_IONS  2100
#define  MSG_UNDO_LBLATMCHG  2101
#define  MSG_UNDO_LBLATMCHIR  2102
#define  MSG_UNDO_LBLATMELM  2103
#define  MSG_UNDO_LBLATMFIX  2104
#define  MSG_UNDO_LBLATMNAM  2105
#define  MSG_UNDO_LBLATMNUM  2106
#define  MSG_UNDO_LBLATMOFF  2107
#define  MSG_UNDO_LBLATMRESIDUE  2108
#define  MSG_UNDO_LBLATMRESNAME  2109
#define  MSG_UNDO_LBLATMRESSEQ  2110
#define  MSG_UNDO_LBLATMTYP  2111
#define  MSG_UNDO_MEASANG  2112
#define  MSG_UNDO_MEASDIST  2113
#define  MSG_UNDO_MEASPLANG  2114
#define  MSG_UNDO_MEASTOR  2115
#define  MSG_UNDO_MERGE  2116
#define  MSG_UNDO_MOLFIX  2117
#define  MSG_UNDO_MOLMERGE  2118
#define  MSG_UNDO_MOPAC  2119
#define  MSG_UNDO_NEW  2120
#define  MSG_UNDO_NORMALIZE  2121
#define  MSG_UNDO_OPEN  2122
#define  MSG_UNDO_OVERLAY  2123
#define  MSG_UNDO_PASTE  2124
#define  MSG_UNDO_PEPTBUILD  2125
#define  MSG_UNDO_POT  2126
#define  MSG_UNDO_RENRES  2127
#define  MSG_UNDO_RESCHANGE  2128
#define  MSG_UNDO_RMATM  2129
#define  MSG_UNDO_RMATMS  2130
#define  MSG_UNDO_RMCENTROIDS  2131
#define  MSG_UNDO_RMGRAPHOBJ  2132
#define  MSG_UNDO_RMHYD  2133
#define  MSG_UNDO_RMINVISATM  2134
#define  MSG_UNDO_RMMOL  2135
#define  MSG_UNDO_RMMON  2136
#define  MSG_UNDO_RMRES  2137
#define  MSG_UNDO_RMSEG  2138
#define  MSG_UNDO_RMSURF  2139
#define  MSG_UNDO_RMSURFALL  2140
#define  MSG_UNDO_RMWAT  2141
#define  MSG_UNDO_RESETV  2142
#define  MSG_UNDO_ROTATE  2143
#define  MSG_UNDO_SCALE  2144
#define  MSG_UNDO_SEGFIX  2145
#define  MSG_UNDO_SEGMERGE  2146
#define  MSG_UNDO_SELALL  2147
#define  MSG_UNDO_SELBACK  2148
#define  MSG_UNDO_SELCUST  2149
#define  MSG_UNDO_SELCUSTPROX  2150
#define  MSG_UNDO_SELCUSTRANGE  2151
#define  MSG_UNDO_SELCUSTSLICE  2152
#define  MSG_UNDO_SELHYD  2153
#define  MSG_UNDO_SELINV  2154
#define  MSG_UNDO_SELMOL  2155
#define  MSG_UNDO_SELNOCONSTR  2156
#define  MSG_UNDO_SELNOHYD  2157
#define  MSG_UNDO_SELNONE  2158
#define  MSG_UNDO_SELNOWAT  2159
#define  MSG_UNDO_SELSEG  2160
#define  MSG_UNDO_SELSIDECHAINS  2161
#define  MSG_UNDO_SELSUPATM  2162
#define  MSG_UNDO_SELWAT  2163
#define  MSG_UNDO_SIDECHAINS  2164
#define  MSG_UNDO_SURF  2165
#define  MSG_UNDO_SURFCOLOR  2166
#define  MSG_UNDO_SURFDOTSIZE  2167
#define  MSG_UNDO_SURFGRAD  2168
#define  MSG_UNDO_SURFHIDEALL  2169
#define  MSG_UNDO_SURFSHOW  2170
#define  MSG_UNDO_SURFSHOWALL  2171
#define  MSG_UNDO_SURFTRANS  2172
#define  MSG_UNDO_TRANSLATE  2173
#define  MSG_UNDO_UNSELCUST  2174
#define  MSG_UNDO_UNSELCUSTPROX  2175
#define  MSG_UNDO_UNSELCUSTRANGE  2176
#define  MSG_UNDO_UNSELCUSTSLICE  2177
#define  MSG_ERR_UNDO_CREATE  2178
#define  MSG_ERR_UNDO_INTERNAL  2179
#define  MSG_ERR_AUTODOCK_CORR  2180
#define  MSG_BT_CALCULATE  2181
#define  MSG_BT_DELETE  2182
#define  MSG_BT_RESTART  2183
#define  MSG_BT_UNSELECT  2184
#define  MSG_DELETE  2185
#define  MSG_ERR_EXCEPT_DUMP  2186
#define  MSG_DLGNAMD_CFGEDIT_TITLE  2187
#define  MSG_DLGNAMD_CK_AUTOINPUT  2188
#define  MSG_DLGNAMD_CK_BINOUTPUT  2189
#define  MSG_DLGNAMD_CK_BINRESTART  2190
#define  MSG_DLGNAMD_CK_BINVEL  2191
#define  MSG_DLGNAMD_CK_CFGIGNFILES  2192
#define  MSG_DLGNAMD_CK_IMDIGNORE  2193
#define  MSG_DLGNAMD_CK_IMDON  2194
#define  MSG_DLGNAMD_CK_IMDWAIT  2195
#define  MSG_DLGNAMD_FILTER_ALL  2196
#define  MSG_DLGNAMD_FILTER_DCD  2197
#define  MSG_DLGNAMD_FILTER_FFTW  2198
#define  MSG_DLGNAMD_FILTER_INP  2199
#define  MSG_DLGNAMD_FILTER_NAMD  2200
#define  MSG_DLGNAMD_FILTER_PDB  2201
#define  MSG_DLGNAMD_FILTER_PDBB  2202
#define  MSG_DLGNAMD_FILTER_PSF  2203
#define  MSG_DLGNAMD_FILTER_VEL  2204
#define  MSG_DLGNAMD_GRP_CONJUGATE  2205
#define  MSG_DLGNAMD_GRP_FIXED  2206
#define  MSG_DLGNAMD_GRP_HARMONIC  2207
#define  MSG_DLGNAMD_GRP_IMD  2208
#define  MSG_DLGNAMD_GRP_LANGEVIN  2209
#define  MSG_DLGNAMD_GRP_MULTIPLETIMESTEP  2210
#define  MSG_DLGNAMD_GRP_OTHERCONFIG  2211
#define  MSG_DLGNAMD_GRP_OTHERPARAM  2212
#define  MSG_DLGNAMD_GRP_OUTPAR  2213
#define  MSG_DLGNAMD_GRP_REASSIGN  2214
#define  MSG_DLGNAMD_GRP_RESCALING  2215
#define  MSG_DLGNAMD_GRP_TCOUPLE  2216
#define  MSG_DLGNAMD_GRP_TIMESTEP  2217
#define  MSG_DLGNAMD_GRP_VELQUENCHING  2218
#define  MSG_DLGNAMD_LBL_14SCALING  2219
#define  MSG_DLGNAMD_LBL_CELLVECT1  2220
#define  MSG_DLGNAMD_LBL_CELLVECT2  2221
#define  MSG_DLGNAMD_LBL_CELLVECT3  2222
#define  MSG_DLGNAMD_LBL_CELLORIGIN  2223
#define  MSG_DLGNAMD_LBL_COMMOTION  2224
#define  MSG_DLGNAMD_LBL_CONFIG  2225
#define  MSG_DLGNAMD_LBL_CONSEXP  2226
#define  MSG_DLGNAMD_LBL_CONSKCOL  2227
#define  MSG_DLGNAMD_LBL_CONSKFILE  2228
#define  MSG_DLGNAMD_LBL_CONSREF  2229
#define  MSG_DLGNAMD_LBL_CONSSCALING  2230
#define  MSG_DLGNAMD_LBL_CONSSELECT  2231
#define  MSG_DLGNAMD_LBL_CONSTRAINTS  2232
#define  MSG_DLGNAMD_LBL_COORDINATES  2233
#define  MSG_DLGNAMD_LBL_CYLAXIS  2234
#define  MSG_DLGNAMD_LBL_CYLL1  2235
#define  MSG_DLGNAMD_LBL_CYLL2  2236
#define  MSG_DLGNAMD_LBL_CUTOFF  2237
#define  MSG_DLGNAMD_LBL_DCDFILE  2238
#define  MSG_DLGNAMD_LBL_DCDFREQ  2239
#define  MSG_DLGNAMD_LBL_DCDUNITCELL  2240
#define  MSG_DLGNAMD_LBL_DIELECTRIC  2241
#define  MSG_DLGNAMD_LBL_ENABLED  2242
#define  MSG_DLGNAMD_LBL_EXCLUDE  2243
#define  MSG_DLGNAMD_LBL_EXTENDEDSYS  2244
#define  MSG_DLGNAMD_LBL_FFTWESTIMATE  2245
#define  MSG_DLGNAMD_LBL_FFTWUSEWISDOM  2246
#define  MSG_DLGNAMD_LBL_FFTWWISDOMFILE  2247
#define  MSG_DLGNAMD_LBL_FIRSTTIMESTEP  2248
#define  MSG_DLGNAMD_LBL_FIXEDATOMS  2249
#define  MSG_DLGNAMD_LBL_FIXEDATOMSCOL  2250
#define  MSG_DLGNAMD_LBL_FIXEDATOMSFILE  2251
#define  MSG_DLGNAMD_LBL_FIXEDATOMSFORCES  2252
#define  MSG_DLGNAMD_LBL_FORCEFIELD  2253
#define  MSG_DLGNAMD_LBL_FULLDIRECT  2254
#define  MSG_DLGNAMD_LBL_FULLELECTFREQ  2255
#define  MSG_DLGNAMD_LBL_HGROUPCUTOFF  2256
#define  MSG_DLGNAMD_LBL_IMDFREQ  2257
#define  MSG_DLGNAMD_LBL_IMDPORT  2258
#define  MSG_DLGNAMD_LBL_LANGEVIN  2259
#define  MSG_DLGNAMD_LBL_LANGEVINCOL  2260
#define  MSG_DLGNAMD_LBL_LANGEVINDAMP  2261
#define  MSG_DLGNAMD_LBL_LANGEVINFILE  2262
#define  MSG_DLGNAMD_LBL_LANGEVINDHYDROG  2263
#define  MSG_DLGNAMD_LBL_LANGEVINTEMP  2264
#define  MSG_DLGNAMD_LBL_LIMITDIST  2265
#define  MSG_DLGNAMD_LBL_LONGSPLITTING  2266
#define  MSG_DLGNAMD_LBL_MARGIN  2267
#define  MSG_DLGNAMD_LBL_MAXIMUMMOVE  2268
#define  MSG_DLGNAMD_LBL_MERGECROSSTERMS  2269
#define  MSG_DLGNAMD_LBL_MINBABYSTEP  2270
#define  MSG_DLGNAMD_LBL_MINLINEGOAL  2271
#define  MSG_DLGNAMD_LBL_MINTINYSTEP  2272
#define  MSG_DLGNAMD_LBL_MOLLYITERATIONS  2273
#define  MSG_DLGNAMD_LBL_MOLLYTOLERANCE  2274
#define  MSG_DLGNAMD_LBL_MTSALGORITHM  2275
#define  MSG_DLGNAMD_LBL_NONBONDEDFREQ  2276
#define  MSG_DLGNAMD_LBL_NONBONDEDSCALING  2277
#define  MSG_DLGNAMD_LBL_NUMSTEPS  2278
#define  MSG_DLGNAMD_LBL_OUTPUTENE  2279
#define  MSG_DLGNAMD_LBL_OUTPUTMOMENTA  2280
#define  MSG_DLGNAMD_LBL_OUTPUTNAME  2281
#define  MSG_DLGNAMD_LBL_OUTPUTPAIRLISTS  2282
#define  MSG_DLGNAMD_LBL_OUTPUTPRESS  2283
#define  MSG_DLGNAMD_LBL_OUTPUTTIMING  2284
#define  MSG_DLGNAMD_LBL_PAIRLISTDIST  2285
#define  MSG_DLGNAMD_LBL_PAIRLISTGROW  2286
#define  MSG_DLGNAMD_LBL_PAIRLISTMINPROCS  2287
#define  MSG_DLGNAMD_LBL_PAIRLISTSHRINK  2288
#define  MSG_DLGNAMD_LBL_PAIRLISTTRIGGER  2289
#define  MSG_DLGNAMD_LBL_PAIRLISTPERCYCLE  2290
#define  MSG_DLGNAMD_LBL_PARAMETERS  2291
#define  MSG_DLGNAMD_LBL_PARAMFORMAT  2292
#define  MSG_DLGNAMD_LBL_PME  2293
#define  MSG_DLGNAMD_LBL_PMEGRIDPOINTS  2294
#define  MSG_DLGNAMD_LBL_PMEGRIDSIZEX  2295
#define  MSG_DLGNAMD_LBL_PMEGRIDSIZEY  2296
#define  MSG_DLGNAMD_LBL_PMEGRIDSIZEZ  2297
#define  MSG_DLGNAMD_LBL_PMEGRIDSPACING  2298
#define  MSG_DLGNAMD_LBL_PMEINTERP  2299
#define  MSG_DLGNAMD_LBL_PMEPROCESSORS  2300
#define  MSG_DLGNAMD_LBL_PMETOLERANCE  2301
#define  MSG_DLGNAMD_LBL_REASSIGNFREQ  2302
#define  MSG_DLGNAMD_LBL_REASSIGNHOLD  2303
#define  MSG_DLGNAMD_LBL_REASSIGNINCR  2304
#define  MSG_DLGNAMD_LBL_REASSIGNTEMP  2305
#define  MSG_DLGNAMD_LBL_RESCALEFREQ  2306
#define  MSG_DLGNAMD_LBL_RESCALETEMP  2307
#define  MSG_DLGNAMD_LBL_RESTARTFREQ  2308
#define  MSG_DLGNAMD_LBL_RESTARTNAME  2309
#define  MSG_DLGNAMD_LBL_RESTARTSAVE  2310
#define  MSG_DLGNAMD_LBL_RIGIDBONDS  2311
#define  MSG_DLGNAMD_LBL_RIGIDDIE  2312
#define  MSG_DLGNAMD_LBL_RIGIDITERATIONS  2313
#define  MSG_DLGNAMD_LBL_RIGIDTOLERANCE  2314
#define  MSG_DLGNAMD_LBL_SEED  2315
#define  MSG_DLGNAMD_LBL_SPHERCENTER  2316
#define  MSG_DLGNAMD_LBL_SPHEREXP1  2317
#define  MSG_DLGNAMD_LBL_SPHEREXP2  2318
#define  MSG_DLGNAMD_LBL_SPHERK1  2319
#define  MSG_DLGNAMD_LBL_SPHERK2  2320
#define  MSG_DLGNAMD_LBL_SPHERR1  2321
#define  MSG_DLGNAMD_LBL_SPHERR2  2322
#define  MSG_DLGNAMD_LBL_SPLITPATCH  2323
#define  MSG_DLGNAMD_LBL_STEPSPERCYCLE  2324
#define  MSG_DLGNAMD_LBL_STRUCTURE  2325
#define  MSG_DLGNAMD_LBL_SWITCHDIST  2326
#define  MSG_DLGNAMD_LBL_SWITCHING  2327
#define  MSG_DLGNAMD_LBL_TCOUPLE  2328
#define  MSG_DLGNAMD_LBL_TCOUPLEFILE  2329
#define  MSG_DLGNAMD_LBL_TCOUPLECOL  2330
#define  MSG_DLGNAMD_LBL_TCOUPLETEMP  2331
#define  MSG_DLGNAMD_LBL_TEMPERATURE  2332
#define  MSG_DLGNAMD_LBL_TIMESTEPSIZE  2333
#define  MSG_DLGNAMD_LBL_USEMOLLY  2334
#define  MSG_DLGNAMD_LBL_USESETTLE  2335
#define  MSG_DLGNAMD_LBL_VDWGEOSIGMA  2336
#define  MSG_DLGNAMD_LBL_VELDCDFILE  2337
#define  MSG_DLGNAMD_LBL_VELDCDFREQ  2338
#define  MSG_DLGNAMD_LBL_VELOCITIES  2339
#define  MSG_DLGNAMD_LBL_WRAPALL  2340
#define  MSG_DLGNAMD_LBL_WRAPNEAREST  2341
#define  MSG_DLGNAMD_LBL_WRAPWATER  2342
#define  MSG_DLGNAMD_LBL_XSTFILE  2343
#define  MSG_DLGNAMD_LBL_ZEROMOMENTUM  2344
#define  MSG_DLGNAMD_OD_CONSKFILE  2345
#define  MSG_DLGNAMD_OD_CONSREF  2346
#define  MSG_DLGNAMD_OD_COORDINATES  2347
#define  MSG_DLGNAMD_OD_FFTWWISDOMFILE  2348
#define  MSG_DLGNAMD_OD_FIXEDATOMSFILE  2349
#define  MSG_DLGNAMD_OD_LANGEVINFILE  2350
#define  MSG_DLGNAMD_OD_NAMDFILE  2351
#define  MSG_DLGNAMD_OD_PARAMETERS  2352
#define  MSG_DLGNAMD_OD_STRUCTURE  2353
#define  MSG_DLGNAMD_OD_TCOUPLEFILE  2354
#define  MSG_DLGNAMD_OD_VELOCITIES  2355
#define  MSG_DLGNAMD_PM_CFGDEFAULT  2356
#define  MSG_DLGNAMD_PM_CFGIGNFILES  2357
#define  MSG_DLGNAMD_PM_CFGLOAD  2358
#define  MSG_DLGNAMD_PM_CFGSAVE  2359
#define  MSG_DLGNAMD_PM_PAGEBASIC  2360
#define  MSG_DLGNAMD_PM_PAGEBC  2361
#define  MSG_DLGNAMD_PM_PAGECONSTRAINTS  2362
#define  MSG_DLGNAMD_PM_PAGECYLINDRICAL  2363
#define  MSG_DLGNAMD_PM_PAGEDYNAMICS  2364
#define  MSG_DLGNAMD_PM_PAGEINPUT  2365
#define  MSG_DLGNAMD_PM_PAGEMINIMIZATION  2366
#define  MSG_DLGNAMD_PM_PAGEOTHER  2367
#define  MSG_DLGNAMD_PM_PAGEOUTPUT  2368
#define  MSG_DLGNAMD_PM_PAGEPERIODIC  2369
#define  MSG_DLGNAMD_PM_PAGEPME  2370
#define  MSG_DLGNAMD_PM_PAGESIMSPACE  2371
#define  MSG_DLGNAMD_PM_PAGESPHERICAL  2372
#define  MSG_DLGNAMD_PM_PAGETEMPERATURE  2373
#define  MSG_DLGNAMD_PM_REFRESH  2374
#define  MSG_DLGNAMD_PRESET_FILE  2375
#define  MSG_DLGNAMD_PRESET_NAME  2376
#define  MSG_DLGNAMD_PRESET_RENAME  2377
#define  MSG_DLGNAMD_RG_CALC  2378
#define  MSG_DLGNAMD_RG_CALC_CONJ  2379
#define  MSG_DLGNAMD_RG_CALC_DYN  2380
#define  MSG_DLGNAMD_RG_CALC_VEL  2381
#define  MSG_DLGNAMD_RG_RUNMODE  2382
#define  MSG_DLGNAMD_RG_RUNMODE_INT  2383
#define  MSG_DLGNAMD_RG_RUNMODE_PREP  2384
#define  MSG_DLGNAMD_RG_RUNMODE_SEP  2385
#define  MSG_DLGNAMD_SD_DCDFILE  2386
#define  MSG_DLGNAMD_SD_NAMDFILE  2387
#define  MSG_DLGNAMD_SD_OUTPUTNAME  2388
#define  MSG_DLGNAMD_SD_RESTARTNAME  2389
#define  MSG_DLGNAMD_SD_VELDCDFILE  2390
#define  MSG_EZGEOM_WARN_BONDORD  2391
#define  MSG_MAIN_MENU_LBLATM_CHIRAL  2392
#define  MSG_MAIN_MENU_NAMD  2393
#define  MSG_MAIN_MENU_RMHYDAPOL  2394
#define  MSG_MAIN_MENU_NAMDHELP  2395
#define  MSG_ERR_INCHI_INTERNAL  2396
#define  MSG_ERR_INCHI_PROMPT  2397
#define  MSG_WARN_INCHI_PROMPT  2398
#define  MSG_INFO_GCMR  2399
#define  MSG_INFOXML_ATMTYPES  2400
#define  MSG_INFOXML_CHIRAL  2401
#define  MSG_INFOXML_CHIRALNUM  2402
#define  MSG_INFOXML_EZBONDS  2403
#define  MSG_INFOXML_GCMR  2404
#define  MSG_INFOXML_EZNUM  2405
#define  MSG_INFOXML_HBONDACC  2406
#define  MSG_INFOXML_HBONDDON  2407
#define  MSG_INFOXML_MOLNAME  2408
#define  MSG_INFOXML_REALMOLS  2409
#define  MSG_INFOXML_RINGS  2410
#define  MSG_INFOXML_TORNUM  2411
#define  MSG_INFOXML_TORFLEXNUM  2412
#define  MSG_LOADER_INCHI  2413
#define  MSG_LOADER_PDBQT  2414
#define  MSG_LOADER_MOPZMAT  2415
#define  MSG_ERR_MAIN_DBEXMODE  2416
#define  MSG_ERR_MAIN_DBINVREC  2417
#define  MSG_ERR_MAIN_DBNOTFOUND  2418
#define  MSG_ERR_MAIN_DBTOOMANY  2419
#define  MSG_ERR_MAIN_RMHYDROG  2420
#define  MSG_ERR_MAIN_TORTYPE  2421
#define  MSG_MOPAC_LBLMIS  2422
#define  MSG_MOPAC_LBLUNUSED  2423
#define  MSG_ERR_NAMD_COORDS  2424
#define  MSG_ERR_NAMD_ENDIAN  2425
#define  MSG_ERR_NAMD_ENERGIES  2426
#define  MSG_ERR_NAMD_FULLELECTFREQUENCY  2427
#define  MSG_ERR_NAMD_LOAD_INVARG  2428
#define  MSG_ERR_NAMD_LOAD_INVFLOAT  2429
#define  MSG_ERR_NAMD_LOAD_INVINT  2430
#define  MSG_ERR_NAMD_LOAD_INVVECT  2431
#define  MSG_ERR_NAMD_LOAD_MISPAR  2432
#define  MSG_ERR_NAMD_LOAD_PROMPT  2433
#define  MSG_ERR_NAMD_LOAD_UNKARGTYP  2434
#define  MSG_ERR_NAMD_MUTEX  2435
#define  MSG_ERR_NAMD_NONBONDEDFREQ  2436
#define  MSG_ERR_NAMD_NOTINSTALLED  2437
#define  MSG_ERR_NAMD_PAIRLISTDIST  2438
#define  MSG_ERR_NAMD_PARFILE  2439
#define  MSG_ERR_NAMD_PMEGRIDSIZEX  2440
#define  MSG_ERR_NAMD_PMEGRIDSIZEY  2441
#define  MSG_ERR_NAMD_PMEGRIDSIZEZ  2442
#define  MSG_ERR_NAMD_READFORCE  2443
#define  MSG_ERR_NAMD_SENDFORCE  2444
#define  MSG_ERR_NAMD_SWITCHDIST  2445
#define  MSG_ERR_NAMD_TCPPORT  2446
#define  MSG_POPUP_MENU_EZGEOM  2447
#define  MSG_WARN_PDB_ALTLOC  2448
#define  MSG_RECOVERY_MESSAGE  2449
#define  MSG_RECOVERY_TITLE  2450
#define  MSG_ERR_RENAME  2451
#define  MSG_UNDO_EZGEOM  2452
#define  MSG_UNDO_NAMD  2453
#define  MSG_UNDO_RMHYDAPOL  2454
#define  MSG_UPDATE  2455
#define  MSG_UTILS_RMHPROGAPOL  2456
#define  MSG_ERR_ARGPAR_FRMINV  2457
#define  MSG_ERR_ARGPAR_FRMINVFIRST  2458
#define  MSG_ERR_ARGPAR_FRMINVLAST  2459
#define  MSG_ERR_ARGPAR_FRMMIS  2460
#define  MSG_ERR_ARGPAR_FRMRNG  2461
#define  MSG_BT_CHANGE  2462
#define  MSG_BT_FILTER  2463
#define  MSG_DCD_EXTRACTFRM  2464
#define  MSG_DCD_PROCFRAME_ZZ  2465
#define  MSG_DLGCOLORSET_LBL_COLBUMP  2466
#define  MSG_DLGDBEXP_CB_NORMALIZE  2467
#define  MSG_DLGDBEXP_CB_MINCONJ  2468
#define  MSG_DLGDBEXP_CB_MINSTEEP  2469
#define  MSG_DLGDBEXP_GB_OPTMIN  2470
#define  MSG_DLGDBEXP_LBL_MINSTEPS  2471
#define  MSG_DLGDBEXP_LBL_MINTOLER  2472
#define  MSG_DLGDBEXP_PROG_COPY  2473
#define  MSG_DLGDBEXP_RG_OPTADDH  2474
#define  MSG_DLGDBEXP_RG_DOALWAYS  2475
#define  MSG_DLGDBEXP_RG_DONEEDED  2476
#define  MSG_DLGDBEXP_RG_DONEVER  2477
#define  MSG_DLGDBEXP_RG_OPTTO3D  2478
#define  MSG_DLGDBFILTER_BT_GETSQL  2479
#define  MSG_DLGDBFILTER_LV_FIELDS  2480
#define  MSG_DLGDBFILTER_LV_QUERY  2481
#define  MSG_DLGDBFILTER_OD_FILTER  2482
#define  MSG_DLGDBFILTER_OD_TITLE  2483
#define  MSG_DLGDBFILTER_SD_TITLE  2484
#define  MSG_DLGDBFILTER_TS_FILTER  2485
#define  MSG_DLGDBFILTER_TS_SQL  2486
#define  MSG_DLGDBFILTER_TITLE  2487
#define  MSG_DLGDBOPEN_CB_FSQLITE  2488
#define  MSG_DLGHBOND_SAVETITLE  2489
#define  MSG_DLGHBOND_CB_SAVE  2490
#define  MSG_DLGHBOND_SAVEFILTER  2491
#define  MSG_BT_CONFIGDEFAULT_HINT  2492
#define  MSG_DLGNAMD_BT_CONFIGLOAD_HINT  2493
#define  MSG_DLGNAMD_BT_CONFIGSAVE_HINT  2494
#define  MSG_DLGNAMD_CK_CFGIGNFILES_HINT  2495
#define  MSG_DLGNAMD_LB_CONFIG_HINT  2496
#define  MSG_DLGNAMD_LBL_BINCOORDINATES  2497
#define  MSG_DLGNAMD_OD_BINCOORDINATES  2498
#define  MSG_DLGNAMD_CFGLOAD_PROGRESS  2499
#define  MSG_SEL_TBI_TORAUTOD  2500
#define  MSG_SEL_TBS_TORADD  2501
#define  MSG_ERR_FFIELD_INVELEM  2502
#define  MSG_ERR_FFIELD_INVOP  2503
#define  MSG_ERR_FFIELD_MULTIOP  2504
#define  MSG_ERR_FFIELD_REQOP  2505
#define  MSG_GLMEAS_BUMPCHK  2506
#define  MSG_GLMEAS_BUMPCHKNOBMP  2507
#define  MSG_MAIN_MENU_ISOTOPIC  2508
#define  MSG_MAIN_MENU_SELNOAPOLHYD  2509
#define  MSG_POPUP_MENU_BUMPCHK  2510
#define  MSG_POPUP_MENU_BUMPCHKRT  2511
#define  MSG_INFOXML_ISOTOPIC  2512
#define  MSG_INFOXML_TORADOCKNUM  2513
#define  MSG_ERR_ISOTOPIC_GRAPHLBLX  2514
#define  MSG_ERR_ISOTOPIC_GRAPHLBLY  2515
#define  MSG_ERR_ISOTOPIC_GRAPHTITLE  2516
#define  MSG_ERR_ISOTOPIC_FRMELEM  2517
#define  MSG_ERR_ISOTOPIC_FRMINV  2518
#define  MSG_ERR_ISOTOPIC_FRMINVCHAR  2519
#define  MSG_ERR_ISOTOPIC_FRMINVINDEX  2520
#define  MSG_ERR_ISOTOPIC_TABEMPTY  2521
#define  MSG_ERR_ISOTOPIC_TABINVMASS  2522
#define  MSG_ERR_ISOTOPIC_TABNOTFOUND  2523
#define  MSG_ERR_ISOTOPIC_TABUNKFORMAT  2524
#define  MSG_ERR_CPUS  2525
#define  MSG_ERR_TRAJ_INVHANDLE  2526
#define  MSG_UNDO_BUMPCHK  2527
#define  MSG_UNDO_SELNOAPOLHYD  2528
#define  MSG_ATMSEL_BT_PREBACKBONE  2529
#define  MSG_BNDCHKLEN_BONDS  2530
#define  MSG_BNDCHKLEN_NUMBER  2531
#define  MSG_BNDCHKLEN_NOINVBOND  2532
#define  MSG_ERR_CANO_PROMPT  2533
#define  MSG_ERR_DBENGINE_INVHANDLE  2534
#define  MSG_ERR_DBENGINE_SQL  2535
#define  MSG_ERR_DBENGINE_SQLSUPP  2536
#define  MSG_ANA_CB_IGNOREWAT_HINT  2537
#define  MSG_ANA_CB_SHOWPLOT_HINT  2538
#define  MSG_ANA_CB_VISONLY_HINT  2539
#define  MSG_DLGDBEXP_CB_BNDORD  2540
#define  MSG_DLGINTER_CB_CLIP  2541
#define  MSG_DLGPREFS_CB_OCLDEVACCEL  2542
#define  MSG_DLGPREFS_CB_OCLDEVAUTO  2543
#define  MSG_DLGPREFS_CB_OCLDEVCPU  2544
#define  MSG_DLGPREFS_CB_OCLDEVDEF  2545
#define  MSG_DLGPREFS_CB_OCLDEVGPU  2546
#define  MSG_DLGPREFS_CB_OCLDEVNOTUSED  2547
#define  MSG_DLGPREFS_LBL_OCLDEV  2548
#define  MSG_DLGPREFS_LBL_OCLSTATUS  2549
#define  MSG_DLGPREFS_LBL_OCLSTATOP  2550
#define  MSG_DLGPREFS_LBL_OCLSTATNOACC  2551
#define  MSG_DLGSMILES_LBL_SMILES  2552
#define  MSG_DLGSMILES_RG_DEST  2553
#define  MSG_DLGSMILES_RG_DEST_CURWKS  2554
#define  MSG_DLGSMILES_RG_DEST_NEWWKS  2555
#define  MSG_DLGSMILES_TITLE  2556
#define  MSG_VIEWPREFS_RGB10  2557
#define  MSG_EXCEL_EXPORTDB  2558
#define  MSG_MAIN_MENU_BUILDSMILES  2559
#define  MSG_MAIN_MENU_BNDLEN  2560
#define  MSG_MAIN_MENU_SELBACKDNA  2561
#define  MSG_INFOXML_MIMASS  2562
#define  MSG_INFOXML_SMILES  2563
#define  MSG_LOADER_PQR  2564
#define  MSG_SCORE_CLIPBOARD  2565
#define  MSG_SCORE_ENERGIES  2566
#define  MSG_SCORE_ENE_BROTO  2567
#define  MSG_SCORE_ENE_BROTO2  2568
#define  MSG_SCORE_ENE_BROTO3  2569
#define  MSG_SCORE_ENE_BROTOF  2570
#define  MSG_SCORE_ENE_CHARMM  2571
#define  MSG_SCORE_ENE_CHARMM22  2572
#define  MSG_SCORE_ENE_CHARMM36  2573
#define  MSG_SCORE_ENE_CVFF  2574
#define  MSG_SCORE_ENE_ELECT  2575
#define  MSG_SCORE_ENE_ELECTDD  2576
#define  MSG_SCORE_NAME_BROTO  2577
#define  MSG_SCORE_NAME_BROTO2  2578
#define  MSG_SCORE_NAME_BROTO3  2579
#define  MSG_SCORE_NAME_BROTOF  2580
#define  MSG_SCORE_NAME_CHARMM  2581
#define  MSG_SCORE_NAME_CHARMM22  2582
#define  MSG_SCORE_NAME_CHARMM36  2583
#define  MSG_SCORE_NAME_CVFF  2584
#define  MSG_SCORE_NAME_ELECT  2585
#define  MSG_SCORE_NAME_ELECTDD  2586
#define  MSG_TURBO  2587
#define  MSG_TURBO_DATABASE  2588
#define  MSG_TURBO_TRAJECTORY  2589
#define  MSG_ERR_SCORE_ENESEL  2590
#define  MSG_ERR_SCORE_ENETYP  2591
#define  MSG_ERR_SCORE_LIG  2592
#define  MSG_ERR_SCORE_PAR  2593
#define  MSG_ERR_SCORE_ONEMOL  2594
#define  MSG_ERR_SCORE_POT  2595
#define  MSG_UNDO_CHKBNDLEN  2596
#define  MSG_UNDO_SMILES  2597
#define  MSG_ERR_AMMP_SAVE_CHKFF  2598
#define  MSG_ERR_DBENGINE_DAT  2599
#define  MSG_ERR_DBENGINE_ODBC_CONNECT  2600
#define  MSG_ERR_DBENGINE_ODBC_HANDLE  2601
#define  MSG_ERR_DBENGINE_ODBC_VERSION  2602
#define  MSG_ERR_DBENGINE_SQLITE_OPEN  2603
#define  MSG_ERR_DBENGINE_SQLITE_SETPAR  2604
#define  MSG_ERR_DBENGINE_SQLITE_STRUCT  2605
#define  MSG_ERR_DBENGINE_SQLITE_VEGATAB  2606
#define  MSG_ERR_DBENGINE_UPDDATA  2607
#define  MSG_ERR_DBENGINE_ZIP_OPEN  2608
#define  MSG_ERR_DBREP_END  2609
#define  MSG_ERR_DBREP_FILENAME  2610
#define  MSG_ERR_DBREP_HEAD  2611
#define  MSG_ERR_DBREP_IMGHEIGHT  2612
#define  MSG_ERR_DBREP_IMGWIDTH  2613
#define  MSG_ERR_DBREP_LOAD  2614
#define  MSG_ERR_DBREP_MISENDOFSEC  2615
#define  MSG_ERR_DBREP_NUMOFARGS  2616
#define  MSG_ERR_DBREP_PROMPT  2617
#define  MSG_ERR_DBREP_SMILES  2618
#define  MSG_ERR_DBREP_TAIL  2619
#define  MSG_ERR_DCD_ATMNUM  2620
#define  MSG_DLGCOLORSET_GRP_COLFLEXTOR  2621
#define  MSG_DLGCOLORSET_LBL_COLFTFLEX  2622
#define  MSG_DLGCOLORSET_LBL_COLFTRIGID  2623
#define  MSG_DLGDBEXP_HC_REPORT  2624
#define  MSG_DLGDBEXP_PN_PREVIEW  2625
#define  MSG_DLGDBEXP_TS_OPTIONS1  2626
#define  MSG_DLGDBEXP_TS_PREVIEW  2627
#define  MSG_DLGDBEXP_TS_REPORTS  2628
#define  MSG_DLGDBOPEN_CB_ACCESS  2629
#define  MSG_DLGIMGSAVE_RG_IMAGETYPE  2630
#define  MSG_DLGMAIN_RM_LAYOUT  2631
#define  MSG_DLGMINIED_CASEINSENS  2632
#define  MSG_DLGMINIED_CASESENS  2633
#define  MSG_DLGMINIED_TEXTFIND  2634
#define  MSG_DLGMOPAC_CK_RMFILES  2635
#define  MSG_DLGMOPAC_GRP_FILES  2636
#define  MSG_DLGMOPAC_LBL_INPUT  2637
#define  MSG_DLGNAMD_CK_REMOVEFILES  2638
#define  MSG_DLGSHUTDOWN_LBL_COUNTDOWN  2639
#define  MSG_DLGSHUTDOWN_TITLE  2640
#define  MSG_DLGUPDATE_LBL_FILE  2641
#define  MSG_DLGUPDATE_LBL_FILENUM  2642
#define  MSG_DLGUPDATE_LBL_SIZE  2643
#define  MSG_DLGUPDATE_TITLE  2644
#define  MSG_ERR_INDIGO_PROMPT  2645
#define  MSG_ERR_R2D_UNKFORMAT  2646
#define  MSG_MAIN_MENU_COLBYFT  2647
#define  MSG_MAIN_MENU_COLBYFTAUTODOCK  2648
#define  MSG_MAIN_MENU_COLBYFTNORM  2649
#define  MSG_MAIN_MENU_FINDDOWN  2650
#define  MSG_MAIN_MENU_FINDUP  2651
#define  MSG_MAIN_MENU_LAYDELETE  2652
#define  MSG_MAIN_MENU_LAYSELECT  2653
#define  MSG_MAIN_MENU_LAYSAVE  2654
#define  MSG_POPUP_MENU_ATOM  2655
#define  MSG_POPUP_MENU_ATMADD_H  2656
#define  MSG_POPUP_MENU_BOND  2657
#define  MSG_POPUP_MENU_CHAIN  2658
#define  MSG_POPUP_MENU_CHANGE  2659
#define  MSG_POPUP_MENU_HIDE  2660
#define  MSG_POPUP_MENU_LABEL  2661
#define  MSG_POPUP_MENU_LENGTH  2662
#define  MSG_POPUP_MENU_MOLECULE  2663
#define  MSG_POPUP_MENU_REMOVE  2664
#define  MSG_POPUP_MENU_RESIDUE  2665
#define  MSG_POPUP_MENU_SEGMENT  2666
#define  MSG_POPUP_MENU_SHOW  2667
#define  MSG_POPUP_MENU_SHUTDOWN  2668
#define  MSG_HELP_TITLE  2669
#define  MSG_INFOXML_INCHI  2670
#define  MSG_INFOXML_INCHIKEY  2671
#define  MSG_NAMD_CALCSTART  2672
#define  MSG_NAMD_CALCFINISH  2673
#define  MSG_UNDO_COLBYFT  2674
#define  MSG_UNDO_MEASBOND  2675
#define  MSG_UNDO_RMCHAIN  2676
#define  MSG_UNDO_SELATM  2677
#define  MSG_UNDO_SELCHAIN  2678
#define  MSG_UNDO_SELRES  2679
#define  MSG_UNDO_UNSELATM  2680
#define  MSG_UNDO_UNSELCHAIN  2681
#define  MSG_UNDO_UNSELRES  2682
#define  MSG_UNDO_UNSELMOL  2683
#define  MSG_UNDO_UNSELSEG  2684
#define  MSG_UPDATE_AVAILABLE  2685
#define  MSG_UPDATE_INSTALL  2686
#define  MSG_UPDATE_MULTIPROC  2687
#define  MSG_UPDATE_NOUPDATE  2688
#define  MSG_ERR_UPDATE_ALRRUN  2689
#define  MSG_ERR_UPDATE_CONN  2690
#define  MSG_ERR_UPDATE_CORRFILE  2691
#define  MSG_ERR_UPDATE_CORRPACK  2692
#define  MSG_ERR_UPDATE_DATA  2693
#define  MSG_ERR_UPDATE_DIR  2694
#define  MSG_ERR_UPDATE_DOWNLOAD  2695
#define  MSG_ERR_UPDATE_RUNCPY  2696
#define  MSG_ERR_UPDATE_SCRIPT  2697
#define  MSG_ERR_UPDATE_SCRIPTRUN  2698
#define  MSG_ERR_UPDATE_URLINV  2699
#define  MSG_ERR_UPDATE_URLMIS  2700
#define  MSG_ERR_WKSLOCKMOL  2701
#define  MSG_DLGAMMP_CONV3D  2702
#define  MSG_DLGIUPAC_LBL_IUPAC  2703
#define  MSG_DLGIUPAC_TITLE  2704
#define  MSG_DLGDBOPEN_CB_FMOL2  2705
#define  MSG_MAIN_MENU_BUILDIUPAC  2706
#define  MSG_ERR_IUPAC_NAME  2707
#define  MSG_ERR_JAVA_NOTINSTALLED  2708
#define  MSG_LOADER_NAMDBIN  2709
#define  MSG_ERR_MAIN_MEASURETRJ  2710
#define  MSG_ERR_OPSIN_NOTINSTALLED  2711
#define  MSG_ERR_OPSIN_CANTRUN  2712
#define  MSG_ERR_SPILLO_MISLIG  2713
#define  MSG_TCPIO_PROGRESS  2714
#define  MSG_TCPIO_PROGRESSSIZE  2715
#define  MSG_ERR_TCPIO_DOWNLOAD  2716
#define  MSG_ERR_TCPIO_HTTPINIT  2717
#define  MSG_ERR_TCPIO_PARSEURL  2718
#define  MSG_ANA_CB_OVALITY  2719
#define  MSG_DCD_STARTOVAL  2720
#define  MSG_GRAPH_OVALITY  2721
#define  MSG_ERR_CDX_STRUCT  2722
#define  MSG_ATMSEL_CB_CENTER  2723
#define  MSG_DLGRMMOL_CB_CENTER  2724
#define  MSG_LOADER_CDX  2725
#define  MSG_LOADER_CLIPBOARD  2726
#define  MSG_LOADER_MOLFROMMEM  2727
#define  MSG_MAIN_MENU_CENTERVIEW  2728
#define  MSG_TBTOOLS_CENTERV  2729
#define  MSG_UNDO_CENTER  2730
#define  MSG_UNDO_CENTERV  2731
#define  MSG_DLGAMMP_LBL_TSRCEND  2732
#define  MSG_DLGAMMP_LV_TSRCTOR_END  2733
#define  MSG_DLGAMMP_PM_TSRCFILL  2734
#define  MSG_DLGAMMP_PM_TSRCFILL_BASE  2735
#define  MSG_DLGAMMP_PM_TSRCFILL_RANGE  2736
#define  MSG_DLGAMMP_PM_TSRCFILL_CUR  2737
#define  MSG_DLGAMMP_PM_TSRCFILL_DEF  2738
#define  MSG_MAIN_MENU_DOCKING  2739
#define  MSG_MAIN_MENU_DOWNLOAD  2740
#define  MSG_MAIN_MENU_CONV2D  2741
#define  MSG_MAIN_MENU_CONV3D  2742
#define  MSG_VIEWPREFS_ZCLIPNEAR  2743
#define  MSG_ERR_GRAMM_FRMNUM  2744
#define  MSG_ERR_GRAMM_LIGRANGE  2745
#define  MSG_ERR_GRAMM_LIGSEL  2746
#define  MSG_WARN_GRAMM_POSES  2747
#define  MSG_UNDO_2DCONVERSION  2748
#define  MSG_UNDO_3DCONVERSION  2749
#define  MSG_DLGCOMBI_BT_BUILD_HINT  2750
#define  MSG_DLGCOMBI_BT_GET_HINT  2751
#define  MSG_DLGCOMBI_BUILDING  2752
#define  MSG_DLGCOMBI_CB_CANONICAL  2753
#define  MSG_DLGCOMBI_CB_RMREDMOL  2754
#define  MSG_DLGCOMBI_CB_SYSTEMATIC  2755
#define  MSG_DLGCOMBI_HINT  2756
#define  MSG_DLGCOMBI_LBL_NAME  2757
#define  MSG_DLGCOMBI_LV_FRAGS_DB  2758
#define  MSG_DLGCOMBI_LV_FRAGS_NUM  2759
#define  MSG_DLGCOMBI_LV_FRAGS_HINT  2760
#define  MSG_DLGCOMBI_LV_FRAGS_POS  2761
#define  MSG_DLGCOMBI_OD_SMILES  2762
#define  MSG_DLGCOMBI_OD_SMILES_FLT  2763
#define  MSG_DLGCOMBI_PM_FRAGSLIB  2764
#define  MSG_DLGCOMBI_PN_PREVIEWLBL  2765
#define  MSG_DLGCOMBI_PN_SCAFFOLDLBL  2766
#define  MSG_DLGCOMBI_REMOVING  2767
#define  MSG_DLGCOMBI_LBL_SCAFFOLD  2768
#define  MSG_DLGCOMBI_SAVING  2769
#define  MSG_DLGCOMBI_SB_INSERTR_HINT  2770
#define  MSG_DLGCOMBI_SD_SMILES  2771
#define  MSG_DLGCOMBI_SD_SMILES_FLT  2772
#define  MSG_DLGCOMBI_SP_HINT  2773
#define  MSG_DLGCOMBI_STR_NAME  2774
#define  MSG_DLGCOMBI_STR_PREVIEW_HINT  2775
#define  MSG_DLGCOMBI_STR_SCAFFOLD_HINT  2776
#define  MSG_DLGCOMBI_TB_PREVIEW_HINT  2777
#define  MSG_DLGCOMBI_TITLE  2778
#define  MSG_ERR_DLGCOMBI_INVSMI  2779
#define  MSG_DLGDBOPEN_CB_FSMI  2780
#define  MSG_MAIN_MENU_BUILDCMBSMILES  2781
#define  MSG_LIGEN_SRFGRIDEXCL  2782
#define  MSG_LIGEN_SRFGRIDHBACC  2783
#define  MSG_LIGEN_SRFGRIDHBDON  2784
#define  MSG_LIGEN_SRFGRIDHYDRO  2785
#define  MSG_LIGEN_SRFGRIDREC  2786
#define  MSG_LIGEN_SRFGRIDSOLV  2787
#define  MSG_LIGEN_SRFGRIDVACANT  2788
#define  MSG_LIGEN_SRFPHARMHBACC  2789
#define  MSG_LIGEN_SRFPHARMHBDON  2790
#define  MSG_LIGEN_SRFPHARMHYDRO  2791
#define  MSG_ERR_LIGEN_FMTGRID  2792
#define  MSG_ERR_LIGEN_FMTPHARM  2793
#define  MSG_ERR_LIGEN_FMTPOCKET  2794
#define  MSG_LOADER_LIGENGRID  2795
#define  MSG_LOADER_LIGENGRIDBIN  2796
#define  MSG_LOADER_LIGENPHARM  2797
#define  MSG_LOADER_LIGENPHARMBIN  2798
#define  MSG_LOADER_LIGENPOCKET  2799
#define  MSG_LOADER_LIGENPOCKETBIN  2800
#define  MSG_MAIN_MENU_DATADIR  2801
#define  MSG_DLGDBEXP_CB_CHARGE  2802
#define  MSG_DLGDBEXP_PM_MOLCSV  2803
#define  MSG_DLGDBEXP_PROG_SIMIL  2804
#define  MSG_DLGDBEXP_SD_EXPFILTER  2805
#define  MSG_DLGDBEXP_SD_EXPTITLE  2806
#define  MSG_DLGDBFILTER_CB_SIMFPENC_BASE64  2807
#define  MSG_DLGDBFILTER_CB_SIMFPENC_HEX  2808
#define  MSG_DLGDBFILTER_CB_SIMSAVE  2809
#define  MSG_DLGDBFILTER_CSVHDR  2810
#define  MSG_DLGDBFILTER_GRP_SIMPAR  2811
#define  MSG_DLGDBFILTER_GRP_TVERSKY  2812
#define  MSG_DLGDBFILTER_LBL_ALPHA  2813
#define  MSG_DLGDBFILTER_LBL_BETA  2814
#define  MSG_DLGDBFILTER_LBL_SIMFPENC  2815
#define  MSG_DLGDBFILTER_LBL_SIMTHRESH  2816
#define  MSG_DLGDBFILTER_RG_FPTYPE  2817
#define  MSG_DLGDBFILTER_RG_FPTYPE_FULL  2818
#define  MSG_DLGDBFILTER_RG_FPTYPE_RESSUB  2819
#define  MSG_DLGDBFILTER_RG_FPTYPE_SIM  2820
#define  MSG_DLGDBFILTER_RG_FPTYPE_SUB  2821
#define  MSG_DLGDBFILTER_RG_FPTYPE_TAUSUB  2822
#define  MSG_DLGDBFILTER_RG_METHOD  2823
#define  MSG_DLGDBFILTER_SD_CSVFILTER  2824
#define  MSG_DLGDBFILTER_SD_CSVTITLE  2825
#define  MSG_DLGDBFILTER_TS_SIMILARITY  2826
#define  MSG_DLGMOPAC_CK_CONSTRAINTS  2827
#define  MSG_DLGMOPAC_HINT_MOPAC2007  2828
#define  MSG_DLGDBOPEN_CB_ACCESS2007  2829
#define  MSG_DLGADDHYD_CB_ADDHYDROG  2830
#define  MSG_DLGADDHYD_CB_IONIZE  2831
#define  MSG_DLGADDHYD_GRP_ACTIONS  2832
#define  MSG_DLGADDHYD_GRP_OPTIONS  2833
#define  MSG_DLGADDHYD_LBL_IONIZEPH  2834
#define  MSG_DLGADDHYD_LBL_IONIZETOL  2835
#define  MSG_DLGCLUSTER_RG_CLUSTPOSBOX  2836
#define  MSG_DLGDBEXP_CB_IONIZE  2837
#define  MSG_DLGDBEXP_CB_OPTGEOGEN  2838
#define  MSG_DLGDBEXP_CB_OPTSTEGEN  2839
#define  MSG_DLGDBEXP_LBL_IONIZETOL  2840
#define  MSG_DLGDBEXP_RG_OPTTAU  2841
#define  MSG_DLGDBEXP_RG_OPTTAU_INCHI  2842
#define  MSG_DLGDBEXP_RG_OPTTAU_NONE  2843
#define  MSG_DLGDBEXP_RG_OPTTAU_RSMARTS  2844
#define  MSG_DLGDBEXP_TS_OPTIONS2  2845
#define  MSG_ERR_FFIELD_SMARTSFILE  2846
#define  MSG_ERR_FFIELD_SMARTSINV  2847
#define  MSG_GEOISO_MOLNAME  2848
#define  MSG_INFO_HLB  2849
#define  MSG_INFOXML_HLBD  2850
#define  MSG_INFOXML_HLBG  2851
#define  MSG_INFOXML_HLBM  2852
#define  MSG_INFOXML_HLBPSA  2853
#define  MSG_IONIZE_PROG  2854
#define  MSG_ERR_IONIZE_NOPAR  2855
#define  MSG_STEISO_MOLNAME  2856
#define  MSG_TAU_MOLNAME  2857
#define  MSG_UNDO_GEOISOMER  2858
#define  MSG_UNDO_IONIZATION  2859
#define  MSG_UNDO_IUPAC  2860
#define  MSG_UNDO_SELCUSTSMARTS  2861
#define  MSG_UNDO_STEISOMER  2862
#define  MSG_UNDO_TAUTOMER  2863
#define  MSG_UNDO_UNSELCUSTSMARTS  2864
#define  MSG_ERR_DBENGINE_COLADD  2865
#define  MSG_ERR_DBENGINE_COLDELETE  2866
#define  MSG_ERR_DBENGINE_COLRENAME  2867
#define  MSG_ERR_DBENGINE_COLUPDATE  2868
#define  MSG_ERR_DBENGINE_PASTE_LINEMATCH  2869
#define  MSG_ERR_DBENGINE_PASTE_COLNUM  2870
#define  MSG_ERR_DBENGINE_PASTE_DATATYPE  2871
#define  MSG_ERR_DBENGINE_PASTE_DATATYPEINC  2872
#define  MSG_ERR_DBENGINE_PASTE_NOTHING  2873
#define  MSG_DLGABOUT_SYSTEM_WIN64  2874
#define  MSG_DLGABOUT_SYSTEM_WIN32  2875
#define  MSG_DLGDBEXP_BT_PASTECOLS  2876
#define  MSG_DLGDBEXP_COLDELETE  2877
#define  MSG_DLGDBEXP_COLPASTE  2878
#define  MSG_DLGDBEXP_COLRENAME  2879
#define  MSG_DLGDBEXP_RG_OPTCHARGE  2880
#define  MSG_DLGDBOPEN_ASKEXIST  2881
#define  MSG_DLGDBOPEN_DBEMPTY  2882
#define  MSG_DLGDBOPEN_DBFILENAME  2883
#define  MSG_DBREP_SAVING  2884
#define  MSG_LOADER_EMPIREARC  2885
#define  MSG_UNDO_RMVISATM  2886
#define  MSG_POPUP_MENU_WINDOWFSCREEN  2887
#define  MSG_DLGCONSOLE_LOG_FILENAME  2888
#define  MSG_DLGCONSOLE_PAUSED  2889
#define  MSG_DLGCONSOLE_PM_CLEARALL  2890
#define  MSG_DLGCONSOLE_PM_RECPAUSE  2891
#define  MSG_DLGCONSOLE_PM_RECRESUME  2892
#define  MSG_DLGCONSOLE_PM_PRINTBUF  2893
#define  MSG_DLGCONSOLE_PM_RECORD  2894
#define  MSG_DLGCONSOLE_PRINTJOB  2895
#define  MSG_DLGCONSOLE_PM_SAVEBUF  2896
#define  MSG_DLGCONSOLE_REDIRECTED  2897
#define  MSG_DLGCONSOLE_SD_FILTER  2898
#define  MSG_DLGCONSOLE_SD_TIT_RECORD  2899
#define  MSG_DLGCONSOLE_SD_TIT_SAVE  2900
#define  MSG_DLGIMGSAVE_IMAGETYPE_360  2901
#define  MSG_DLGIMGSAVE_IMAGETYPE_2D  2902
#define  MSG_DLGIMGSAVE_IMAGETYPE_3D  2903
#define  MSG_DLGIMGSAVE_IMAGETYPE_CUBE  2904
#define  MSG_DLGJOY_CK_ENABLED  2905
#define  MSG_DLGJOY_LBL_3DXBEEPER  2906
#define  MSG_DLGJOY_LBL_3DXFIRMWARE  2907
#define  MSG_DLGJOY_LBL_3DXPERIOD  2908
#define  MSG_DLGJOY_LBL_3DXROTX  2909
#define  MSG_DLGJOY_LBL_3DXROTY  2910
#define  MSG_DLGJOY_LBL_3DXROTZ  2911
#define  MSG_DLGJOY_LBL_3DXTRANSX  2912
#define  MSG_DLGJOY_LBL_3DXTRANSY  2913
#define  MSG_DLGJOY_LBL_3DXTRANSZ  2914
#define  MSG_DLGJOY_TS_MOUSE3D  2915
#define  MSG_ERR_PBUFFER_REN360_BUILD  2916
#define  MSG_ERR_PBUFFER_REN360_INJMETA  2917
#define  MSG_ERR_PBUFFER_REN360_PROG  2918
#define  MSG_ERR_PBUFFER_REN360_PRJ  2919
#define  MSG_ERR_PBUFFER_REN360_PRJ_BACK  2920
#define  MSG_ERR_PBUFFER_REN360_PRJ_BOTTOM  2921
#define  MSG_ERR_PBUFFER_REN360_PRJ_FRONT  2922
#define  MSG_ERR_PBUFFER_REN360_PRJ_LEFT  2923
#define  MSG_ERR_PBUFFER_REN360_PRJ_RIGHT  2924
#define  MSG_ERR_PBUFFER_REN360_PRJ_TOP  2925
#define  MSG_ERR_PBUFFER_REN360_SAVE  2926
#define  MSG_ATOM_SORTBYRES  2927
#define  MSG_ERR_COM_MOLDELETE  2928
#define  MSG_ERR_COM_SEGDELETE  2929
#define  MSG_DLGCOLORSET_CB_SKYSWAP  2930
#define  MSG_DLGDBEXP_CB_OPTKEEPLARGE  2931
#define  MSG_DLGDBEXP_CB_OPTRMIONS  2932
#define  MSG_DLGSSTR_LBL_CAPPING  2933
#define  MSG_DLGSSTR_ME_PATTERN_HINT  2934
#define  MSG_DLGSSTR_ME_SEQ_HINT  2935
#define  MSG_MAIN_MENU_SORTATMBYRES  2936
#define  MSG_MAIN_MENU_RMCIONS  2937
#define  MSG_SKYBOX_BUILDINGFACES  2938
#define  MSG_SKYBOX_CACHING  2939
#define  MSG_SKYBOX_LOADING  2940
#define  MSG_ERR_SKYBOX_AR  2941
#define  MSG_ERR_SKYBOX_COLOR  2942
#define  MSG_ERR_SKYBOX_DATAFORMAT  2943
#define  MSG_ERR_SKYBOX_FACEBMP  2944
#define  MSG_ERR_SKYBOX_INIT  2945
#define  MSG_ERR_SKYBOX_VERTEX  2946
#define  MSG_ERR_SKYBOX_VERTEXDUP  2947
#define  MSG_INFOXML_ANGLES  2948
#define  MSG_INFOXML_BONDS  2949
#define  MSG_INFOXML_ATMTYPEFREQ  2950
#define  MSG_ERR_INFOXML_KEYWORD  2951
#define  MSG_OPENVR_DISABLED  2952
#define  MSG_OPENVR_INFO  2953
#define  MSG_ERR_OPENVR_INIT  2954
#define  MSG_UNDO_RMCIONS  2955
#define  MSG_UNDO_SORTATMBYRES  2956
#define  MSG_UTILS_RMCIONSPROG  2957
#define  MSG_ERR_MAIN_CAPCTERM  2958
#define  MSG_ERR_MAIN_CAPNTERM  2959
#define  MSG_MAIN_MENU_BUILDDNA  2960
#define  MSG_ANA_CB_RMSDH  2961
#define  MSG_ANA_CB_RMSDHUNGARIAN  2962
#define  MSG_ANA_CB_RMSDHUNGARIAN_H  2963
#define  MSG_ANA_CB_RMSDSUPER  2964
#define  MSG_ANA_CB_RMSDSUPERH  2965
#define  MSG_DLGDNA_CB_ADDHYDROG  2966
#define  MSG_DLGDNA_NUCLEICACID  2967
#define  MSG_DLGDNA_GRP_OPTIONS  2968
#define  MSG_DLGDNA_RG_STRAND  2969
#define  MSG_DLGDNA_RG_STRAND_35  2970
#define  MSG_DLGDNA_RG_STRAND_53  2971
#define  MSG_DLGDNA_RG_STRAND_BOTH  2972
#define  MSG_DLGDNA_RG_TYPE  2973
#define  MSG_DLGDNA_RG_TYPE_ADNA  2974
#define  MSG_DLGDNA_RG_TYPE_ABDNA  2975
#define  MSG_DLGDNA_RG_TYPE_LBDNA  2976
#define  MSG_DLGDNA_RG_TYPE_SBDNA  2977
#define  MSG_DLGDNA_RG_TYPE_ARNA  2978
#define  MSG_DLGDNA_RG_TYPE_APRNA  2979
#define  MSG_DLGDNA_TITLE  2980
#define  MSG_DLGPREFS_LBL_ENECONTDIST  2981
#define  MSG_ERR_DNABUILD_INVSEQ  2982
#define  MSG_DNA_PROG  2983
#define  MSG_INFOXML_AAKEY1  2984
#define  MSG_INFOXML_AAKEY3  2985
#define  MSG_INFOXML_AACOMPNUM  2986
#define  MSG_INFOXML_AACOMPNUMPERC  2987
#define  MSG_INFOXML_AACOMPMASS  2988
#define  MSG_INFOXML_AACOMPMASSPERC  2989
#define  MSG_INFOXML_AAMASS  2990
#define  MSG_INFOXML_AAMASSPERC  2991
#define  MSG_INFOXML_AANUM  2992
#define  MSG_SCORE_ENE_CONTACTS  2993
#define  MSG_SCORE_ENE_CONTACTS_FP  2994
#define  MSG_SCORE_NAME_CONTACTS  2995
#define  MSG_UNDO_DNABUILD  2996
#define  MSG_MAIN_MENU_SELSUPRES  2997
#define  MSG_UNDO_SELSUPRES  2998
#define  MSG_INFOXML_COORDTYP  2999
#define  MSG_WARN_AMBERNC_CONVVER  3000


typedef struct
{
  unsigned int   Id;
  const char     *Str;
} LOCARRAY;


#ifdef VEGA_ARRAY

static const LOCARRAY VEGA_Str[] =
{
    { MSG_TRANSLATION, "Translation by Alessandro Pedretti" },
    { MSG_ERR_ARGPAR_MATCH, "The atom description %d has multiple matches" },
    { MSG_ERR_ARGPAR_INPFILE, "The atom description %d doesn't match the input file" },
    { MSG_ERR_ARGPAR_ATMNUM, "The atom descrpition %d exceeds the total number of atoms" },
    { MSG_ERR_ARGPAR_DUPDESC, "Duplicate description for atom %d and %d" },
    { MSG_ERR_ARGPAR_INCDESC, "Incorrect atom description" },
    { MSG_ERR_ARGPAR_TOOLONG, "Atom description too long" },
    { MSG_ERR_ARGPAR_ILLATMNUM, "Illegal atom number" },
    { MSG_ERR_ARGPAR_ILLNUMMES, "Illegal number of atoms in measure" },
    { MSG_ERR_ARGPAR_UNKMES, "Unknown measure mode" },
    { MSG_ERR_ARGPAR_TOOFEW, "Too few atoms to measure" },
    { MSG_ERR_BIOSYM_NOTSUPPORTED, "Biosym subformat not supported" },
    { MSG_CHARGE_ASSPROG, "* Assigning atom charges" },
    { MSG_ERR_CHARGE_TEMDESCDUP, "#Description duplicated at template line %d" },
    { MSG_CHARGE_TOTCHG, "  Total charge: %.4f\n" },
    { MSG_ERR_CHARGE_TEMRESEXP, "#ResName expected at template line %d" },
    { MSG_CHARGE_PROGPMT, "* Searching for residues" },
    { MSG_ERR_CHARGE_TEMFORMAT, "Unknown format of charge template" },
    { MSG_ERR_CHARGE_TEMRESLONG, "Residue name too long at template line %d" },
    { MSG_ERR_CHARGE_TEMRESMIS, "Missing residue name at template line %d" },
    { MSG_CONNECT_PROGPMT, "  Building connectivity" },
    { MSG_CONNECT_NORMPROG, "* Normalizing coordinates" },
    { MSG_ERR_CONNECT_UNKVDW, "  Unknown VDW radius" },
    { MSG_ERR_CONNECT_UNACCBD, "Unacceptable bond order" },
    { MSG_DCD_STARTAREA, "  Starting area      : %.1f \n" },
    { MSG_DCD_STARTSURFDIA, "  Starting surf. dia.: %.1f \n" },
    { MSG_DCD_STARTPSA, "  Starting PSA       : %.1f \n" },
    { MSG_DCD_ASSATMTYPES, "* Assignig atom types" },
    { MSG_DCD_STARTVLOGP, "  Starting VlogP     : %.4f\n" },
    { MSG_DCD_STARTVOL, "  Starting volume    : %.1f \n" },
    { MSG_DCD_STARTVOLDIA, "  Starting vol. dia. : %.1f \n" },
    { MSG_DCD_TRJINFO, "  Total frames       : %d\n"\
	"  Frame step         : %d fs\n"\
	"  Simulation time    : %d ps\n"\
	"  Total atoms        : %d\n" },
    { MSG_DCD_LOWESTENE, "  Lowest energy frame: %d (%.4f Kcal/mol)\n" },
    { MSG_ECD_SAVPLOT, "* Saving plot" },
    { MSG_DCD_PROCFRAME, "  Processing frame %d/%d (%.4f)" },
    { MSG_DCD_RESULTS, "  Range              : %.4f (%d) <-> %.4f (%d)\n"\
	"  Average value      : %.4f %.4f\n" },
    { MSG_ERR_DCD_UNCHARGED, "Unassigned atomic charges.\n"\
	"Use the -c[TEMPLATE] option to fix them" },
    { MSG_ERR_DCD_UNSUPPOUT, "Unsupported output format for trajectory" },
    { MSG_ERR_DCD_UNKTRAJ, "Unknown or incompatible trajectory file" },
    { MSG_ENER_ASSPROG, "* Assigning CVFF parameters" },
    { MSG_ENER_CALCPROG, "* Calcualting CVFF interaction energy" },
    { MSG_ENER_SORTPROG, "* Sorting energies" },
    { MSG_ENER_RESULTS, "\n\n"\
	"*********************************************\n"\
	"**** CVFF interaction energy calculation ****\n"\
	"*********************************************\n\n"\
	"Ligand name..........: %.4s:%.4s\n"\
	"Ligand atoms.........: %d\n"\
	"Dielectric constant..: %.3f\n\n"\
	"Total................: %9.4f Kcal/mol\n"\
	"Non-bond.............: %9.4f Kcal/mol\n"\
	"Coulombic............: %9.4f Kcal/mol\n\n\n"\
	" Res    Num     R6-R12    Coulomb     Total\n"\
	"============================================\n" },
    { MSG_ERR_ENER_MORERES, "More than one residue is needed" },
    { MSG_ERR_ENER_RESNOTFOUND, "Residue not found" },
    { MSG_ERR_ENER_UNKATMTYPE, "Unknown atom type %.8s" },
    { MSG_ERR_ENER_NONBOND, "Non-bond parameters not found" },
    { MSG_ERR_ENER_FFNOTFOUND, "Can't open template %s" },
    { MSG_ERR_FASTA_NOPROT, "Input file is not protein or DNA/RNA" },
    { MSG_FFIELD_FIXPROG, "* Fixing atomic potentials" },
    { MSG_FFIELD_RINGPROG, "* Searching for rings" },
    { MSG_ERR_FFIELD_ILLNUM, "Illegal number of arguments" },
    { MSG_ERR_FFIELD_UNBPAR, "Unbalenced parenthesis" },
    { MSG_ERR_FFIELD_SUBLVL, "Sublevel overflow in ATDL description" },
    { MSG_ERR_FFIELD_ATYPTOOLONG, "Atom type too long" },
    { MSG_ERR_FFIELD_INCATM, "Incorrect atom description" },
    { MSG_ERR_FFIELD_INCSUBATM, "Incorrect subatom description" },
    { MSG_ERR_FFIELD_NOTFOUND, "\"%s\".\nTemplate file not found" },
    { MSG_ERR_FFIELD_CANTASSIGN, "  Can't assign potential to" },
    { MSG_ERR_FFIELD_FFERRLINE, "at line #%d -> \"%s\"\n%s" },
    { MSG_ERR_FFIELD_FFERRLINETEMP, "at line #%d of template file\n%s" },
    { MSG_ERR_FFIELD_ILLAROM, "Illegal ATDL aromatic indicator" },
    { MSG_ERR_FFIELD_ILLRING, "Illegal ATDL ring indicator" },
    { MSG_ERR_FFIELD_ILLBONDORD, "Illegal bond order in ATDL description" },
    { MSG_GASTEIGER_TOTCHG, "  Total charge: %.2f\n" },
    { MSG_ERR_IFF_INVFILE, "Invalid IFF/RIFF file" },
    { MSG_ERR_IFF_NOTIFF, "Not an IFF/RIFF %-4.4s file" },
    { MSG_ERR_IFF_CORRUPTED, "Corrupted IFF/RIFF file" },
    { MSG_INFO_TITLE, "\n\n"\
	"************************************\n"\
	"**** Information about molecule ****\n"\
	"************************************\n\n" },
    { MSG_INFO_PROP1, "Atoms..............: %d\n"\
	"Heavy atoms........: %d\n"\
	"Centroids..........: %d\n"\
	"Residues...........: %d\n"\
	"Molecules..........: %d\n"\
	"Waters.............: %d\n"\
	"Formula............: %s\n"\
	"Molecular weight...: %.3f Daltons\n"\
	"Monoisotopic mass..: %.6lf Daltons\n"\
	"Gyration radius....: %.4f \n"\
	"Geometry center....: %.4f %.4f %.4f\n"\
	"Mass center........: %.4f %.4f %.4f\n"\
	"Appx. dimensions...: %.4f %.4f %.4f\n" },
    { MSG_INFO_CHARGE, "Total charge.......: %.4f\n"\
	"Dipole.............: %.4f Debye\n" },
    { MSG_INFO_SURFAREA, "Surf. area (%3.2f)..: %.1f  (ds=%.1f )\n" },
    { MSG_INFO_PSA, "Polar area (PSA)...: %.1f  (apolar=%.1f )\n" },
    { MSG_INFO_VOLUME, "Volume.............: %.1f  (dv=%.1f )\n" },
    { MSG_INFO_OVALITY, "Ovality............:" },
    { MSG_INFO_OVALNOTAVAIL, "Not available (use -g 0)\n" },
    { MSG_INFO_LOGPCRIPPEN, "logP (Crippen).....: %.4f\n"\
	"Lipole (Crippen)...: %.4f\n" },
    { MSG_INFO_LOGPBROTO, "logP (Broto).......: %.4f\n"\
	"Lipole (Broto).....: %.4f\n" },
    { MSG_INFO_VLOGP, "Virtual logP.......:" },
    { MSG_INFO_NOTAVAIL, "Not available" },
    { MSG_INFO_AACOMP, "Aminoacidic charge.: %d (aminoacid composition)\n"\
	"\n\nAminoacidic composition:\n\n"\
	" Res    N.   N. %%     Mass    Mass %%\n"\
	"====================================\n" },
    { MSG_INFO_TOOATM, "\nToo many atoms for logP, surface and volume calculations.\n"\
	"See the key \"mAXATMINFO\" in the \"prefs\" file." },
    { MSG_LOADER_MSF, "Quanta .msf" },
    { MSG_LOADER_CRD, "CHARMM CRD" },
    { MSG_LOADER_SYBREG, "Sybyl Rgn" },
    { MSG_LOADER_IFF, "IFF/RIFF MOLE" },
    { MSG_LOADER_CAR, "Accelrys .car" },
    { MSG_LOADER_PDB, "PDB" },
    { MSG_LOADER_CSSR, "CSSR" },
    { MSG_LOADER_QMC, "QMC" },
    { MSG_LOADER_HIN, "HyperChem HIN" },
    { MSG_LOADER_MOPARC, "Mopac Arc" },
    { MSG_LOADER_MOPINT, "Mopac internal" },
    { MSG_LOADER_MOL2, "Sybyl Mol2" },
    { MSG_LOADER_GROMOS, "Gromos" },
    { MSG_LOADER_XYZ, "XYZ" },
    { MSG_ERR_LOADER_CRDASSFILE, "Can't load the associated molecular file" },
    { MSG_ERR_LOADER_CRDMESURE, "Measure type not specified (-m option)" },
    { MSG_ERR_LOADER_UNKFORMAT, "Unknown input format" },
    { MSG_ERR_LOADER_NOTFOUND, "Input file not found" },
    { MSG_ERR_LOCALE_ERROR, "ERROR" },
    { MSG_ERR_LOGP_UNRECFILE, "Unrecognized parameter file \"%s.par\"" },
    { MSG_ERR_LOGP_NOTALLPAR, "logP %s.\nNot all atoms are parametrized" },
    { MSG_ERR_LOGP_NOTFOUND, "Can't find \"%s.par\"" },
    { MSG_MAIN_DCDPROG, "* Analyzing the trajectory" },
    { MSG_MAIN_DONEPROG, "Done.\n" },
    { MSG_MAIN_VLOGP, "Virtual logP by Bernard Testa et al.\n" },
    { MSG_MAIN_OSCPU, "%s version\n\n" },
    { MSG_MAIN_SYNOPSIS, "Synopsis: vega INPUT ... -o[OUT.PACK] -f[OUTPUT_FORMAT] -p[FORCE_FIELD]\n"\
	"          -s[POINTS] -g[RADIUS] -c[TEMPLATE] -k[KEYWORDS] -a[RES_NUM]\n"\
	"          -d[DIELECTRIC] -e[MOLNUM] -i[SHELL RAD SHAPE] -j[TORSIONS]\n"\
	"          -l[MOLTYPE] -m[KEYWORDS] -q[METHOD] -r[MODE] -t[SECSTRUCT]\n"\
	"          -v[CPUS] -x[MODE (ID)] -z[NTERM CTERM] -0bhnuwy\n"\
	"\n"\
	" 0 -> ignore locale settings of the decimal separator\n"\
	" a -> renumber residues starting from RES_NUM\n"\
	" b -> don't save the connectivity\n"\
	" c -> charge template (FORMAL, GASTEIGER, ...)\n"\
	" d -> dielectric constant for energy calculation\n"\
	" e -> molecule number for score calculation (0 = last)\n"\
	" f -> output format\n"\
	" g -> probe radius for SAS\n"\
	" h -> show this help\n"\
	" i -> solvate the molecule\n"\
	" j -> define the torsions (ALL, AUTODOCK, FLEX)\n"\
	" k -> keywords for InfoXML, MopInt and Score\n"\
	" l -> add the hydrogens (GEN, GENBO, NA, NABO, PROT, PROTBO)\n"\
	" m -> keywords for trajectory analysis\n"\
	" n -> normalize coordinates\n"\
	" o -> output file name\n"\
	" p -> define force field to apply\n"\
	" q -> fix the bond order (ALL, RINGS)\n"\
	" r -> remove hydrogens (ALL, APOLAR)\n"\
	" s -> point density for SAS\n"\
	" t -> change the protein secondary structure\n"\
	" u -> add the side chains to a protein\n"\
	" v -> number of CPUs (0 = all)\n"\
	" w -> remove waters\n"\
	" x -> extract/list molecule/s from the database (LIST; NAME name; NUM number)\n"\
	" y -> find the molecules in the assembly\n"\
	" z -> N-term and C-term capping for peptide (default: NONE NONE)\n"\
	"\n"\
	"INPUT formats:\n"\
	" Alchemy, Amber NC, Amber Mdcrd, AMMP, Arc, AutoDock 4 DLG, BioDock, CAR,\n"\
	" CHARMM CRD, CIF, CML, CML 2.0, CPMD XYZ, CRT, CHARMM DCD, Chem3D, ChemDraw\n"\
	" CDX, ChemSol, CSSR, EMPIRE, ESCHER NG, Fasta, GAMESS, Gaussian In/Out,\n"\
	" GRAMM, Gromacs/Gromos mol, Gromacs TRR, Gromacs XTC, HIN, IFF, InChI, LiGen\n"\
	" pocket, MDL, MDL V3000, Mol2, Mopac cartesian, Mopac Gaussian Z-matrix, Mopac\n"\
	" internal, MSF, NAMD binary, PDB, PDBA, PDBF, PDBL, PDBQT, PQR, PQRXML, PSFX,\n"\
	" QMC, Quanta CSR, RIFF, SDF, TINKER XYZ, XYZ, ZIP.\n"\
	"\n"\
	"OUTput formats:\n"\
	" Calc:       Info, InfoXML, Score.\n"\
	" Map:        BiosymSrf, ComfaFld, CsvIlm, CsvLogP, CsvMep, CsvSrf,\n"\
	"             QuantaIlm, QuantaLogP, QuantaMep, QuantaSrf.\n"\
	" Molecule:   Alchemy, AMMP, Biosym, ChemSol, CIF, CML, CML2, CPMDXYZ, CRD,\n"\
	"             CRT, CSSR, Fasta, GAMESS, GaussIn, Gromos, GromosNm, IFF,\n"\
	"             InChI, InChIAux, InChIKey, Indigo, MdlMol, MdlMol3, mmCIF,\n"\
	"             Mol2, MopCar, MopInt, MSF, NamdBin, OldBiosym, PDB, PDB2,\n"\
	"             PDBOMM, PDBQ, PDBA, PDBF, PDBL, PDBNOTSTD, PDBQT, PQR, PQRXML,\n"\
	"             PSFX, QMC, RIFF, SMILES, SpilloRBS, VINA, XYZ.\n"\
	" Plot:       BinPlt, CSV, QuantaPlt.\n"\
	" Trajectory: TrjAmber, TrjAmberNc, TrjDCD, YrjIFF, TrjMol2, TrjPDB, TrjTrr,\n"\
	"             TrjXtc.\n"\
	" VRML:       Vrml, VrmlPts, VrmlCpk, VrmlSol.\n"\
	"\n"\
	"PACKer formats:\n"\
	" bz2 (BZip2), gz (GZip), pp (PowerPacker), z (Z-Compress).\n"\
	"\n"\
	"Score functions (-f Score -k):\n"\
	" Charmm, Charmm22, Charmm36, Contacts, CVFF, Elect, ElectDD, MLPInS,\n"\
	"  MLPInS2, MLPInS3, MLPInSF.\n"\
	"\n"\
	"TRAJECTORY keywords (-m):\n"\
	" Angle A1 A2 A3, Dipole, Distance A1 A2, Extract F1 [F2], GyrRad, ILM,\n"\
	" LipoleBr, LipoleCr, Ovality, PlaneAng A1 A2 A3 A4 A5 A6, PSA, RMSD,\n"\
	" RMSDH, RMSDALN, RMSDALNH, RMSDSYMCOR, RMSDSYMCORH, Surface A1 ...,\n"\
	" SurfDia A1 ..., Torsion A1 A2 A3 A4, VlogP, VolDia, Volume.\n"\
	"\n"\
	"Secondary structure keywords (-t):\n"\
	" AlphaHelix, LeftHelix, 310Helix, PiHelix, Beta, BetaAnti, BetaPar.\n"\
	" or\n"\
	" TOR=VALUE TOR=VALUE ...\n"\
	" where TOR is the torsion name (Phi, Psi, Omega) and VALUE is the\n"\
	" torsion value in degree.\n"\
	"\n"\
	"Peptide capping keywords (-z):\n"\
	" NTERM: NONE, H3N+, HCONH, H3CCONH\n"\
	" CTERM: NONE, O-, OH, OCH3, OC2H5, NH2\n\n" },
    { MSG_ERR_MAIN_UNKOPT, "Unknown option" },
    { MSG_ERR_MAIN_ILLSTRRES, "Illegal starting residue" },
    { MSG_ERR_MAIN_UNSUPCHGTEMP, "Unsupported charge template" },
    { MSG_ERR_MAIN_CHGFILENOTFOUND, "Can't find charge template" },
    { MSG_ERR_MAIN_ILLDIEL, "Illegal dielectric constant" },
    { MSG_ERR_MAIN_ILLRESNAME, "Illegal residue name" },
    { MSG_ERR_MAIN_UNKOUTFOR, "Unknown output format" },
    { MSG_ERR_MAIN_ILLPROBERAD, "Illegal probe radius" },
    { MSG_ERR_MAIN_ILLSOLRAD, "Illegal solvatation radius" },
    { MSG_ERR_MAIN_UNKSHAPE, "Unknown shape type" },
    { MSG_ERR_MAIN_SHELLFILE, "Shell file not found" },
    { MSG_ERR_MAIN_FFNOTSTD, "Force field template is not in standard format" },
    { MSG_ERR_MAIN_FFNOTFOUND, "Unable to find the force field template" },
    { MSG_ERR_MAIN_ILLSASDEN, "Illegal SAS density" },
    { MSG_ERR_MAIN_NOSTDOUT, "Stdout not allowed with selected output format" },
    { MSG_ERR_MAIN_NOFFTAG, "The output format doesn't support force field tag" },
    { MSG_ERR_MAIN_NOCHARGE, "The output format doesn't support atomic charges" },
    { MSG_ERR_MAIN_CSSR, "CSSR/QMC format doesn't support more than 9999 atoms" },
    { MSG_ERR_MAIN_FILEMISS, "Missing input file name" },
    { MSG_ERR_MAIN_FLOATNUM, "The argument must be an unsigned float number" },
    { MSG_ERR_MAIN_INTNUM, "The argument must be an unsigned integer number" },
    { MSG_ERR_MAIN_SUBARG, "Missing sub-argument" },
    { MSG_ERR_MAIN_RGNNOMOL, "Region file without a molecule" },
    { MSG_ERR_MAIN_ABORT, "Aborted." },
    { MSG_MLP_VLOGP, "  Virtual logP.......: %.4f\n" },
    { MSG_ERR_MLP_NOVLOGP, "\n"\
	"The Virtual logP calculation is not available in this\n"\
	"release. Please contact the Author to obtain the full\n"\
	"version" },
    { MSG_MOL2_ASSIGNPROG, "  Assigning the atom types" },
    { MSG_ERR_MOL2_CORRUPTED, "Corrupted or incoplete Mol2 file" },
    { MSG_ERR_MOL2_ILLTEMP, "Illegal force field template" },
    { MSG_ERR_MOL2_NOTFOUND, "Can't find the force field template" },
    { MSG_ERR_MOPAC_ATMFAULT, "The faulty atom is the number %d\n"\
	"Atoms %d, %d and %d are within %f angstroms of a straight line" },
    { MSG_ERR_MOPAC_BADCONV, "Bad cartesian to internal conversion" },
    { MSG_ERR_MSF_NOTCOMP, "The MSF file to overwrite is not compatible" },
    { MSG_ERR_MSF_NOTMSF, "The output file to overwrite is not in MSF format" },
    { MSG_PDB_FATFOUND, "  PDB Fat extension found" },
    { MSG_SAVER_SAVEPROG, "* Saving" },
    { MSG_SAVER_ATTYPROG, "* Assignig atom types" },
    { MSG_SAVER_LOGPBROTO, "  LogP (Broto)   : %.4f\n"\
	"  Lipole (Broto) : %.4f\n" },
    { MSG_SAVER_WARN_NOTCHARGE, "Charges not assigned" },
    { MSG_SAVER_WARN_FFNOTASS, "Force field not assigned" },
    { MSG_ERR_SAVER_PLOT, "Plot files are usable only with a trajectory" },
    { MSG_SOLVATE_SOLVATING, "* Solvating the molecule\n"\
	"  Reading shell file" },
    { MSG_SOLVATE_ADDED, "  Added %d solvent molecules\n" },
    { MSG_ERR_SOLVATE_SHELLDIM, "The molecule exceeds the shell dimensions" },
    { MSG_ERR_SOLVATE_READING, "while reading shell file" },
    { MSG_ERR_SOLVATE_CANTOPEN, "Can't open shell file" },
    { MSG_ERR_SOLVATE_NOTHING, "  Nothing to add" },
    { MSG_SURFACE_BUILDILM, "* Building ILM surface" },
    { MSG_SURFACE_BUILDMEP, "* Building MEP surface" },
    { MSG_SURFACE_BUILDVLOGP, "* Building MLP surface" },
    { MSG_SURFACE_BUILDPSA, "* Building PSA surface" },
    { MSG_SURFACE_BUILDVDW, "* Building VdW/SAS surface" },
    { MSG_SURFACE_RANGES, "  Range of values....: %.4f -> %.4f\n"\
	"  Average value......: %.4f %.4f\n" },
    { MSG_ERR_SURFACE_ILM, "A water cluster is required for the ILM calculation" },
    { MSG_UTILS_LOADPROG, "* Loading file %d as %s\n" },
    { MSG_UTILS_RMHPROG, "* Removing hydrogens" },
    { MSG_UTILS_RMWPROG, "* Removing waters" },
    { MSG_UTILS_RENRESPROG, "* Renumbering residues" },
    { MSG_ERR_UTILS_DOSERR, "DOS - %s" },
    { MSG_ERR_UTILS_OUTOFMEM, "Out of memory" },
    { MSG_ERR_UTILS_VEGADIR, "Please set the \"VEGADIR\" environment variable" },
    { MSG_WARN_UTILS_WARN, "  WARNING:" },
    { MSG_ERR_SYBRGN_POINTS, "Illegal number of points in rgn file" },
    { MSG_ERR_SYBRGN_NUMSTPS, "Illegal number of steps in rgn file" },
    { MSG_ERR_SYBRGN_STPSIZE, "Illegal step size in rgn file" },
    { MSG_ERR_SYBRGN_CORRUPTED, "Corrupted rgn file" },
    { MSG_ERR_PKPP_ERROR, "PowerPacker.\n%s" },
    { MSG_ERR_PKPP_NOPPLIB, "Can't open the powerpacker.library" },
    { MSG_ERR_PKPP_TOOSHORT, "PowerPacker.\n"\
	"The unpacked file length is too short" },
    { MSG_ERR_GZIP_ERROR, "GZip.\n%s" },
    { MSG_ERR_BZIP2_ERROR, "BZip2.\n%s" },
    { MSG_ERR_Z32_ERROR, "Z de/compress.\n%s" },
    { MSG_ERR_COMFA_RGN, "Sybyl rgn file not loaded" },
    { MSG_ERR_COMFA_FIELD, "Field type not specified" },
    { MSG_PK_PROG, "* Packing file" },
    { MSG_ERR_MAIN_PKNOSTDOUT, "Stdout not allowed with packed output" },
    { MSG_ERR_MAIN_UNKPACKMET, "Unknown packing method" },
    { MSG_ERR_MAIN_CANTPACK, "Can't compress the output file" },
    { MSG_ERR_PKPP_NOCRUNCH, "Sorry, the PowerPacker compression is available for AmigaOS only" },
    { MSG_ERR_ENER_FFNOTASS, "Force field not assigned.\n"\
	"You need to fix it with -p[FORCE_FIELD] option" },
    { MSG_MAIN_OPENGLSTART, "* OpenGL Interface started\n" },
    { MSG_MAIN_ASSVDW, "  Assigning VdW radii" },
    { MSG_GLMAIN_EXITREQ, "Do you want really exit from VEGA ?" },
    { MSG_MAIN_MENU_FILE, "&File" },
    { MSG_MAIN_MENU_NEW, "&New" },
    { MSG_MAIN_MENU_OPEN, "&Open" },
    { MSG_MAIN_MENU_SAVE, "&Save As..." },
    { MSG_MAIN_MENU_SAVE2, "&Save" },
    { MSG_MAIN_MENU_PRINT, "&Print" },
    { MSG_MAIN_MENU_SAVEBMP, "Save &image" },
    { MSG_MAIN_MENU_COMP, "Compression" },
    { MSG_MAIN_MENU_CMPNONE, "None" },
    { MSG_MAIN_MENU_CMPBZIP2, "BZip2" },
    { MSG_MAIN_MENU_CMPGZIP, "GZip" },
    { MSG_MAIN_MENU_CMPZCOMP, "Z Compress" },
    { MSG_MAIN_MENU_CONN, "Co&nnectivity" },
    { MSG_MAIN_MENU_DEMO, "&Demo mode" },
    { MSG_MAIN_MENU_DMRUN, "&Run" },
    { MSG_MAIN_MENU_DMSTOP, "&Stop" },
    { MSG_MAIN_MENU_DMMUSIC, "&Music" },
    { MSG_MAIN_MENU_TEXT, "Sub&titles" },
    { MSG_MAIN_MENU_EXIT, "&Exit" },
    { MSG_MAIN_MENU_EDIT, "&Edit" },
    { MSG_MAIN_MENU_CUT, "Cu&t" },
    { MSG_MAIN_MENU_COPY, "&Copy" },
    { MSG_MAIN_MENU_PASTE, "&Paste" },
    { MSG_MAIN_MENU_SELECTALL, "&Select all" },
    { MSG_MAIN_MENU_COPYALL, "C&opy all" },
    { MSG_MAIN_MENU_FIND, "&Find" },
    { MSG_MAIN_MENU_REMOVE, "&Remove" },
    { MSG_MAIN_MENU_RMHYD, "&Hydrogens" },
    { MSG_MAIN_MENU_RMWAT, "&Water" },
    { MSG_MAIN_MENU_RMSURF, "All &surfaces" },
    { MSG_MAIN_MENU_RENRES, "&Renumber residues" },
    { MSG_MAIN_MENU_NORM, "&Normalize" },
    { MSG_MAIN_MENU_COPYBMP, "Copy &special" },
    { MSG_MAIN_MENU_VIEW, "&View" },
    { MSG_MAIN_MENU_DISPLAY, "&Display" },
    { MSG_MAIN_MENU_DSPWIRE, "&Wireframe" },
    { MSG_MAIN_MENU_DSPVDW, "C&PK wire" },
    { MSG_MAIN_MENU_DSPVDWS, "&CPK solid" },
    { MSG_MAIN_MENU_DSPCPK, "B&all && stick wire" },
    { MSG_MAIN_MENU_DSPCPKS, "&Ball && stick solid" },
    { MSG_MAIN_MENU_COLOR, "&Color" },
    { MSG_MAIN_MENU_COLBYATM, "By &atom" },
    { MSG_MAIN_MENU_COLBYMOL, "By &molecule" },
    { MSG_MAIN_MENU_LBLATM, "&Label atom" },
    { MSG_MAIN_MENU_LBLATM_OFF, "&Off" },
    { MSG_MAIN_MENU_LBLATM_NAME, "&Name" },
    { MSG_MAIN_MENU_LBLATM_ELEM, "&Element" },
    { MSG_MAIN_MENU_LBLATM_NUM, "N&umber" },
    { MSG_MAIN_MENU_LBLATM_TYPE, "&Type" },
    { MSG_MAIN_MENU_LBLATM_CHARGE, "&Charge" },
    { MSG_MAIN_MENU_RESETVIEW, "&Reset" },
    { MSG_MAIN_MENU_ANIM, "&Animation" },
    { MSG_MAIN_MENU_LIGHT, "&Light" },
    { MSG_MAIN_MENU_MOLINFO, "I&nformation" },
    { MSG_MAIN_MENU_CALC, "&Calculate" },
    { MSG_MAIN_MENU_CALCCHRG, "&Charge && Pot." },
    { MSG_MAIN_MENU_CALCCLUST, "&Cluster" },
    { MSG_MAIN_MENU_CALCMOPAC, "&Mopac" },
    { MSG_MAIN_MENU_ANALYSIS, "&Analysis" },
    { MSG_MAIN_MENU_BIODOCK, "&BioDock" },
    { MSG_MAIN_MENU_SURFACE, "&Surface" },
    { MSG_MAIN_MENU_SRFCALC, "&Calculate" },
    { MSG_MAIN_MENU_SRFSAVE, "&Save" },
    { MSG_MAIN_MENU_SRFCOLOR, "C&olor" },
    { MSG_MAIN_MENU_SRFREMOVE, "&Remove" },
    { MSG_MAIN_MENU_TOOLS, "&Tools" },
    { MSG_MAIN_MENU_TLSGRAPH, "&GraphED" },
    { MSG_MAIN_MENU_TLSMINIED, "&MiniED" },
    { MSG_MAIN_MENU_TLSWINDD, "Win&DD" },
    { MSG_MAIN_MENU_TLSTASK, "&Task Manager" },
    { MSG_MAIN_MENU_TLSPRFED, "&Preferences" },
    { MSG_MAIN_MENU_HELP, "&Help" },
    { MSG_MAIN_MENU_CONTENTS, "&VEGA ZZ manual" },
    { MSG_MAIN_MENU_MOPACHELP, "&Mopac 7 manual" },
    { MSG_MAIN_MENU_HELPKEYS, "&Keys" },
    { MSG_MAIN_MENU_WEB, "VEGA on the &Web" },
    { MSG_MAIN_MENU_ABOUT, "&About" },
    { MSG_POPUP_MENU_MOUSE, "M&ouse mode" },
    { MSG_POPUP_MENU_MOUSEROT, "&Rotate" },
    { MSG_POPUP_MENU_MOUSETR, "&Translate" },
    { MSG_POPUP_MENU_MOUSESCL, "&Scale" },
    { MSG_GLDEMO_ERR_MCI, "Unknown MCI error. Return code 0x%8x" },
    { MSG_MENU_NEW, "This operation clear all data\n"\
	"Do you want continue ?" },
    { MSG_MENU_CALCSTOP, "Do you want kill the selected process ?" },
    { MSG_MENU_OPEN, "Open molecule(s)" },
    { MSG_MENU_SAVE, "Save molecule" },
    { MSG_MENU_SAVEBMP, "Save image" },
    { MSG_MENU_ERR_HELP, "Can't open help file" },
    { MSG_MENU_ERR_CANTEXE, "Can't execute %s" },
    { MSG_LOADER_QNTSRF, "Quanta Srf" },
    { MSG_ERR_LODER_OPENGLONLY, "Available with OpenGL output only" },
    { MSG_ERR_GL_RUNNING, "VEGA OpenGL port %d is already running" },
    { MSG_ERR_GL_CONTEXT, "Can't create a GL device context" },
    { MSG_ERR_GL_PIXELFORM1, "Can't find a suitable pixelformat" },
    { MSG_ERR_GL_PIXELFORM2, "Can't set the pixel format" },
    { MSG_ERR_GL_CREATECONTEXT, "Can't create a GL rendering context" },
    { MSG_ERR_GL_ACTCONTEXT, "Can't activate the GL rendering context" },
    { MSG_ERR_GL_FAIL, "OpenGL initialization failed" },
    { MSG_ERR_GL_WNDCLASS, "Failed to register the window class" },
    { MSG_ERR_GL_WNDCREAT, "Window creation error" },
    { MSG_DLGSRFCLC_TITLE, "Surface management" },
    { MSG_DLGSRFCLC_LBL_TYPE, "Type:" },
    { MSG_DLGSRFCLC_LBL_PROBERAD, "Probe rad.:" },
    { MSG_DLGSRFCLC_LBL_DENSITY, "Density:" },
    { MSG_DLGSRFCLC_BT_CALC, "C&alculate" },
    { MSG_BT_ABORT, "&Abort" },
    { MSG_BT_CANCEL, "&Cancel" },
    { MSG_BT_CLOSE, "&Close" },
    { MSG_BT_DONE, "&Done" },
    { MSG_BT_FIX, "&Fix" },
    { MSG_BT_OK, "&Ok" },
    { MSG_BT_RUN, "&Run" },
    { MSG_BT_SHOW, "&Show" },
    { MSG_BT_START, "&Start" },
    { MSG_BT_STOP, "&Stop" },
    { MSG_BT_EDIT, "&Edit" },
    { MSG_DLGSRFCOL_TITLE, "Surface Color" },
    { MSG_DLGSRFCOL_RG_TITLE, "Color by" },
    { MSG_DLGSRFCOL_RG_PROPERTY, "&Property" },
    { MSG_DLGSRFCOL_RG_ATOM, "&Atom" },
    { MSG_DLGSRFCOL_RG_COLOR, "&Color" },
    { MSG_DLGSRFCOL_BT_CHANGE, "C&hange" },
    { MSG_DLGMOPAC_TITLE, "Mopac calculation" },
    { MSG_DLGMOPAC_LBL_CALC, "Calculate:" },
    { MSG_DLGMOPAC_LBL_CHARGE, "Charge:" },
    { MSG_DLGMOPAC_LBL_OTHER, "Other:" },
    { MSG_DLGMOPAC_GRP_PARAM, "Parameters" },
    { MSG_DLGMOPAC_GRP_SWITCHES, "Switches" },
    { MSG_DLGMOPAC_START, "* Starting Mopac process" },
    { MSG_ERR_DLGMOPAC_MOPACEXE, "Can't find Mopac executable" },
    { MSG_ERR_DLGMOPAC_NOARC, "Abnormal termination.\n"\
	"For more information, see the .mno or .out file" },
    { MSG_ERR_DLGMOPAC_TOOATM, "Too many atoms (%d) for the calculation"\
	" (Max %d heavy atoms, %d hydrogens)." },
    { MSG_DLGCHG_TITLE, "Charges & Potential" },
    { MSG_DLGCHG_FFIELD, "Force field:" },
    { MSG_DLGCHG_CHARGES, "Charges:" },
    { MSG_DLGCHG_ERR_NOTASS, "Atomic charges not assigned" },
    { MSG_DLGABOUT_TITLE, "About VEGA ZZ" },
    { MSG_DLGABOUT_COPYRIGHT, "All rights reserved" },
    { MSG_DLGABOUT_MUSIC, "Demo music by Anvil" },
    { MSG_DLGCLUSTER_TITLE, "Cluster Calculation" },
    { MSG_DLGCLUSTER_LBL_SOLVENT, "Solvent:" },
    { MSG_DLGCLUSTER_RG_BOX, "&Box" },
    { MSG_DLGCLUSTER_RG_SPHERE, "&Sphere" },
    { MSG_DLGCLUSTER_RG_LAYER, "&Layer" },
    { MSG_DLGCLUSTER_LBL_SIDE, "Sides:" },
    { MSG_DLGCLUSTER_LBL_RADIUS, "Radius:" },
    { MSG_DLGCLUSTER_LBL_THICKNESS, "Thickness:" },
    { MSG_DLGKEYS_TITLE, "Hot keys" },
    { MSG_DLGKEYS_TITDES, "Hot key definitions:" },
    { MSG_DLGKEYS_DESC, "A -> Turn on/off the animation mode\n"\
	"L -> Switch on/off the light\n"\
	"Q -> Go to the previous workspace\n"\
	"R, S, T -> Rotate, scale, translate\n"\
	"SPC -> Stop the animation\n"\
	"V -> Change the display mode\n"\
	"W -> Go to the next workspace\n"\
	"F12 -> Turn on/off the full screen mode\n"\
	"\n"\
	"Arrows:\n"\
	"Up, down -> X rotate\n"\
	"Left, right -> Y rotate\n"\
	"Pg.Up, Pg.Down -> Z rotate" },
    { MSG_PRFED_TITLE, "VEGA Preferences" },
    { MSG_MOLINFO_TITLE, "Information about the molecule" },
    { MSG_MOLINFO_MAXATM, "Max. Atoms:" },
    { MSG_EDFFIELD_TITLE, "%s ATDL Template" },
    { MSG_EDCHARGE_TITLE, "%s Parameters" },
    { MSG_GRAPH_YAXTITLE, "Value" },
    { MSG_GRAPH_BT_PLOT1, "Plot &1" },
    { MSG_GRAPH_BT_PLOT2, "Plot &2" },
    { MSG_GRAPH_DLG_SAVE, "Save plot" },
    { MSG_GRAPH_DLG_OPEN, "Open plot" },
    { MSG_GRAPH_MM_ZOOM, "&Zoom" },
    { MSG_GRAPH_MM_ZOOMIN, "&In" },
    { MSG_GRAPH_MM_ZOOMOUT, "&Out" },
    { MSG_GRAPH_MM_ZOOMRESET, "&Reset" },
    { MSG_GRAPH_MM_CLUSTGRAPH, "&Cluster Chart" },
    { MSG_GRAPH_MM_STATVAL, "&Statistical Values" },
    { MSG_GRAPHVAL_TITLE, "Statistical Values" },
    { MSG_GRAPHVAL_LBL_SAMPLES, "Samples:" },
    { MSG_GRAPHVAL_LBL_MIN, "Min. Value:" },
    { MSG_GRAPHVAL_LBL_MAX, "Max. Value:" },
    { MSG_GRAPHVAL_LBL_AVERAGE, "Average:" },
    { MSG_GRAPHVAL_LBL_STDDEV, "Standard Dev.:" },
    { MSG_GRAPHCLUST_TITLE, "Cluster Chart Options" },
    { MSG_GRAPHCLUST_MODE, "Clustering Mode" },
    { MSG_GRAPHCLUST_MAX, "Max.:" },
    { MSG_GRAPHCLUST_MIN, "Min.:" },
    { MSG_GRAPHCLUST_BT_ADD, "&Add" },
    { MSG_GRAPHCLUST_BT_UPDATE, "&Update" },
    { MSG_GRAPHCLUST_BT_REMOVE, "&Remove" },
    { MSG_GRAPHCLUST_BT_LOAD, "&Load" },
    { MSG_GRAPHCLUST_BT_REMALL, "Re&move All" },
    { MSG_GRAPHCLUST_BT_SAVE, "&Save" },
    { MSG_GRAPHCLUST_NUMBER, "&Number" },
    { MSG_GRAPHCLUST_RANGES, "&Ranges" },
    { MSG_GRAPHFILT_TITLE, "Custom Filter" },
    { MSG_GRAPHFILT_SKIP, "Skip Lines:" },
    { MSG_GRAPHFILT_SEPARATOR, "Separator:" },
    { MSG_GRAPHFILT_COLUMNX, "X Column:" },
    { MSG_GRAPHFILT_COLUMNY, "Y Column:" },
    { MSG_ANA_TITLE, "Trajectory analysis" },
    { MSG_ANA_GRP_FILE, "File name" },
    { MSG_ANA_GRP_INFO, "Information" },
    { MSG_ANA_LBL_FRAMES, "Frames:" },
    { MSG_ANA_LBL_FRMSTEP, "Time step:" },
    { MSG_ANA_LBL_SIMTIME, "Simulation time:" },
    { MSG_ANA_LBL_LOWENE, "Best frame:" },
    { MSG_ANA_LBL_RANGE, "Range:" },
    { MSG_ANA_LBL_TEMP, "Temperature:" },
    { MSG_ANA_LBL_STARTTIME, "Start Time:" },
    { MSG_ANA_TS_SELECT, "&Selection" },
    { MSG_ANA_TS_CALC, "&Calculation" },
    { MSG_ANA_TS_MEASURE, "&Measure" },
    { MSG_ANA_TS_ANIM, "&Animation" },
    { MSG_ANA_LBL_FRAME, "Frame number:" },
    { MSG_ANA_BT_FIRST, "&First" },
    { MSG_ANA_BT_LAST, "&Last" },
    { MSG_ANA_BT_LOWEST, "&Lowest" },
    { MSG_ANA_LBL_OUTPUT, "Output:" },
    { MSG_ANA_LBL_PROPERTY, "Property:" },
    { MSG_ANA_CB_IGNOREWAT, "&Exclude invisible atoms" },
    { MSG_ANA_CB_SHOWPLOT, "&Show plot" },
    { MSG_ANA_LBL_MEASURE, "Measure:" },
    { MSG_ANA_LBL_START, "Start:" },
    { MSG_ANA_LBL_END, "End:" },
    { MSG_ANA_LBL_SKIP, "Skip frame:" },
    { MSG_ANA_CB_LOOP, "&Loop" },
    { MSG_ANA_LBL_SPEED, "Speed" },
    { MSG_ANA_CB_DISTANCE, "Distance" },
    { MSG_ANA_CB_ANGLE, "Angle" },
    { MSG_ANA_CB_TORSION, "Torsion" },
    { MSG_ANA_CB_PLANE, "Plane Angle" },
    { MSG_ANA_CB_MULTIPLE, "Multiple" },
    { MSG_ANA_CB_DIPOLE, "Dipole moment" },
    { MSG_ANA_CB_MLP, "Molecular lipophilicity (MLP)" },
    { MSG_ANA_CB_PSA, "Polar surface area (PSA)" },
    { MSG_ANA_CB_SRF, "Surface area" },
    { MSG_ANA_CB_SRFDIA, "Surface diameter" },
    { MSG_ANA_CB_VOL, "Volume" },
    { MSG_ANA_CB_VOLDIA, "Volume diameter" },
    { MSG_ANA_OPENTRJ, "Open trajectory" },
    { MSG_ERR_ANA_SELECT, "Incorrect selection" },
    { MSG_PRGBAR_TITLE, "Calculation progress" },
    { MSG_RENRES_TITLE, "Residue renumbering" },
    { MSG_RENRES_LBL_START, "Start from:" },
    { MSG_RENRES_BT_REN, "&Renumber" },
    { MSG_SEL_TITLE, "Selection tool" },
    { MSG_SEL_BT_DWNATM, "&Down" },
    { MSG_SEL_BT_NEW, "&New" },
    { MSG_SEL_BT_REMATM, "&Remove" },
    { MSG_SEL_BT_REMOVE, "&Remove" },
    { MSG_SEL_BT_UPATM, "&Up" },
    { MSG_SEL_LBL_NAME, "Name:" },
    { MSG_SEL_LBL_SELATM, "Selected Atoms:" },
    { MSG_SEL_LBL_SELECT, "Selections:" },
    { MSG_BIODOCK_TITLE, "BioDock - Automatic Docking System" },
    { MSG_BIODOCK_OPENREC, "Open Receptor:" },
    { MSG_BIODOCK_OPENLIG, "Open Ligand:" },
    { MSG_BIODOCK_OPENPRJ, "Open Project:" },
    { MSG_BIODOCK_SAVEPRJ, "Save Project:" },
    { MSG_BIODOCK_MM_EXIT, "&Exit" },
    { MSG_BIODOCK_MM_NOW, "&Now !" },
    { MSG_BIODOCK_MM_OPEN, "&Open" },
    { MSG_BIODOCK_MM_PROJ, "&Project" },
    { MSG_BIODOCK_MM_RESET, "&Reset" },
    { MSG_BIODOCK_MM_SAVEAS, "&Save As..." },
    { MSG_BIODOCK_MM_RUN, "&Run" },
    { MSG_BIODOCK_LBL_PRJNAME, "Project Name:" },
    { MSG_BIODOCK_LBL_RECNAME, "Receptor File:" },
    { MSG_BIODOCK_LBL_LIGNAME, "Ligand File:" },
    { MSG_BIODOCK_LBL_FFIELD, "Force Field:" },
    { MSG_BIODOCK_LBL_DIEL, "Dielectric:" },
    { MSG_BIODOCK_LBL_FRAMES, "Frames:" },
    { MSG_BIODOCK_CB_HBOND, "Calculate H-Bond" },
    { MSG_BIODOCK_CB_DISTDEP, "Distance Dep." },
    { MSG_BIODOCK_CB_AUTOPAR, "Automatic Param." },
    { MSG_BIODOCK_GB_RANGES, "Ranges" },
    { MSG_BIODOCK_LBL_TRANS, "Translations:" },
    { MSG_BIODOCK_LBL_ROT, "Rotations:" },
    { MSG_BIODOCK_CB_CENTER, "Center:" },
    { MSG_BIODOCK_LBL_ACTIONS, "Actions:" },
    { MSG_BIODOCK_CB_DOCK, "Docking" },
    { MSG_BIODOCK_CB_EXTRA, "Extraction" },
    { MSG_BIODOCK_CB_OPTIM, "Optimization" },
    { MSG_BIODOCK_GB_ADVANCED, "Advanced" },
    { MSG_BIODOCK_LBL_OVERLAP, "Overlap:" },
    { MSG_BIODOCK_LBL_CUTOFF, "Cutoff:" },
    { MSG_BIODOCK_LBL_CUT, "Cut Sphere:" },
    { MSG_BIODOCK_LBL_TPRI, "Thread Pri:" },
    { MSG_BIODOCK_LBL_ACLUST, "Clustering Value:" },
    { MSG_BIODOCK_LBL_TRASH, "Trash Energy:" },
    { MSG_BIODOCK_LBL_REFRESH, "Refresh Traj. (sec.):" },
    { MSG_BIODOCK_RG_METHOD, "Method" },
    { MSG_BIODOCK_LBL_STEPS, "Steps:" },
    { MSG_BIODOCK_LBL_RMS, "RMS:" },
    { MSG_BIODOCK_LBL_SAVEMIN, "Save Best Frame" },
    { MSG_BIODOCK_LBL_CONV, "Conv. Factor:" },
    { MSG_BIODOCK_LBL_INTER, "Inter Frames:" },
    { MSG_BIODOCK_CB_EXBESTE, "Best Energy" },
    { MSG_BIODOCK_CB_EXBESTF, "Best Frames" },
    { MSG_BIODOCK_CB_EXTRASH, "Energy Threshold" },
    { MSG_BIODOCK_CB_EXSEL, "Selection" },
    { MSG_BIODOCK_CB_EXALL, "All Frames" },
    { MSG_BIODOCK_LBL_MODE, "Mode:" },
    { MSG_BIODOCK_LBL_ENER, "Energy threshold:" },
    { MSG_BIODOCK_LBL_FRMNUM, "Number of Frames:" },
    { MSG_BIODOCK_LBL_SEL, "Selection:" },
    { MSG_BIODOCK_STRBOX_TIT, "BioDock Macro" },
    { MSG_BIODOCK_ERR_PROMPT, "Error in BioDock project file at line %d:\n" },
    { MSG_BIODOCK_ERR_TOOLINES, "The input file exceeds 65535 lines" },
    { MSG_BIODOCK_ERR_UNKCOM, "Unknown command \"%s\"" },
    { MSG_BIODOCK_ERR_BEGMIS, "Missing BEGIN command" },
    { MSG_BIODOCK_ERR_DUPCOM, "Command %s duplicated at line %d" },
    { MSG_BIODOCK_ERR_ARGMIS, "Missing argument" },
    { MSG_BIODOCK_ERR_MACROOPENED, "Field already opened in macro input" },
    { MSG_BIODOCK_ERR_MACROCLOSED, "Field already closed in macro input" },
    { MSG_BIODOCK_ERR_MACRONOTOPENED, "Field not opened in macro input" },
    { MSG_BIODOCK_ERR_MACRONOTCLOSED, "Field not closed in macro input" },
    { MSG_BIODOCK_ERR_MACROFREQ, "Field required" },
    { MSG_BIODOCK_ERR_NUMINT, "Integer number required" },
    { MSG_BIODOCK_ERR_NUMPOS, "Positive integer required" },
    { MSG_BIODOCK_ERR_NUMFLOAT, "Float number required" },
    { MSG_BIODOCK_ERR_NUMBOOL, "Boolean value required (TRUE/FALSE)" },
    { MSG_BIODOCK_ERR_NULLNUM, "The specified value can't be zero" },
    { MSG_BIODOCK_ERR_NULLSTR, "Null string not allowed" },
    { MSG_BIODOCK_ERR_UNTERMSTR, "Unterminated string" },
    { MSG_BIODOCK_ERR_TRANSPOS, "The translation ranges must be positive" },
    { MSG_BIODOCK_ERR_ROTPOS, "The rotation ranges must be positive" },
    { MSG_BIODOCK_ERR_CONVFACT, "The value must be lower than 1" },
    { MSG_BIODOCK_ERR_MODULE, "Unknown module" },
    { MSG_BIODOCK_ERR_MISCOM, "Missing %s command" },
    { MSG_BIODOCK_ERR_NUMFRM, "Illegal number of frames (must be > 1)" },
    { MSG_BIODOCK_ERR_FRMSELMIS, "Missing selection of frames" },
    { MSG_BIODOCK_ERR_FRMSELINV, "Invalid selection of frames" },
    { MSG_BIODOCK_ERR_UNKEXTRA, "Unknown extraction mode" },
    { MSG_BIODOCK_ERR_UNKOPTMET, "Unknown optimization method" },
    { MSG_STRBOX_TITLE, "Enter a string ..." },
    { MSG_TASK_TITLE, "Active processes" },
    { MSG_TASK_BT_KILL, "&Kill" },
    { MSG_TASK_LBL_NAME, "Name" },
    { MSG_TASK_LBL_PID, "PID" },
    { MSG_TASK_MM_FILE, "&File" },
    { MSG_TASK_MM_REFRESH, "&Refresh" },
    { MSG_TASK_MM_EXIT, "&Exit" },
    { MSG_TASK_MM_OPTIONS, "&Options" },
    { MSG_TASK_MM_FORCE, "&Force" },
    { MSG_TASK_MM_FILTER, "F&ilter" },
    { MSG_TASK_MM_ACTIONS, "&Actions" },
    { MSG_TASK_MM_KILL, "&Kill now !" },
    { MSG_GLPRF_ERR_DEFAULT, "Using default GL parameters" },
    { MSG_MAIN_MENU_DSPVDWD, "CP&K dotted" },
    { MSG_MAIN_MENU_DSPLIQS, "&Stick solid" },
    { MSG_POPUP_MENU_WINDOW, "&Window" },
    { MSG_GLMAIN_WINTITLE, "VEGA ZZ %s - Port %d" },
    { MSG_ERR_MAIN_INVALIDPORT, "Invalid port number" },
    { MSG_ERR_MAIN_WIN32ONLY, "Option available only in Win32 version" },
    { MSG_DLGCLUSTER_RG_SHAPE, "Type" },
    { MSG_ERR_DEVIL_PROMPT, "DevIL image library error:\n%s" },
    { MSG_GLMAIN_CONTITLE, "VEGA Console - Port %d" },
    { MSG_MAIN_MENU_COLBYCHAIN, "By &chain" },
    { MSG_MAIN_MENU_COLBYCHARGE, "By ch&arge" },
    { MSG_MAIN_MENU_COLBYRES, "By &residue" },
    { MSG_MAIN_MENU_SELECT, "&Select" },
    { MSG_MAIN_MENU_SELALL, "&All" },
    { MSG_MAIN_MENU_SELBACK, "Protein &backbone" },
    { MSG_MAIN_MENU_SELNOHYD, "No &hydrogens" },
    { MSG_MAIN_MENU_SELINV, "&Invert" },
    { MSG_MAIN_MENU_SELNOWAT, "No &water" },
    { MSG_MAIN_MENU_DSPSET, "&Settings" },
    { MSG_PDB_ATDLFOUND, "  PDB ATDL extension found" },
    { MSG_PDB_ORACFOUND, "  PDB Orac subformat found" },
    { MSG_VIEWPREFS_CYLRAD, "Cylinder rad.:" },
    { MSG_VIEWPREFS_CYLRES, "Cylinder res.:" },
    { MSG_VIEWPREFS_DEFAULT, "Default" },
    { MSG_VIEWPREFS_DOTSIZE, "Dot size:" },
    { MSG_VIEWPREFS_SPHERERES, "Resolution:" },
    { MSG_VIEWPREFS_SPHERESCALE, "Sphere scale:" },
    { MSG_VIEWPREFS_TITLE, "View settings" },
    { MSG_VIEWPREFS_WIRESMOOTH, "Smooth mode" },
    { MSG_VIEWPREFS_WIRETICK, "Thickness:" },
    { MSG_VIEWPREFS_PGCPK, "Ball stick" },
    { MSG_VIEWPREFS_PGLIQ, "Stick" },
    { MSG_VIEWPREFS_PGVDW, "CPK" },
    { MSG_VIEWPREFS_PGWIREFRAME, "Wireframe" },
    { MSG_VIEWPREFS_ZCLIPFAR, "Z clip f.:" },
    { MSG_DLGABOUT_MAIL, "E-Mail" },
    { MSG_DLGABOUT_MORE, "More" },
    { MSG_ATMSEL_GB_RANGE, "Range" },
    { MSG_ATMSEL_LBL_ATOMCOL, "Atom:" },
    { MSG_ATMSEL_LBL_CHAINCOL, "Chain:" },
    { MSG_ATMSEL_LBL_RESCOL, "Residue:" },
    { MSG_ATMSEL_LBL_RESNCOL, "Number:" },
    { MSG_ATMSEL_LBL_SELECT, "Selection:" },
    { MSG_ATMSEL_RG_ATMNUM, "&Atom number" },
    { MSG_ATMSEL_RG_MODE, "Mode" },
    { MSG_ATMSEL_RG_RESNUM, "&Residue number" },
    { MSG_ATMSEL_TITLE, "Atom selection" },
    { MSG_ATMSEL_TS_ATOM, "&Atoms" },
    { MSG_ATMSEL_TS_RANGE, "&Range" },
    { MSG_COLTBL_TITLE, "Color table" },
    { MSG_PLUG_TITLE, "VEGA Plug-in System" },
    { MSG_PLUG_BT_ABOUT, "&About" },
    { MSG_PLUG_BT_CONFIG, "&Configure" },
    { MSG_PLUG_BT_HELP, "&Help" },
    { MSG_GLDEMO_ERR_NOTINST, "Demo not installed.\nPlease use the setup to install it." },
    { MSG_ERR_COM_COLUNK, "\"%s\" unknown color" },
    { MSG_ERR_COM_EXCRANGEI, "%s command: argument %d exceeds the range (%d -> %d)" },
    { MSG_ERR_COM_EXCRANGEF, "%s command: argument %d exceeds the range (%f -> %f)" },
    { MSG_ERR_COM_FEWARGS, "%s command with too few arguments (required %d)" },
    { MSG_ERR_COM_TOOARGS, "%s command with too many arguments (required %d)" },
    { MSG_ERR_COM_ILLARG, "%s command: illegal type or value in argument %d" },
    { MSG_ERR_COM_NOCOM, "Command not specified" },
    { MSG_ERR_COM_NOMOL, "%s command: no molecules loaded" },
    { MSG_ERR_COM_NOTIMP, "Command not yet implemented" },
    { MSG_ERR_COM_NOTRJ, "%s command: trajectory not opened" },
    { MSG_ERR_COM_UNKCOM, "%s unknown command" },
    { MSG_ERR_COM_UNKERR, "Unknown error" },
    { MSG_ERR_COM_UNKKEYWORD, "%s command: unknown keyword in argument %d" },
    { MSG_ERR_DCD_NOMOL, "No molecule loaded" },
    { MSG_ERR_FMOD_PROMPT, "Fmod sound system error:\n%s" },
    { MSG_GLMAIN_CPUTYPE, "%d logical processors detected\n" },
    { MSG_GLMAIN_ENABLEOPT, "HyperDrive %d.%d initialized (%s)\n" },
    { MSG_GLMEAS_ANGLE, "Angle: %.4f\n" },
    { MSG_GLMEAS_DIST, "Distance: %.4f \n" },
    { MSG_GLMEAS_PICKATM, "Picked atom: %4d %.4s:%.4s:%.4s:%c:%d\n" },
    { MSG_GLMEAS_PLANG, "Angle between two planes: %.4f\n" },
    { MSG_GLMEAS_TORSION, "Torsion: %.4f\n" },
    { MSG_MAIN_MENU_LASTERR, "Last &error" },
    { MSG_MAIN_MENU_PLUGCONF, "Pl&ug-in configuration" },
    { MSG_MAIN_MENU_PLUGMANAGE, "&Manage" },
    { MSG_MAIN_MENU_COLSEL, "&Selection" },
    { MSG_MAIN_MENU_SELNONE, "&None" },
    { MSG_MAIN_MENU_SELCUSTOM, "&Custom" },
    { MSG_POPUP_MENU_JOYENABLE, "&Joystick enabled" },
    { MSG_POPUP_MENU_PICK, "&Measure" },
    { MSG_POPUP_MENU_PICKNONE, "&None" },
    { MSG_POPUP_MENU_PICKANG, "A&ngle" },
    { MSG_POPUP_MENU_PICKATOM, "&Atom" },
    { MSG_POPUP_MENU_PICKDIST, "&Distance" },
    { MSG_POPUP_MENU_PICKPLANG, "&Plane angle" },
    { MSG_POPUP_MENU_PICKTOR, "&Torsion" },
    { MSG_LOADER_MDLMOL, "MDL Molfile" },
    { MSG_ERR_LOCALE_ERRCODE, "Code %d\n%s" },
    { MSG_ERR_LOCALE_LASTERR, "LAST ERROR" },
    { MSG_ERR_LOCALE_NOERROR, "No error" },
    { MSG_ERR_WIN_CANTLOADLIB, "Can't load the \"%s\"" },
    { MSG_ERR_WIN_CANTGETADDR, "Can't get the address of the function \"%s\" (%s)" },
    { MSG_DLGCOPY_TITLE, "Copy special" },
    { MSG_DLGCOPY_BT_COPY, "Copy" },
    { MSG_DLGCOPY_RG_MODE, "Mode" },
    { MSG_DLGINTER_TITLE, "Evaluation of the interactions" },
    { MSG_DLGINTER_BT_EVALUATE, "&Evaluate" },
    { MSG_DLGINTER_CB_COLLIGATM, "Ligand by atom" },
    { MSG_DLGINTER_CB_COLMOL, "By molecule" },
    { MSG_DLGINTER_CB_COLRES, "By residue" },
    { MSG_DLGINTER_CB_LIGAND, "Ligand" },
    { MSG_DLGINTER_CB_RESIDUES, "Best residues" },
    { MSG_DLGINTER_CB_WATER, "Water" },
    { MSG_DLGINTER_GRP_COLOR, "Color" },
    { MSG_DLGINTER_GRP_ENERGY, "Energy" },
    { MSG_DLGINTER_GRP_VISUAL, "Visualization" },
    { MSG_DLGINTER_LBL_CUTOFF, "Cutoff ():" },
    { MSG_DLGINTER_LBL_DIEL, "Dielectric:" },
    { MSG_DLGINTER_LBL_RESIDUE, "Ligand:" },
    { MSG_DLGINTER_LBL_TRASH, "Threshold (%):" },
    { MSG_ERR_SRFNOMOL, "No molecule loaded" },
    { MSG_LOADER_GAMESS, "GAMESS cartesian" },
    { MSG_LOADER_INSSRF, "Insight Srf" },
    { MSG_MAIN_MENU_CONSOLE, "C&onsole" },
    { MSG_MAIN_MENU_INTERACT, "&Interactions" },
    { MSG_ERR_COM_EXCRANGEUI, "Argument %d exceeds the range (%u -> %u)" },
    { MSG_ERR_COM_NOSRF, "No surface present" },
    { MSG_ERR_CON_SAVEBUF, "Unable to save the buffer" },
    { MSG_BT_NO, "&No" },
    { MSG_BT_YES, "&Yes" },
    { MSG_MAIN_MENU_EDITATM, "&Atom/residue/chain" },
    { MSG_DLGEDITATM_TITLE, "Edit" },
    { MSG_DLGEDITATM_BT_APPLY, "Apply" },
    { MSG_DLGEDITATM_LBL_ATMTYPE, "Atom type:" },
    { MSG_DLGEDITATM_LBL_CHAIN, "Chain:" },
    { MSG_DLGEDITATM_LBL_CHARGE, "Charge:" },
    { MSG_DLGEDITATM_LBL_CURCHAIN, "Current:" },
    { MSG_DLGEDITATM_LBL_CURRES, "Current:" },
    { MSG_DLGEDITATM_LBL_ELEM, "Element:" },
    { MSG_DLGEDITATM_LBL_NUMBER, "Number:" },
    { MSG_DLGEDITATM_LBL_NAME, "Name:" },
    { MSG_DLGEDITATM_LBL_RESNAME, "Res. name:" },
    { MSG_DLGEDITATM_LBL_RESSEQ, "Res. number:" },
    { MSG_DLGEDITATM_RG_ALLATM, "All atoms" },
    { MSG_DLGEDITATM_RG_APPLYTO, "Apply to" },
    { MSG_DLGEDITATM_RG_VIS, "Visible only" },
    { MSG_DLGEDITATM_TS_ATOM, "Atom" },
    { MSG_DLGEDITATM_TS_CHAIN, "Chain" },
    { MSG_DLGEDITATM_TS_RESIDUE, "Residue" },
    { MSG_DLGMULTISEL_TITLE, "Select an item ..." },
    { MSG_VIEWPREFS_SMARTATOMS, "Atoms:" },
    { MSG_VIEWPREFS_SMARTMOVE, "Smart move" },
    { MSG_ANA_BT_ENEGRAPH, "Energy Graph" },
    { MSG_ANA_CB_ILM, "Molecular Hydropathicity Index (ILM)" },
    { MSG_ANA_CB_VISONLY, "&Consider selected atoms only" },
    { MSG_ANA_INTENERGY, "Interaction energy" },
    { MSG_ANA_LBL_ENERGY, "Energy:" },
    { MSG_ANA_POTENERGY, "Potential energy" },
    { MSG_ANA_SELCONF, "Selected conformation:" },
    { MSG_BIODOCK_END, "\n* End of BioDock calculation" },
    { MSG_BIODOCK_START, "* Starting BioDock process" },
    { MSG_ERR_BIODOCK_BIODOCKEXE, "Can't find BioDock executable" },
    { MSG_ERR_BIODOCK_BIOOUT, "Unrecognized BioDock output" },
    { MSG_ERR_BIODOCK_BIOPRJ, "Can't open BioDock input file" },
    { MSG_ERR_BIODOCK_STRUCTOUT, "BioDock output corrupted" },
    { MSG_GLMEAS_CENTER, "* New molecule rotation center at: %.4f %.4f %.4f\n" },
    { MSG_MAIN_MENU_COLBYSEG, "By s&egment" },
    { MSG_POPUP_MENU_PICKCENTER, "&Center" },
    { MSG_GRAPH_MM_SELCONF, "&Sel. conformation" },
    { MSG_ERR_GRAPH_POINTRANGE, "Point out of range (0-%d)" },
    { MSG_LOADER_ALCHEMY, "Alchemy" },
    { MSG_LOADER_CSVSRF, "CSV Srf" },
    { MSG_MINIED_SAVECHG, "Do you want save the changes ?" },
    { MSG_DLG3DCTRL_BT_ROTX, "X Rotation" },
    { MSG_DLG3DCTRL_BT_ROTY, "Y Rotation" },
    { MSG_DLG3DCTRL_BT_ROTZ, "Z Rotation" },
    { MSG_DLG3DCTRL_BT_SCALE, "Scale" },
    { MSG_DLG3DCTRL_TITLE, "3D Controls" },
    { MSG_DLG3DCTRL_BT_TRANSX, "X Translation" },
    { MSG_DLG3DCTRL_BT_TRANSY, "Y Translation" },
    { MSG_DLG3DCTRL_BT_TRANSZ, "Z Translation" },
    { MSG_BT_ADD, "&Add" },
    { MSG_BT_ALL, "&All" },
    { MSG_BT_APPLY, "&Apply" },
    { MSG_BT_AUTO, "&Auto" },
    { MSG_BT_BUILD, "&Build" },
    { MSG_BT_MERGE, "&Merge" },
    { MSG_BT_MOVE, "&Move" },
    { MSG_BT_NEW, "&New" },
    { MSG_BT_NONE, "&None" },
    { MSG_BT_REMOVE, "&Remove" },
    { MSG_BT_RESET, "&Reset" },
    { MSG_BT_SELECT, "&Select" },
    { MSG_BT_SWAP, "&Swap" },
    { MSG_OVERWRITE, "File already present. Do you want overwrite it ?" },
    { MSG_ANA_CB_LIPBROTO, "Lipole (Broto & Moreau)" },
    { MSG_ANA_CB_LIPCRIPPEN, "Lipole (Ghose & Crippen)" },
    { MSG_ERR_ANA_GRAPH, "The energy graph can't be showed" },
    { MSG_DLGATMADD_LBL_OTHER, "Other" },
    { MSG_DLGATMADD_LBL_SELECTED, "Selected:" },
    { MSG_DLGATMADD_RG_ELEM, "Element" },
    { MSG_DLGATMADD_RG_HYBRID, "Hybrization" },
    { MSG_DLGATMADD_TITLE, "Add/remove atom" },
    { MSG_ATMSEL_BT_HYDROG, "&Hydrogens" },
    { MSG_ATMSEL_BT_WATER, "&Water" },
    { MSG_ATMSEL_LBL_MOLCOL, "Mol.:" },
    { MSG_ATMSEL_LBL_PROXRAD, "Radius:" },
    { MSG_ATMSEL_RG_PROXMODE, "Around" },
    { MSG_ATMSEL_RG_PROXMODEATM, "&Atom" },
    { MSG_ATMSEL_RG_PROXMODECHAIN, "&Chain" },
    { MSG_ATMSEL_RG_PROXMODEMOL, "&Molecule" },
    { MSG_ATMSEL_RG_PROXMODERES, "&Residue" },
    { MSG_ATMSEL_RG_PROXWHAT, "What" },
    { MSG_ATMSEL_RG_PROXWHATATM, "&Atoms" },
    { MSG_ATMSEL_RG_PROXWHATRES, "&Residues" },
    { MSG_ATMSEL_RG_SELMODE, "Selection" },
    { MSG_ATMSEL_RG_SELMODEADD, "&Additive" },
    { MSG_ATMSEL_RG_SELMODENORM, "&Normal" },
    { MSG_ATMSEL_TS_PROXIMITY, "&Proximity" },
    { MSG_BOND_ASSIGNPROG, "* Assigning bond types" },
    { MSG_DLGBOND_LBL_CONNTOL, "Bond tolerance (%):" },
    { MSG_DLGBOND_LBL_NONE, "None" },
    { MSG_DLGBOND_LBL_SELBOND, "Selected bond:" },
    { MSG_DLGBOND_RG_ALLATM, "All atoms" },
    { MSG_DLGBOND_RG_BND, "Bond type" },
    { MSG_DLGBOND_RG_BNDDOUBLE, "Double" },
    { MSG_DLGBOND_RG_BNDPARDOUBLE, "Partial double" },
    { MSG_DLGBOND_RG_BNDSINGLE, "Single" },
    { MSG_DLGBOND_RG_BNDTRIPLE, "Triple" },
    { MSG_DLGBOND_RG_CONNALLATM, "Connectivity of all atoms" },
    { MSG_DLGBOND_RG_CONNSELATM, "Connectivity of selected atoms" },
    { MSG_DLGBOND_RG_FINDALLATM, "Find the bond types" },
    { MSG_DLGBOND_RG_FINDSELATM, "Find the bond types (sel. atoms)" },
    { MSG_DLGBOND_RG_ONEBOND, "One bond" },
    { MSG_DLGBOND_RG_SELATM, "Selected atoms only" },
    { MSG_DLGBOND_RG_WHAT, "What ?" },
    { MSG_DLGBOND_TITLE, "Bonds" },
    { MSG_DLGBOND_TS_ADD, "&Add" },
    { MSG_DLGBOND_TS_CHANGE, "&Change" },
    { MSG_DLGBOND_TS_REMOVE, "&Remove" },
    { MSG_WARN_DLGBOND_ALREADYCONN, "The two atoms are already connected\n" },
    { MSG_WARN_DLGBOND_ALREADYNOTCONN, "The two atoms are already unbonded\n" },
    { MSG_WARN_DLGBOND_NOTCONN, "The two atoms aren't bonded\n" },
    { MSG_DLGCENTR_HC_ATOMS, "Atoms" },
    { MSG_DLGCENTR_LBL_NAME, "Name:" },
    { MSG_DLGCENTR_TITLE, "Centroid definition" },
    { MSG_WARN_DLGCENTR_ALREADY, "Atom already picked" },
    { MSG_WARN_DLGCENTR_NOCENTR, "You can't pick another centroid" },
    { MSG_DLGCLUSTBLD_BEGIN, "\n* Starting the cluster building\n" },
    { MSG_DLGCLUSTBLD_CB_ENABLE, "Enable" },
    { MSG_DLGCLUSTBLD_CB_MINIMIZE, "Minimize bumps" },
    { MSG_DLGCLUSTBLD_CB_REMOVE, "Remove molecules" },
    { MSG_DLGCLUSTBLD_CB_SAME, "Same" },
    { MSG_DLGCLUSTBLD_CB_VIEWCELL, "View 3D cell" },
    { MSG_DLGCLUSTBLD_GRP_BOXSIZE, "Cell size ()" },
    { MSG_DLGCLUSTBLD_GRP_COPIES, "Cell copies" },
    { MSG_DLGCLUSTBLD_GRP_EXTRA, "Extra" },
    { MSG_DLGCLUSTBLD_GRP_OPTIM, "Optimization" },
    { MSG_DLGCLUSTBLD_GRP_RNDROT, "Random rot." },
    { MSG_DLGCLUSTBLD_LBL_OVERLAP, "Overlap (%):" },
    { MSG_DLGCLUSTBLD_LBL_RMS, "RMS:" },
    { MSG_DLGCLUSTBLD_LBL_STEPS, "Steps:" },
    { MSG_DLGCLUSTBLD_PROGBAR, "Placing the molecules ..." },
    { MSG_DLGCLUSTBLD_RMMOL, "  Molecule removed\n" },
    { MSG_DLGCLUSTBLD_TITLE, "Cluster builder" },
    { MSG_DLGCLUSTBLD_TOTMOL, "\n%d solvent molecules placed in the cluster\n\n" },
    { MSG_DLGCLUSTBLD_UNABLEOPTIM, "  Unable to optimize the molecule %d\n" },
    { MSG_ERR_COM_EXCRANGEC, "Argument %d has an illegal number of characters (%u -> %u)" },
    { MSG_DCD_STARTLIPOLE, "  Starting lipole    : %.4f\n" },
    { MSG_ERR_DCD_NOOUT, "Output file not specified" },
    { MSG_ERR_ESCHER_CORRFILE, "Corrupted ESCHER file" },
    { MSG_ERR_ESCHER_NUMSOL, "Illegal number of solutions in ESCHER file" },
    { MSG_ERR_ESCHER_UNKFORMAT, "Unknown ESCHER file format" },
    { MSG_DLGEDITATM_CB_COLOR, "Color atoms" },
    { MSG_DLGFIXATM_GRP_COLORS, "Colors" },
    { MSG_DLGFIXATM_GRP_PARAM, "Parameters" },
    { MSG_DLGFIXATM_LBL_VALUE, "Value:" },
    { MSG_DLGFIXATM_RG_MODE, "Mode" },
    { MSG_DLGFIXATM_RG_MODEFIX, "Fix" },
    { MSG_DLGFIXATM_RG_MODEFREE, "Free" },
    { MSG_DLGFIXATM_RG_MODEVALUE, "Value" },
    { MSG_DLGFIXATM_RG_SELECTION, "Selection" },
    { MSG_DLGFIXATM_RG_SELALLATM, "All atoms" },
    { MSG_DLGFIXATM_RG_SELBACKBONE, "Protein backbone" },
    { MSG_DLGFIXATM_RG_SELVISATM, "Visible atoms" },
    { MSG_DLGFIXATM_TITLE, "Constraint options" },
    { MSG_GAMESS_ENERGY, "  Total energy: %.4f\n" },
    { MSG_GLMEAS_CENTERWARN, "It will be active moving the molecule\n" },
    { MSG_MAIN_MENU_BIOTOOLS, "&Bioinformatics" },
    { MSG_MAIN_MENU_BIORES, "&Web resources" },
    { MSG_MAIN_MENU_SIMIL, "Simi&larity" },
    { MSG_MAIN_MENU_CHANGE, "C&hange" },
    { MSG_MAIN_MENU_CHGTOR, "Bond/Angle/&Torsion" },
    { MSG_MAIN_MENU_ADD, "&Add" },
    { MSG_MAIN_MENU_ATOM, "&Atom" },
    { MSG_MAIN_MENU_BONDS, "&Bonds" },
    { MSG_MAIN_MENU_CENTROID, "C&entroid" },
    { MSG_MAIN_MENU_IONS, "&Ions" },
    { MSG_MAIN_MENU_BUILD, "&Build" },
    { MSG_MAIN_MENU_BLDCLUST, "&Solvent cluster" },
    { MSG_MAIN_MENU_COORDS, "C&oordinates" },
    { MSG_MAIN_MENU_APPLYTRANS, "Apply &transf." },
    { MSG_MAIN_MENU_INVERTZ, "Invert &Z" },
    { MSG_MAIN_MENU_RMCENTR, "&Centroids" },
    { MSG_MAIN_MENU_RMINVISATM, "Invisible a&toms" },
    { MSG_MAIN_MENU_RMMOL, "&Molecule" },
    { MSG_MAIN_MENU_RMSEG, "&Segment" },
    { MSG_MAIN_MENU_SEG, "&Segments" },
    { MSG_MAIN_MENU_SEGFIX, "&Fix" },
    { MSG_MAIN_MENU_SEGMERGE, "&Merge" },
    { MSG_MAIN_MENU_FIXSAV, "Cons&traints" },
    { MSG_MAIN_MENU_MERGE, "&Merge" },
    { MSG_MAIN_MENU_3DCTRL, "&3D controls" },
    { MSG_MAIN_MENU_COLBYCONSTR, "By c&onstraint" },
    { MSG_MAIN_MENU_LBLATM_FIX, "&Fixing val." },
    { MSG_MAIN_MENU_LBLATM_RESNAME, "Residue na&me" },
    { MSG_MAIN_MENU_LBLATM_RESNAMSEQ, "&Residue" },
    { MSG_MAIN_MENU_LBLATM_RESSEQ, "Residue num&ber" },
    { MSG_MAIN_MENU_SELMOL, "&Molecule" },
    { MSG_MAIN_MENU_SELSEG, "&Segment" },
    { MSG_INFO_PREDCHARGE, "Predicted charge...: %d (composition of charged fragments)\n" },
    { MSG_ION_EXCLUSIONMAP, "  Building the exclusion map ...\n" },
    { MSG_ION_GRIDNODES, "    Grid nodes left after the exclusion: %d\n\n" },
    { MSG_ION_GRIDSIZE, "  Grid size: %d x %d x %d (%d nodes)\n" },
    { MSG_ION_MOLDIM, "  Molecule dimensions:\n"\
	"    X = %f - %f\n"\
	"    Y = %f - %f\n"\
	"    Z = %f - %f\n\n" },
    { MSG_ION_PLACED, "  Palced ions:\n" },
    { MSG_ION_POSITION, "    # %d (E= %f) at grid point %d (%d, %d, %d)\n" },
    { MSG_ION_START, "* Ion arrangment started" },
    { MSG_DLGEDITATM_LBL_FIX, "Contraint:" },
    { MSG_DLGIONS_BT_NEUTRALIZE, "Neutralize" },
    { MSG_DLGIONS_FIXCHARGES, "The molecule doesn't have got the atomic charges.\n"\
	"Do you want fix them ?" },
    { MSG_DLGIONS_LBL_BOXTHICK, "Box thickness:" },
    { MSG_DLGIONS_LBL_CHARGE, "Charge:" },
    { MSG_DLGIONS_LBL_ELEMENT, "Element:" },
    { MSG_DLGIONS_LBL_EXCLATOMRAD, "Atom/ion closest distance:" },
    { MSG_DLGIONS_LBL_EXCLIONRAD, "Ion/ion closest distance:" },
    { MSG_DLGIONS_LBL_GRIDSTEP, "Grid step:" },
    { MSG_DLGIONS_LBL_ION, "Ion:" },
    { MSG_DLGIONS_LBL_MOLCHARGE, "Total molecule charge:" },
    { MSG_DLGIONS_LBL_NIONS, "Ions to add:" },
    { MSG_DLGIONS_TITLE, "Add ions" },
    { MSG_DLGMERGE_CB_ATMNAMES, "Atom names" },
    { MSG_DLGMERGE_CB_ATMTYPES, "Atom types" },
    { MSG_DLGMERGE_CB_CHAINID, "Chain IDs" },
    { MSG_DLGMERGE_CB_CHARGES, "Atom charges" },
    { MSG_DLGMERGE_CB_CONN, "Connectivity" },
    { MSG_DLGMERGE_CB_CONSTRAINTS, "Constraints" },
    { MSG_DLGMERGE_CB_COORD, "Coordinates" },
    { MSG_DLGMERGE_CB_ELEM, "Elements" },
    { MSG_DLGMERGE_CB_MOLID, "Molecule IDs" },
    { MSG_DLGMERGE_CB_RESNAMES, "Residue names" },
    { MSG_DLGMERGE_CB_RESSEQ, "Residue numbers" },
    { MSG_DLGMERGE_CB_SEGMENTS, "Segments" },
    { MSG_DLGMERGE_LBL_SEL, "Select:" },
    { MSG_DLGMERGE_OPEN, "Merge the molecule with ..." },
    { MSG_DLGMERGE_TITLE, "What merge ?" },
    { MSG_DLGMERGE_ERR_NOTMERGE, "  Nothing to merge\n" },
    { MSG_DLGMERGE_ERR_UNALLELEM, "One or more selected elements to merge aren't present in the file" },
    { MSG_DLGMERGE_ERR_UNCOMPMOL, "Uncompatible molecules. Nothing to merge" },
    { MSG_DLGMERGE_WARN_CONT, "The molecule to merge has got the correct\n"\
	"number of atoms, but it appears different.\n"\
	"Do you want continue ?" },
    { MSG_DLGMVMOL_CONNMOL, "\n* Controls connected to molecule %d\n" },
    { MSG_DLGMVMOL_CONNWORLD, "\n* Controls connected to world\n" },
    { MSG_DLGMVMOL_TITLE, "Move molecule" },
    { MSG_DLGOVER_BT_RETRY, "&Revert" },
    { MSG_DLGOVER_BT_SUPERIMP, "&Superimpose" },
    { MSG_DLGOVER_HC_SEL, "Selection" },
    { MSG_DLGOVER_HC_SOURCE, "Source" },
    { MSG_DLGOVER_HC_TARGET, "Target" },
    { MSG_DLGOVER_RMS, "* RMS distance between the selected atom pairs: %.3f\n" },
    { MSG_DLGOVER_TITLE, "Molecular similarity" },
    { MSG_WARN_PDB_NOCONN, "Connectivity not saved. The total atom number exceeds 99999" },
    { MSG_DLGRMMOL_HC_ATOMS, "Atoms" },
    { MSG_DLGRMMOL_HC_END, "Last atm" },
    { MSG_DLGRMMOL_HC_MOLECULE, "Molecule" },
    { MSG_DLGRMMOL_HC_RESIDUE, "Residues" },
    { MSG_DLGRMMOL_HC_SEGMENT, "Segment" },
    { MSG_DLGRMMOL_HC_START, "First atm" },
    { MSG_DLGRMMOL_TITLE, "Remove molecule(s)" },
    { MSG_DLGRMSEG_TITLE, "Remove segment(s)" },
    { MSG_DLGSELMOL_TITLE, "Molecules" },
    { MSG_DLGSELSEG_TITLE, "Segments" },
    { MSG_DLGTOR_BT_CHANGE, "-    Change    +" },
    { MSG_DLGTOR_HC_TORSION, "Element" },
    { MSG_DLGTOR_LBL_CONTROL, "Interactive control:" },
    { MSG_DLGTOR_LBL_TORVAL, "Torsion value (degree):" },
    { MSG_DLGTOR_TITLE, "Change bond/angle/torsion" },
    { MSG_GASTEIGER_ERR_NOPAR, "  No parameters for %.8s atom type\n" },
    { MSG_GLMEAS_ALREADYPICKED, "Atom already picked\n" },
    { MSG_GLMEAS_RESET, "Restart the picking from the beginning\n" },
    { MSG_GLMEAS_ERR_ATMMULTI, "One or more atoms are multiple selected" },
    { MSG_GLMEAS_ERR_ATMNOTFND, "Atom %d not found" },
    { MSG_ION_PROGBAR_ENER, "Computing energy" },
    { MSG_ION_PROGBAR_INIT, "Initialization" },
    { MSG_ION_PROGBAR_PLACE, "Placing ions" },
    { MSG_POPUP_MENU_MOVE, "M&ove" },
    { MSG_POPUP_MENU_MOVEALL, "&All" },
    { MSG_POPUP_MENU_MOVEMOL, "&Molecule" },
    { MSG_POPUP_MENU_PICKNOMON, "Remove &monitors" },
    { MSG_POPUP_MENU_PICKRESET, "&Reset picking" },
    { MSG_REM_ATOMS, "* %d atoms removed\n" },
    { MSG_REM_NOATOM, "* No atom removed\n" },
    { MSG_SEG_FOUND, "* %d segments found\n" },
    { MSG_SEG_NOTFOUND, "* No segment found\n" },
    { MSG_SEL_OPENTITLE, "Open selection ..." },
    { MSG_SEL_SAVETITLE, "Save selection ..." },
    { MSG_SEL_ERR_DONTMATCH, "The atom %s doesn't match the molecule/s. The selection \"%s\" will be removed" },
    { MSG_SEL_ERR_ILLNUMARG, "Illegal number of arguments at line %d" },
    { MSG_SEL_ERR_UNKTAG, "Unknown tag \"%s\" at line %d" },
    { MSG_VIEWPREFS_WIRESMULTIVEC, "Multivector" },
    { MSG_DLG3DCTRL_LBL_WRKSPC, "Workspaces:" },
    { MSG_DLG3DCTRL_PM_WKSNEW, "&New" },
    { MSG_DLG3DCTRL_PM_WKSREMOVE, "&Remove" },
    { MSG_DLG3DCTRL_PM_WKSREMOVEALL, "Remove &all" },
    { MSG_ADDH_PROG, "* Adding hydrogens" },
    { MSG_ADDH_WARN, "  The atom %s has an unusual geometry. The atom type will be corrected.\n" },
    { MSG_BT_CLOSEALL, "Close all" },
    { MSG_BT_GET, "&Get" },
    { MSG_BT_FINISH, "&Finish" },
    { MSG_BT_NEXT, "&Next" },
    { MSG_BT_OPEN, "&Open" },
    { MSG_BT_PREVIOUS, "Pre&vious" },
    { MSG_BT_PUT, "&Put" },
    { MSG_BT_RENAME, "&Rename" },
    { MSG_BT_RESCAN, "R&escan" },
    { MSG_BT_UPDATE, "&Update" },
    { MSG_DBENGINE_CREATE, "* Database \"%s\" successfully created\n" },
    { MSG_DBENGINE_GET, "* Molecule \"%s\" extracted from the database\n" },
    { MSG_DBENGINE_OPEN, "* Database \"%s\" open\n"\
	"  Creating index ...\n" },
    { MSG_DBENGINE_OPENMOL, "  Found %d molecules\n" },
    { MSG_DBENGINE_PUT, "* Molecule  \"%s\" added to the database\n" },
    { MSG_DBENGINE_REMOVE, "* Molecule \"%s\" removed from the database\n" },
    { MSG_DBENGINE_REN, "* Molecule  \"%s\" renamed to \"%s\"\n" },
    { MSG_DBENGINE_UPDATE, "* Molecule \"%s\" updated in the database\n" },
    { MSG_ERR_DBENGINE_CREATE, "Can't create the database" },
    { MSG_ERR_DBENGINE_GET, "Unable to get the molecule \"%s\" from the database" },
    { MSG_ERR_DBENGINE_MOLLIST, "Unable to read the database index" },
    { MSG_ERR_DBENGINE_OPEN, "Database already opened" },
    { MSG_ERR_DBENGINE_PUT, "Unable to put the molecule \"%s\" into the database" },
    { MSG_ERR_DBENGINE_REMOVE, "Unable to remove the molecule \"%s\" from the database" },
    { MSG_ERR_DBENGINE_REN, "Unable to rename the molecule \"%s\"" },
    { MSG_ERR_DBENGINE_UNKFOR, "Unknown database format" },
    { MSG_ERR_DBENGINE_UPDATE, "Unable to update the molecule \"%s\" in the database" },
    { MSG_ERR_DBENGINE_ZIP32INIT, "The Zip32.dll can't be initialized" },
    { MSG_WARN_DBENGINE_PUT, "Nothing to put into the database" },
    { MSG_WARN_DBENGINE_UPDATE, "Nothing to update in the database" },
    { MSG_DLGADDFRAG_HC_DBASE, "Group" },
    { MSG_DLGADDFRAG_HC_FRAG, "Fragment" },
    { MSG_DLGADDFRAG_LBL_FRAGMENT, "Choose the fragment to add" },
    { MSG_DLGADDFRAG_LBL_PICKFRAGH, "Pick a fragment hydrogen" },
    { MSG_DLGADDFRAG_LBL_PICKEDFRAG, "Fragment:" },
    { MSG_DLGADDFRAG_LBL_PICKEDFRAGH, "Frag. hydrog.:" },
    { MSG_DLGADDFRAG_LBL_PICKMOLH, "Pick a molecule hydrogen" },
    { MSG_DLGADDFRAG_LBL_PICKEDMOLH, "Mol. hydrog.:" },
    { MSG_DLGADDFRAG_TITLE, "Add fragment/s" },
    { MSG_DLGADDFRAG_LBL_TOR, "Torsion:" },
    { MSG_DLGADDFRAG_LBL_TORSION, "Set the torsion" },
    { MSG_DLGADDHYD_CB_IUPAC, "Use &IUPAC atom nomenclature" },
    { MSG_DLGADDHYD_RG_HYDPOS, "Position of hydrogens" },
    { MSG_DLGADDHYD_RG_HYDPOS_ATM, "After each heavy atom" },
    { MSG_DLGADDHYD_RG_HYDPOS_RES, "Residue end" },
    { MSG_DLGADDHYD_RG_MOLTYPE, "Molecule type" },
    { MSG_DLGADDHYD_RG_MOLTYPE_GEN, "Generic organic" },
    { MSG_DLGADDHYD_RG_MOLTYPE_NA, "Nucleic acid" },
    { MSG_DLGADDHYD_RG_MOLTYPE_PROT, "Protein" },
    { MSG_DLGADDHYD_TITLE, "Add hydrogens" },
    { MSG_ANA_CB_RMSD, "RMSD" },
    { MSG_DLGANA_OPENTRJFILTER, "All supported files|*.arc;*.csr;*.dcd;*.dlg;*.iff;*.mdcrd;*.ml2;*.mol;*.mol2;*.nc;*.pdb;*.pdbqt*.trr;*.xtc|"\
	"Accelrys archive (*.arc)|*.arc|"\
	"Amber (*.mdcrd)|*.mdcrd|"\
	"Amber NetCDF (*.nc)|*.nc|"\
	"AutoDock (*.dlg)|*.dlg|"\
	"CHARMM/NAMD (*.dcd)|*.dcd|"\
	"CIF/mmCIF (*.cif)|*.cif|"\
	"GRAMM (*.res)|*.res|"\
	"Gromacs TRR (*.trr)|*.trr|"\
	"Gromacs XTC (*.xtc)|*.xtc|"\
	"IFF/RIFF (*.iff)|*.iff|"\
	"MDL Mol (*.mol)|*.mol|"\
	"Mol2 (*.ml2;*.mol;*.mol2)|*.ml2;*.mol;*.mol2|"\
	"PDB MultiModel (*.pdb)|*.pdb|"\
	"PDBQT MultiModel (*.pdbqt)|*.pdbqt|"\
	"Quanta CSR (*.csr)|*.csr" },
    { MSG_DLGCENTR_CB_FIXED, "Keep fixed" },
    { MSG_DLGCLUSTER_GRP_COORD, "Center coord." },
    { MSG_DLGCLUSTER_RG_CLUSTPOS, "Cluster position" },
    { MSG_DLGCLUSTER_RG_CLUSTPOSCUST, "Custom" },
    { MSG_DLGCLUSTER_RG_CLUSTPOSGEO, "Geometry center" },
    { MSG_DLGDBEXP_DEFMOLNAME, "Molecule" },
    { MSG_DLGDBEXP_HC_DBASE, "Database" },
    { MSG_DLGDBEXP_HC_MOL, "Molecule" },
    { MSG_DLGDBEXP_RG_MODE, "Get mode" },
    { MSG_DLGDBEXP_RG_MODEADD, "Add" },
    { MSG_DLGDBEXP_RG_MODEREP, "Replace" },
    { MSG_DLGDBEXP_RG_MODEWKS, "Workspace" },
    { MSG_DLGDBEXP_REMOVE, "Are you sure to remove the selected molecule/s ?" },
    { MSG_DLGDBEXP_TITLE, "Database explorer" },
    { MSG_DLGDBOPEN_BT_CREATE, "&Create" },
    { MSG_DLGDBOPEN_CB_FDIR, "Directory" },
    { MSG_DLGDBOPEN_CB_FSDF, "SDF file" },
    { MSG_DLGDBOPEN_CB_FZIP, "Zip archive" },
    { MSG_DLGDBOPEN_CK_MOLCONN, "Co&nnectivity" },
    { MSG_DLGDBOPEN_CK_MOLFIX, "Con&straints" },
    { MSG_DLGDBOPEN_FILTER, "All supported databases|%s;*.accdb;*.db;*.dsn;*.iup;*.mdb;*.mmd;*.ml2;*.mol2;*.sdf;*.sdb;*.smi;*.txt;*.zip|"\
	"Access (*.mdb;*.accdb)|*.mdb;*.accdb|"\
	"Directory (%s)|%s|"\
	"IUPAC (*.iup;*.txt)|*.iup;*.txt|"\
	"Merck MMD  (*.mmd)|*.mmd|"\
	"Mol2 (*.ml2;*.mol2)|*ml2;*.mol2|"\
	"ODBC data source (*.dsn)|*.dsn|"\
	"SDF file (*.sdf)|*.sdf|"\
	"SMILES (*.smi;*.txt)|*.smi;*.txt|"\
	"SQLite (*.db;*.sdb)|*.db;*.sdb|"\
	"Zip archive (*.zip)|*.zip" },
    { MSG_DLGDBOPEN_LBL_FORMAT, "Format:" },
    { MSG_DLGDBOPEN_LBL_MOLCOMPR, "Compr.:" },
    { MSG_DLGDBOPEN_LBL_MOLTYPE, "Mol. typ:" },
    { MSG_DLGDBOPEN_LBL_NEWDB, "New database:" },
    { MSG_DLGDBOPEN_TITLE, "Open database/s ..." },
    { MSG_GRAPH_OPENFILTER, "All supported files (autodetect)|*.*|"\
	"CHARMM (.ene)|*.ene|"\
	"Comma Separated Values (*.csv)|*.csv|"\
	"Data Interchange Format (*.dif)|*.dif|"\
	"Gromacs (.log)|*.log|"\
	"Quanta Plot (*.plt)|*.plt|"\
	"Custom Format|*.*" },
    { MSG_GRAPH_SAVEFILTER, "Comma Separated Values (*.csv)|*.csv|"\
	"Data Interchange Format (*.dif)|*.dif|"\
	"Quanta Plot (*.plt)|*.plt|"\
	"Raw binary (*.raw)|*.raw" },
    { MSG_ERR_GRAPH_ILLID, "Illegal GraphEd ID" },
    { MSG_ERR_GRAPH_NOCUR, "No GraphEd window opened" },
    { MSG_DLGINTER_CB_RESWORSE, "Worse residues" },
    { MSG_DLGINTER_CB_COLRESENE, "By residue energy" },
    { MSG_DLGMOLPLACE_BT_APPEND, "&Append" },
    { MSG_DLGMOLPLACE_BT_REPLACE, "&Replace" },
    { MSG_DLGMOLPLACE_BT_WORKSPACE, "New &workspace" },
    { MSG_DLGMOLPLACE_CB_SAME, "Same for all molecules" },
    { MSG_DLGMOLPLACE_TITLE, "Molecule placing" },
    { MSG_DLGREMRES_CB_KEEPOPEN, "Keep the window opened" },
    { MSG_DLGREMRES_HC_CHAIN, "Chain" },
    { MSG_DLGREMRES_HC_MOLECULE, "Molecule" },
    { MSG_DLGREMRES_HC_RESNAM, "Residue" },
    { MSG_DLGREMRES_HC_RESSEQ, "Number" },
    { MSG_DLGREMRES_TITLE, "Remove residue(s)" },
    { MSG_GLDEMO_LBLATOM, "Atom name labels" },
    { MSG_GLDEMO_LBLCHG, "Atomic charge labels" },
    { MSG_GLDEMO_LBLELEM, "Element labels" },
    { MSG_GLDEMO_LOADING, "Loading ..." },
    { MSG_GLDEMO_MOL1, "Diazepam" },
    { MSG_GLDEMO_MOL2, "Nicotine" },
    { MSG_GLDEMO_MOL3, "Inosine monophosphate dehydrogenase" },
    { MSG_GLDEMO_MOL3_ATOMS, "8598 atoms" },
    { MSG_GLDEMO_MOL3_VECTORS, "More than 20.000 vectors" },
    { MSG_GLDEMO_MOL3_WATERS, "497 water molecules" },
    { MSG_GLDEMO_SOLVAT, "Fast water solvatation" },
    { MSG_GLDEMO_SONGTITLE, "VEGA ZZ theme \"Path to nowhere\" composed by Anvil" },
    { MSG_GLDEMO_SURFMEP, "MEP surface" },
    { MSG_GLDEMO_SURFVDW, "Van der Waals surface" },
    { MSG_GLDEMO_VIEWCPKSOLID, "Ball & stick" },
    { MSG_GLDEMO_VIEWCPKWIRE, "Ball & stick wireframe" },
    { MSG_GLDEMO_VIEWLIQUORICE, "Stick" },
    { MSG_GLDEMO_VIEWVDWDOT, "CPK dotted" },
    { MSG_GLDEMO_VIEWVDWSOLID, "CPK solid" },
    { MSG_GLDEMO_VIEWVDWWIRE, "CPK wireframe" },
    { MSG_ERR_HYB_ATMNAMEMIS, "Missing atom name at line %d" },
    { MSG_ERR_HYB_ATMNAMETOOLONG, "Atom name too long (max 4 char.) at line %d" },
    { MSG_ERR_HYB_ATMTYPMIS, "Missing atom type at line %d" },
    { MSG_ERR_HYB_ATMTYPTOOLONG, "Atom type too long (max 8 char.) at line %d" },
    { MSG_ERR_HYB_RESNAMETOOLONG, "Residue name too long (max 4 char.) at line %d" },
    { MSG_ERR_HYB_UNKFORMAT, "Unknown template format" },
    { MSG_ERR_HYB_WARNCODEILL, "Illegal warning code at line %d" },
    { MSG_ERR_HYB_WARNCODEMIS, "Missing warning code at line %d" },
    { MSG_ERR_MAIN_MOLTYPE, "Unknown molecule type to add the hydrogens" },
    { MSG_MAIN_MENU_ADDFRAG, "&Fragments" },
    { MSG_MAIN_MENU_ADDHYD, "&Hydrogens" },
    { MSG_MAIN_MENU_MOLECULES, "Mol&ecules" },
    { MSG_MAIN_MENU_RMRES, "&Residue" },
    { MSG_MAIN_MENU_DATABASE, "D&atabase" },
    { MSG_MAIN_MENU_DBEXPLORE, "&Explore" },
    { MSG_MAIN_MENU_DBOPEN, "&Open" },
    { MSG_WKS_CLIP, "Molecule from clipboard" },
    { MSG_WKS_EMPTY, "Empty" },
    { MSG_WKS_REMCONT, "The workspace isn't empty. Do you want continue ?" },
    { MSG_WKS_REMALLCONT, "One or more workspaces aren't empty. Do you want continue ?" },
    { MSG_LOADER_CHEM3D, "Chem3D" },
    { MSG_MOL_FOUND, "* %d molecules found\n" },
    { MSG_MOL_MERGE, "* All molecules merged in one" },
    { MSG_MOL_NOMERGE, "* No molecules to merge" },
    { MSG_MOL_NOTFOUND, "* No molecule found\n" },
    { MSG_SEG_MERGE, "* All segments merged in one" },
    { MSG_SEG_NOMERGE, "* No segments to merge" },
    { MSG_WARN_SOLVATE_SHELLDIM, "The molecule exceeds the shell dimensions. Do you want continue ?" },
    { MSG_BT_DEFAULT, "&Default" },
    { MSG_CHKCHIR_HEADER, "\nList of D-aminoacids:\n"\
	"=====================\n" },
    { MSG_CHKCHIR_NODFOUND, "\nNo D-aminoacid found\n" },
    { MSG_CHKCHIR_WARNING, "Aminoacids signed with * are L but have got a negative zeta angle as D.\n"\
	"With most probability the geometry is distorted.\n\n" },
    { MSG_CHKINTER_HEADER, "\nIntersected rings:\n"\
	"==================\n" },
    { MSG_CHKINTER_NODFOUND, "\nNo intersected ring found\n" },
    { MSG_CHKRESSEQ_TOTINTER, "\nTotal intersected rings: %d\n" },
    { MSG_CHKPEPT_HEADER, "\nList of cis-bonds:\n"\
	"==================\n" },
    { MSG_CHKPEPT_NOCISFOUND, "\nNo cis-bond found\n" },
    { MSG_CHKRESSEQ_HEADER, "\nMissing residues:\n"\
	"=================\n" },
    { MSG_CHKRESSEQ_NOMIS, "\nNo missing residues\n" },
    { MSG_CHKRESSEQ_TOTMIS, "\nTotal missing residues: %d\n" },
    { MSG_ERR_COM_SRFINVNUM, "Invalid surface ID" },
    { MSG_BT_ADVANCED, "&Advanced" },
    { MSG_BT_HIDEALL, "&Hide all" },
    { MSG_BT_LOAD, "&Load" },
    { MSG_BT_REMALL, "Re&move all" },
    { MSG_BT_SAVE, "&Save" },
    { MSG_BT_SHOWALL, "Sh&ow all" },
    { MSG_LBL_NO, "No" },
    { MSG_LBL_YES, "Yes" },
    { MSG_OPEN_FILTER, "All supported files (*.*)\000*.*\000" },
    { MSG_OPEN_FILTER_TEXT, "Text files (*.txt)|*.txt|"\
	"All files (*.*)|*.*" },
    { MSG_DLGACT_BT_COPY, "&Copy to clip." },
    { MSG_DLGACT_BT_CONNECT, "Click &here !" },
    { MSG_DLGACT_LBL_MES0, "Your copy of VEGA isn't operative, but this isn't a problem because"\
	" you can follow a simple activation procedure that it's totally FREE." },
    { MSG_DLGACT_LBL_MES1, "Connect to the DDL registration server:" },
    { MSG_DLGACT_LBL_MES2, "Type the product key in the form (copy and paste):" },
    { MSG_DLGACT_LBL_MES3, "Fill the other requested fields." },
    { MSG_DLGACT_LBL_MES4, "When you receive the confirmation e-mail, download the license"\
	" file and open or copy it to the following directory:" },
    { MSG_DLGACT_LBL_MES5, "Congratulations ! Now you can start your personal copy of the VEGA"\
	" package. Please remember that if you find problems in the activation"\
	" procedure, you can contact the Authors as reported in the manual." },
    { MSG_DLGACT_LICMES, "Licensed to:\n%s\n\n" },
    { MSG_DLGACT_TITLE, "VEGA ZZ activation" },
    { MSG_ERR_DLGACT_EXPIRED, "Sorry, the license is expired" },
    { MSG_ERR_DLGACT_INVCRC, "Invalid activation key CRC" },
    { MSG_ERR_DLGACT_INVHOST, "Invalid activation key host" },
    { MSG_ERR_DLGACT_INVSIGN, "Invalid activation key signature" },
    { MSG_ANA_CB_GYRRAD, "Gyration radius" },
    { MSG_DLGATMADD_RG_PLACE, "Where to place in the atom list" },
    { MSG_DLGATMADD_RG_PLACEATM, "After the selected atom" },
    { MSG_DLGATMADD_RG_PLACEMOL, "At the molecule end" },
    { MSG_DLGATMADD_RG_PLACERES, "At the residue end" },
    { MSG_DLGADDFRAG_WKS_NAME, "Fragment" },
    { MSG_ATMSEL_CB_COLOR, "Change color" },
    { MSG_ATMSEL_CB_DISNOCHANGE, "Don't change" },
    { MSG_ATMSEL_GRP_DISPLAY, "Display" },
    { MSG_DLGBOND_LBL_SELANGLE, "Selecte bonds:" },
    { MSG_DLGBOND_TS_SWAP, "&Swap" },
    { MSG_DLGCLUSTER_SRFNAME, "Cluster shape" },
    { MSG_DLGCLUSTER_CB_SHOWSHAPE, "Show the shape" },
    { MSG_DLGCOLORSET_CB_LOGOP_BOTTOMC, "Bottom center" },
    { MSG_DLGCOLORSET_CB_LOGOP_BOTTOML, "Bottom left" },
    { MSG_DLGCOLORSET_CB_LOGOP_BOTTOMR, "Bottom right" },
    { MSG_DLGCOLORSET_CB_LOGOP_CENTERC, "Centered" },
    { MSG_DLGCOLORSET_CB_LOGOP_CENTERL, "Center left" },
    { MSG_DLGCOLORSET_CB_LOGOP_CENTERR, "Center right" },
    { MSG_DLGCOLORSET_CB_LOGOP_NONE, "None" },
    { MSG_DLGCOLORSET_CB_LOGOP_TOPC, "Top center" },
    { MSG_DLGCOLORSET_CB_LOGOP_TOPL, "Top left" },
    { MSG_DLGCOLORSET_CB_LOGOP_TOPR, "Top right" },
    { MSG_DLGCOLORSET_CB_SKYENABLED, "Enabled" },
    { MSG_DLGCOLORSET_GRP_ATMLABELS, "Labels" },
    { MSG_DLGCOLORSET_GRP_SKYVISION, "SkyVision" },
    { MSG_DLGCOLORSET_LBL_ATMLABELS, "Atom:" },
    { MSG_DLGCOLORSET_LBL_ATMUNKPOT, "Unknown pot.:" },
    { MSG_DLGCOLORSET_LBL_BACKCOL, "Background color:" },
    { MSG_DLGCOLORSET_LBL_COLANGLE, "Angle:" },
    { MSG_DLGCOLORSET_LBL_COLBR, "Bromine:" },
    { MSG_DLGCOLORSET_LBL_COLC, "Carbon:" },
    { MSG_DLGCOLORSET_LBL_COLCL, "Chlorine:" },
    { MSG_DLGCOLORSET_LBL_COLDIST, "Distance:" },
    { MSG_DLGCOLORSET_LBL_COLF, "Fluorine:" },
    { MSG_DLGCOLORSET_LBL_COLGEN, "Generic:" },
    { MSG_DLGCOLORSET_LBL_COLH, "Hydrogen:" },
    { MSG_DLGCOLORSET_LBL_COLI, "Iodine:" },
    { MSG_DLGCOLORSET_LBL_COLN, "Nitrogen:" },
    { MSG_DLGCOLORSET_LBL_COLO, "Oxygen:" },
    { MSG_DLGCOLORSET_LBL_COLP, "Phosph.:" },
    { MSG_DLGCOLORSET_LBL_COLPLANG, "Plane angle:" },
    { MSG_DLGCOLORSET_LBL_COLPLANGCUBE, "Plane ang. cube:" },
    { MSG_DLGCOLORSET_LBL_COLS, "Sulfur:" },
    { MSG_DLGCOLORSET_LBL_COLSHAPE, "Shape:" },
    { MSG_DLGCOLORSET_LBL_COLSUPLINE, "Supp. line:" },
    { MSG_DLGCOLORSET_LBL_COLTEXT, "Text:" },
    { MSG_DLGCOLORSET_LBL_COLTOR, "Torsion:" },
    { MSG_DLGCOLORSET_LBL_COLXC, "Centroid:" },
    { MSG_DLGCOLORSET_LBL_LOGO, "Logo:" },
    { MSG_DLGCOLORSET_LBL_SKYOFFSETX, "X offset:" },
    { MSG_DLGCOLORSET_LBL_SKYOFFSETY, "Y offset:" },
    { MSG_DLGCOLORSET_LBL_SKYROTX, "X rotation:" },
    { MSG_DLGCOLORSET_LBL_SKYROTY, "Y rotation:" },
    { MSG_DLGCOLORSET_LBL_SKYROTZ, "Z rotation:" },
    { MSG_DLGCOLORSET_LBL_SKYTEX, "Texture" },
    { MSG_DLGCOLORSET_TITLE, "Color settings" },
    { MSG_DLGCOLORSET_TS_ATOMS, "Atoms" },
    { MSG_DLGCOLORSET_TS_BACKGROUND, "Background" },
    { MSG_DLGCOLORSET_TS_MONITORS, "Monitors" },
    { MSG_DLGCOLORSET_TS_RESIDUES, "Residues" },
    { MSG_DLGDBEXP_MOLNAME, "Molecule name:" },
    { MSG_DLGENETYPE_ANGLE, "Angle energy" },
    { MSG_DLGENETYPE_BOND, "Bond energy" },
    { MSG_DLGENETYPE_BOUNDARY, "Boundary energy" },
    { MSG_DLGENETYPE_DIHEDRAL, "Dihedral" },
    { MSG_DLGENETYPE_ELECTROSTATIC, "Electrostatic energy" },
    { MSG_DLGENETYPE_IMPROPER, "Improper energy" },
    { MSG_DLGENETYPE_KINETIC, "Kinetic energy" },
    { MSG_DLGENETYPE_MISC, "Misc energy" },
    { MSG_DLGENETYPE_POTENTIAL, "Potential energy" },
    { MSG_DLGENETYPE_TEMPERATURE, "Temperature" },
    { MSG_DLGENETYPE_TITLE, "Select plot type" },
    { MSG_DLGENETYPE_TOTAL, "Total energy" },
    { MSG_DLGENETYPE_VANDERWAALS, "Van der Waals energy" },
    { MSG_DLGFIXATM_RG_SELINVISATM, "Invisible atoms" },
    { MSG_DLGIMGSAVE_CB_RES_CUSTOM, "Custom" },
    { MSG_DLGIMGSAVE_CK_EMBOSS, "Emboss" },
    { MSG_DLGIMGSAVE_CK_GAUSSBLUR, "Gaussian blur" },
    { MSG_DLGIMGSAVE_CK_GRAYSCALE, "Gray scale" },
    { MSG_DLGIMGSAVE_GRP_EFFECTS, "Effects" },
    { MSG_DLGIMGSAVE_GRP_RESOLUTION, "Resolution" },
    { MSG_DLGIMGSAVE_RG_RMODE, "Render mode" },
    { MSG_DLGIMGSAVE_RG_RMODE_HARD, "Hardware" },
    { MSG_DLGIMGSAVE_RG_RMODE_SNAP, "Snapshot" },
    { MSG_DLGIMGSAVE_RG_RMODE_SOFT, "Software" },
    { MSG_LIGHT_CB_AMBIENT, "Enabled" },
    { MSG_LIGHT_CB_LIGHTEN, "Enabled" },
    { MSG_LIGHT_CB_LIGHTINGEN, "Lighting enabled" },
    { MSG_LIGHT_CB_VECTORSPEC, "Vector reflection" },
    { MSG_LIGHT_GRP_AMBIENT, "Ambient light" },
    { MSG_LIGHT_GRP_COLORS, "Light colors" },
    { MSG_LIGHT_GRP_MATERIAL, "Material properties" },
    { MSG_LIGHT_GRP_POSITION, "Position" },
    { MSG_LIGHT_LBL_AMBIENTCOL, "Color:" },
    { MSG_LIGHT_LBL_DIFFUSECOL, "Diffuse:" },
    { MSG_LIGHT_LBL_MATSHINY, "Shiny:" },
    { MSG_LIGHT_LBL_MATSPECULAR, "Specular:" },
    { MSG_LIGHT_LBL_SPECULARCOL, "Specular:" },
    { MSG_LIGHT_RG_LIGHTS, "Sources" },
    { MSG_LIGHT_RG_LIGHT, "Light" },
    { MSG_LIGHT_TITLE, "Lighting" },
    { MSG_DLGMAIN_OD_FILTER, "Alchemy (*.alc;*.mol)\000*.alc;*.mol\000"\
	"Amber (*.mdcrd)\000*.mdcrd\000"\
	"Amber NetCDF (*.nc)\000*.nc\000"\
	"AMMP (*.amp;*.ammp)\000*.amp;*.ammp\000"\
	"AutoDock 4 DLG (*.dlg)\000*.dlg\000"\
	"AutoDock 4/Vina PDBQT (*.pdbqt)\000*.pdbqt\000"\
	"BioDock output (*.out)\000*.out\000"\
	"CHARMM (*.crd)\000*.CRD\000"\
	"CHARMM/NAMD trajectory (*.dcd)\000*.dcd\000"\
	"Chem3D (*.c3d;*.cc1;*.cc2)\000*.c3d;*.cc1;*.cc2\000"\
	"ChemDraw (*.cdx)\000*.cdx\000"\
	"ChemSol (*.cs)\000*.cs\000"\
	"CIF/mmCIF (*.cif)\000*.cif\000"\
	"CML (*.cml;*.xml)\000*.cml;*.xml\000"\
	"CPMD XYZ (*.xyz)\000*.xyz\000"\
	"Cssr (*.cssr;*.csr)\000*.cssr;*.csr\000"\
	"CSV surface (*.csv)\000*.csv\000"\
	"CRT (*.crt)\000*.crt\000"\
	"Escher NG (*.sol)\000*.sol\000"\
	"Fasta (*.fas;*.seq;*.txt)\000*.fas;*.seq;*.txt\000"\
	"Gaussian input (*.gjf)\000*.gjf\000"\
	"Gaussian output (*.log)\000*.log\000"\
	"Gamess (*.inp)\000*.inp\000"\
	"GRAMM (*.res)\000*.res\000"\
	"Gromacs/Gromos (*.gro)\000*.gro\000"\
	"Gromacs TRR trajectory (*.trr)\000*.trr\000"\
	"Gromacs XTC trajectory (*.xtc)\000*.xtc\000"\
	"HyperChem (*.hin)\000*.hin\000"\
	"IFF/RIFF (*.iff)\000*.iff\000"\
	"LiGen (*.bin;*.txt)\000*.bin;*.txt\000"\
	"InChI (*.inchi;*.inc)\000*.inchi;*.inc\000"\
	"Insight (*.car;*.cor)\000*.car;*.cor\000"\
	"Insight archive (*.arc)\000*.arc\000"\
	"Insight surface (*.srf)\000*.srf\000"\
	"MDL Mol (*.mol)\000*.mol\000"\
	"Mol2 (*.ml2;*.mol;*.mol2)\000*.ml2;*.mol;*.mol2\000"\
	"Mopac (*.arc;*.dat)\000*.arc;*.dat\000"\
	"NAMD binary (*.coor)\000*.coor\000"\
	"PDB (*.pdb;*.ent;*.pdba;*.pdbf;*.pdbl;*.pqr)\000*.pdb;*.ent;*.pdba;*.pdbf;*.pdbl;*.pqr\000"\
	"PQR XML (*.xml)\000*.pqr\000"\
	"Quanta CSR (*.csr)\000*.csr\000"\
	"Quanta MSF (*.msf)\000*.msf\000"\
	"Quanta surface (*.srf)\000*.srf\000"\
	"QMC (*.qmc)\000*.qmc\000"\
	"Spillo RBS (*.pdb)\000*.pdb\000"\
	"TINKER XYZ (*.xyz)\000*.xyz\000"\
	"X-Plor PSF (*.psf)\000*.psf\000"\
	"XYZ (*.xyz)\000*.xyz\000"\
	"Compressed files (*.bz2;*.gz;*.pp;*.Z)\000*.bz2;*.gz;*.pp;*.Z\000"\
	"All files (*.*)\000*.*\000" },
    { MSG_DLGMAIN_OD_FILTERMRG, "Alchemy (*.alc;*.mol)\000*.alc;*.mol\000"\
	"AMMP (*.amp;*.ammp)\000*.amp;*.ammp\000"\
	"AutoDock 4/Vina PDBQT (*.pdbqt)\000*.pdbqt\000"\
	"CHARMM (*.crd)\000*.CRD\000"\
	"Chem3D (*.c3d;*.cc1;*.cc2)\000*.c3d;*.cc1;*.cc2\000"\
	"ChemSol (*.cs)\000*.cs\000"\
	"CIF/mmCIF (*.cif)\000*.cif\000"\
	"CML (*.cml;*.xml)\000*.cml;*.xml\000"\
	"CPMD XYZ (*.xyz)\000*.xyz\000"\
	"Cssr (*.cssr;*.csr)\000*.cssr;*.csr\000"\
	"CRT (*.crt)\000*.crt\000"\
	"Fasta (*.fas;*.seq;*.txt)\000*.fas;*.seq;*.txt\000"\
	"Gamess (*.inp)\000*.inp\000"\
	"Gaussian input (*.gjf)\000*.gjf\000"\
	"Gaussian output (*.log)\000*.log\000"\
	"Gromacs/Gromos (*.gro)\000*.gro\000"\
	"HyperChem (*.hin)\000*.hin\000"\
	"IFF/RIFF (*.iff)\000*.iff\000"\
	"Insight (*.car;*.cor)\000*.car;*.cor\000"\
	"MDL Mol (*.mol)\000*.mol\000"\
	"Mol2 (*.ml2;*.mol;*.mol2)\000*.ml2;*.mol;*.mol2\000"\
	"Mopac (*.arc;*.dat)\000*.arc;*.dat\000"\
	"NAMD binary (*.coor)\000*.coor\000"\
	"PDB (*.pdb;*.ent;*.pdba;*.pdbf;*.pdbl;*.pqr)\000*.pdb;*.ent;*.pdba;*.pdbf;*.pdbl;*.pqr\000"\
	"PQR XML (*.xml)\000*.xml\000"\
	"Quanta CSR (*.csr)\000*.csr\000"\
	"Quanta MSF (*.msf)\000*.msf\000"\
	"QMC (*.qmc)\000*.qmc\000"\
	"Spillo RBS (*.pdb)\000*.pdb\000"\
	"TINKER XYZ (*.xyz)\000*.xyz\000"\
	"X-Plor PSF (*.psf)\000*.psf\000"\
	"XYZ (*.xyz)\000*.xyz\000"\
	"Compressed files (*.bz2;*.gz;*.pp;*.Z)\000*.bz2;*.gz;*.pp;*.Z\000"\
	"All files (*.*)\000*.*\000" },
    { MSG_DLGMAIN_MOVE_ALL, "All" },
    { MSG_DLGMAIN_MOVE_MOL, "Molecule" },
    { MSG_DLGMAIN_MOVE_SEG, "Segment" },
    { MSG_DLGMAIN_MOVE_RES, "Residue" },
    { MSG_DLGMAIN_SB_ANGLE, "Angle" },
    { MSG_DLGMAIN_SB_ATOM, "Atom" },
    { MSG_DLGMAIN_SB_CENTER, "Center" },
    { MSG_DLGMAIN_SB_DISTANCE, "Distance" },
    { MSG_DLGMAIN_SB_PLANG, "Plane angle" },
    { MSG_DLGMAIN_SB_ROTATE, "Rotate" },
    { MSG_DLGMAIN_SB_SCALE, "Scale" },
    { MSG_DLGMAIN_SB_TORSION, "Torsion" },
    { MSG_DLGMAIN_SB_TRANSLATE, "Translate" },
    { MSG_DLGMAIN_SD_FILTERBMP, "Bitmap (*.bmp)|*.bmp|"\
	"Gif (*.gif)|*.gif|"\
	"Encapsulated PostScript (*.eps)|*.eps|"\
	"Enhanced Metafile (*.emf)|*.emf|"\
	"Jpeg (*.jpg;*.jpe;*.jpeg)|*.jpg;*.jpe;*.jpeg|"\
	"LaTex (*.tex)|*.tex|"\
	"PCX (*.pcx)|*.pcx|"\
	"PDF (*.pdf)|*.pdf|"\
	"Portable Network Graphics (*.png)|*.png|"\
	"PostScript (*.ps)|*.ps|"\
	"POV-Ray (*.pov)|*.pov|"\
	"Pnm (*.pnm;*.pbm;*.pgm;*.ppm)|*.ppm;*.pbm;*.pgm;*.pnm|"\
	"Raw Data (*.raw)|*.raw|"\
	"Silicon Graphics (*.sgi)|*.sgi|"\
	"STL (*.stl)|*.stl|"\
	"STL binary (*.stb)|*.stb|"\
	"SVG (*.svg)|*.svg|"\
	"Targa (*.tga)|*.tga|"\
	"Tiff (*.tif)|*.tif|"\
	"VRML (*.wrl)|*.wrl|"\
	"VRML compressed (*.wrz)|*.wrz" },
    { MSG_DLGMAIN_SD_FILTERSAVE, "PDB 2.2 (*.pdb)|*.pdb|"\
	"PDB (*.pdb)|*.pdb|"\
	"PDB non-std (*.pdb)|*.pdb|"\
	"PDB Atdl (*.pdb;*.pdba)|*.pdb;*.pdba|"\
	"PDB Fat (*.pdb;*.pdbf)|*.pdb;*.pdbf|"\
	"PDB Large (*.pdb;*.pdbl)|*.pdb;*.pdbl|"\
	"PDB OpenMM (*.pdb)|*.pdb|"\
	"PDBQ (*.pdb)|*.pdb|"\
	"PQR (*.pqr)|*.pqr|"\
	"PQR XML (*.xml)|*.xml|"\
	"Insight (*.car)|*.car;*.cor|"\
	"Old Insight (*.car)|*.car;*.cor|"\
	"Alchemy (*.alc;*.mol)|*.alc;*.mol|"\
	"AMMP (*.amp)|*.amp|"\
	"AutoDock 4 PDBQT (*.pdbqt)|*.pdbqt|"\
	"Autodock Vina PDBQT (*.pdbqt)|*.pdbqt|"\
	"ChemSol (*.cs)|*.cs|"\
	"CIF (*.cif)|*.cif|"\
	"mmCIF (*.cif)|*.cif|"\
	"CML 2.0 (*.cml)|*.cml|"\
	"CML (*.cml)|*.cml|"\
	"CPMD XYZ (*.xyz)|*.xyz|"\
	"CRD (*.crd)|*.crd|"\
	"CRT (*.crt)|*.crt|"\
	"Cssr (*.cssr;*.csr)|*.cssr;*.csr|"\
	"Fasta (*.fas)|*.fas|"\
	"Gamess (*.inp)|*.inp|"\
	"Gaussian cartesian (*.gjf)|*.gjf|"\
	"Gromacs/Gromos (*.gro)|*.gro|"\
	"Gromacs/Gromos nm (*.gro)|*.gro|"\
	"IFF/RIFF (*.iff)|*.iff|"\
	"InChI (*.inchi;*.inc)|*.inchi;*.inc|"\
	"InChI + Aux (*.inchi;*.inc)|*.inchi;*.inc|"\
	"InChIKey (*.ikey)|*.ikey|"\
	"Indigo layered code (*.indigo)|*.indigo|"\
	"Info XML (*.xml)|*.xml|"\
	"MDL Mol (*.mol)|*.mol|"\
	"MDL Mol V3000 (*.mol)|*.mol|"\
	"Mol2 (*.mol2;*.mol;*.ml2)|*.mol2;*.mol;*.ml2|"\
	"Mopac (*.dat;*.arc)|*.dat;*.arc|"\
	"Mopac cartesian (*.dat;*.arc)|*.dat;*.arc|"\
	"NAMD binary (*.coor)|*.coor|"\
	"Quanta MSF (*.msf)|*.msf|"\
	"QMC (*.qmc)|*.qmc|"\
	"SMILES (*.smi)|*.smi|"\
	"Spillo RBS (*.pdb)|*.pdb|"\
	"X-Plor PSF (*.psf)|*.psf|"\
	"XYZ (*.xyz)|*.xyz" },
    { MSG_DLGMAIN_SD_FILTERSRF, "Quanta (*.srf)|*.srf|"\
	"Comma Separated Values (*.csv)|*.csv|"\
	"IFF/RIFF (*.iff)|*.iff|"\
	"Insight (*.srf)|*.srf|"\
	"Raw binary (*.raw)|*.raw" },
    { MSG_DLGMAIN_TBT_MENU_TITLE, "Main menu" },
    { MSG_DLGMAIN_TBT_STANDARD_TITLE, "Standard" },
    { MSG_DLGMAIN_TBT_TOOLS_TITLE, "Tools" },
    { MSG_DLGMINIED_OD_FILTER, "All files (*.*)|*.*|"\
	"Text files (*.txt)|*.txt" },
    { MSG_DLGMINIED_OD_TITLE, "Open text file" },
    { MSG_DLGMINIED_SD_FILTER, "Text files (*.txt)|*.txt|"\
	"All files (*.*)|*.*" },
    { MSG_DLGMINIED_SD_TITLE, "Save text file" },
    { MSG_MOLSAVE_GRP_OPTIONS, "Options" },
    { MSG_DLGMVRES_CONNRES, "\n* Controls connected to residue %s\n" },
    { MSG_DLGMVRES_TITLE, "Move residue" },
    { MSG_DLGMVSEG_CONNSEG, "\n* Controls connected to segment %d\n" },
    { MSG_DLGMVSEG_TITLE, "Move segment" },
    { MSG_DLGPROGBAR_WAIT, "Halting the calculation ..." },
    { MSG_DLGRMMOL_CB_VISSEL, "Highlight the selection" },
    { MSG_DLGRMMOL_HC_RESEND, "Last res." },
    { MSG_DLGRMMOL_HC_RESSTART, "First res." },
    { MSG_DLGRMMOL_LBL_SELECTED, "Selected" },
    { MSG_DLGRMMOL_LBL_UNSELECTED, "Unselected" },
    { MSG_SEL_TBI_TORALL, "&All" },
    { MSG_SEL_TBI_TORFLEX, "&Flexible" },
    { MSG_DLGSRFCLC_BT_COLGRADRANGE, "Fill range" },
    { MSG_DLGSRFCLC_CB_COLBYGRAD, "Color by gradient" },
    { MSG_DLGSRFCLC_CB_COLGRADRANGE, "Auto range" },
    { MSG_DLGSRFCLC_CB_DEFOGLLISTS, "Use OpenGL lists (more rendering speed)" },
    { MSG_DLGSRFCLC_CB_DEFTRANSEN, "Solid transparent surfaces" },
    { MSG_DLGSRFCLC_CB_OGLLIST, "Use OpenGL list" },
    { MSG_DLGSRFCLC_CB_TRANSEN, "Enable transparency" },
    { MSG_DLGSRFCLC_HC_SRFNAME, "Name" },
    { MSG_DLGSRFCLC_HC_SRFNUM, "Num." },
    { MSG_DLGSRFCLC_HC_SRFPOINTS, "Points" },
    { MSG_DLGSRFCLC_HC_SRFVIS, "Visib." },
    { MSG_DLGSRFCLC_LBL_COLCUST, "Custom color:" },
    { MSG_DLGSRFCLC_LBL_COLGRADNUM, "Colors:" },
    { MSG_DLGSRFCLC_LBL_DEFCOLOR, "Surface color" },
    { MSG_DLGSRFCLC_LBL_DOTSIZE, "Dot size:" },
    { MSG_DLGSRFCLC_LBL_MESHSIZE, "Mesh size:" },
    { MSG_DLGSRFCLC_LBL_TRANSVAL, "Transparency:" },
    { MSG_DLGSRFCLC_NEWSRFNAM, "New surface name:" },
    { MSG_DLGSRFCLC_PM_COLGRADINV, "&Invert" },
    { MSG_DLGSRFCLC_PM_COLGRADSHL, "Shift &left" },
    { MSG_DLGSRFCLC_PM_COLGRADSHR, "Shift &right" },
    { MSG_DLGSRFCLC_PM_COLGRADPRESETS, "&Presets" },
    { MSG_DLGSRFCLC_PM_COLGRADDEF, "Default" },
    { MSG_DLGSRFCLC_PM_SRFHIDE, "&Hide" },
    { MSG_DLGSRFCLC_PM_SRFREMOVE, "R&emove" },
    { MSG_DLGSRFCLC_PM_SRFREN, "&Rename" },
    { MSG_DLGSRFCLC_PM_SRFSHOW, "&Show" },
    { MSG_DLGSRFCLC_RG_COLMET, "Method" },
    { MSG_DLGSRFCLC_RG_COLMETATM, "Atom" },
    { MSG_DLGSRFCLC_RG_COLMETCHAIN, "Chain" },
    { MSG_DLGSRFCLC_RG_COLMETCUST, "Custom" },
    { MSG_DLGSRFCLC_RG_COLMETMOL, "Molecule" },
    { MSG_DLGSRFCLC_RG_COLMETRES, "Residue" },
    { MSG_DLGSRFCLC_RG_COLMETSEG, "Segment" },
    { MSG_DLGSRFCLC_RG_COLMETSRF, "Surface" },
    { MSG_DLGSRFCLC_RG_VISMODDOT, "Dots" },
    { MSG_DLGSRFCLC_RG_VISMODMESH, "Mesh" },
    { MSG_DLGSRFCLC_RG_VISMODSOLID, "Solid" },
    { MSG_DLGSRFCLC_TS_COLOR, "Color" },
    { MSG_DLGSRFCLC_TS_GRADIENT, "Gradient" },
    { MSG_DLGSRFCLC_TS_NEW, "New" },
    { MSG_DLGSRFCLC_TS_SETTINGS, "Settings" },
    { MSG_DLGSRFCLC_TS_TRANS, "Transparency" },
    { MSG_DLGTOR_LBL_ANGVAL, "Angle value (degree):" },
    { MSG_DLGTRJADV_BT_SETFRMCUR, "&Set current" },
    { MSG_DLGTRJADV_GRP_FILENAME, "Reference file name" },
    { MSG_DLGTRJADV_GRP_REFFRAME, "Reference frame" },
    { MSG_DLGTRJADV_RG_RMSDFRAME, "Trajectory frame" },
    { MSG_DLGTRJADV_RG_RMSDMODE, "RMSD type" },
    { MSG_DLGTRJADV_RG_RMSDMOLFILE, "Molecule file" },
    { MSG_DLGTRJADV_RG_RMSDPREV, "Previous frame (velocity)" },
    { MSG_DLGTRJADV_TITLE, "RMSD advanced options" },
    { MSG_VIEWPREFS_ANTIALIAS, "Vector antialiasing" },
    { MSG_VIEWPREFS_CB_FONTTYPEBMP, "Bitmap" },
    { MSG_VIEWPREFS_CB_FONTTYPESTROKE, "Stroke" },
    { MSG_VIEWPREFS_CB_FONTTYPETT, "True Type" },
    { MSG_VIEWPREFS_CB_STEREOACTIVEGLASSES, "Active glasses" },
    { MSG_VIEWPREFS_CB_STEREOANAFBLUERED, "Red blue" },
    { MSG_VIEWPREFS_CB_STEREOANAFCYANRED, "Cyan red" },
    { MSG_VIEWPREFS_CB_STEREOANAFGREENRED, "Green red" },
    { MSG_VIEWPREFS_CB_STEREOANAFREDBLUE, "Red blue" },
    { MSG_VIEWPREFS_CB_STEREOANAFREDCYAN, "Red cyan" },
    { MSG_VIEWPREFS_CB_STEREOANAFREDGREEN, "Red green" },
    { MSG_VIEWPREFS_CB_STEREOANAGLYPH, "Anaglyph" },
    { MSG_VIEWPREFS_CB_STEREONONE, "None" },
    { MSG_VIEWPREFS_CB_TUBEBIQUAD, "Biquadratic interpolation" },
    { MSG_VIEWPREFS_DEPTHCUE, "Depth cueing" },
    { MSG_VIEWPREFS_LBL_FONT, "Font:" },
    { MSG_VIEWPREFS_LBL_FONTRES, "Resolution:" },
    { MSG_VIEWPREFS_LBL_FONTSIZE, "Size:" },
    { MSG_VIEWPREFS_LBL_FONTSIZETT, "Scale:" },
    { MSG_VIEWPREFS_LBL_FONTTYPE, "Type:" },
    { MSG_VIEWPREFS_LBL_SPLINERES, "Spline res.:" },
    { MSG_VIEWPREFS_LBL_STEREOANAFILTER, "Anaglyph filter:" },
    { MSG_VIEWPREFS_LBL_STEREOEYEADJUST, "Shift adjust:" },
    { MSG_VIEWPREFS_LBL_STEREOEYEOFFSET, "Eye offset:" },
    { MSG_VIEWPREFS_LBL_STEREOMODE, "Mode:" },
    { MSG_VIEWPREFS_PGFONTS, "Fonts" },
    { MSG_VIEWPREFS_PGMAIN, "Main" },
    { MSG_VIEWPREFS_PGSTEREO, "Stereo" },
    { MSG_VIEWPREFS_PGTRACE, "Trace" },
    { MSG_VIEWPREFS_PGTUBE, "Tube" },
    { MSG_VIEWPREFS_RESTART, "In order to apply the changes, VEGA must"\
	"be restarted.\n"\
	"Do you want proceed ?" },
    { MSG_ERR_DCD_RMSDNOSRF, "Surface file not allowed as reference structure" },
    { MSG_ERR_DCD_RMSDNOTRAJ, "Trajectory file not allowed as reference structure" },
    { MSG_DCD_ERR_RMSDUNCOMPMOL, "Uncompatible reference structure" },
    { MSG_MENU_SAVESURF, "Save surface" },
    { MSG_MLP_DOTONLY, "  Virtual logP.......: Available for dotted surface with 1.5  probe radius only\n" },
    { MSG_MAIN_MENU_PROCHECK, "&Protein check" },
    { MSG_MAIN_MENU_CHKAACHIR, "&AA chirality" },
    { MSG_MAIN_MENU_PEPTBOND, "Peptidic bonds" },
    { MSG_MAIN_MENU_PROMISRES, "&Missing res." },
    { MSG_MAIN_MENU_RINGINTER, "&Ring inter." },
    { MSG_MAIN_MENU_SWAPBONDS, "&Swap bonds" },
    { MSG_MAIN_MENU_DSPLIQ, "S&tick wire" },
    { MSG_MAIN_MENU_DSPTRACES, "Tr&ace" },
    { MSG_MAIN_MENU_DSPTUBES, "T&ube" },
    { MSG_ERR_PBUFFER_CANTALLOCATE, "Can't allocate the pixel buffer.\n"\
	"Insufficient graphic memory for the\n"\
	"selected resolution" },
    { MSG_ERR_PBUFFER_CANTINIT, "Can't initialize the bitmap rendering" },
    { MSG_ERR_PBUFFER_EXT, "Unable to get the address of one or\n"\
	"more OpenGL extensions" },
    { MSG_ERR_PBUFFER_NOOGLEXTENSIONS, "OpenGL extensions not supported by\n"\
	"your graphic card" },
    { MSG_ERR_PBUFFER_NOPIXELFORMAT, "Sorry, your graphic card doesn't support\n"\
	"the WGL_ARB_pixel_format extension" },
    { MSG_ERR_PBUFFER_NOTAVAIL, "Sorry, your graphic card doesn't support\n"\
	"the WGL_ARB_pbuffer extension" },
    { MSG_ERR_PBUFFER_UNKPROB, "Unknown problem during the pixel\n"\
	"buffer initialization" },
    { MSG_POPUP_MENU_MOVESEG, "&Segment" },
    { MSG_POPUP_MENU_MOVERES, "&Residue" },
    { MSG_GLPRINT_JOB, "VEGA ZZ print job" },
    { MSG_LOADER_CRT, "IUMSC CRT" },
    { MSG_ERR_MAIN_MDLMOL, "This format doesn't support more than 999 atoms" },
    { MSG_MAIN_MENU_ADDCHAINS, "&Side chains" },
    { MSG_MAIN_MENU_HLPCHKUPD, "Check for &updates" },
    { MSG_MAIN_MENU_TBSTANDARD, "&Standard" },
    { MSG_MAIN_MENU_TBTOOLS, "&Tools" },
    { MSG_MAIN_MENU_TOOLBARS, "&Tool bars" },
    { MSG_WARN_MAIN_STEREO, "The graphic card doesn't support active stereo view" },
    { MSG_MOL2_MOREMOL, "  The file contains more than one molecule. It will be loaded the first one.\n" },
    { MSG_SURFACE_BUILDDEEP, "* Building deep surface" },
    { MSG_TBTOOLS_ADDATM, "Add atoms" },
    { MSG_TBTOOLS_ADDFRAG, "Add fragments" },
    { MSG_TBTOOLS_ANALYSIS, "Trajectory analysis" },
    { MSG_TBTOOLS_CALCCHRG, "Charges & potential" },
    { MSG_TBTOOLS_CALCMOPAC, "Run Mopac" },
    { MSG_TBTOOLS_CALCSRF, "Surface calculation" },
    { MSG_TBTOOLS_EDITATM, "Edit atom/residue/chain" },
    { MSG_TBTOOLS_SELCUSTOM, "Custom selection" },
    { MSG_ATMSEL_LBL_SLICEPOS, "Position:" },
    { MSG_ATMSEL_LBL_SLICETHICK, "Thickness:" },
    { MSG_ATMSEL_TS_SLICE, "&Slice" },
    { MSG_ATMSEL_RG_SLICEPLANE, "Plane" },
    { MSG_GRAPH_EXCEL_STEP, "Step" },
    { MSG_GRAPH_EXCEL_VALUE, "Value" },
    { MSG_GRAPH_MM_EXCEL, "&Export to Excel" },
    { MSG_ERR_GRAPH_EXCEL, "Unable to export to Excel" },
    { MSG_ERR_GRAPH_EXCEL_TOOMANYREC, "Too many records to export to Excel (max 65535)" },
    { MSG_ERR_GRAPH_OLE, "Unable to initialize the OLE interface" },
    { MSG_MAIN_MENU_EXCEL, "Export to E&xcel" },
    { MSG_EXCEL_EXPORT, "Exporting atoms ..." },
    { MSG_EXCEL_INIT, "Initializing ..." },
    { MSG_EXCEL_MOLLBL_ATOM, "Atom" },
    { MSG_EXCEL_MOLLBL_CHAIN, "Chain" },
    { MSG_EXCEL_MOLLBL_CHARGE, "Charge" },
    { MSG_EXCEL_MOLLBL_CONSTRINT, "Constraint" },
    { MSG_EXCEL_MOLLBL_ELEMENT, "Element" },
    { MSG_EXCEL_MOLLBL_NUMBER, "Number" },
    { MSG_EXCEL_MOLLBL_POTENTIAL, "Potential" },
    { MSG_EXCEL_MOLLBL_RESIDUE, "Residue" },
    { MSG_EXCEL_MOLLBL_RESNUM, "Residue #" },
    { MSG_BT_APPEND, "&Append" },
    { MSG_BT_FIND, "&Find" },
    { MSG_BT_OVERWRITE, "&Overwrite" },
    { MSG_BT_SYNCHRONIZE, "S&ynchronize" },
    { MSG_ERR_AVI_ADD, "Can't add a new frame to the video file." },
    { MSG_ERR_AVI_CREATE, "Can't create the video file." },
    { MSG_ERR_AVI_SET, "Unable to use the selected codec." },
    { MSG_DCD_SAVEPROG, "* Saving the trajectory" },
    { MSG_ERR_DCD_SAMEFILE, "The source and the destination trajectory can't be the same" },
    { MSG_DLGADDFRAG_CB_INHERITH, "Inherit residue name" },
    { MSG_ANA_CB_SHOWFRAME, "Show &frame" },
    { MSG_DLGCODEC_CB_ASPRATIO, "4/3 aspect ratio" },
    { MSG_DLGCODEC_DEPTHS, "Valid depths:" },
    { MSG_DLGCODEC_DEPTHWARN, "* It doesn't support 32 bit depth required by Snapshot and Hardware rendering modes" },
    { MSG_DLGCODEC_GRP_INFO, "Video codec information" },
    { MSG_DLGCODEC_GRP_RATE, "Frame rate" },
    { MSG_DLGCODEC_HC_CODEC, "Codec" },
    { MSG_DLGCODEC_LB_CODEC_UNCOMP, "(Uncompressed RGB)" },
    { MSG_DLGCODEC_LBL_BITRATE, "Bit rate (Kbps):" },
    { MSG_DLGCODEC_LBL_DELTAFRAME, "Delta frame:" },
    { MSG_DLGCODEC_LBL_DRIVERNAME, "Driver name:" },
    { MSG_DLGCODEC_LBL_FOURCC, "FOURCC code:" },
    { MSG_DLGCODEC_LBL_QUALITY, "Quality:" },
    { MSG_DLGCODEC_LBL_RATE, "Fps:" },
    { MSG_DLGCODEC_LBL_REST, "Format restrictions:" },
    { MSG_DLGCODEC_RESTHEIGHT, "Height must be a multiple of %d" },
    { MSG_DLGCODEC_RESTWIDTH, "Width must be a multiple of %d" },
    { MSG_DLGCODEC_RESTUNKHEIGHT, "Height: unknown (%dx%d worked)" },
    { MSG_DLGCODEC_RESTUNKWIDTH, "Width: unknown (%dx%d worked)" },
    { MSG_DLGCODEC_SIZEWARN, "The file size could be very large." },
    { MSG_DLGCODEC_TITLE, "Codec configuration" },
    { MSG_ERR_DLGCODEC_FORMAT, "Couldn't find compatible format.\n"\
	"Possible reasons:\n"\
	"* Codec may only support YUV.\n"\
	"* Codec might be locked.\n"\
	"* Codec might be decompression only." },
    { MSG_ERR_DLGCODEC_MULTX, "The height is incompatible with the codec. It must be multiple of %d." },
    { MSG_ERR_DLGCODEC_MULTY, "The width is incompatible with the codec. It must be multiple of %d." },
    { MSG_ERR_DLGCODEC_SIZE, "* The window size is incompatible with the codec.\n"\
	"  It will be adjusted to %dx%d.\n" },
    { MSG_DLGCOLORSET_GRP_COLHBTYPES, "H-bond atom types" },
    { MSG_DLGCOLORSET_LBL_COLHBACC, "Acceptor:" },
    { MSG_DLGCOLORSET_LBL_COLHBDON, "Donor:" },
    { MSG_DLGCOLORSET_LBL_COLHBOTHER, "Other:" },
    { MSG_DLGCOLORSET_TS_HBONDS, "Other" },
    { MSG_DLGDBEXP_CB_COLBYATM, "Col. by atom" },
    { MSG_DLGDBEXP_FD_MOLTITLE, "Find molecule/s" },
    { MSG_DLGDBEXP_LARGEDB, "This operation on a large database requires a lot of time.\n"\
	"Do yo want to proceed anyway ?" },
    { MSG_DLGDBEXP_NEWMOLNAME, "New molecule name:" },
    { MSG_DLGDBEXP_SB_DATABASES, "Databases:" },
    { MSG_DLGDBEXP_SB_MOLECULES, "Molecules:" },
    { MSG_GRAPH_AMPLITUDE, "Amplitude" },
    { MSG_GRAPH_ANGLE, "Angle (Degree)" },
    { MSG_GRAPH_AREA, "Area ()" },
    { MSG_GRAPH_CLUSTERS, "clusters" },
    { MSG_GRAPH_DERIVATIVE, "Derivative" },
    { MSG_GRAPH_DISTANCE, "Distance ()" },
    { MSG_GRAPH_DIAMETER, "Diameter ()" },
    { MSG_GRAPH_LOGP, "logP" },
    { MSG_GRAPH_NEWXLABEL, "New X axis label:" },
    { MSG_GRAPH_NEWYLABEL, "New X axis label:" },
    { MSG_GRAPH_NEWTITLE, "New graph title:" },
    { MSG_GRAPH_POPULATION, "Population" },
    { MSG_GRAPH_RADIUS, "Radius ()" },
    { MSG_GRAPH_RMSD, "RMSD ()" },
    { MSG_GRAPH_SPECTRUM, "Frequency spectrum" },
    { MSG_GRAPH_SPECTRUMAX, "Spectrum (%)" },
    { MSG_GRAPH_TIME, "Frame" },
    { MSG_GRAPH_TORSION, "Torsion (Degree)" },
    { MSG_GRAPH_VOLUME, "Volume ()" },
    { MSG_GRAPH_MM_LABELS, "&Labels" },
    { MSG_GRAPH_MM_CALCDER, "&Derivative" },
    { MSG_GRAPH_MM_CALCNR, "&Noise reduction" },
    { MSG_GRAPH_MM_CALCSPEC, "&Freq. spectrum" },
    { MSG_GRAPH_MM_LBLTITLE, "&Title" },
    { MSG_GRAPH_MM_LBLXAXIS, "&X axis" },
    { MSG_GRAPH_MM_LBLYAXIS, "&Y axis" },
    { MSG_ERR_GRAPH_NODATA, "No data in plot" },
    { MSG_DLGHBOND_CB_LBLDIST, "Distance ()" },
    { MSG_DLGHBOND_CB_LBLENE, "Energy (Kcal/mol)" },
    { MSG_DLGHBOND_CB_VISONLY, "Selected atoms only" },
    { MSG_DLGHBOND_GRP_ACC, "Acceptors" },
    { MSG_DLGHBOND_GRP_DON, "Donors" },
    { MSG_DLGHBOND_GRP_LABELS, "Labels" },
    { MSG_DLGHBOND_GRP_PARAM, "Parameters" },
    { MSG_DLGHBOND_LBL_PARAEXP, "Attractive exp.:" },
    { MSG_DLGHBOND_LBL_PARCOSEXPAA, "AA cos exp.:" },
    { MSG_DLGHBOND_LBL_PARCOSEXPHA, "HA cos exp.:" },
    { MSG_DLGHBOND_LBL_PARCUTOFF, "Cutoff ():" },
    { MSG_DLGHBOND_LBL_PARREXP, "Repulsive exp.:" },
    { MSG_DLGHBOND_LBL_PARTHRES, "Thresh. (Kcal/mol):" },
    { MSG_DLGHBOND_RESULTS, "  H-bond acceptors : %d\n"\
	"  H-bond donors    : %d\n"\
	"  Number of H-bonds: %d\n"\
	"  H-bond energy    : %.4f Kcal/mol\n\n" },
    { MSG_DLGHBOND_START, "* Searching for hydrogen bonds" },
    { MSG_DLGHBOND_TITLE, "Hydrogen bonds" },
    { MSG_DLGIMGSAVE_LBL_ANTIALIAS, "AntiAlias:" },
    { MSG_DLGMAIN_FRAME, "Frame: %d/%d" },
    { MSG_DLGMAIN_SD_FILTERAVI, "|AVI Windows codec (*.avi)|*.avi|"\
	"AVI XVID libavcodec (*.avi)|*.avi|"\
	"Matroska video H.264 (*.mkv)|*.mkv|"\
	"MP4 H.264 (*.mp4)|*.mp4|"\
	"MPEG-1 (*.mpeg)|*.mpeg|"\
	"MPEG-2 transport stream (*.ts)|*.ts|"\
	"MPEG-2 video object for DVD (*.vob)|*.vob|"\
	"MPEG-4 AVC H.264 (*.264)|*.264|"\
	"MPEG-H HEVC H.265 (*.265)|*.265|"\
	"QuickTime H.264 (*.mov)|*.mov" },
    { MSG_DLGMAIN_SD_FILTERTRJ, "Amber (*.mdcrd)|*.mdcrd|"\
	"Amber NetCDF (*.nc)|*.nc|"\
	"CHARMM/NAMD (*.dcd)|*.dcd|"\
	"IFF/RIFF (*.iff)|*.iff|"\
	"Mol2 multi model (*.mol2;*.mol;*.ml2)|*.mol2;*.mol;*.ml2|"\
	"PDB multi model (*.pdb)|*.pdb|"\
	"Gromacs TRR (*.trr)|*.trr|"\
	"Gromacs XTC (*.xtc)|*.xtc" },
    { MSG_DLGMOPAC_FILEFILTER, "Mopac (*.dat)\000*.dat\000"\
	"All files (*.*)\000*.*\000" },
    { MSG_ERR_DLGMOPAC_FRACCHARGE, "The total charge isn't an integer number.\n"\
	"Do you want proceed anyway ?" },
    { MSG_DLGMPEG_CB_AUTOMAXBR, "Auto max bitrate" },
    { MSG_DLGMPEG_CB_EXTREME, "Extreme && slow settings" },
    { MSG_DLGMPEG_CB_GOPFORCE, "Force closed GOP" },
    { MSG_DLGMPEG_CB_HQ, "High quality" },
    { MSG_DLGMPEG_CB_INTERLACE, "Interlaced encoding" },
    { MSG_DLGMPEG_CB_SCENEDETECT, "Scene detection" },
    { MSG_DLGMPEG_CB_PRODEFAULT, "Default" },
    { MSG_DLGMPEG_CB_PRODVDNTSC, "Digital Versatile Disk (DVD) NTSC" },
    { MSG_DLGMPEG_CB_PRODVDPAL, "Digital Versatile Disk (DVD) PAL" },
    { MSG_DLGMPEG_CB_PROSVCDNTSC, "Super Video CD (SVCD) NTSC" },
    { MSG_DLGMPEG_CB_PROSVCDPAL, "Super Video CD (SVCD) PAL" },
    { MSG_DLGMPEG_CB_PROVCDNTSC, "Video CD (VCD) NTSC" },
    { MSG_DLGMPEG_CB_PROVCDPAL, "Video CD (VCD) PAL" },
    { MSG_DLGMPEG_CB_TOPFIELD, "Top field first" },
    { MSG_DLGMPEG_CB_TRELLIS, "Trellis quantization" },
    { MSG_DLGMPEG_CB_VBR, "Variable bitrate" },
    { MSG_DLGMPEG_GRP_ADVANCED, "Advanced settings" },
    { MSG_DLGMPEG_GRP_BITRATE, "Bitrate" },
    { MSG_DLGMPEG_GRP_GOP, "GOP" },
    { MSG_DLGMPEG_GRP_PROFILES, "Profiles" },
    { MSG_DLGMPEG_LBL_GOPSIZE, "GOP size:" },
    { MSG_DLGMPEG_LBL_MAXBFRM, "Max B-Frames:" },
    { MSG_DLGMPEG_LBL_MAXBITRATE, "Max bitrate (Kbps):" },
    { MSG_DLGMPEG_LBL_PROFILE, "Choose:" },
    { MSG_DLGMPEG_RG_ASPECTRATIO, "Aspect ratio" },
    { MSG_DLGMPEG_TITLE, "MPEG settings" },
    { MSG_TRJSAVE_CK_ACTIVEONLY, "Active only" },
    { MSG_TRJSAVE_CK_SWAPENDIAN, "Big endian" },
    { MSG_TRJSAVE_GRP_ADVANCED, "Advanced options" },
    { MSG_TRJSAVE_GRP_FRAMES, "Frames" },
    { MSG_TRJSAVE_GRP_OPTIONS, "Options" },
    { MSG_TRJSAVE_GRP_PRECISION, "FP precision" },
    { MSG_TRJSAVE_LBL_END, "End:" },
    { MSG_TRJSAVE_LBL_SKIP, "Skip:" },
    { MSG_TRJSAVE_LBL_START, "Start:" },
    { MSG_TRJSAVE_OVERWRITE, "The trajectory file is already present.\n"\
	"Please choose the operation:" },
    { MSG_TRJSAVE_PROGRESS, "Saving frame %d" },
    { MSG_VIEWPREFS_CB_FASTREFRESH, "Fast refresh" },
    { MSG_VIEWPREFS_LBL_ANIMSPEED, "Animation timing:" },
    { MSG_VIEWPREFS_LBL_FRAMERATE, "Animation frame rate (fps):" },
    { MSG_VIEWPREFS_PGREFRESH, "Refresh" },
    { MSG_MENU_SAVETRJ, "Save trajectory" },
    { MSG_MAIN_MENU_AACONTACTS, "AA &contacts" },
    { MSG_MAIN_MENU_SAVETRAJ, "Save t&rajectory" },
    { MSG_MAIN_MENU_COLBYHBOND, "By &H-bond" },
    { MSG_MAIN_MENU_SELSUPATM, "S&up. atoms" },
    { MSG_POPUP_MENU_HBONDS, "&Hydrogen bonds" },
    { MSG_WKS_MOLECULE, "Molecule" },
    { MSG_LOADER_TINKER_XYZ, "TINKER XYZ" },
    { MSG_SURFACE_BUILDHBACC, "* Building H-bond acceptor surface" },
    { MSG_SURFACE_BUILDHBDON, "* Building H-bond donor surface" },
    { MSG_ERR_DLGACT_IFFCSIZE, "Invalid chunk size in the license file" },
    { MSG_ERR_DLGACT_IFFFORMAT, "The license isn't in the requested IFF format" },
    { MSG_ERR_DLGACT_IFFSIZE, "Incorrect license file size" },
    { MSG_ERR_DLGACT_INVCRYP, "The license file has an invalid encryption" },
    { MSG_ERR_DLGACT_NOIFF, "The license file isn't in IFF format" },
    { MSG_ERR_DLGACT_READ, "While reading the license file" },
    { MSG_ERR_DLGACT_UNKNOWN, "Unknown license error (code %d)" },
    { MSG_DLGDBEXP_PM_MOLCOPYLIST, "Copy list" },
    { MSG_DLGDBEXP_PM_MOLSAVELIST, "Save list" },
    { MSG_DLGDBEXP_SD_FILEFILTER, "Text file (*.txt)\000*.txt\000"\
	"All files (*.*)\000*.*\000" },
    { MSG_DLGDBEXP_SD_TITLE, "Save the molecule list ..." },
    { MSG_ERR_ENER_PARKEYUNK, "Unknown parameter key %s" },
    { MSG_DLGJOY_ASS_AR, "R-axis" },
    { MSG_DLGJOY_ASS_AU, "U-axis" },
    { MSG_DLGJOY_ASS_AV, "V-axis" },
    { MSG_DLGJOY_ASS_AX, "X-axis" },
    { MSG_DLGJOY_ASS_AY, "Y-axis" },
    { MSG_DLGJOY_ASS_AZ, "Z-axis" },
    { MSG_DLGJOY_ASS_BUTTON, "Button %d" },
    { MSG_DLGJOY_ASS_CENTER, "Center" },
    { MSG_DLGJOY_ASS_NO, "No" },
    { MSG_DLGJOY_ASS_NONE, "None" },
    { MSG_DLGJOY_ASS_POVX, "X-Pov" },
    { MSG_DLGJOY_ASS_POVY, "Y-Pov" },
    { MSG_DLGJOY_ASS_YES, "Yes" },
    { MSG_DLGJOY_LBL_CAPSAXIS, "Axis:" },
    { MSG_DLGJOY_LBL_CAPSBUT, "Buttons:" },
    { MSG_DLGJOY_LBL_CAPSDRIVER, "Driver:" },
    { MSG_DLGJOY_LBL_CAPSMID, "Manufacturer ID:" },
    { MSG_DLGJOY_LBL_CAPSOEM, "Driver OEM:" },
    { MSG_DLGJOY_LBL_CAPSPID, "Product ID:" },
    { MSG_DLGJOY_LBL_CAPSPOV, "Point of view:" },
    { MSG_DLGJOY_LBL_CAPSPOV4D, "POV 4 directions:" },
    { MSG_DLGJOY_LBL_CAPSPOVCTS, "POV continuous:" },
    { MSG_DLGJOY_LBL_CAPSR, "Rudder:" },
    { MSG_DLGJOY_LBL_CAPSU, "U-coordinate:" },
    { MSG_DLGJOY_LBL_CAPSV, "V-coordinate:" },
    { MSG_DLGJOY_LBL_CAPSZ, "Z-coordinate:" },
    { MSG_DLGJOY_LBL_CFGAXL, "X-rotate down:" },
    { MSG_DLGJOY_LBL_CFGAXR, "X-rotate up:" },
    { MSG_DLGJOY_LBL_CFGAYD, "Y-rotate right:" },
    { MSG_DLGJOY_LBL_CFGAYU, "Y-rotate left:" },
    { MSG_DLGJOY_LBL_CFGAZL, "Z-rotate left:" },
    { MSG_DLGJOY_LBL_CFGAZR, "Z-rotate right:" },
    { MSG_DLGJOY_LBL_CFGMAN, "Click with the left mouse button to change the joystick functions:" },
    { MSG_DLGJOY_LBL_CFGSD, "Scale decrease:" },
    { MSG_DLGJOY_LBL_CFGSI, "Scale increase:" },
    { MSG_DLGJOY_LBL_CFGTXL, "X-translate left:" },
    { MSG_DLGJOY_LBL_CFGTXR, "X-translate right:" },
    { MSG_DLGJOY_LBL_CFGTYD, "Y-translate down:" },
    { MSG_DLGJOY_LBL_CFGTYU, "Y-translate up:" },
    { MSG_DLGJOY_LBL_CFGTZF, "Z-translate far:" },
    { MSG_DLGJOY_LBL_CFGTZN, "Z-translate near:" },
    { MSG_DLGJOY_LBL_JOYDEVICE, "Device:" },
    { MSG_DLGJOY_LBL_JOYPARSAMP, "Sleep time (ms):" },
    { MSG_DLGJOY_LBL_JOYPARSENSAX, "X-rotatate sensibility:" },
    { MSG_DLGJOY_LBL_JOYPARSENSAY, "Y-rotatate sensibility:" },
    { MSG_DLGJOY_LBL_JOYPARSENSAZ, "Z-rotatate sensibility:" },
    { MSG_DLGJOY_LBL_JOYPARSENSSC, "Scale sensibility:" },
    { MSG_DLGJOY_LBL_JOYPARSENSTX, "X-translate sensibility:" },
    { MSG_DLGJOY_LBL_JOYPARSENSTY, "Y-translate sensibility:" },
    { MSG_DLGJOY_LBL_JOYPARSENSTZ, "Z-translate sensibility:" },
    { MSG_DLGJOY_LBL_JOYPARTRESH, "Axis threshold (%):" },
    { MSG_DLGJOY_LBL_MOUSEPARSENSA, "Rotation sensibility:" },
    { MSG_DLGJOY_LBL_MOUSEPARSENSS, "Scale sensibility:" },
    { MSG_DLGJOY_LBL_MOUSEPARSENSW, "Wheel sensibility:" },
    { MSG_DLGJOY_LBL_MOUSEPOSX, "X-position:" },
    { MSG_DLGJOY_LBL_MOUSEPOSY, "Y-position:" },
    { MSG_DLGJOY_LBL_TESTAR, "R-axis:" },
    { MSG_DLGJOY_LBL_TESTAU, "U-axis:" },
    { MSG_DLGJOY_LBL_TESTAV, "V-axis:" },
    { MSG_DLGJOY_LBL_TESTAX, "X-axis:" },
    { MSG_DLGJOY_LBL_TESTAY, "Y-axis:" },
    { MSG_DLGJOY_LBL_TESTAZ, "Z-axis:" },
    { MSG_DLGJOY_LBL_TESTBT, "Buttons:" },
    { MSG_DLGJOY_LBL_TESTPOV, "Point of view:" },
    { MSG_DLGJOY_LBL_TESTPX, "X-point of view:" },
    { MSG_DLGJOY_LBL_TESTPY, "Y-point of view:" },
    { MSG_DLGJOY_TS_JOYCAPS, "Capabilities" },
    { MSG_DLGJOY_TS_JOYCONFIG, "Configuration" },
    { MSG_DLGJOY_TS_JOYPARAM, "Parameters" },
    { MSG_DLGJOY_TS_JOYSTICK, "Joystick" },
    { MSG_DLGJOY_TS_JOYTEST, "Test" },
    { MSG_DLGJOY_TS_MOUSE, "Mouse" },
    { MSG_DLGJOY_TITLE, "Configuration of human interface devices (HID)" },
    { MSG_VIEWPREFS_CB_AXISSTYLESOLID, "Solid" },
    { MSG_VIEWPREFS_CB_AXISSTYLEVECTOR, "Vector" },
    { MSG_VIEWPREFS_LBL_AXISCOLL, "Labels:" },
    { MSG_VIEWPREFS_LBL_AXISPOS, "Position:" },
    { MSG_VIEWPREFS_LBL_AXISRES, "Resolution:" },
    { MSG_VIEWPREFS_LBL_AXISSCALE, "Scale %:" },
    { MSG_VIEWPREFS_LBL_AXISSTYLE, "Style:" },
    { MSG_VIEWPREFS_LBL_AXISWIDTH, "Width:" },
    { MSG_VIEWPREFS_LBL_MULTISAMPLE, "Antialiasing:" },
    { MSG_VIEWPREFS_TS_AXIS, "Axis" },
    { MSG_MAIN_MENU_TLSHID, "&HID configuration" },
    { MSG_ERR_GL2VRML_PROMPT, "Gl2Vrml library error:\n%s." },
    { MSG_CHARGE_TEMRESFIX, "  Unable to fix the charges of the residue %.4s:%.4s:%c:%d\n" },
    { MSG_ERR_CHARGE_TEMATMLONG, "Atom name too long at template line %d" },
    { MSG_ERR_CHARGE_TEMATMMIS, "Missing atom name at template line %d" },
    { MSG_ERR_CHARGE_TEMBONDINV, "Invalid number of bonds at template line %d" },
    { MSG_ERR_CHARGE_TEMBONDMIS, "Missing number of bonds at template line %d" },
    { MSG_ERR_CHARGE_TEMCALLITS, "A macro/residue can't call itself at template line %d" },
    { MSG_ERR_CHARGE_TEMDEL, "Unable to delete atom %s at template line %d" },
    { MSG_ERR_CHARGE_TEMDESCLONG, "Description too long at template line %d (max. %d chars)" },
    { MSG_ERR_CHARGE_TEMGRPINV, "Invalid group ID at template line %d" },
    { MSG_ERR_CHARGE_TEMGRPMIS, "Missing group ID at template line %d" },
    { MSG_ERR_CHARGE_TEMIDDUP, "Duplicated ID at template line %d" },
    { MSG_ERR_CHARGE_TEMIDLONG, "ID too long at template line %d (max. %d chars)" },
    { MSG_ERR_CHARGE_TEMIDMIS, "Missing ID at template line %d" },
    { MSG_ERR_CHARGE_TEMIDUNI, "Non-unique ID at template line %d" },
    { MSG_ERR_CHARGE_TEMIDUNK, "Unknown ID at template line %d" },
    { MSG_ERR_DLGCLUSTBLD_TOOBIG, "Too many cell copies" },
    { MSG_DLGADDFRAG_CB_RESSEQ, "Inc. residue num." },
    { MSG_DLGADDHYD_CB_BNDORD, "Use the bond order to assign the atom types" },
    { MSG_DLGDBOPEN_CK_MOL64BIT, "&64 bit" },
    { MSG_DLGPREFS_CB_CPUS, "Use the maximum number of CPUs" },
    { MSG_DLGPREFS_CB_GLASSWIN, "Enable the glass windows" },
    { MSG_DLGPREFS_CB_SOUND, "Use sound effects" },
    { MSG_DLGPREFS_LBL_CPUS, "Reserved CPUs:" },
    { MSG_DLGPREFS_LBL_ENECUTOFF, "Cut-off distance ():" },
    { MSG_DLGPREFS_LBL_ENEDIEL, "Dielectric constant:" },
    { MSG_DLGPREFS_LBL_ENENONBOND, "Non-bond energy evaluation:" },
    { MSG_DLGPREFS_LBL_ENETRESH, "Energy threshold (%):" },
    { MSG_DLGPREFS_LBL_GLASSWINMAX, "Max alpha:" },
    { MSG_DLGPREFS_LBL_GLASSWINMIN, "Min alpha:" },
    { MSG_DLGPREFS_LBL_GLASSWINOPEN, "Open alpha:" },
    { MSG_DLGPREFS_LBL_GLASSWINSPEED, "Fade time (ms):" },
    { MSG_DLGPREFS_LBL_GRAPHZOOM, "Chart zoom factor (%):" },
    { MSG_DLGPREFS_LBL_LANGUAGE, "Language:" },
    { MSG_DLGPREFS_LBL_MOPACKEY, "Mopac keywords:" },
    { MSG_DLGPREFS_LBL_SOLRAD, "Solvatation radius ():" },
    { MSG_DLGPREFS_LBL_SRFDOTS, "Dot density:" },
    { MSG_DLGPREFS_LBL_SRFMESH, "Mesh size:" },
    { MSG_DLGPREFS_LBL_SURFACE, "Surface parameters:" },
    { MSG_DLGPREFS_LBL_VOLUME, "Volume parameters:" },
    { MSG_DLGPREFS_LBL_SRFPROBERAD, "Probe radius ():" },
    { MSG_DLGPREFS_TITLE, "Preferences" },
    { MSG_DLGPREFS_TS_ENERGY, "Energy" },
    { MSG_DLGPREFS_TS_GLASSWIN, "Glass windows" },
    { MSG_DLGPREFS_TS_MAIN, "Main" },
    { MSG_DLGPREFS_TS_MISC, "Misc" },
    { MSG_DLGPREFS_TS_SURFACE, "Surface" },
    { MSG_DLGPROFIT_BT_WAIT, "Wait" },
    { MSG_DLGPROFIT_MESSAGE, "Welcome to VEGA ZZ !\r\n"\
	"\r\n"\
	"This software is activated for business/profit use only. If you find it useful for"\
	" your job, you should consider to buy the Support Pack.\r\n"\
	"The Support Pack is meant as an economical contribute to the VEGA ZZ developers who,"\
	" in the last years, made their know-how accessible to the whole scientific community."\
	" This contribute is required to allow the VEGA ZZ package to evolve and to start new"\
	" molecular modelling projects.\r\n"\
	"\r\n"\
	"The VEGA ZZ Support Pack includes:\r\n"\
	"\r\n"\
	"- The Activation Key for one year (the standard key expires every 30 days).\r\n"\
	"- Deactivation of this startup window.\r\n"\
	"- Technical support access (normally not allowed to business companies). Help requests"\
	" and suggestions will be considered to improve software features and performances.\r\n"\
	"- The possibility to use the next generation softwares not already officially released.\r\n"\
	"\r\n"\
	"For more information, plase contact:\r\n"\
	"\r\n"\
	"Alessandro Pedretti\r\n"\
	"Dipartimento di Scienze Farmaceutiche \"Pietro Pratesi\"\r\n"\
	"Universit degli Studi di Milano\r\n"\
	"Via Mangiagalli, 25\r\n"\
	"I-20133 Milano - Italy\r\n"\
	"Tel. +39 02 503 19332\r\n"\
	"Fax. +39 02 503 19359\r\n"\
	"E-Mail: info@vegazz.net\r\n" },
    { MSG_ERR_GAUSS_CORR, "The Guassian file is corrupted" },
    { MSG_MAIN_MENU_RMGRAPHOBJ, "&Graphic objects" },
    { MSG_MAIN_MENU_UNDO, "&Undo" },
    { MSG_ERR_IFF_READ, "Unable to read the IFF/RIFF file. It may be corrupted" },
    { MSG_ERR_IFF_WRITE, "Unable to write the IFF/RIFF file" },
    { MSG_LOADER_GAUSSIN, "Gaussian input file" },
    { MSG_LOADER_GAUSSOUT, "Gaussian output file" },
    { MSG_ERR_VGL_BEGINGROUPEXP, "VglBeginGroup expected" },
    { MSG_ERR_VGL_INVGROUPID, "Invalid group ID" },
    { MSG_ERR_VGL_MTXSTACK, "Matrix stack out of bounds" },
    { MSG_ERR_VGL_PRIMITIVE, "Primitive already initialized" },
    { MSG_ERR_VGL_TOOMANYGROUPS, "Too many groups" },
    { MSG_MAIN_MENU_CALCAMMP, "A&mmp" },
    { MSG_AMMP_BACKRUN, "Ammp calculation started in background\n" },
    { MSG_ERR_AMMP_COMLOST, "Ammp communication lost" },
    { MSG_ERR_AMMP_CONNECT, "Unable to connect to %s.\n%s (%d)" },
    { MSG_ERR_AMMP_DATA, "Unable to send data to Ammp" },
    { MSG_ERR_AMMP_EXE, "Can't find the Ammp executable" },
    { MSG_ERR_AMMP_HOSTUSED, "The host %d, pool %s is already in use" },
    { MSG_ERR_AMMP_NOFREEHOST, "No free host available" },
    { MSG_ERR_AMMP_MOLECULE, "Unable to send the molecule to Ammp" },
    { MSG_ERR_AMMP_PIPEINIT, "Unable to initialize the Ammp communication pipe" },
    { MSG_ERR_AMMP_SAVE_SP4, "Can't load the SP4 parameters" },
    { MSG_WARN_AMMP_BONDORD, "The bond order seems to be uncorrectly assigned.\nDo you want proceed always ?" },
    { MSG_WARN_AMMP_SAVE_ATT, "The %.8s atom type isn't present in the SP4 parameter file" },
    { MSG_BT_SEND, "&Send" },
    { MSG_ERR_COM_CALCRUN, "Calculation already running" },
    { MSG_ERR_DLGACT_LIVE, "The live activation key can't work with\n"\
	"a full installation. Please request a new\n"\
	"license." },
    { MSG_DLGAMMP_BT_EDITPAR, "Parameters" },
    { MSG_DLGAMMP_BT_EDITTEM, "Template" },
    { MSG_DLGAMMP_CB_POTHYBRID, "Hybrid" },
    { MSG_DLGAMMP_CB_POTTORSION, "Torsion" },
    { MSG_DLGAMMP_FILEFILTER, "Ammp (*.amp)\000*.amp\000"\
	"All files (*.*)\000*.*\000" },
    { MSG_DLGAMMP_GRP_EDIT, "Force field" },
    { MSG_DLGAMMP_GRP_POTOTHER, "Other" },
    { MSG_DLGAMMP_LBL_CURHOST, "Current:" },
    { MSG_DLGAMMP_LBL_CURHOSTDESC, "Description:" },
    { MSG_DLGAMMP_LBL_CUTOFF, "Long range cutoff ():" },
    { MSG_DLGAMMP_LBL_DTOLER, "Dtoler:" },
    { MSG_DLGAMMP_LBL_HOSTS, "Hosts:" },
    { MSG_DLGAMMP_LBL_HOSTPOOLS, "Host pools:" },
    { MSG_DLGAMMP_LBL_LAMBDA, "Lambda (homotopy):" },
    { MSG_DLGAMMP_LBL_MINRMS, "Toler:" },
    { MSG_DLGAMMP_LBL_MINSTEPS, "Minimization steps:" },
    { MSG_DLGAMMP_LBL_MINUPDATE, "Graphic update:" },
    { MSG_DLGAMMP_LBL_MXCUT, "Short range cutoff ():" },
    { MSG_DLGAMMP_LBL_MXDQ, "Update full electrostatic ():" },
    { MSG_DLGAMMP_LBL_NCNGDEL, "Conjugate steps:" },
    { MSG_DLGAMMP_LBL_NDEEP, "Number of solutions:" },
    { MSG_DLGAMMP_LBL_NSTEEP, "Steepest steps:" },
    { MSG_DLGAMMP_LBL_SEED, "Random number seed:" },
    { MSG_DLGAMMP_LBL_SIGMA, "Scale factor:" },
    { MSG_DLGAMMP_LBL_VSTART, "Starting variance:" },
    { MSG_DLGAMMP_RG_CONST, "Constraints" },
    { MSG_DLGAMMP_RG_CONST_ATMFIX, "Atom fixing" },
    { MSG_DLGAMMP_RG_CONST_NONE, "None" },
    { MSG_DLGAMMP_RG_CONST_TETHER, "Tethering" },
    { MSG_DLGAMMP_RG_MINMETHOD_CONJG, "Conjugate gradients" },
    { MSG_DLGAMMP_RG_MINMETHOD_GENETIC, "Genetic algorithm" },
    { MSG_DLGAMMP_RG_MINMETHOD_NEWTO, "Quasi-Newton" },
    { MSG_DLGAMMP_RG_MINMETHOD_POLY, "Polytope simplex" },
    { MSG_DLGAMMP_RG_MINMETHOD_RIGID, "Rigid-body" },
    { MSG_DLGAMMP_RG_MINMETHOD_SINPT, "Single point" },
    { MSG_DLGAMMP_RG_MINMETHOD_STEEP, "Steepest descent" },
    { MSG_DLGAMMP_RG_MINMETHOD_TRUNC, "Truncated Newton" },
    { MSG_DLGAMMP_RG_MINMETHOD_TRUST, "Trust" },
    { MSG_DLGAMMP_RG_POTANGLE, "Angle" },
    { MSG_DLGAMMP_RG_POTANGLE_UFFC, "UFF cosine" },
    { MSG_DLGAMMP_RG_POTBOND, "Bond" },
    { MSG_DLGAMMP_RG_POTBOND_HOMO, "Homotopy" },
    { MSG_DLGAMMP_RG_POTBOND_MM3, "MM3" },
    { MSG_DLGAMMP_RG_POTBOND_NONE, "None" },
    { MSG_DLGAMMP_RG_POTBOND_STD, "Standard" },
    { MSG_DLGAMMP_RG_POTNONBOND, "Non-bond" },
    { MSG_DLGAMMP_RG_POTNONBOND_DEBYE, "Debye screened" },
    { MSG_DLGAMMP_RG_POTNONBOND_HSHELL, "Hard shell" },
    { MSG_DLGAMMP_RG_POTNONBOND_SCREEN, "Screened" },
    { MSG_DLGAMMP_RG_POTNONBOND_SHADOW, "Shadow" },
    { MSG_DLGAMMP_SP4PARAM, "SP4 parameters" },
    { MSG_DLGAMMP_STOPCALC, "Do you want really stop the calculation ?" },
    { MSG_DLGAMMP_TITLE, "Ammp minimization" },
    { MSG_DLGAMMP_TS_CONSOLE, "Console" },
    { MSG_DLGAMMP_TS_HOSTS, "Hosts" },
    { MSG_DLGAMMP_TS_MIN, "Minimization" },
    { MSG_DLGAMMP_TS_PARAM, "Parameters" },
    { MSG_DLGAMMP_TS_POT, "Potential" },
    { MSG_DLGHOST_BT_KEYGEN, "Key gen." },
    { MSG_DLGHOST_CB_ENCRYPTION, "Encryption" },
    { MSG_DLGHOST_CB_TYPE_UNK, "Unknown" },
    { MSG_DLGHOST_LBL_ADDRESS, "Address:" },
    { MSG_DLGHOST_LBL_DESC, "Description:" },
    { MSG_DLGHOST_LBL_KEY, "WarpKey:" },
    { MSG_DLGHOST_LBL_NAME, "Name:" },
    { MSG_DLGHOST_LBL_PASSWD1, "Password:" },
    { MSG_DLGHOST_LBL_PASSWD2, "Confirm:" },
    { MSG_DLGHOST_LBL_PORT, "Port:" },
    { MSG_DLGHOST_LBL_THREADS, "Threads:" },
    { MSG_DLGHOST_LBL_TYPE, "Type:" },
    { MSG_DLGHOST_LBL_USER, "User name:" },
    { MSG_DLGHOST_SAVE, "Hosts saved to disk" },
    { MSG_DLGHOST_TITLE, "Host configuration" },
    { MSG_DLGHOST_ERR_ADDRESS, "The host address can't be empty" },
    { MSG_DLGHOST_ERR_DUPNAME, "The host name is already assigned and it must be unique" },
    { MSG_DLGHOST_ERR_NAME, "The host name can't be empty" },
    { MSG_DLGHOST_ERR_PORT, "Invalid TCP/IP port number. It must be in 1 to 65535 range" },
    { MSG_DLGHOST_ERR_PWDDIFF, "Password differs" },
    { MSG_DLGHOST_ERR_PWDREQ, "Password required" },
    { MSG_DLGHOST_ERR_REMOVE, "The host is in use and it can't be removed" },
    { MSG_DLGHOST_ERR_SAVE, "Unable to save the hosts" },
    { MSG_DLGHOST_ERR_THREADS, "Invalid number of threads" },
    { MSG_DLGHOST_ERR_KEYSIZE, "Invalid encryption key size" },
    { MSG_DLGHOST_WARN_SAVE, "The host configuration is not saved.\n"\
	"Do you want save it ?" },
    { MSG_DLGJOY_LBL_MOUSEBT, "Buttons" },
    { MSG_DLGJOY_LBL_MOUSEBTL, "L\ne\nf\nt" },
    { MSG_DLGJOY_LBL_MOUSEBTM, "M\ni\nd\nd\nl\ne" },
    { MSG_DLGJOY_LBL_MOUSEBTR, "R\ni\ng\nh\nt" },
    { MSG_DLGJOY_LBL_WHEEL, "Wheel" },
    { MSG_DLGJOY_LBL_WHEELB, "Backward" },
    { MSG_DLGJOY_LBL_WHEELF, "Forward" },
    { MSG_DLGJOY_LBL_WHEELVAL, "Val.:" },
    { MSG_ERR_UNCHARGED, "Unassigned atomic charges.\n"\
	"Use Calculate -> Charges && Pot. to fix them" },
    { MSG_MAIN_MENU_STOPCALC, "Stop calculation" },
    { MSG_MAIN_MENU_TLSHOST, "Host configuration" },
    { MSG_HOSTS_POOL_ALL, "All hosts" },
    { MSG_HOSTS_POOL_LOCAL, "Local hosts" },
    { MSG_HOSTS_POOL_REMOTE, "Remote hosts" },
    { MSG_LOADER_AMMP, "AMMP" },
    { MSG_LOADER_CPMD_XYZ, "CPMD XYZ" },
    { MSG_LOADER_STOPCALC, "To do this operation, you must stop the calculation\n"\
	"Do you want proceed ?" },
    { MSG_BT_CONTINUE, "C&ontinue" },
    { MSG_ERR_AMMP_SAVE_ASSFF, "Can't assign the force field parameters" },
    { MSG_ERR_AMMP_SAVE_FF, "Can't load the force field parameter file" },
    { MSG_ERR_CML_ABORT, "XML aborted at line %d, column %d" },
    { MSG_ERR_CML_PARSE, "XML error at line %d, column %d.\n%s" },
    { MSG_ERR_CML_PARSER, "Unable to create the XML parser" },
    { MSG_ERR_CML_STACK, "Unable to allocate the XML stack" },
    { MSG_MAIN_MENU_CALCAMMPMIN, "&Minimization" },
    { MSG_MAIN_MENU_CALCAMMPTSRC, "&Conformational search" },
    { MSG_DLGAMMP_BT_TSRCEDIT, "Edit torsions" },
    { MSG_DLGAMMP_CB_TSCRMETHOD_BJUMP, "Boltzmann jump" },
    { MSG_DLGAMMP_CB_TSCRMETHOD_RANDOM, "Random" },
    { MSG_DLGAMMP_CB_TSCRMETHOD_SYSTEM, "Systematic" },
    { MSG_DLGAMMP_CK_TSRCKEEPMIN, "Minimized" },
    { MSG_DLGAMMP_CK_TSRCKEEPPREV, "Previous" },
    { MSG_DLGAMMP_CK_TSRCMIN, "Minimize all conformations" },
    { MSG_DLGAMMP_GRP_TSRC, "Search parameters" },
    { MSG_DLGAMMP_GRP_TSRCKEEP, "Apply to" },
    { MSG_DLGAMMP_GRP_TSRCPAR, "Torsion parameters" },
    { MSG_DLGAMMP_HINT_BT_CLONE, "Clone the file names" },
    { MSG_DLGAMMP_HINT_BT_OPEN, "Open the file requester" },
    { MSG_DLGAMMP_LBL_FFIELD, "Type:" },
    { MSG_DLGAMMP_LBL_FILEENE, "Energy:" },
    { MSG_DLGAMMP_LBL_FILEOUT, "Output:" },
    { MSG_DLGAMMP_LBL_FILETRAJ, "Trajectory:" },
    { MSG_DLGAMMP_LBL_FILEVEL, "Velocities:" },
    { MSG_DLGAMMP_LBL_TSRCBASE, "Base:" },
    { MSG_DLGAMMP_LBL_TSRCITER, "Steps:" },
    { MSG_DLGAMMP_LBL_TSRCMETHOD, "Method:" },
    { MSG_DLGAMMP_LBL_TSRCRMSD, "RMSD:" },
    { MSG_DLGAMMP_LBL_TSRCSTEPS, "Steps:" },
    { MSG_DLGAMMP_LBL_TSRCTEMP, "Temp.:" },
    { MSG_DLGAMMP_LBL_TSRCWIN, "Window:" },
    { MSG_DLGAMMP_LV_TSRCTOR_BASE, "Base" },
    { MSG_DLGAMMP_LV_TSRCTOR_STEPS, "Steps" },
    { MSG_DLGAMMP_LV_TSRCTOR_TOR, "Torsion" },
    { MSG_DLGAMMP_LV_TSRCTOR_WIN, "Window" },
    { MSG_DLGAMMP_PM_TSRCCHKALL, "Check all" },
    { MSG_DLGAMMP_PM_TSRCCHKSEL, "Check selected" },
    { MSG_DLGAMMP_PM_TSRCSELALL, "Select all" },
    { MSG_DLGAMMP_PM_TSRCUNCHKALL, "Uncheck all" },
    { MSG_DLGAMMP_PM_TSRCUNCHKSEL, "Uncheck selected" },
    { MSG_DLGAMMP_PM_TSRCUNSELALL, "Unselect all" },
    { MSG_DLGAMMP_SD_FILTER_ENE, "Ammp energy (*.ene)|*.ene" },
    { MSG_DLGAMMP_SD_FILTER_OUT, "Ammp output (*.out)|*.out" },
    { MSG_DLGAMMP_SD_FILTER_VEL, "Ammp velocities (*.vel)|*.vel" },
    { MSG_DLGAMMP_SD_TITLE_ENE, "Save Ammp energy ..." },
    { MSG_DLGAMMP_SD_TITLE_OUT, "Save Ammp output ..." },
    { MSG_DLGAMMP_SD_TITLE_VEL, "Save Ammp velocities ..." },
    { MSG_DLGAMMP_TITLETSRC, "Ammp conformational search" },
    { MSG_DLGAMMP_TS_TSRC, "Conformational search" },
    { MSG_DLGANA_CK_CLUSTACTIVE, "Active atoms only" },
    { MSG_DLGANA_CK_CLUSTENE, "Save energy" },
    { MSG_DLGANA_CK_CLUSTSORTENE, "Sort by energy" },
    { MSG_DLGANA_CLUST_HDR, "\n"\
	"ID         Energy             Conf.\n"\
	"====================================\n" },
    { MSG_DLGANA_CLUST_NUM, "\nNumber of clusters: %d\n" },
    { MSG_DLGANA_RG_CLUSTMETCOORD, "Coordinates" },
    { MSG_DLGANA_RG_CLUSTMETTOR, "Torsion" },
    { MSG_DLGANA_RG_CLUSTMETTORRMSD, "Torsion RMSD" },
    { MSG_DLGANA_TS_CLUST, "Cl&uster" },
    { MSG_DLGCODEC_LBL_SLOWMOTION, "Slow motion:" },
    { MSG_DLGCOLORSET_LBL_COLHBOND, "H-bond:" },
    { MSG_DLGMOLOPEN_CB_FFIELD_NONE, "None" },
    { MSG_DLGMOLOPEN_LBL_FFIELD, "Force field param.:" },
    { MSG_DLGMOPAC_BT_CHGCALC, "Calc." },
    { MSG_DLGPARMIS_MM_AUTOASSIGN, "Auto assign" },
    { MSG_DLGPARMIS_MM_VIEW_COLOR, "Color atoms" },
    { MSG_DLGPARMIS_MM_VIEW_LABELS, "Labels" },
    { MSG_DLGPARMIS_MM_VIEW_WIRE, "Wireframe" },
    { MSG_DLGPARMIS_NBOND_HPOLAR, "H polar" },
    { MSG_DLGPARMIS_NBOND_CAROM, "C aromatic" },
    { MSG_DLGPARMIS_NBOND_NAROM, "N aromatic" },
    { MSG_DLGPARMIS_NBOND_METAL, "Metal" },
    { MSG_DLGPARMIS_PM_ANGLE_SP3, "Generic sp3" },
    { MSG_DLGPARMIS_PM_ANGLE_SP2, "Generic sp2" },
    { MSG_DLGPARMIS_PM_ANGLE_SP, "Generic sp" },
    { MSG_DLGPARMIS_PM_DIHEDRAL_DOUBLE, "Double bond" },
    { MSG_DLGPARMIS_PM_DIHEDRAL_PARDOUBLE, "Partial double bond" },
    { MSG_DLGPARMIS_PM_DIHEDRAL_SINGLE, "Single bond" },
    { MSG_DLGPARMIS_PM_DIHEDRAL_TRIPLE, "Triple bond" },
    { MSG_DLGPARMIS_PM_IMPROPER_FREE, "Make it free" },
    { MSG_DLGPARMIS_PM_IMPROPER_PLANAR, "Make it planar" },
    { MSG_DLGPARMIS_ROW_ANGLE, "Angle" },
    { MSG_DLGPARMIS_ROW_BOND, "Bond" },
    { MSG_DLGPARMIS_ROW_DIHEDRAL, "Torsion" },
    { MSG_DLGPARMIS_ROW_IMPROPER, "Hybrid" },
    { MSG_DLGPARMIS_ROW_NBOND, "Non-bond" },
    { MSG_DLGPARMIS_SD_PARAM_FILTER, "CHARMM parameter (*.inp)|*.inp" },
    { MSG_DLGPARMIS_SD_PARAM_TITLE, "Save CHARMM parameters ..." },
    { MSG_DLGPARMIS_SG_ATMTYP, "Atom type" },
    { MSG_DLGPARMIS_SG_PARAM, "Parameter" },
    { MSG_DLGPARMIS_TITLE, "Missing parameter table" },
    { MSG_DLGSEL_MM_REMALL, "&Remove all" },
    { MSG_DLGSEL_OD_FILTER, "VEGA ZZ selection (*.sel)|*.sel" },
    { MSG_DLGSRFCLC_CB_DEFOGLBUFFER, "Use vertex buffer (more rendering speed)" },
    { MSG_DLGSRFCLC_CB_OGLBUFFER, "Use vertex buffer" },
    { MSG_ERR_HOST_CONN, "Connection refused" },
    { MSG_ERR_HOST_CREATEPIPE, "Can't create pipe" },
    { MSG_ERR_HOST_RECV, "Can't receive the data" },
    { MSG_ERR_HOST_RUN, "Unable to run the application" },
    { MSG_ERR_HOST_SEND, "Can't send the data" },
    { MSG_ERR_HOST_SOCKET, "Can't create the socket" },
    { MSG_ERR_HOST_USER, "Invalid user name or password" },
    { MSG_ERR_HOST_UNKNOWN, "Unknown error" },
    { MSG_LOADER_CHEMSOL, "ChemSol" },
    { MSG_LOADER_CML, "CML" },
    { MSG_LOADER_PQRXML, "PQR XML" },
    { MSG_LOADER_PSFX, "X-Plor PSF" },
    { MSG_MAIN_MENU_DELETE, "&Delete" },
    { MSG_TRJCLUST_PROGRMSD, "Computing the RMS differences" },
    { MSG_TRJCLUST_PROGMEASTOR, "Measuring the torsions" },
    { MSG_TRJCLUST_PROGTOR, "Clustering the torsions" },
    { MSG_WARN_PSFX, "All atoms are placed at (0, 0, 0) coordinates" },
    { MSG_BOND_FIXAROMATIC, "* Fixing aromatic rings" },
    { MSG_ERR_BOND_FIXTYPE, "Unknown bond order fix method" },
    { MSG_INFOXML_AREA, "Surface area" },
    { MSG_INFOXML_AREAPRORAD, "Probe radius" },
    { MSG_INFOXML_ASA, "Apolar surface area" },
    { MSG_INFOXML_ATOMS, "Number of atoms" },
    { MSG_INFOXML_CENTGEO, "Geometry center" },
    { MSG_INFOXML_CENTMASS, "Center of mass" },
    { MSG_INFOXML_CENTROIDS, "Number of centroids" },
    { MSG_INFOXML_CHAINS, "Number of chains" },
    { MSG_INFOXML_CHARGE, "Charge" },
    { MSG_INFOXML_DIMENSIONS, "Dimensions" },
    { MSG_INFOXML_DIPOLE, "Dipole moment" },
    { MSG_INFOXML_FORMULA, "Formula" },
    { MSG_INFOXML_GYRRAD, "Radius of gyration" },
    { MSG_INFOXML_HEAVYATOMS, "Number of heavy atoms" },
    { MSG_INFOXML_LIPOLEBROTO, "Lipole (Broto and Moreau)" },
    { MSG_INFOXML_LOGPBROTO, "logP (Broto and Moreau)" },
    { MSG_INFOXML_LIPOLECRIPPEN, "Lipole (Ghose and Crippen)" },
    { MSG_INFOXML_LOGPCRIPPEN, "logP (Ghose and Crippen)" },
    { MSG_INFOXML_LOGPVIRTUAL, "Virtual logP" },
    { MSG_INFOXML_MOLECULES, "Number of molecules" },
    { MSG_INFOXML_OVALITY, "Ovality index" },
    { MSG_INFOXML_PSA, "Polar surface area" },
    { MSG_INFOXML_RESIDUES, "Number of residues" },
    { MSG_INFOXML_SDIAM, "Surface diameter" },
    { MSG_INFOXML_SEGMENTS, "Number of segments" },
    { MSG_INFOXML_VDIAM, "Volume diameter" },
    { MSG_INFOXML_VOLUME, "Volume" },
    { MSG_INFOXML_WATERS, "Number of waters" },
    { MSG_INFOXML_WEIGHT, "Molecular weight" },
    { MSG_INFOXML_HYDROGENS, "Number of hydrogens" },
    { MSG_ERR_INFOXML_NOTHING, "Nothing to calculate" },
    { MSG_LOADER_CIF, "CIF/mmCIF" },
    { MSG_LOADER_MDLMOLV3000, "MDL Extended Molfile" },
    { MSG_DLGADDFRAG_BT_FINISH, "&Finish >>" },
    { MSG_DLGADDFRAG_BT_NEXT, "&Next >>" },
    { MSG_DLGADDFRAG_BT_PREV, "<< Pre&vious" },
    { MSG_DLGBOND_RG_FIXALLAROM, "Fix aromatic rings" },
    { MSG_DLGBOND_RG_FIXSELAROM, "Fix aromatic rings (sel. atoms)" },
    { MSG_MAIN_MENU_MOPAC2007HELP, "&Mopac 2016 manual" },
    { MSG_MAIN_MENU_TLSVEGAOL, "VEGA On-line" },
    { MSG_ADDCHAIN_ADDPROG, "* Adding the side chains" },
    { MSG_APPLYTRANS_PROG, "* Applying the transformation matrix" },
    { MSG_CHIRAL_WARN_BONDORD, "The chirality may be wrong because the bond order isn't correctly assigned" },
    { MSG_BT_MORE, "&More" },
    { MSG_BT_TERMINATE, "&Terminate" },
    { MSG_ANA_CB_BOND, "Bond" },
    { MSG_DLGAMMP_CB_FILEENE, "Energy:" },
    { MSG_DLGAMMP_CB_FILEOUT, "Output:" },
    { MSG_DLGAMMP_CB_FILETRAY, "Trajectory:" },
    { MSG_DLGAMMP_CB_FILEVEL, "Velocities:" },
    { MSG_DLGAMMP_GRP_OUTFILES, "Output files" },
    { MSG_DLGEXCEPT_LBL_ERROR, "Error message:" },
    { MSG_DLGEXCEPT_MESSAGE_1, "VEGA ZZ has detected an internal error and become unstable. "\
	"It's strongly recommended to save your molecules in new files "\
	"and restart the software. If you continue to use it in this "\
	"condition, you could lost the data." },
    { MSG_DLGEXCEPT_MESSAGE_2, "The internal error is too severe and it's impossible to recover it. "\
	"VEGA ZZ will be restarted." },
    { MSG_DLGEXCEPT_TITLE, "Internal error" },
    { MSG_DLGMOPAC_CK_UPDATE, "Update the 3D view" },
    { MSG_DLGMOPAC_HINT_BT_CHGCALC, "Calculate the total charge" },
    { MSG_DLGMOPAC_HINT_CK_1SCF, "Single point energy calculation" },
    { MSG_DLGMOPAC_HINT_CK_GEOK, "Override some safety checks" },
    { MSG_DLGMOPAC_HINT_CK_MMOK, "Use molecular mechanics correction to CONH bonds" },
    { MSG_DLGMOPAC_HINT_CK_PRECISE, "Increase the criteria for terminating all optimizations,\n"\
	"by a factor, normally 100." },
    { MSG_DLGSSTR_CB_ADDCHAINS, "Add the side chains" },
    { MSG_DLGSSTR_CB_ADDHYDROG, "Add the hydrogens" },
    { MSG_DLGSSTR_CB_FIXANG, "Fix bond angles" },
    { MSG_DLGSSTR_RG_TYP, "Structure type" },
    { MSG_DLGSSTR_PEPTIDE, "Peptide" },
    { MSG_DLGSSTR_PROG, "* Building the peptide" },
    { MSG_DLGSSTR_RG_TYP310, "3.10 helix (3)" },
    { MSG_DLGSSTR_RG_TYPALPHA, "Alpha helix (H)" },
    { MSG_DLGSSTR_RG_TYPBETA, "Beta strand (E)" },
    { MSG_DLGSSTR_RG_TYPBETAANTI, "Antiparallel beta strand (A)" },
    { MSG_DLGSSTR_RG_TYPBETAPAR, "Parallel beta strand (B)" },
    { MSG_DLGSSTR_RG_TYPCUSTOM, "Custom (U)" },
    { MSG_DLGSSTR_RG_TYPLEFT, "Left handed helix (L)" },
    { MSG_DLGSSTR_RG_TYPPI, "Pi helix (P)" },
    { MSG_DLGSSTR_TITLE_BUILD, "Build a peptide" },
    { MSG_DLGSSTR_TITLE_CHANGE, "Change the secondary structure" },
    { MSG_MAIN_MENU_BUILDPEPT, "&Peptide" },
    { MSG_MAIN_MENU_BOND_TYPE, "&Bond type" },
    { MSG_MAIN_MENU_REDO, "&Redo" },
    { MSG_MAIN_MENU_CHGSECSTRUCT, "Secondary struct." },
    { MSG_MAIN_MENU_SELNOCONSTR, "&Unconstrained atoms" },
    { MSG_ERR_MAIN_NOTHINGTOSAVE, "Nothing to save" },
    { MSG_LOADER_FASTA, "Fasta" },
    { MSG_DLGOVER_TBI_SELACTIVEONLY, "A&ctive only" },
    { MSG_DLGOVER_TBI_SELALL, "&All pairs" },
    { MSG_DLGOVER_TBI_SELDIRASC, "&Ascending" },
    { MSG_DLGOVER_TBI_SELDIRBOTH, "&Both" },
    { MSG_DLGOVER_TBI_SELDIRDESC, "&Descending" },
    { MSG_DLGOVER_TBI_SELSIM, "&Similar pairs" },
    { MSG_DLGOVER_TBS_SELDIR, "&Direction" },
    { MSG_DLGOVER_TBS_SELECT, "&Select" },
    { MSG_DLGPREFS_CB_UNDOWORLD, "Undo the world transformations" },
    { MSG_DLGPREFS_LBL_UNDOLVL, "Max undo levels:" },
    { MSG_SEL_TBS_TORPROT, "Add &protein torsions" },
    { MSG_SSTR_PROGPMT, "* Changing the secondary structure" },
    { MSG_ERR_SSTR_NOPROT, "The molecule is not a protein" },
    { MSG_ERR_SSTR_NOTOR, "No valid torsions" },
    { MSG_ERR_SSTR_TORNAME, "Invalid torsion name. It must be Omega, Phi and Psi" },
    { MSG_DLGTOR_BT_DEFAULT, "Default length" },
    { MSG_DLGTOR_CK_MOVEALL, "Move all" },
    { MSG_DLGTOR_LBL_BNDLEN, "Bond length ():" },
    { MSG_LOADER_MOPCAR, "Mopac cartesian" },
    { MSG_UNDO_ADDATOM, "add atom" },
    { MSG_UNDO_AMMP, "Ammp" },
    { MSG_UNDO_AMMPBOND, "invalid bonds" },
    { MSG_UNDO_ANIM, "animation" },
    { MSG_UNDO_APPLYTRANSF, "apply transf." },
    { MSG_UNDO_ATMCHANGE, "change atom" },
    { MSG_UNDO_BONDADD, "add bond" },
    { MSG_UNDO_BONDCHANGE, "change bond" },
    { MSG_UNDO_BONDCHANGEALL, "change bonds" },
    { MSG_UNDO_BONDCONN, "connectivity" },
    { MSG_UNDO_BONDFIXAROM, "fix arom. rings" },
    { MSG_UNDO_BONDFIXTYPES, "fix bond types" },
    { MSG_UNDO_BONDRM, "remove bond" },
    { MSG_UNDO_BONDRMCONN, "remove conn." },
    { MSG_UNDO_BONDSWAP, "swap bonds" },
    { MSG_UNDO_CENTROID, "add centroid" },
    { MSG_UNDO_CHAINCHANGE, "change chain" },
    { MSG_UNDO_CHARGE, "charge" },
    { MSG_UNDO_CHARGEPOT, "charge && pot." },
    { MSG_UNDO_CHGANGLE, "change angle" },
    { MSG_UNDO_CHGBOND, "change bond len." },
    { MSG_UNDO_CHGTORSION, "change torsion" },
    { MSG_UNDO_CHGSECSTRUCT, "secondary struct." },
    { MSG_UNDO_CHKAACHIR, "AA chirality" },
    { MSG_UNDO_CHKPEPTBOND, "peptidic bonds" },
    { MSG_UNDO_CLUSTBUILD, "build cluster" },
    { MSG_UNDO_CLUSTER, "add cluster" },
    { MSG_UNDO_CHKRINGS, "ring inter." },
    { MSG_UNDO_COLBYATM, "color by atom" },
    { MSG_UNDO_COLBYCHAIN, "color by chain" },
    { MSG_UNDO_COLBYCHARGE, "color by charge" },
    { MSG_UNDO_COLBYCONSTR, "color by constraints" },
    { MSG_UNDO_COLBYATMHBOND, "color by H-bond" },
    { MSG_UNDO_COLBYMOL, "color by molecule" },
    { MSG_UNDO_COLBYRES, "color by residue" },
    { MSG_UNDO_COLBYSEG, "color by segment" },
    { MSG_UNDO_COLSEL, "color selection" },
    { MSG_UNDO_DBGET, "database get" },
    { MSG_UNDO_DSPCPK, "ball && stick solid" },
    { MSG_UNDO_DSPCPKW, "ball && stick wire" },
    { MSG_UNDO_DSPLIQ, "stick solid" },
    { MSG_UNDO_DSPLIQW, "stick wire" },
    { MSG_UNDO_DSPTRACE, "trace" },
    { MSG_UNDO_DSPTUBE, "tube" },
    { MSG_UNDO_DSPVDW, "CPK solid" },
    { MSG_UNDO_DSPVDWD, "CPK dotted" },
    { MSG_UNDO_DSPVDWW, "CPK wire" },
    { MSG_UNDO_DSPWIRE, "wireframe" },
    { MSG_UNDO_FIX, "constraints" },
    { MSG_UNDO_FRAGMENT, "fragment" },
    { MSG_UNDO_HBOND, "H-bonds" },
    { MSG_UNDO_HYDROG, "add hydrogens" },
    { VG_UNDO_INTER, "interactions" },
    { MSG_UNDO_INVZCOORD, "inv. Z coord." },
    { MSG_UNDO_IONS, "add ions" },
    { MSG_UNDO_LBLATMCHG, "label atom charge" },
    { MSG_UNDO_LBLATMCHIR, "label atom chirality" },
    { MSG_UNDO_LBLATMELM, "label atom element" },
    { MSG_UNDO_LBLATMFIX, "label atom fix" },
    { MSG_UNDO_LBLATMNAM, "label atom name" },
    { MSG_UNDO_LBLATMNUM, "label atom number" },
    { MSG_UNDO_LBLATMOFF, "label atom off" },
    { MSG_UNDO_LBLATMRESIDUE, "label residue" },
    { MSG_UNDO_LBLATMRESNAME, "label res. name" },
    { MSG_UNDO_LBLATMRESSEQ, "label res. number" },
    { MSG_UNDO_LBLATMTYP, "label atom type" },
    { MSG_UNDO_MEASANG, "measure angle" },
    { MSG_UNDO_MEASDIST, "measure distance" },
    { MSG_UNDO_MEASPLANG, "measure plane ang." },
    { MSG_UNDO_MEASTOR, "measure torsion" },
    { MSG_UNDO_MERGE, "merge" },
    { MSG_UNDO_MOLFIX, "molecule fix" },
    { MSG_UNDO_MOLMERGE, "molecule merge" },
    { MSG_UNDO_MOPAC, "Mopac" },
    { MSG_UNDO_NEW, "new" },
    { MSG_UNDO_NORMALIZE, "normalize" },
    { MSG_UNDO_OPEN, "open" },
    { MSG_UNDO_OVERLAY, "superimpose" },
    { MSG_UNDO_PASTE, "paste" },
    { MSG_UNDO_PEPTBUILD, "new peptide" },
    { MSG_UNDO_POT, "potential" },
    { MSG_UNDO_RENRES, "renumber residues" },
    { MSG_UNDO_RESCHANGE, "change residue" },
    { MSG_UNDO_RMATM, "remove atom" },
    { MSG_UNDO_RMATMS, "remove atoms" },
    { MSG_UNDO_RMCENTROIDS, "remove centroids" },
    { MSG_UNDO_RMGRAPHOBJ, "remove graphic objects" },
    { MSG_UNDO_RMHYD, "remove hydrogens" },
    { MSG_UNDO_RMINVISATM, "remove invisible atoms" },
    { MSG_UNDO_RMMOL, "remove molecule" },
    { MSG_UNDO_RMMON, "remove monitors" },
    { MSG_UNDO_RMRES, "remove residue" },
    { MSG_UNDO_RMSEG, "remove segment" },
    { MSG_UNDO_RMSURF, "remove surface" },
    { MSG_UNDO_RMSURFALL, "remove all surf." },
    { MSG_UNDO_RMWAT, "remove water" },
    { MSG_UNDO_RESETV, "reset view" },
    { MSG_UNDO_ROTATE, "rotate" },
    { MSG_UNDO_SCALE, "scale" },
    { MSG_UNDO_SEGFIX, "segment fix" },
    { MSG_UNDO_SEGMERGE, "segment merge" },
    { MSG_UNDO_SELALL, "select all" },
    { MSG_UNDO_SELBACK, "select backbone" },
    { MSG_UNDO_SELCUST, "select custom" },
    { MSG_UNDO_SELCUSTPROX, "select proximity" },
    { MSG_UNDO_SELCUSTRANGE, "select range" },
    { MSG_UNDO_SELCUSTSLICE, "select slice" },
    { MSG_UNDO_SELHYD, "select hydrogens" },
    { MSG_UNDO_SELINV, "invert selection" },
    { MSG_UNDO_SELMOL, "select molecule" },
    { MSG_UNDO_SELNOCONSTR, "select unconstrained atoms" },
    { MSG_UNDO_SELNOHYD, "hide hydrogens" },
    { MSG_UNDO_SELNONE, "select none" },
    { MSG_UNDO_SELNOWAT, "select no water" },
    { MSG_UNDO_SELSEG, "select segment" },
    { MSG_UNDO_SELSIDECHAINS, "select side chains" },
    { MSG_UNDO_SELSUPATM, "select sup. atoms" },
    { MSG_UNDO_SELWAT, "select water" },
    { MSG_UNDO_SIDECHAINS, "side chains" },
    { MSG_UNDO_SURF, "surface" },
    { MSG_UNDO_SURFCOLOR, "surface color" },
    { MSG_UNDO_SURFDOTSIZE, "surface dot size" },
    { MSG_UNDO_SURFGRAD, "surface gradient" },
    { MSG_UNDO_SURFHIDEALL, "hide all surf." },
    { MSG_UNDO_SURFSHOW, "show/hide surface" },
    { MSG_UNDO_SURFSHOWALL, "show all surf." },
    { MSG_UNDO_SURFTRANS, "surface trans." },
    { MSG_UNDO_TRANSLATE, "translate" },
    { MSG_UNDO_UNSELCUST, "unselect custom" },
    { MSG_UNDO_UNSELCUSTPROX, "unselect proximity" },
    { MSG_UNDO_UNSELCUSTRANGE, "unselect range" },
    { MSG_UNDO_UNSELCUSTSLICE, "unselect slice" },
    { MSG_ERR_UNDO_CREATE, "Can't create a new undo level" },
    { MSG_ERR_UNDO_INTERNAL, "Internal error.\n"\
	"The undo queue will be reinitialized" },
    { MSG_ERR_AUTODOCK_CORR, "Corrupted AutoDock output" },
    { MSG_BT_CALCULATE, "&Calculate" },
    { MSG_BT_DELETE, "&Delete" },
    { MSG_BT_RESTART, "&Restart" },
    { MSG_BT_UNSELECT, "&Unselect" },
    { MSG_DELETE, "Do you really want delete the file ?" },
    { MSG_ERR_EXCEPT_DUMP, "Unable to create the one or more recovery files" },
    { MSG_DLGNAMD_CFGEDIT_TITLE, "Edit NAMD input - %s" },
    { MSG_DLGNAMD_CK_AUTOINPUT, "Automatic input" },
    { MSG_DLGNAMD_CK_BINOUTPUT, "Binary output" },
    { MSG_DLGNAMD_CK_BINRESTART, "Binary restart" },
    { MSG_DLGNAMD_CK_BINVEL, "Binary velocities" },
    { MSG_DLGNAMD_CK_CFGIGNFILES, "Ignore file names" },
    { MSG_DLGNAMD_CK_IMDIGNORE, "Ignore forces" },
    { MSG_DLGNAMD_CK_IMDON, "Update graphics" },
    { MSG_DLGNAMD_CK_IMDWAIT, "Wait connection" },
    { MSG_DLGNAMD_FILTER_ALL, "All files (*.*)|*.*" },
    { MSG_DLGNAMD_FILTER_DCD, "CHARMM/NAMD (*.dcd)|*.dcd" },
    { MSG_DLGNAMD_FILTER_FFTW, "FFTW wisdom file (*.txt)|*.txt" },
    { MSG_DLGNAMD_FILTER_INP, "Parameters (*.inp;*.prm)|*.inp;*.prm|"\
	"All files (*.*)|*.*" },
    { MSG_DLGNAMD_FILTER_NAMD, "NAMD input (*.namd)|*.namd|"\
	"All files (*.*)|*.*" },
    { MSG_DLGNAMD_FILTER_PDB, "PDB (*.pdb;*.ent;*.pdba;*.pdbf;*.pdbl;*.pqr)|*.pdb;*.ent;*.pdba;*.pdbf;*.pdbl;*.pqr|"\
	"All files (*.*)|*.*" },
    { MSG_DLGNAMD_FILTER_PDBB, "Binary PDB (*.coor)|*.coor|"\
	"All files (*.*)|*.*" },
    { MSG_DLGNAMD_FILTER_PSF, "X-Plor PSF (*.psf)|*.psf" },
    { MSG_DLGNAMD_FILTER_VEL, "Velocities (*.vel)|*.vel" },
    { MSG_DLGNAMD_GRP_CONJUGATE, "Conjugate gradients parameters" },
    { MSG_DLGNAMD_GRP_FIXED, "Fixed atoms parameters" },
    { MSG_DLGNAMD_GRP_HARMONIC, "Harmonic constraints parameters" },
    { MSG_DLGNAMD_GRP_IMD, "Interactive molecular dynamics" },
    { MSG_DLGNAMD_GRP_LANGEVIN, "Langevin" },
    { MSG_DLGNAMD_GRP_MULTIPLETIMESTEP, "Multiple timestep parameters" },
    { MSG_DLGNAMD_GRP_OTHERCONFIG, "Configurations" },
    { MSG_DLGNAMD_GRP_OTHERPARAM, "Other parameters" },
    { MSG_DLGNAMD_GRP_OUTPAR, "Output file parameters" },
    { MSG_DLGNAMD_GRP_REASSIGN, "Reassignment" },
    { MSG_DLGNAMD_GRP_RESCALING, "Rescaling" },
    { MSG_DLGNAMD_GRP_TCOUPLE, "Coupling" },
    { MSG_DLGNAMD_GRP_TIMESTEP, "Timestep parameters" },
    { MSG_DLGNAMD_GRP_VELQUENCHING, "Velocity quenching parameters" },
    { MSG_DLGNAMD_LBL_14SCALING, "Scaling factor for 1-4 interactions:" },
    { MSG_DLGNAMD_LBL_CELLVECT1, "Basis vector 1:" },
    { MSG_DLGNAMD_LBL_CELLVECT2, "Basis vector 2:" },
    { MSG_DLGNAMD_LBL_CELLVECT3, "Basis vector 3:" },
    { MSG_DLGNAMD_LBL_CELLORIGIN, "Cell origin:" },
    { MSG_DLGNAMD_LBL_COMMOTION, "Allow initial center of mass motion:" },
    { MSG_DLGNAMD_LBL_CONFIG, "Presettings:" },
    { MSG_DLGNAMD_LBL_CONSEXP, "Constraints exponent:" },
    { MSG_DLGNAMD_LBL_CONSKCOL, "PDB colum with force contants:" },
    { MSG_DLGNAMD_LBL_CONSKFILE, "Constants file:" },
    { MSG_DLGNAMD_LBL_CONSREF, "Ref. pos. file:" },
    { MSG_DLGNAMD_LBL_CONSSCALING, "Scaling factor for energy function:" },
    { MSG_DLGNAMD_LBL_CONSSELECT, "Restraint the Cartesian coord. only:" },
    { MSG_DLGNAMD_LBL_CONSTRAINTS, "Constraints:" },
    { MSG_DLGNAMD_LBL_COORDINATES, "Coordinate PDB file:" },
    { MSG_DLGNAMD_LBL_CYLAXIS, "Cylinder axis alignment:" },
    { MSG_DLGNAMD_LBL_CYLL1, "1st distance ():" },
    { MSG_DLGNAMD_LBL_CYLL2, "2nd distance ():" },
    { MSG_DLGNAMD_LBL_CUTOFF, "Cutoff ():" },
    { MSG_DLGNAMD_LBL_DCDFILE, "Trajectory file:" },
    { MSG_DLGNAMD_LBL_DCDFREQ, "Trajectory frequency:" },
    { MSG_DLGNAMD_LBL_DCDUNITCELL, "Write unit cell data:" },
    { MSG_DLGNAMD_LBL_DIELECTRIC, "Dielectric constant:" },
    { MSG_DLGNAMD_LBL_ENABLED, "Enabled:" },
    { MSG_DLGNAMD_LBL_EXCLUDE, "Exclusion policy:" },
    { MSG_DLGNAMD_LBL_EXTENDEDSYS, "XSC file name:" },
    { MSG_DLGNAMD_LBL_FFTWESTIMATE, "Estimate to optimize FFTW:" },
    { MSG_DLGNAMD_LBL_FFTWUSEWISDOM, "Use FFTW wisdom archive file:" },
    { MSG_DLGNAMD_LBL_FFTWWISDOMFILE, "Wisdom file:" },
    { MSG_DLGNAMD_LBL_FIRSTTIMESTEP, "Starting timestep value:" },
    { MSG_DLGNAMD_LBL_FIXEDATOMS, "Fixed atoms:" },
    { MSG_DLGNAMD_LBL_FIXEDATOMSCOL, "PDB colum with fixed atoms:" },
    { MSG_DLGNAMD_LBL_FIXEDATOMSFILE, "Fixed atoms file:" },
    { MSG_DLGNAMD_LBL_FIXEDATOMSFORCES, "Calculate forces between fixed atoms:" },
    { MSG_DLGNAMD_LBL_FORCEFIELD, "Force field:" },
    { MSG_DLGNAMD_LBL_FULLDIRECT, "Calculate full electrostatic directly:" },
    { MSG_DLGNAMD_LBL_FULLELECTFREQ, "Timesteps between full elect. evaluations:" },
    { MSG_DLGNAMD_LBL_HGROUPCUTOFF, "Group-based distance testing ():" },
    { MSG_DLGNAMD_LBL_IMDFREQ, "Frequency:" },
    { MSG_DLGNAMD_LBL_IMDPORT, "TCP/IP port:" },
    { MSG_DLGNAMD_LBL_LANGEVIN, "Langevin dynamics:" },
    { MSG_DLGNAMD_LBL_LANGEVINCOL, "PDB colum with coefficients:" },
    { MSG_DLGNAMD_LBL_LANGEVINDAMP, "Damping coefficient (1/ps):" },
    { MSG_DLGNAMD_LBL_LANGEVINFILE, "Parameters:" },
    { MSG_DLGNAMD_LBL_LANGEVINDHYDROG, "Apply to hydrogens:" },
    { MSG_DLGNAMD_LBL_LANGEVINTEMP, "Temperature (K):" },
    { MSG_DLGNAMD_LBL_LIMITDIST, "Max. distance for limit interaction ():" },
    { MSG_DLGNAMD_LBL_LONGSPLITTING, "Long and short range forces split:" },
    { MSG_DLGNAMD_LBL_MARGIN, "Extra length in patch dimension ():" },
    { MSG_DLGNAMD_LBL_MAXIMUMMOVE, "Max. atom move ():" },
    { MSG_DLGNAMD_LBL_MERGECROSSTERMS, "Add crossterm energy to dihedral:" },
    { MSG_DLGNAMD_LBL_MINBABYSTEP, "Max. initilal step for line minimizer:" },
    { MSG_DLGNAMD_LBL_MINLINEGOAL, "Gradient reduction factor:" },
    { MSG_DLGNAMD_LBL_MINTINYSTEP, "First initilal step for line minimizer:" },
    { MSG_DLGNAMD_LBL_MOLLYITERATIONS, "Max. MOLLY iterations:" },
    { MSG_DLGNAMD_LBL_MOLLYTOLERANCE, "Tolerance for MOLLY errors:" },
    { MSG_DLGNAMD_LBL_MTSALGORITHM, "MTS algorithm type:" },
    { MSG_DLGNAMD_LBL_NONBONDEDFREQ, "Timesteps between nonbonded evaluations:" },
    { MSG_DLGNAMD_LBL_NONBONDEDSCALING, "Scaling factor for nonbonded forces:" },
    { MSG_DLGNAMD_LBL_NUMSTEPS, "Number of timesteps:" },
    { MSG_DLGNAMD_LBL_OUTPUTENE, "Output frequency:" },
    { MSG_DLGNAMD_LBL_OUTPUTMOMENTA, "Momentum output frequency:" },
    { MSG_DLGNAMD_LBL_OUTPUTNAME, "Output file:" },
    { MSG_DLGNAMD_LBL_OUTPUTPAIRLISTS, "How often to print warnings:" },
    { MSG_DLGNAMD_LBL_OUTPUTPRESS, "Pressure output frequency:" },
    { MSG_DLGNAMD_LBL_OUTPUTTIMING, "Timing output frequency:" },
    { MSG_DLGNAMD_LBL_PAIRLISTDIST, "Distance for inclusion in pair list ():" },
    { MSG_DLGNAMD_LBL_PAIRLISTGROW, "Tolerance increasing fraction:" },
    { MSG_DLGNAMD_LBL_PAIRLISTMINPROCS, "Min. procs for pairlist:" },
    { MSG_DLGNAMD_LBL_PAIRLISTSHRINK, "Tolerance reduction fraction:" },
    { MSG_DLGNAMD_LBL_PAIRLISTTRIGGER, "Tolerance trigger:" },
    { MSG_DLGNAMD_LBL_PAIRLISTPERCYCLE, "Regenerate x times per cycle:" },
    { MSG_DLGNAMD_LBL_PARAMETERS, "Parameter file:" },
    { MSG_DLGNAMD_LBL_PARAMFORMAT, "Format:" },
    { MSG_DLGNAMD_LBL_PME, "Use Particle Mesh Evald:" },
    { MSG_DLGNAMD_LBL_PMEGRIDPOINTS, "Grid points:" },
    { MSG_DLGNAMD_LBL_PMEGRIDSIZEX, "Number of grid points in X dimension:" },
    { MSG_DLGNAMD_LBL_PMEGRIDSIZEY, "Number of grid points in Y dimension:" },
    { MSG_DLGNAMD_LBL_PMEGRIDSIZEZ, "Number of grid points in Z dimension:" },
    { MSG_DLGNAMD_LBL_PMEGRIDSPACING, "Max. space between grid points:" },
    { MSG_DLGNAMD_LBL_PMEINTERP, "PME interpolation order:" },
    { MSG_DLGNAMD_LBL_PMEPROCESSORS, "Processors for FFT and recip. sum:" },
    { MSG_DLGNAMD_LBL_PMETOLERANCE, "PME direct space tolerance:" },
    { MSG_DLGNAMD_LBL_REASSIGNFREQ, "Temperature reassignment frequency:" },
    { MSG_DLGNAMD_LBL_REASSIGNHOLD, "Holding temperature (K):" },
    { MSG_DLGNAMD_LBL_REASSIGNINCR, "Temperature increment (K):" },
    { MSG_DLGNAMD_LBL_REASSIGNTEMP, "Temperature for equilibration (K):" },
    { MSG_DLGNAMD_LBL_RESCALEFREQ, "Temperature rescaling frequency:" },
    { MSG_DLGNAMD_LBL_RESCALETEMP, "Temperature for equilibration (K):" },
    { MSG_DLGNAMD_LBL_RESTARTFREQ, "Restart frequency:" },
    { MSG_DLGNAMD_LBL_RESTARTNAME, "Restart file prefix:" },
    { MSG_DLGNAMD_LBL_RESTARTSAVE, "Timestep in restart:" },
    { MSG_DLGNAMD_LBL_RIGIDBONDS, "ShakeH type:" },
    { MSG_DLGNAMD_LBL_RIGIDDIE, "Exit if a ShakeH error occurs:" },
    { MSG_DLGNAMD_LBL_RIGIDITERATIONS, "Max. ShakeH iterations:" },
    { MSG_DLGNAMD_LBL_RIGIDTOLERANCE, "Bond-length error for ShakeH ():" },
    { MSG_DLGNAMD_LBL_SEED, "Random number seed:" },
    { MSG_DLGNAMD_LBL_SPHERCENTER, "Center:" },
    { MSG_DLGNAMD_LBL_SPHEREXP1, "1st exponent:" },
    { MSG_DLGNAMD_LBL_SPHEREXP2, "2nd exponent:" },
    { MSG_DLGNAMD_LBL_SPHERK1, "1st force constant:" },
    { MSG_DLGNAMD_LBL_SPHERK2, "2nd force constant:" },
    { MSG_DLGNAMD_LBL_SPHERR1, "1st radius ():" },
    { MSG_DLGNAMD_LBL_SPHERR2, "2nd radius ():" },
    { MSG_DLGNAMD_LBL_SPLITPATCH, "How to assign atoms to patches:" },
    { MSG_DLGNAMD_LBL_STEPSPERCYCLE, "Timesteps per cycle:" },
    { MSG_DLGNAMD_LBL_STRUCTURE, "PSF file:" },
    { MSG_DLGNAMD_LBL_SWITCHDIST, "Activation switching distance ():" },
    { MSG_DLGNAMD_LBL_SWITCHING, "Switching function:" },
    { MSG_DLGNAMD_LBL_TCOUPLE, "Temperature coupling:" },
    { MSG_DLGNAMD_LBL_TCOUPLEFILE, "Parameters:" },
    { MSG_DLGNAMD_LBL_TCOUPLECOL, "PDB colum with coefficients:" },
    { MSG_DLGNAMD_LBL_TCOUPLETEMP, "Temperature (K):" },
    { MSG_DLGNAMD_LBL_TEMPERATURE, "Temperature (K):" },
    { MSG_DLGNAMD_LBL_TIMESTEPSIZE, "Timestep size (fs):" },
    { MSG_DLGNAMD_LBL_USEMOLLY, "Modified impulse method (MOLLY):" },
    { MSG_DLGNAMD_LBL_USESETTLE, "Use SETTLE for waters:" },
    { MSG_DLGNAMD_LBL_VDWGEOSIGMA, "Use geometric mean for L-J sigmas:" },
    { MSG_DLGNAMD_LBL_VELDCDFILE, "Velocity trajectory file:" },
    { MSG_DLGNAMD_LBL_VELDCDFREQ, "Velocity frequency:" },
    { MSG_DLGNAMD_LBL_VELOCITIES, "Velocities file:" },
    { MSG_DLGNAMD_LBL_WRAPALL, "Wrap all:" },
    { MSG_DLGNAMD_LBL_WRAPNEAREST, "Wrap nearest:" },
    { MSG_DLGNAMD_LBL_WRAPWATER, "Wrap water:" },
    { MSG_DLGNAMD_LBL_XSTFILE, "XST trajectory:" },
    { MSG_DLGNAMD_LBL_ZEROMOMENTUM, "Remove center of mass drift for PME:" },
    { MSG_DLGNAMD_OD_CONSKFILE, "Open PDB file with force constants ..." },
    { MSG_DLGNAMD_OD_CONSREF, "Open PDB file with reference positions ..." },
    { MSG_DLGNAMD_OD_COORDINATES, "Open PDB file with coordinates ..." },
    { MSG_DLGNAMD_OD_FFTWWISDOMFILE, "Open FFTW wisdom archive ..." },
    { MSG_DLGNAMD_OD_FIXEDATOMSFILE, "Open PDB file with fixed atoms ..." },
    { MSG_DLGNAMD_OD_LANGEVINFILE, "Open PDB file with langevin parameters ..." },
    { MSG_DLGNAMD_OD_NAMDFILE, "Open NAMD input file ..." },
    { MSG_DLGNAMD_OD_PARAMETERS, "Open parameter file ..." },
    { MSG_DLGNAMD_OD_STRUCTURE, "Open structure file ..." },
    { MSG_DLGNAMD_OD_TCOUPLEFILE, "Open PDB file with temperature coupling parameters ..." },
    { MSG_DLGNAMD_OD_VELOCITIES, "Open velocities ..." },
    { MSG_DLGNAMD_PM_CFGDEFAULT, "&Default settings" },
    { MSG_DLGNAMD_PM_CFGIGNFILES, "&Ignore file names" },
    { MSG_DLGNAMD_PM_CFGLOAD, "&Load settings" },
    { MSG_DLGNAMD_PM_CFGSAVE, "&Save settings" },
    { MSG_DLGNAMD_PM_PAGEBASIC, "&Basic" },
    { MSG_DLGNAMD_PM_PAGEBC, "&BC" },
    { MSG_DLGNAMD_PM_PAGECONSTRAINTS, "&Constraints" },
    { MSG_DLGNAMD_PM_PAGECYLINDRICAL, "&Cylindrical" },
    { MSG_DLGNAMD_PM_PAGEDYNAMICS, "&Dynamics" },
    { MSG_DLGNAMD_PM_PAGEINPUT, "&Input" },
    { MSG_DLGNAMD_PM_PAGEMINIMIZATION, "&Minimization" },
    { MSG_DLGNAMD_PM_PAGEOTHER, "&Other" },
    { MSG_DLGNAMD_PM_PAGEOUTPUT, "&Output" },
    { MSG_DLGNAMD_PM_PAGEPERIODIC, "&Periodic" },
    { MSG_DLGNAMD_PM_PAGEPME, "&PME" },
    { MSG_DLGNAMD_PM_PAGESIMSPACE, "&Simulation space partitioning" },
    { MSG_DLGNAMD_PM_PAGESPHERICAL, "&Spherical" },
    { MSG_DLGNAMD_PM_PAGETEMPERATURE, "&Temperature" },
    { MSG_DLGNAMD_PM_REFRESH, "Re&fresh list" },
    { MSG_DLGNAMD_PRESET_FILE, "New preset" },
    { MSG_DLGNAMD_PRESET_NAME, "Put the preset name:" },
    { MSG_DLGNAMD_PRESET_RENAME, "New preset name:" },
    { MSG_DLGNAMD_RG_CALC, "Calculation type" },
    { MSG_DLGNAMD_RG_CALC_CONJ, "Conjugate gradient minimization" },
    { MSG_DLGNAMD_RG_CALC_DYN, "Molecular dynamics" },
    { MSG_DLGNAMD_RG_CALC_VEL, "Velocity quenching minimization" },
    { MSG_DLGNAMD_RG_RUNMODE, "Run mode" },
    { MSG_DLGNAMD_RG_RUNMODE_INT, "Run NAMD in interactive mode" },
    { MSG_DLGNAMD_RG_RUNMODE_PREP, "Prepare the input files only" },
    { MSG_DLGNAMD_RG_RUNMODE_SEP, "Run NAMD as separate process" },
    { MSG_DLGNAMD_SD_DCDFILE, "Save DCD file ..." },
    { MSG_DLGNAMD_SD_NAMDFILE, "Save NAMD input file ..." },
    { MSG_DLGNAMD_SD_OUTPUTNAME, "Output prefix ..." },
    { MSG_DLGNAMD_SD_RESTARTNAME, "Restart prefix ..." },
    { MSG_DLGNAMD_SD_VELDCDFILE, "Save velocity tarjectory ..." },
    { MSG_EZGEOM_WARN_BONDORD, "The E/Z geometry may be wrong because the bond order isn't correctly assigned" },
    { MSG_MAIN_MENU_LBLATM_CHIRAL, "C&hirality" },
    { MSG_MAIN_MENU_NAMD, "&NAMD" },
    { MSG_MAIN_MENU_RMHYDAPOL, "&Apolar hydrog." },
    { MSG_MAIN_MENU_NAMDHELP, "&NAMD user's guide" },
    { MSG_ERR_INCHI_INTERNAL, "InChI library internal error" },
    { MSG_ERR_INCHI_PROMPT, "InChI error: %s." },
    { MSG_WARN_INCHI_PROMPT, "InChI %s" },
    { MSG_INFO_GCMR, "Molar refractivity.: %.4f\n" },
    { MSG_INFOXML_ATMTYPES, "Atom types" },
    { MSG_INFOXML_CHIRAL, "Chiral atoms" },
    { MSG_INFOXML_CHIRALNUM, "Number of chiral atoms" },
    { MSG_INFOXML_EZBONDS, "E/Z bonds" },
    { MSG_INFOXML_GCMR, "Molar refractivity" },
    { MSG_INFOXML_EZNUM, "Number of E/Z bonds" },
    { MSG_INFOXML_HBONDACC, "Number of H-bond acceptors" },
    { MSG_INFOXML_HBONDDON, "Number of H-bond donors" },
    { MSG_INFOXML_MOLNAME, "Molecule name" },
    { MSG_INFOXML_REALMOLS, "Real number of molecules" },
    { MSG_INFOXML_RINGS, "Number of rings" },
    { MSG_INFOXML_TORNUM, "Number of torsions" },
    { MSG_INFOXML_TORFLEXNUM, "Number of flexible torsions" },
    { MSG_LOADER_INCHI, "InChI" },
    { MSG_LOADER_PDBQT, "PDBQT" },
    { MSG_LOADER_MOPZMAT, "Mopac Guassian Z-matrix Arc" },
    { MSG_ERR_MAIN_DBEXMODE, "Unknown database extraction mode" },
    { MSG_ERR_MAIN_DBINVREC, "Invalid record number" },
    { MSG_ERR_MAIN_DBNOTFOUND, "Record not found" },
    { MSG_ERR_MAIN_DBTOOMANY, "Unable to process more than one database" },
    { MSG_ERR_MAIN_RMHYDROG, "Unknown method to remove the hydrogens" },
    { MSG_ERR_MAIN_TORTYPE, "Unknown method to define the torsions" },
    { MSG_MOPAC_LBLMIS, "  Missing label %s\n" },
    { MSG_MOPAC_LBLUNUSED, "  Unused label %s (%f)\n" },
    { MSG_ERR_NAMD_COORDS, "Unable to read the remote coordinates" },
    { MSG_ERR_NAMD_ENDIAN, "Unable to ascertain relative endianness of remote host" },
    { MSG_ERR_NAMD_ENERGIES, "Unable to read the energies" },
    { MSG_ERR_NAMD_FULLELECTFREQUENCY, "The number of timesteps between full electrostatic evaluations "\
	"(fullElectFrequency) must be factor of the timesteps per cycle "\
	"(stepspercycle)" },
    { MSG_ERR_NAMD_LOAD_INVARG, "Invalid argument" },
    { MSG_ERR_NAMD_LOAD_INVFLOAT, "Invalid floating point number" },
    { MSG_ERR_NAMD_LOAD_INVINT, "Invalid integer number" },
    { MSG_ERR_NAMD_LOAD_INVVECT, "Invalid vector" },
    { MSG_ERR_NAMD_LOAD_MISPAR, "Missing argument" },
    { MSG_ERR_NAMD_LOAD_PROMPT, "\"%s %s\" at line %d.\n"\
	"%s" },
    { MSG_ERR_NAMD_LOAD_UNKARGTYP, "Unknown argument type" },
    { MSG_ERR_NAMD_MUTEX, "Unable to create mutex" },
    { MSG_ERR_NAMD_NONBONDEDFREQ, "The timesteps between nonbonded evaluation (nonbondedFreq) must "\
	"be factor of the The number of timesteps between full electrostatic "\
	"evaluations (fullElectFrequency)" },
    { MSG_ERR_NAMD_NOTINSTALLED, "NAMD not installed.\n"\
	"For more details, see the manual" },
    { MSG_ERR_NAMD_PAIRLISTDIST, "The distance between pairs for inclusion in pair lists (pailistdist) "\
	"must be >= cutoff" },
    { MSG_ERR_NAMD_PARFILE, "Unable to open %s parameter file" },
    { MSG_ERR_NAMD_PMEGRIDSIZEX, "The number of points in X direction (PMEGridSizeX) must be "\
	"factor of 2, 3, 5" },
    { MSG_ERR_NAMD_PMEGRIDSIZEY, "The number of points in Y direction (PMEGridSizeY) must be "\
	"factor of 2, 3, 5" },
    { MSG_ERR_NAMD_PMEGRIDSIZEZ, "The number of points in Z direction (PMEGridSizeZ) must be "\
	"factor of 2, 3, 5" },
    { MSG_ERR_NAMD_READFORCE, "Unable to read indices and forces" },
    { MSG_ERR_NAMD_SENDFORCE, "Unable to send indices and forces" },
    { MSG_ERR_NAMD_SWITCHDIST, "The distance at which to activate switching/splitting function for "\
	"electrostatic and van der Waals calculations (switchdist) must be "\
	">= cutoff" },
    { MSG_ERR_NAMD_TCPPORT, "Unable to find a free TCP/IP port" },
    { MSG_POPUP_MENU_EZGEOM, "&E/Z geometry" },
    { MSG_WARN_PDB_ALTLOC, "Alternate location found. The first loaction will be loaded" },
    { MSG_RECOVERY_MESSAGE, "There are melecules recovered from a previous crash" },
    { MSG_RECOVERY_TITLE, "VEGA ZZ recovery system" },
    { MSG_ERR_RENAME, "File name already existing" },
    { MSG_UNDO_EZGEOM, "E/Z geometry" },
    { MSG_UNDO_NAMD, "NAMD" },
    { MSG_UNDO_RMHYDAPOL, "remove apolar hydrog." },
    { MSG_UPDATE, "Do you really want update the file ?" },
    { MSG_UTILS_RMHPROGAPOL, "* Removing apolar hydrogens" },
    { MSG_ERR_ARGPAR_FRMINV, "Invalid frame number" },
    { MSG_ERR_ARGPAR_FRMINVFIRST, "The first frame exceeds the total number of frames" },
    { MSG_ERR_ARGPAR_FRMINVLAST, "The last frame exceeds the total number of frames" },
    { MSG_ERR_ARGPAR_FRMMIS, "First frame missing" },
    { MSG_ERR_ARGPAR_FRMRNG, "Invalid range of frames" },
    { MSG_BT_CHANGE, "C&hange" },
    { MSG_BT_FILTER, "F&ilter" },
    { MSG_DCD_EXTRACTFRM, "  Extracting frame %6u\n" },
    { MSG_DCD_PROCFRAME_ZZ, "Frame %d/%d (%.2f)" },
    { MSG_DLGCOLORSET_LBL_COLBUMP, "Bump:" },
    { MSG_DLGDBEXP_CB_NORMALIZE, "Normalize the coordinates" },
    { MSG_DLGDBEXP_CB_MINCONJ, "Do conjugate gradients" },
    { MSG_DLGDBEXP_CB_MINSTEEP, "Do steepest descent" },
    { MSG_DLGDBEXP_GB_OPTMIN, "Minimization" },
    { MSG_DLGDBEXP_LBL_MINSTEPS, "Steps:" },
    { MSG_DLGDBEXP_LBL_MINTOLER, "Toler:" },
    { MSG_DLGDBEXP_PROG_COPY, "Copying %d/%d" },
    { MSG_DLGDBEXP_RG_OPTADDH, "Add hydrogens" },
    { MSG_DLGDBEXP_RG_DOALWAYS, "Always" },
    { MSG_DLGDBEXP_RG_DONEEDED, "When needed" },
    { MSG_DLGDBEXP_RG_DONEVER, "Never" },
    { MSG_DLGDBEXP_RG_OPTTO3D, "2D to 3D" },
    { MSG_DLGDBFILTER_BT_GETSQL, "Get SQL" },
    { MSG_DLGDBFILTER_LV_FIELDS, "Fields" },
    { MSG_DLGDBFILTER_LV_QUERY, "Conditions" },
    { MSG_DLGDBFILTER_OD_FILTER, "SQL file (*.sql)|*.sql" },
    { MSG_DLGDBFILTER_OD_TITLE, "Open SQL file ..." },
    { MSG_DLGDBFILTER_SD_TITLE, "Save SQL file ..." },
    { MSG_DLGDBFILTER_TS_FILTER, "&Filter" },
    { MSG_DLGDBFILTER_TS_SQL, "&SQL" },
    { MSG_DLGDBFILTER_TITLE, "Database SQL filter" },
    { MSG_DLGDBOPEN_CB_FSQLITE, "SQLite" },
    { MSG_DLGHBOND_SAVETITLE, "Save the hydrogen bonds ..." },
    { MSG_DLGHBOND_CB_SAVE, "Save the results" },
    { MSG_DLGHBOND_SAVEFILTER, "Comma Separated Values (*.csv)|*.csv|"\
	"Data Interchange Format (*.dif)|*.dif" },
    { MSG_BT_CONFIGDEFAULT_HINT, "Revert to default configuration" },
    { MSG_DLGNAMD_BT_CONFIGLOAD_HINT, "Load the configuration from file" },
    { MSG_DLGNAMD_BT_CONFIGSAVE_HINT, "Save the configuration to file" },
    { MSG_DLGNAMD_CK_CFGIGNFILES_HINT, "Ignore the file names loading the configuration" },
    { MSG_DLGNAMD_LB_CONFIG_HINT, "Double click to apply the preset" },
    { MSG_DLGNAMD_LBL_BINCOORDINATES, "Binary PDB file:" },
    { MSG_DLGNAMD_OD_BINCOORDINATES, "Open binary coordinate file ..." },
    { MSG_DLGNAMD_CFGLOAD_PROGRESS, "* NAMD configuration loaded" },
    { MSG_SEL_TBI_TORAUTOD, "A&utoDock" },
    { MSG_SEL_TBS_TORADD, "Add &torsions" },
    { MSG_ERR_FFIELD_INVELEM, "Invalid element in ATDL description" },
    { MSG_ERR_FFIELD_INVOP, "Invalid operator in ATDL description" },
    { MSG_ERR_FFIELD_MULTIOP, "Multiple operators" },
    { MSG_ERR_FFIELD_REQOP, "Operator required" },
    { MSG_GLMEAS_BUMPCHK, "Bumps found: %d\n" },
    { MSG_GLMEAS_BUMPCHKNOBMP, "No bump found\n" },
    { MSG_MAIN_MENU_ISOTOPIC, "&Isotopic distribution" },
    { MSG_MAIN_MENU_SELNOAPOLHYD, "No &apolar hydrogens" },
    { MSG_POPUP_MENU_BUMPCHK, "&Bump check" },
    { MSG_POPUP_MENU_BUMPCHKRT, "&RT bump check" },
    { MSG_INFOXML_ISOTOPIC, "Isotopic distribution" },
    { MSG_INFOXML_TORADOCKNUM, "Number of AutoDock flexible torsions" },
    { MSG_ERR_ISOTOPIC_GRAPHLBLX, "Mass (Daltons)" },
    { MSG_ERR_ISOTOPIC_GRAPHLBLY, "Peak size (%)" },
    { MSG_ERR_ISOTOPIC_GRAPHTITLE, "Isotopic distribution" },
    { MSG_ERR_ISOTOPIC_FRMELEM, "Invalid element in formula" },
    { MSG_ERR_ISOTOPIC_FRMINV, "Invalid formula" },
    { MSG_ERR_ISOTOPIC_FRMINVCHAR, "Invalid character in formula" },
    { MSG_ERR_ISOTOPIC_FRMINVINDEX, "Invalid index in formula" },
    { MSG_ERR_ISOTOPIC_TABEMPTY, "The table of isotopes is empty" },
    { MSG_ERR_ISOTOPIC_TABINVMASS, "The table of isotopes contains an invalid mass" },
    { MSG_ERR_ISOTOPIC_TABNOTFOUND, "File of isotopes not found" },
    { MSG_ERR_ISOTOPIC_TABUNKFORMAT, "File of isotopes in unknown format" },
    { MSG_ERR_CPUS, "Invalid number of CPUs (max. %d)" },
    { MSG_ERR_TRAJ_INVHANDLE, "Invalid handle" },
    { MSG_UNDO_BUMPCHK, "Bump check" },
    { MSG_UNDO_SELNOAPOLHYD, "hide apol. hydrog." },
    { MSG_ATMSEL_BT_PREBACKBONE, "Backbone" },
    { MSG_BNDCHKLEN_BONDS, "Bonds with invalid length:" },
    { MSG_BNDCHKLEN_NUMBER, "Number of bond length violations:" },
    { MSG_BNDCHKLEN_NOINVBOND, "No bond with invalid length" },
    { MSG_ERR_CANO_PROMPT, "Cano error: %s." },
    { MSG_ERR_DBENGINE_INVHANDLE, "Invalid database handle" },
    { MSG_ERR_DBENGINE_SQL, "Invalid SQL code" },
    { MSG_ERR_DBENGINE_SQLSUPP, "The database doesn't support SQL" },
    { MSG_ANA_CB_IGNOREWAT_HINT, "The invisible atoms will be totally ignored" },
    { MSG_ANA_CB_SHOWPLOT_HINT, "Show the calculation results in a plot" },
    { MSG_ANA_CB_VISONLY_HINT, "For areas: the invisible atoms will be considered\n"\
	"to evaluate the overlap of VdW spheres" },
    { MSG_DLGDBEXP_CB_BNDORD, "Fix bond order" },
    { MSG_DLGINTER_CB_CLIP, "Copy the result to clipboard" },
    { MSG_DLGPREFS_CB_OCLDEVACCEL, "Accelerator" },
    { MSG_DLGPREFS_CB_OCLDEVAUTO, "Automatic detection" },
    { MSG_DLGPREFS_CB_OCLDEVCPU, "CPU" },
    { MSG_DLGPREFS_CB_OCLDEVDEF, "Default" },
    { MSG_DLGPREFS_CB_OCLDEVGPU, "GPU" },
    { MSG_DLGPREFS_CB_OCLDEVNOTUSED, "Not used" },
    { MSG_DLGPREFS_LBL_OCLDEV, "OpenCL device:" },
    { MSG_DLGPREFS_LBL_OCLSTATUS, "OpenCL status:" },
    { MSG_DLGPREFS_LBL_OCLSTATOP, "Operative" },
    { MSG_DLGPREFS_LBL_OCLSTATNOACC, "Acceleration not available" },
    { MSG_DLGSMILES_LBL_SMILES, "SMILES:" },
    { MSG_DLGSMILES_RG_DEST, "Destination" },
    { MSG_DLGSMILES_RG_DEST_CURWKS, "Current workspace" },
    { MSG_DLGSMILES_RG_DEST_NEWWKS, "New workspace" },
    { MSG_DLGSMILES_TITLE, "SMILES" },
    { MSG_VIEWPREFS_RGB10, "10 bit pixel format" },
    { MSG_EXCEL_EXPORTDB, "Exporting database ..." },
    { MSG_MAIN_MENU_BUILDSMILES, "&SMILES" },
    { MSG_MAIN_MENU_BNDLEN, "&Bond length" },
    { MSG_MAIN_MENU_SELBACKDNA, "&DNA backbone" },
    { MSG_INFOXML_MIMASS, "Monoisotopic mass" },
    { MSG_INFOXML_SMILES, "SMILES" },
    { MSG_LOADER_PQR, "PQR" },
    { MSG_SCORE_CLIPBOARD, "* Results copied to clipboard" },
    { MSG_SCORE_ENERGIES, "* Interaction energies:" },
    { MSG_SCORE_ENE_BROTO, "  MLPInS......................: %12.4f\n" },
    { MSG_SCORE_ENE_BROTO2, "  MLPInS (square dist.).......: %12.4f\n" },
    { MSG_SCORE_ENE_BROTO3, "  MLPInS (cubic dist.)........: %12.4f\n" },
    { MSG_SCORE_ENE_BROTOF, "  MLPInS (Fermi's eq.)........: %12.4f\n" },
    { MSG_SCORE_ENE_CHARMM, "  CHARMM R6-R12...............: %12.4f Kcal/mol\n" },
    { MSG_SCORE_ENE_CHARMM22, "  CHARMM 22 R6-R12............: %12.4f Kcal/mol\n" },
    { MSG_SCORE_ENE_CHARMM36, "  CHARMM 36 R6-R12............: %12.4f Kcal/mol\n" },
    { MSG_SCORE_ENE_CVFF, "  CVFF R6-R12.................: %12.4f Kcal/mol\n" },
    { MSG_SCORE_ENE_ELECT, "  Electrostatic...............: %12.4f Kcal/mol\n" },
    { MSG_SCORE_ENE_ELECTDD, "  Electrostatic dd............: %12.4f Kcal/mol\n" },
    { MSG_SCORE_NAME_BROTO, "MLPInS" },
    { MSG_SCORE_NAME_BROTO2, "MLPInS square distance" },
    { MSG_SCORE_NAME_BROTO3, "MLPInS cubic distance" },
    { MSG_SCORE_NAME_BROTOF, "MLPInS Fermi's equation" },
    { MSG_SCORE_NAME_CHARMM, "CHARMM R6-R12" },
    { MSG_SCORE_NAME_CHARMM22, "CHARMM 22 R6-R12" },
    { MSG_SCORE_NAME_CHARMM36, "CHARMM 36 R6-R12" },
    { MSG_SCORE_NAME_CVFF, "CVFF R6-R12" },
    { MSG_SCORE_NAME_ELECT, "Electrostatic" },
    { MSG_SCORE_NAME_ELECTDD, "Electrostatic distance dependent" },
    { MSG_TURBO, "Turbo mode ..." },
    { MSG_TURBO_DATABASE, "Processing the database ..." },
    { MSG_TURBO_TRAJECTORY, "Processing the trajectory ..." },
    { MSG_ERR_SCORE_ENESEL, "Energy function not selected" },
    { MSG_ERR_SCORE_ENETYP, "Unknown interaction energy type" },
    { MSG_ERR_SCORE_LIG, "Invalid ligand number" },
    { MSG_ERR_SCORE_PAR, "Unable to assign the parameters needed by\n"\
	"%s" },
    { MSG_ERR_SCORE_ONEMOL, "To calculate the energy, at least two molecules are required" },
    { MSG_ERR_SCORE_POT, "Can't assign the %s potential needed by %s" },
    { MSG_UNDO_CHKBNDLEN, "bond length" },
    { MSG_UNDO_SMILES, "SMILES" },
    { MSG_ERR_AMMP_SAVE_CHKFF, "One or more atoms have an undefined atom type" },
    { MSG_ERR_DBENGINE_DAT, "Unknown format of .dat file" },
    { MSG_ERR_DBENGINE_ODBC_CONNECT, "Unable to connect to ODBC database" },
    { MSG_ERR_DBENGINE_ODBC_HANDLE, "Unable to allocate the ODBC handle" },
    { MSG_ERR_DBENGINE_ODBC_VERSION, "Unable to check the ODBC version" },
    { MSG_ERR_DBENGINE_SQLITE_OPEN, "Unable to open the SQLite database" },
    { MSG_ERR_DBENGINE_SQLITE_SETPAR, "Unable to set the SQLite parameters" },
    { MSG_ERR_DBENGINE_SQLITE_STRUCT, "Unable to attach the SQLite structure database" },
    { MSG_ERR_DBENGINE_SQLITE_VEGATAB, "Missing control data in SQLite database" },
    { MSG_ERR_DBENGINE_UPDDATA, "Unable to update the data" },
    { MSG_ERR_DBENGINE_ZIP_OPEN, "Unable to open the Zip archive" },
    { MSG_ERR_DBREP_END, "Unterminated token" },
    { MSG_ERR_DBREP_FILENAME, "Invalid file name" },
    { MSG_ERR_DBREP_HEAD, "Unable to find the report head. Missing #BEGINLOOP#" },
    { MSG_ERR_DBREP_IMGHEIGHT, "Invalid image height" },
    { MSG_ERR_DBREP_IMGWIDTH, "Invalid image width" },
    { MSG_ERR_DBREP_LOAD, "Unable to load the report template" },
    { MSG_ERR_DBREP_MISENDOFSEC, "Missing end of section (%s)" },
    { MSG_ERR_DBREP_NUMOFARGS, "Illegal number of arguments" },
    { MSG_ERR_DBREP_PROMPT, "%s# at line %d\n%s" },
    { MSG_ERR_DBREP_SMILES, "Invalid SMILES string" },
    { MSG_ERR_DBREP_TAIL, "Unable to find the report tail. Missing #ENDLOOP#" },
    { MSG_ERR_DCD_ATMNUM, "The number of atoms doesn't match the trajectory" },
    { MSG_DLGCOLORSET_GRP_COLFLEXTOR, "Torsions" },
    { MSG_DLGCOLORSET_LBL_COLFTFLEX, "Flexible:" },
    { MSG_DLGCOLORSET_LBL_COLFTRIGID, "Rigid:" },
    { MSG_DLGDBEXP_HC_REPORT, "Report" },
    { MSG_DLGDBEXP_PN_PREVIEW, "Not available" },
    { MSG_DLGDBEXP_TS_OPTIONS1, "&Options 1" },
    { MSG_DLGDBEXP_TS_PREVIEW, "&Preview" },
    { MSG_DLGDBEXP_TS_REPORTS, "&Reports" },
    { MSG_DLGDBOPEN_CB_ACCESS, "Access" },
    { MSG_DLGIMGSAVE_RG_IMAGETYPE, "Image type" },
    { MSG_DLGMAIN_RM_LAYOUT, "Do you want do delete this layout ?" },
    { MSG_DLGMINIED_CASEINSENS, "Case insensitive" },
    { MSG_DLGMINIED_CASESENS, "Case sensitive" },
    { MSG_DLGMINIED_TEXTFIND, "Text to find" },
    { MSG_DLGMOPAC_CK_RMFILES, "Remove output files" },
    { MSG_DLGMOPAC_GRP_FILES, "Files" },
    { MSG_DLGMOPAC_LBL_INPUT, "Input:" },
    { MSG_DLGNAMD_CK_REMOVEFILES, "Remove files after minimization" },
    { MSG_DLGSHUTDOWN_LBL_COUNTDOWN, "Time remaining to power off the system:" },
    { MSG_DLGSHUTDOWN_TITLE, "Power off the system" },
    { MSG_DLGUPDATE_LBL_FILE, "File name:" },
    { MSG_DLGUPDATE_LBL_FILENUM, "File number:" },
    { MSG_DLGUPDATE_LBL_SIZE, "Size:" },
    { MSG_DLGUPDATE_TITLE, "Software update" },
    { MSG_ERR_INDIGO_PROMPT, "Indigo error: %s." },
    { MSG_ERR_R2D_UNKFORMAT, "Unsupported 2D file format" },
    { MSG_MAIN_MENU_COLBYFT, "By &flex torsions" },
    { MSG_MAIN_MENU_COLBYFTAUTODOCK, "&AutoDock" },
    { MSG_MAIN_MENU_COLBYFTNORM, "&Normal" },
    { MSG_MAIN_MENU_FINDDOWN, "Find &down" },
    { MSG_MAIN_MENU_FINDUP, "Find &up" },
    { MSG_MAIN_MENU_LAYDELETE, "Delete layout" },
    { MSG_MAIN_MENU_LAYSELECT, "Select layout" },
    { MSG_MAIN_MENU_LAYSAVE, "Save layout" },
    { MSG_POPUP_MENU_ATOM, "&Atom" },
    { MSG_POPUP_MENU_ATMADD_H, "&Hydrogens" },
    { MSG_POPUP_MENU_BOND, "&Bond" },
    { MSG_POPUP_MENU_CHAIN, "&Chain" },
    { MSG_POPUP_MENU_CHANGE, "&Change" },
    { MSG_POPUP_MENU_HIDE, "&Hide" },
    { MSG_POPUP_MENU_LABEL, "&Label" },
    { MSG_POPUP_MENU_LENGTH, "&Length" },
    { MSG_POPUP_MENU_MOLECULE, "&Molecule" },
    { MSG_POPUP_MENU_REMOVE, "&Remove" },
    { MSG_POPUP_MENU_RESIDUE, "&Residue" },
    { MSG_POPUP_MENU_SEGMENT, "&Segment" },
    { MSG_POPUP_MENU_SHOW, "&Select" },
    { MSG_POPUP_MENU_SHUTDOWN, "Power off the system" },
    { MSG_HELP_TITLE, "VEGA ZZ manual" },
    { MSG_INFOXML_INCHI, "InChI" },
    { MSG_INFOXML_INCHIKEY, "InChIKey" },
    { MSG_NAMD_CALCSTART, "* NAMD calculation started" },
    { MSG_NAMD_CALCFINISH, "* Done" },
    { MSG_UNDO_COLBYFT, "color by flexible torsion" },
    { MSG_UNDO_MEASBOND, "measure bond" },
    { MSG_UNDO_RMCHAIN, "remove chain" },
    { MSG_UNDO_SELATM, "select atom" },
    { MSG_UNDO_SELCHAIN, "select chain" },
    { MSG_UNDO_SELRES, "select residue" },
    { MSG_UNDO_UNSELATM, "unselect atom" },
    { MSG_UNDO_UNSELCHAIN, "unselect chain" },
    { MSG_UNDO_UNSELRES, "unselect residue" },
    { MSG_UNDO_UNSELMOL, "unselect molecule" },
    { MSG_UNDO_UNSELSEG, "unselect segment" },
    { MSG_UPDATE_AVAILABLE, "There are software updates:\n" },
    { MSG_UPDATE_INSTALL, "Do you want install them ?" },
    { MSG_UPDATE_MULTIPROC, "You are running multiple VEGA applications. "\
	"Close them before update.\nDo you want retry ?" },
    { MSG_UPDATE_NOUPDATE, "The software is updated." },
    { MSG_ERR_UPDATE_ALRRUN, "Update already running" },
    { MSG_ERR_UPDATE_CONN, "Internet connection not available" },
    { MSG_ERR_UPDATE_CORRFILE, "Corrupted update file" },
    { MSG_ERR_UPDATE_CORRPACK, "Corrupted package list" },
    { MSG_ERR_UPDATE_DATA, "Unable to download update data" },
    { MSG_ERR_UPDATE_DIR, "Unable to create the download directory" },
    { MSG_ERR_UPDATE_DOWNLOAD, "Unable to download the update file" },
    { MSG_ERR_UPDATE_RUNCPY, "Run.exe not found" },
    { MSG_ERR_UPDATE_SCRIPT, "Unable to save the update script" },
    { MSG_ERR_UPDATE_SCRIPTRUN, "Unable to start the update script" },
    { MSG_ERR_UPDATE_URLINV, "Invalid update URL" },
    { MSG_ERR_UPDATE_URLMIS, "Missing update URL in configuration file" },
    { MSG_ERR_WKSLOCKMOL, "Unable to open the molecule beacuse the workspace is locked" },
    { MSG_DLGAMMP_CONV3D, "Converting to 3D ..." },
    { MSG_DLGIUPAC_LBL_IUPAC, "IUPAC name:" },
    { MSG_DLGIUPAC_TITLE, "IUPAC" },
    { MSG_DLGDBOPEN_CB_FMOL2, "Mol2" },
    { MSG_MAIN_MENU_BUILDIUPAC, "&IUPAC" },
    { MSG_ERR_IUPAC_NAME, "Invalid IUPAC name" },
    { MSG_ERR_JAVA_NOTINSTALLED, "Java not installed" },
    { MSG_LOADER_NAMDBIN, "NAMD binary" },
    { MSG_ERR_MAIN_MEASURETRJ, "-m option and trajectory format can't be used at the same time" },
    { MSG_ERR_OPSIN_NOTINSTALLED, "OPSIN not installed" },
    { MSG_ERR_OPSIN_CANTRUN, "Can't run OPSIN" },
    { MSG_ERR_SPILLO_MISLIG, "Missing ligand" },
    { MSG_TCPIO_PROGRESS, "* Downloaded %d bytes\n" },
    { MSG_TCPIO_PROGRESSSIZE, "* Downloading %d/%d bytes (%.2f%%)\n" },
    { MSG_ERR_TCPIO_DOWNLOAD, "Can't download the file" },
    { MSG_ERR_TCPIO_HTTPINIT, "Can't initialize the HTTP client" },
    { MSG_ERR_TCPIO_PARSEURL, "Can't parse the URL" },
    { MSG_ANA_CB_OVALITY, "Ovality" },
    { MSG_DCD_STARTOVAL, "  Starting ovality   : %.4f\n" },
    { MSG_GRAPH_OVALITY, "Ovality" },
    { MSG_ERR_CDX_STRUCT, "Corrupted ChemDraw CDX structure" },
    { MSG_ATMSEL_CB_CENTER, "Center && zoom" },
    { MSG_DLGRMMOL_CB_CENTER, "Center && zoom" },
    { MSG_LOADER_CDX, "ChemDraw CDX" },
    { MSG_LOADER_CLIPBOARD, "* Loading molecule from clipboard" },
    { MSG_LOADER_MOLFROMMEM, "Molecule from memory" },
    { MSG_MAIN_MENU_CENTERVIEW, "&Center" },
    { MSG_TBTOOLS_CENTERV, "Center view" },
    { MSG_UNDO_CENTER, "rotation center" },
    { MSG_UNDO_CENTERV, "center view" },
    { MSG_DLGAMMP_LBL_TSRCEND, "End:" },
    { MSG_DLGAMMP_LV_TSRCTOR_END, "End" },
    { MSG_DLGAMMP_PM_TSRCFILL, "&Fill" },
    { MSG_DLGAMMP_PM_TSRCFILL_BASE, "&Base with torsion value" },
    { MSG_DLGAMMP_PM_TSRCFILL_RANGE, "Base && &end" },
    { MSG_DLGAMMP_PM_TSRCFILL_CUR, "&Current values" },
    { MSG_DLGAMMP_PM_TSRCFILL_DEF, "&Default" },
    { MSG_MAIN_MENU_DOCKING, "&Docking" },
    { MSG_MAIN_MENU_DOWNLOAD, "D&ownload" },
    { MSG_MAIN_MENU_CONV2D, "Convert to &2D" },
    { MSG_MAIN_MENU_CONV3D, "Convert to &3D" },
    { MSG_VIEWPREFS_ZCLIPNEAR, "Z clip n." },
    { MSG_ERR_GRAMM_FRMNUM, "Invalid number of poses in GRAMM output" },
    { MSG_ERR_GRAMM_LIGRANGE, "Invalid atom range of GRAMM ligand" },
    { MSG_ERR_GRAMM_LIGSEL, "Atom selection of the GRAMM ligand not found" },
    { MSG_WARN_GRAMM_POSES, "Too many poses in GRAMM output file" },
    { MSG_UNDO_2DCONVERSION, "2D conversion" },
    { MSG_UNDO_3DCONVERSION, "3D" },
    { MSG_DLGCOMBI_BT_BUILD_HINT, "Build the SMILES database" },
    { MSG_DLGCOMBI_BT_GET_HINT, "Get the SMILES string from the molecule in the current workspace" },
    { MSG_DLGCOMBI_BUILDING, "* Building the combinatorial database" },
    { MSG_DLGCOMBI_CB_CANONICAL, "Canonical SMILES" },
    { MSG_DLGCOMBI_CB_RMREDMOL, "Remove redundant molecules" },
    { MSG_DLGCOMBI_CB_SYSTEMATIC, "Systematic build" },
    { MSG_DLGCOMBI_HINT, "1) Put the SMILES scaffold.\n"\
	"2) Insert the variable positions.\n"\
	"3) Add one or more fragment databases.\n"\
	"4) Click \"Build\" button." },
    { MSG_DLGCOMBI_LBL_NAME, "Compound name:" },
    { MSG_DLGCOMBI_LV_FRAGS_DB, "Database" },
    { MSG_DLGCOMBI_LV_FRAGS_NUM, "Num." },
    { MSG_DLGCOMBI_LV_FRAGS_HINT, "To associate a database to a variable position,\n"\
	"drag & drop the file or double click the item or\n"\
	"select a database from the library" },
    { MSG_DLGCOMBI_LV_FRAGS_POS, "Position" },
    { MSG_DLGCOMBI_OD_SMILES, "Open SMILES database ..." },
    { MSG_DLGCOMBI_OD_SMILES_FLT, "SMILES database (*.smi;*.txt)|*.smi;*.txt" },
    { MSG_DLGCOMBI_PM_FRAGSLIB, "&Library" },
    { MSG_DLGCOMBI_PN_PREVIEWLBL, "Preview" },
    { MSG_DLGCOMBI_PN_SCAFFOLDLBL, "Scaffold" },
    { MSG_DLGCOMBI_REMOVING, "* Removing the redundant molecules\n" },
    { MSG_DLGCOMBI_LBL_SCAFFOLD, "SMILES scaffold:" },
    { MSG_DLGCOMBI_SAVING, "* Saving \"%s\" database\n" },
    { MSG_DLGCOMBI_SB_INSERTR_HINT, "Instert a variable position in the SMILES string" },
    { MSG_DLGCOMBI_SD_SMILES, "Save SMILES database ..." },
    { MSG_DLGCOMBI_SD_SMILES_FLT, "SMILES database (*.smi)|*.smi" },
    { MSG_DLGCOMBI_SP_HINT, "Move here to change the size of the panels" },
    { MSG_DLGCOMBI_STR_NAME, "Compound" },
    { MSG_DLGCOMBI_STR_PREVIEW_HINT, "Change the compound preview by ID" },
    { MSG_DLGCOMBI_STR_SCAFFOLD_HINT, "Put here the scaffold in SMILES format" },
    { MSG_DLGCOMBI_TB_PREVIEW_HINT, "Move the slider to change the compond preview" },
    { MSG_DLGCOMBI_TITLE, "Combinatorial SMILES builder" },
    { MSG_ERR_DLGCOMBI_INVSMI, "Invalid SMILES string" },
    { MSG_DLGDBOPEN_CB_FSMI, "SMILES file" },
    { MSG_MAIN_MENU_BUILDCMBSMILES, "&Combi SMILES" },
    { MSG_LIGEN_SRFGRIDEXCL, "LiGen grid exclusion" },
    { MSG_LIGEN_SRFGRIDHBACC, "LiGen grid HBacc" },
    { MSG_LIGEN_SRFGRIDHBDON, "LiGen grid HBdon" },
    { MSG_LIGEN_SRFGRIDHYDRO, "LiGen grid hydrophobic" },
    { MSG_LIGEN_SRFGRIDREC, "LiGen grid receptor" },
    { MSG_LIGEN_SRFGRIDSOLV, "LiGen grid solvent" },
    { MSG_LIGEN_SRFGRIDVACANT, "LiGen grid vacant" },
    { MSG_LIGEN_SRFPHARMHBACC, "LiGen pharm. HBacc" },
    { MSG_LIGEN_SRFPHARMHBDON, "LiGen pharm. HBdon" },
    { MSG_LIGEN_SRFPHARMHYDRO, "LiGen pharm. hydrophobic" },
    { MSG_ERR_LIGEN_FMTGRID, "Invalid LiGen grid file" },
    { MSG_ERR_LIGEN_FMTPHARM, "Invalid LiGen pharmacophore file" },
    { MSG_ERR_LIGEN_FMTPOCKET, "Invalid LiGen pocket file" },
    { MSG_LOADER_LIGENGRID, "LiGen grid" },
    { MSG_LOADER_LIGENGRIDBIN, "LiGen binary grid" },
    { MSG_LOADER_LIGENPHARM, "LiGen pharmacopore" },
    { MSG_LOADER_LIGENPHARMBIN, "LiGen binary pharmacopore" },
    { MSG_LOADER_LIGENPOCKET, "LiGen pocket" },
    { MSG_LOADER_LIGENPOCKETBIN, "LiGen binary pocket" },
    { MSG_MAIN_MENU_DATADIR, "Explore data directory" },
    { MSG_DLGDBEXP_CB_CHARGE, "Force charge assignment" },
    { MSG_DLGDBEXP_PM_MOLCSV, "Export to &file" },
    { MSG_DLGDBEXP_PROG_SIMIL, "Comparing %d/%d" },
    { MSG_DLGDBEXP_SD_EXPFILTER, "Attribute-relation file (*.arff)\000*.arff\000"\
	"Attribute-relation file without strings (*.arff)\000*.arff\000"\
	"Comma Separated Values (*.csv)\000*.csv\000" },
    { MSG_DLGDBEXP_SD_EXPTITLE, "Export the database ..." },
    { MSG_DLGDBFILTER_CB_SIMFPENC_BASE64, "Base64" },
    { MSG_DLGDBFILTER_CB_SIMFPENC_HEX, "Hexadecimal" },
    { MSG_DLGDBFILTER_CB_SIMSAVE, "Save data:" },
    { MSG_DLGDBFILTER_CSVHDR, "Molecule;Similarity;Fingerprint\n" },
    { MSG_DLGDBFILTER_GRP_SIMPAR, "Parameters" },
    { MSG_DLGDBFILTER_GRP_TVERSKY, "Tversky parameters" },
    { MSG_DLGDBFILTER_LBL_ALPHA, "Alpha:" },
    { MSG_DLGDBFILTER_LBL_BETA, "Beta:" },
    { MSG_DLGDBFILTER_LBL_SIMFPENC, "Fingerprint encoding:" },
    { MSG_DLGDBFILTER_LBL_SIMTHRESH, "Similarity threshold:" },
    { MSG_DLGDBFILTER_RG_FPTYPE, "Fingerprint type" },
    { MSG_DLGDBFILTER_RG_FPTYPE_FULL, "Full" },
    { MSG_DLGDBFILTER_RG_FPTYPE_RESSUB, "Resonance substructure" },
    { MSG_DLGDBFILTER_RG_FPTYPE_SIM, "Similarity" },
    { MSG_DLGDBFILTER_RG_FPTYPE_SUB, "Substructure" },
    { MSG_DLGDBFILTER_RG_FPTYPE_TAUSUB, "Tautomer substructure" },
    { MSG_DLGDBFILTER_RG_METHOD, "Similarity method" },
    { MSG_DLGDBFILTER_SD_CSVFILTER, "Comma separated values (*.csv)|*.csv" },
    { MSG_DLGDBFILTER_SD_CSVTITLE, "Save CSV file ..." },
    { MSG_DLGDBFILTER_TS_SIMILARITY, "&Similarity" },
    { MSG_DLGMOPAC_CK_CONSTRAINTS, "Use atom constraints" },
    { MSG_DLGMOPAC_HINT_MOPAC2007, "Mopac 2007 or greater required" },
    { MSG_DLGDBOPEN_CB_ACCESS2007, "Access 2007" },
    { MSG_DLGADDHYD_CB_ADDHYDROG, "Add the hydrogens" },
    { MSG_DLGADDHYD_CB_IONIZE, "Ionize the molecule" },
    { MSG_DLGADDHYD_GRP_ACTIONS, "Actions" },
    { MSG_DLGADDHYD_GRP_OPTIONS, "Options" },
    { MSG_DLGADDHYD_LBL_IONIZEPH, "Ionization pH:" },
    { MSG_DLGADDHYD_LBL_IONIZETOL, "Ionization tol.:" },
    { MSG_DLGCLUSTER_RG_CLUSTPOSBOX, "Box center" },
    { MSG_DLGDBEXP_CB_IONIZE, "Ionize" },
    { MSG_DLGDBEXP_CB_OPTGEOGEN, "Generate geometric isomers" },
    { MSG_DLGDBEXP_CB_OPTSTEGEN, "Generate stereoisomers" },
    { MSG_DLGDBEXP_LBL_IONIZETOL, "Tol.:" },
    { MSG_DLGDBEXP_RG_OPTTAU, "Generate tautomers" },
    { MSG_DLGDBEXP_RG_OPTTAU_INCHI, "InChI based" },
    { MSG_DLGDBEXP_RG_OPTTAU_NONE, "None" },
    { MSG_DLGDBEXP_RG_OPTTAU_RSMARTS, "SMARTS templ." },
    { MSG_DLGDBEXP_TS_OPTIONS2, "&Options 2" },
    { MSG_ERR_FFIELD_SMARTSFILE, "Invalid SMARTS file" },
    { MSG_ERR_FFIELD_SMARTSINV, "Invalid SMARTS" },
    { MSG_GEOISO_MOLNAME, "Geometric isomer" },
    { MSG_INFO_HLB, "Davies' HLB........: %.2f\n"\
	"Griffin's HLB......: %.2f\n"\
	"PSA-based HLB......: %.2f\n"\
	"Mean HLB...........: %.2f\n" },
    { MSG_INFOXML_HLBD, "Davies' HLB" },
    { MSG_INFOXML_HLBG, "Griffin's HLB" },
    { MSG_INFOXML_HLBM, "Mean HLB" },
    { MSG_INFOXML_HLBPSA, "PSA-based HLB" },
    { MSG_IONIZE_PROG, "* Ionizing the molercule" },
    { MSG_ERR_IONIZE_NOPAR, "Parameters not available for %s" },
    { MSG_STEISO_MOLNAME, "Stereoisomer" },
    { MSG_TAU_MOLNAME, "Tautomer" },
    { MSG_UNDO_GEOISOMER, "geometric isomer" },
    { MSG_UNDO_IONIZATION, "ionization" },
    { MSG_UNDO_IUPAC, "IUPAC" },
    { MSG_UNDO_SELCUSTSMARTS, "select SMARTS" },
    { MSG_UNDO_STEISOMER, "stereoisomer" },
    { MSG_UNDO_TAUTOMER, "tautomer" },
    { MSG_UNDO_UNSELCUSTSMARTS, "unselect SMARTS" },
    { MSG_ERR_DBENGINE_COLADD, "Can't add \"%s\" column" },
    { MSG_ERR_DBENGINE_COLDELETE, "Can't delete \"%s\" column" },
    { MSG_ERR_DBENGINE_COLRENAME, "Can't rename \"%s\" column" },
    { MSG_ERR_DBENGINE_COLUPDATE, "Can't update the columns" },
    { MSG_ERR_DBENGINE_PASTE_LINEMATCH, "The number of lines (%u) don't match the number of molecules (%u)" },
    { MSG_ERR_DBENGINE_PASTE_COLNUM, "Line %u with %d columns instead of %d" },
    { MSG_ERR_DBENGINE_PASTE_DATATYPE, "Unsupported data type for \"%s\" column" },
    { MSG_ERR_DBENGINE_PASTE_DATATYPEINC, "Incompatible data type (column \"%s\", line %u)" },
    { MSG_ERR_DBENGINE_PASTE_NOTHING, "Nothing to paste" },
    { MSG_DLGABOUT_SYSTEM_WIN64, "Windows x64 version" },
    { MSG_DLGABOUT_SYSTEM_WIN32, "Windows x86 version" },
    { MSG_DLGDBEXP_BT_PASTECOLS, "&Paste cols." },
    { MSG_DLGDBEXP_COLDELETE, "Do you want to delete \"%a\" column ?" },
    { MSG_DLGDBEXP_COLPASTE, "This operation cannot be undoed. Do you want to proceed ?" },
    { MSG_DLGDBEXP_COLRENAME, "New column name:" },
    { MSG_DLGDBEXP_RG_OPTCHARGE, "Assign charges" },
    { MSG_DLGDBOPEN_ASKEXIST, "Database file already exists.\n"\
	"Do you want to replace it ?" },
    { MSG_DLGDBOPEN_DBEMPTY, "Empty" },
    { MSG_DLGDBOPEN_DBFILENAME, "Database file name:" },
    { MSG_DBREP_SAVING, "Saving the report..." },
    { MSG_LOADER_EMPIREARC, "EMPIRE Arc" },
    { MSG_UNDO_RMVISATM, "remove visible atoms" },
    { MSG_POPUP_MENU_WINDOWFSCREEN, "&Full screen" },
    { MSG_DLGCONSOLE_LOG_FILENAME, "VEGA console.log" },
    { MSG_DLGCONSOLE_PAUSED, "(Paused)" },
    { MSG_DLGCONSOLE_PM_CLEARALL, "Clear all" },
    { MSG_DLGCONSOLE_PM_RECPAUSE, "Pause recording" },
    { MSG_DLGCONSOLE_PM_RECRESUME, "Resume recording" },
    { MSG_DLGCONSOLE_PM_PRINTBUF, "Print buffer" },
    { MSG_DLGCONSOLE_PM_RECORD, "Record to file" },
    { MSG_DLGCONSOLE_PRINTJOB, "VEGA ZZ console output" },
    { MSG_DLGCONSOLE_PM_SAVEBUF, "Save buffer" },
    { MSG_DLGCONSOLE_REDIRECTED, "Redirected to file" },
    { MSG_DLGCONSOLE_SD_FILTER, "Log file (*.log)|*.log" },
    { MSG_DLGCONSOLE_SD_TIT_RECORD, "Record log file" },
    { MSG_DLGCONSOLE_SD_TIT_SAVE, "Save console buffer" },
    { MSG_DLGIMGSAVE_IMAGETYPE_360, "360" },
    { MSG_DLGIMGSAVE_IMAGETYPE_2D, "2D" },
    { MSG_DLGIMGSAVE_IMAGETYPE_3D, "3D" },
    { MSG_DLGIMGSAVE_IMAGETYPE_CUBE, "Cube" },
    { MSG_DLGJOY_CK_ENABLED, "Enabled" },
    { MSG_DLGJOY_LBL_3DXBEEPER, "Beeper:" },
    { MSG_DLGJOY_LBL_3DXFIRMWARE, "Firmware:" },
    { MSG_DLGJOY_LBL_3DXPERIOD, "Period:" },
    { MSG_DLGJOY_LBL_3DXROTX, "X rotation:" },
    { MSG_DLGJOY_LBL_3DXROTY, "Y rotation:" },
    { MSG_DLGJOY_LBL_3DXROTZ, "Z rotation:" },
    { MSG_DLGJOY_LBL_3DXTRANSX, "X translation:" },
    { MSG_DLGJOY_LBL_3DXTRANSY, "Y translation:" },
    { MSG_DLGJOY_LBL_3DXTRANSZ, "Z translation:" },
    { MSG_DLGJOY_TS_MOUSE3D, "3D mouse" },
    { MSG_ERR_PBUFFER_REN360_BUILD, "  Building the panorama" },
    { MSG_ERR_PBUFFER_REN360_INJMETA, "  Injecting the metedata" },
    { MSG_ERR_PBUFFER_REN360_PROG, "* Rendering the scene as 360 panorama" },
    { MSG_ERR_PBUFFER_REN360_PRJ, "  Creating the %s projection\n" },
    { MSG_ERR_PBUFFER_REN360_PRJ_BACK, "back" },
    { MSG_ERR_PBUFFER_REN360_PRJ_BOTTOM, "bottom" },
    { MSG_ERR_PBUFFER_REN360_PRJ_FRONT, "front" },
    { MSG_ERR_PBUFFER_REN360_PRJ_LEFT, "left" },
    { MSG_ERR_PBUFFER_REN360_PRJ_RIGHT, "right" },
    { MSG_ERR_PBUFFER_REN360_PRJ_TOP, "top" },
    { MSG_ERR_PBUFFER_REN360_SAVE, "  Saving the picture" },
    { MSG_ATOM_SORTBYRES, "* Sorting atoms by residue" },
    { MSG_ERR_COM_MOLDELETE, "Invalid molecule ID" },
    { MSG_ERR_COM_SEGDELETE, "Invalid segment ID" },
    { MSG_DLGCOLORSET_CB_SKYSWAP, "Swap eyes" },
    { MSG_DLGDBEXP_CB_OPTKEEPLARGE, "Keep only the largest molecule" },
    { MSG_DLGDBEXP_CB_OPTRMIONS, "Remove counterions && waters" },
    { MSG_DLGSSTR_LBL_CAPPING, "Peptide capping" },
    { MSG_DLGSSTR_ME_PATTERN_HINT, "Put here the secondary structure pattern (optional)." },
    { MSG_DLGSSTR_ME_SEQ_HINT, "Put here the amino acid sequence in single character format.\n"\
	"Upper and lower case caharaters are respectively for L- and\n"\
	"D- amino acids." },
    { MSG_MAIN_MENU_SORTATMBYRES, "S&ort atoms by res." },
    { MSG_MAIN_MENU_RMCIONS, "C&ounterions" },
    { MSG_SKYBOX_BUILDINGFACES, "  Creating the faces" },
    { MSG_SKYBOX_CACHING, "  Caching the angles" },
    { MSG_SKYBOX_LOADING, "* Loading the SkyBox textures" },
    { MSG_ERR_SKYBOX_AR, "The texture aspect ratio must be 2:1" },
    { MSG_ERR_SKYBOX_COLOR, "Invalid color component (%d). It must be in 0-255 range" },
    { MSG_ERR_SKYBOX_DATAFORMAT, "The pixel format of the image must be RGB" },
    { MSG_ERR_SKYBOX_FACEBMP, "Can't buid the face bitmaps" },
    { MSG_ERR_SKYBOX_INIT, "Can't initialize the context to process the textures" },
    { MSG_ERR_SKYBOX_VERTEX, "Invalid vertex ID (%d). It must be in 1-8 range" },
    { MSG_ERR_SKYBOX_VERTEXDUP, "Vertex %d already defined" },
    { MSG_INFOXML_ANGLES, "Number of bond angles" },
    { MSG_INFOXML_BONDS, "Number of bonds" },
    { MSG_INFOXML_ATMTYPEFREQ, "Occurences of atom types" },
    { MSG_ERR_INFOXML_KEYWORD, "Unknown keyword \"%s\" for InfoXML file format" },
    { MSG_OPENVR_DISABLED, "* OpenVR rendering disabled\n" },
    { MSG_OPENVR_INFO, "* OpenVR rendering enabled\n"\
	"  Detected VR system..: %s %s %s\n"\
	"  View port size......: %ux%u (AR %.4lf)\n"\
	"  Update frequency....: %.2f Hz\n" },
    { MSG_ERR_OPENVR_INIT, "Unable to initialize the context.\n"\
	"%s" },
    { MSG_UNDO_RMCIONS, "remove counterions" },
    { MSG_UNDO_SORTATMBYRES, "sort atoms by res." },
    { MSG_UTILS_RMCIONSPROG, "* Removing counterions" },
    { MSG_ERR_MAIN_CAPCTERM, "Unknown C-term capping keyword" },
    { MSG_ERR_MAIN_CAPNTERM, "Unknown N-term capping keyword" },
    { MSG_MAIN_MENU_BUILDDNA, "&DNA/RNA" },
    { MSG_ANA_CB_RMSDH, "RMSD with H" },
    { MSG_ANA_CB_RMSDHUNGARIAN, "RMSD sym. cor." },
    { MSG_ANA_CB_RMSDHUNGARIAN_H, "RMSD sym. cor. with H" },
    { MSG_ANA_CB_RMSDSUPER, "RMSD alignment" },
    { MSG_ANA_CB_RMSDSUPERH, "RMSD alignment with H" },
    { MSG_DLGDNA_CB_ADDHYDROG, "Add hydrogens" },
    { MSG_DLGDNA_NUCLEICACID, "Nucleic acid" },
    { MSG_DLGDNA_GRP_OPTIONS, "Options" },
    { MSG_DLGDNA_RG_STRAND, "Strands" },
    { MSG_DLGDNA_RG_STRAND_35, "3' - 5'" },
    { MSG_DLGDNA_RG_STRAND_53, "5' - 3'" },
    { MSG_DLGDNA_RG_STRAND_BOTH, "Both" },
    { MSG_DLGDNA_RG_TYPE, "Type" },
    { MSG_DLGDNA_RG_TYPE_ADNA, "Right handled A-DNA (Arnot)" },
    { MSG_DLGDNA_RG_TYPE_ABDNA, "Right handed B-DNA (Arnot)" },
    { MSG_DLGDNA_RG_TYPE_LBDNA, "Right handed B-DNA (Langridge)" },
    { MSG_DLGDNA_RG_TYPE_SBDNA, "Left handed B-DNA (Sasisekharan)" },
    { MSG_DLGDNA_RG_TYPE_ARNA, "Right handled A-RNA (Arnot)" },
    { MSG_DLGDNA_RG_TYPE_APRNA, "Right handed A-PRIME RNA (Arnott)" },
    { MSG_DLGDNA_TITLE, "Build DNA/RNA" },
    { MSG_DLGPREFS_LBL_ENECONTDIST, "Max. contact distance ():" },
    { MSG_ERR_DNABUILD_INVSEQ, "Invalid DNA/RNA sequence" },
    { MSG_DNA_PROG, "* Building the nucleic acid" },
    { MSG_INFOXML_AAKEY1, "Labels of amino acid (1 character)" },
    { MSG_INFOXML_AAKEY3, "Labels of amino acid (3 characters)" },
    { MSG_INFOXML_AACOMPNUM, "Amino acid composition by occurrence" },
    { MSG_INFOXML_AACOMPNUMPERC, "Amino acid composition by percentage occurrence" },
    { MSG_INFOXML_AACOMPMASS, "Amino acid composition by mass" },
    { MSG_INFOXML_AACOMPMASSPERC, "Amino acid composition by percentage mass (referred to the total protein mass)" },
    { MSG_INFOXML_AAMASS, "Mass of the protein only" },
    { MSG_INFOXML_AAMASSPERC, "Percentage mass of the protein (referred to the total mass)" },
    { MSG_INFOXML_AANUM, "Number of amino acids" },
    { MSG_SCORE_ENE_CONTACTS, "  Contacts....................: %12.0f\n"\
	"  Contacts norm. heavy atoms..: %12.4f\n"\
	"  Contacts norm. mass.........: %12.4f\n" },
    { MSG_SCORE_ENE_CONTACTS_FP, "  Contacts fingerprint........: %12d (0x%05x)\n"\
	"  Contacts amino acid map.....: %s\n" },
    { MSG_SCORE_NAME_CONTACTS, "Contacts" },
    { MSG_UNDO_DNABUILD, "new nucleic acid" },
    { MSG_MAIN_MENU_SELSUPRES, "Sup. &residues" },
    { MSG_UNDO_SELSUPRES, "select sup. residues" },
    { MSG_INFOXML_COORDTYP, "Type of ccordinates" },
    { MSG_WARN_AMBERNC_CONVVER, "Amber NetCDF file has ConventionVersion which is not 1.0 (%s)" },
};
#endif /* VEGA_ARRAY */
